{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "d526d7b4-78c4-469d-9c54-3de46f1848ea",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "INFO: pikepdf C++ to Python logger bridge initialized\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Warning: No languages specified, defaulting to English.\n",
      "✅ Extracted text saved to rsob_240345.txt\n"
     ]
    }
   ],
   "source": [
    "from langchain_unstructured import UnstructuredLoader\n",
    "\n",
    "file = r\"C:\\Users\\deepa\\Downloads\\Deepanshu Bhatt\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\Data\\raw\\pmc_pdfs\\rsob.240345.pdf\"\n",
    "loader = UnstructuredLoader(file, mode=\"elements\", strategy=\"fast\")\n",
    "\n",
    "docs = loader.load()  # this is a list of Document objects\n",
    "\n",
    "# Combine all text\n",
    "full_text = \"\\n\\n\".join([d.page_content for d in docs])\n",
    "\n",
    "# Save to a txt file\n",
    "output_file = \"rsob_240345.txt\"\n",
    "with open(output_file, \"w\", encoding=\"utf-8\") as f:\n",
    "    f.write(full_text)\n",
    "\n",
    "print(f\"✅ Extracted text saved to {output_file}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "bfc869da-8664-4002-8145-68af819d60e9",
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"rsob_240345.txt\", \"r\", encoding = \"utf-8\") as f:\n",
    "    text = f.read()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "998bdd25-eb8b-45f6-a4ab-c5e714042f67",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Warning: No languages specified, defaulting to English.\n",
      "[Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.802, 176.68271000000004), (42.802, 194.68271000000004), (82.97800000000001, 194.68271000000004), (82.97800000000001, 176.68271000000004)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Title', 'element_id': '842992c400388e72e728939fc2d44f9b'}, page_content='Review'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.800000000000004, 213.68271000000004), (42.800000000000004, 265.68271000000004), (191.66005, 265.68271000000004), (191.66005, 213.68271000000004)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '842992c400388e72e728939fc2d44f9b', 'category': 'NarrativeText', 'element_id': '871217bce83cd1614715ff10e539da1d'}, page_content='Cite this article: Cheng CW et al. 2025 PIEZO1 variant implications for biological understanding and human health. Open Biol. 15: 240345. https://doi.org/10.1098/rsob.240345'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.80008000000001, 290.68271000000004), (42.80008000000001, 314.68271000000004), (131.30007999999998, 314.68271000000004), (131.30007999999998, 290.68271000000004)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Title', 'element_id': '2474051350654cfcf444078042e59f51'}, page_content='Received: 11 November 2024 Accepted: 28 May 2025'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.80008000000001, 354.93271), (42.80008000000001, 392.93271), (178.27007999999995, 392.93271), (178.27007999999995, 354.93271)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Title', 'element_id': '54eab0e91f9b69853d67e3b4725acdf4'}, page_content='Subject Areas: genetics, cellular biology, molecular biology, biophysics'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.800110000000004, 417.93271), (42.800110000000004, 469.93271), (193.69012999999995, 469.93271), (193.69012999999995, 417.93271)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '54eab0e91f9b69853d67e3b4725acdf4', 'category': 'UncategorizedText', 'element_id': 'c43c8f0540859f8abb018821d5c9b16a'}, page_content='Keywords: non-selective cation channel, calcium channel, calcium signalling, mechanical force, shear stress, endothelium'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.80015, 500.18271), (42.80015, 538.18271), (137.21015, 538.18271), (137.21015, 500.18271)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Title', 'element_id': '0456f477eaea27c39f306e893c3508cc'}, page_content='Author for correspondence: David J. Beech e-mail: d.j.beech@leeds.ac.uk'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((241.0, 40.5), (241.0, 90.5), (540.16, 90.5), (540.16, 40.5)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Title', 'element_id': '22171180e85cbfc436ba32ea424ab00c'}, page_content='PIEZO1 variant implications for biological understanding and human health'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((240.99525, 107.0), (240.99525, 193.5), (549.41505, 193.5), (549.41505, 107.0)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 14}, {'text': ', 2 ,', 'url': 'aff-kriyadocs-aff1, aff2', 'start_index': 32}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 58}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 74}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 91}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 111}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 130}, {'text': ',', 'url': 'aff-kriyadocs-aff3', 'start_index': 148}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 163}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 179}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 210}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 230}, {'text': ', 4 ,', 'url': 'aff-kriyadocs-aff1, aff4', 'start_index': 251}, {'text': ', 6 ,', 'url': 'aff-kriyadocs-aff1, aff5', 'start_index': 270}, {'text': ',', 'url': 'aff-kriyadocs-aff1', 'start_index': 291}, {'text': ', 2 ,', 'url': 'aff-kriyadocs-aff1, aff2', 'start_index': 308}, {'text': ', 5', 'url': 'aff-kriyadocs-aff1, aff6', 'start_index': 327}, {'text': 'Bailey1 , 6 , Antreas C . Kalli1 , Lara C . Morley1 , 2 , Klaus K . Witte1 , 5 and David J . Beech1', 'url': 'aff-kriyadocs-aff1', 'start_index': 263}], 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '22171180e85cbfc436ba32ea424ab00c', 'category': 'UncategorizedText', 'element_id': '972a46c0f50e49e281dac7586ae707ae'}, page_content='Chew W. Cheng1, Sophie L. Earle1,2, Oleksandr V. Povstyan1, Chloe Randall1, Katie A. Smith1, Marjolaine Debant1, Fraser L. Macrae1, Daniel G. Beech3, Anna McGrane1, Fiona Bartoli1, Eulashini Chuntharpursat-Bon1, Richard M. Cubbon1, Kathryn J. Griffin1,4, Marc A. Bailey1,6, Antreas C. Kalli1, Lara C. Morley1,2, Klaus K. Witte1,5 and David J. Beech1'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((240.99863000000002, 202.70000000000005), (240.99863000000002, 256.29999999999995), (543.57863, 256.29999999999995), (543.57863, 202.70000000000005)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '22171180e85cbfc436ba32ea424ab00c', 'category': 'UncategorizedText', 'element_id': 'd80d40ab32fa0f72463b190df0171dd6'}, page_content='1Leeds Institute of Cardiovascular and Metabolic Medicine, School of Medicine, and 2Academic Department of Obstetrics and Gynaecology, School of Medicine, University of Leeds, Leeds LS2 9JT, UK 3Calderdale and Huddersfield NHS Foundation Trust, Huddersfield HD3 3EA, UK 4Department of Histopathology, and 5Department of Cardiology, Leeds Teaching Hospitals NHS Trust, Leeds, UK 6Leeds Vascular Institute, Leeds General Infirmary, Leeds, UK'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((248.92, 266.23001), (248.92, 275.23001), (408.83202, 275.23001), (408.83202, 266.23001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': '0000 - 0002 - 7813 - 7213', 'url': 'http://orcid.org/0000-0002-7813-7213', 'start_index': 5}, {'text': '0000 - 0002 - 7683 - 9422', 'url': 'http://orcid.org/0000-0002-7683-9422', 'start_index': 31}], 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '22171180e85cbfc436ba32ea424ab00c', 'category': 'UncategorizedText', 'element_id': '89fcba142ad8b083a0b899fda858c41e'}, page_content='AMG, 0000-0002-7813-7213; DJB, 0000-0002-7683-9422'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((245.99574, 291.93590000000006), (245.99574, 540.9359), (547.8460000000001, 540.9359), (547.8460000000001, 291.93590000000006)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '22171180e85cbfc436ba32ea424ab00c', 'category': 'NarrativeText', 'element_id': '3f161f4523db92c4d7241469d461af21'}, page_content='The large membrane protein PIEZO1 assembles as trimers to form exceptional mechanical force-sensing ion channels of eukaryotes. When these channels are activated by force, cell membrane permeability to calcium ions and other ions increases rapidly, coupling force to cell function through ionic control. In humans and other species, PIEZO1 is both widely expressed and functional across major systems that include the cardiovascular, haematological and musculoskeletal systems, thereby serving diverse needs. In this narrative review of the scientific literature, we address what has been learned about PIEZO1 from associations of its gene variation with human characteristics. A particular physiological importance of PIEZO1 is emerging in lymphatics and thus in the control of tissue fluid homeostasis with relevance to the disease conditions of non-immune fetal hydrops and generalized lymphatic dysplasia. Other vascular relevance is seen in lower limb venous varicosities. PIEZO1 may be non-essential in red blood cells but the amplification of its function by gene variation quite selectively alters these cells, leading to haemolytic anaemia and other related disturbances that may be only mildly adverse and confer survival advantage. We speculate on what else might be learned in humans, guided by knowledge from PIEZO1 studies in mice, and describe how knowledge accumulated to date highlights new opportunities for PIEZO1 understanding and pathways to patient benefit.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((241.0, 581.22998), (241.0, 597.22998), (313.752, 597.22998), (313.752, 581.22998)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '22171180e85cbfc436ba32ea424ab00c', 'category': 'ListItem', 'element_id': '7d2337894e02097a79e387fbbd2657f8'}, page_content='1. Introduction'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((240.99597000000009, 605.43587), (240.99597000000009, 734.43587), (552.84202, 734.43587), (552.84202, 605.43587)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': 'figure  1', 'url': 'fig-caption-kriyadocs-F1_0', 'start_index': 235}, {'text': '1', 'url': 'ref-para-kriyadocs-R1', 'start_index': 320}, {'text': '2', 'url': 'ref-para-kriyadocs-R2', 'start_index': 446}, {'text': '–', 'url': 'ref-para-kriyadocs-R7', 'start_index': 447}, {'text': '7', 'url': 'ref-para-kriyadocs-R7', 'start_index': 449}, {'text': '3 ,', 'url': 'ref-para-kriyadocs-R3', 'start_index': 748}, {'text': '7', 'url': 'ref-para-kriyadocs-R7', 'start_index': 750}, {'text': '8', 'url': 'ref-para-kriyadocs-R8', 'start_index': 792}, {'text': '– 10', 'url': 'ref-para-kriyadocs-R10', 'start_index': 793}], 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '22171180e85cbfc436ba32ea424ab00c', 'category': 'NarrativeText', 'element_id': '266cd6b70b0107781116f95d270a58c4'}, page_content='Mechanobiology is the ability of biological molecules, cells and tissues to detect and respond to mechanical forces, thereby ensuring that the mechan- ical states of these structures match the external and internal demands of organisms (figure 1). Mechanobiology has been described as ‘the elephant in the room … overlooked’ [1], but there is now increasing recognition of its importance across animal biology including human health and disease [2– 7]. Moreover, although all biological molecules are potentially mechanically sensitive depending on the intensity and type of stimulus, it has become increasingly clear that specific proteins evolved as dedicated force sensors of physiology, conferring high-fidelity force-dependent responsiveness [3,7]. A key protein of this type is PIEZO1 [8–10]. It has remarkable force sensing and'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((245.29700000000003, 769.90002), (245.29700000000003, 800.90002), (548.8039900000001, 800.90002), (548.8039900000001, 769.90002)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': 'http :// creativecommons . org / licenses / by / 4 . 0 /,', 'url': 'http://creativecommons.org/licenses/by/4.0/', 'start_index': 111}], 'page_number': 1, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Footer', 'element_id': 'dc47c864f6707ab6f07a2358c377057e'}, page_content='© 2025 The Authors. Published by the Royal Society under the terms of the Creative Commons Attribution License http://creativecommons.org/licenses/by/4.0/, which permits unrestricted use, provided the original author and source are credited.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49937, 42.70589000000007), (42.49937, 63.70589000000007), (552.8180000000002, 63.70589000000007), (552.8180000000002, 42.70589000000007)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': '8 ,', 'url': 'ref-para-kriyadocs-R8', 'start_index': 141}, {'text': '10', 'url': 'ref-para-kriyadocs-R10', 'start_index': 143}, {'text': '– 12', 'url': 'ref-para-kriyadocs-R12', 'start_index': 145}], 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Header', 'element_id': '92d37b75024b480c3b341f726390133c'}, page_content='transduction capabilities, serves force sensing needs in many cell and tissue types, and may be a ubiquitous force sensor across eukaryotes [8,10–12].'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49638, 82.39999999999998), (42.49638, 96.39999999999998), (136.82837999999998, 96.39999999999998), (136.82837999999998, 82.39999999999998)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '92d37b75024b480c3b341f726390133c', 'category': 'Title', 'element_id': '2580c763689997ed3ca3ccd80a9f42e7'}, page_content='1.1. PIEZO1 properties'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49590999999998, 105.60589000000004), (42.49590999999998, 174.60588000000007), (552.8049099999997, 174.60588000000007), (552.8049099999997, 105.60589000000004)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': '9', 'url': 'ref-para-kriyadocs-R9', 'start_index': 119}, {'text': '9', 'url': 'ref-para-kriyadocs-R9', 'start_index': 259}, {'text': '9 ,', 'url': 'ref-para-kriyadocs-R9', 'start_index': 410}, {'text': '13', 'url': 'ref-para-kriyadocs-R13', 'start_index': 412}, {'text': 'figure 2a – d', 'url': 'fig-caption-kriyadocs-F2_0', 'start_index': 542}, {'text': '14', 'url': 'ref-para-kriyadocs-R14', 'start_index': 737}, {'text': '15', 'url': 'ref-para-kriyadocs-R15', 'start_index': 763}], 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2580c763689997ed3ca3ccd80a9f42e7', 'category': 'NarrativeText', 'element_id': 'a1c79d9d0c1c9bf2a66b25f156190cfb'}, page_content='PIEZO1 was first defined as a force sensor in patch-clamp recordings from nerve cancer cells cultured in an incubator [9]. The studies were possible because PIEZO1 forms calcium ion (Ca2+)-permeable non-selective cationic channels in the outer cell membrane [9]. The channels open in response to the application of experimentally applied forces such as increased pressure in the patch-clamp recording pipette [9,13]. As the channels open, they allow the flow of millions of ions across the cell membrane, creating a measurable ionic current (figure 2a–d). Although artificial methods of this type can activate the channels in experiments, the channels are also sensitive to physiological forces that arise, for example, from fluid flow [14] and tissue stiffness [15].'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49625999999998, 177.60588000000007), (42.49625999999998, 282.60588000000007), (552.8460300000002, 282.60588000000007), (552.8460300000002, 177.60588000000007)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': 'figure 2b , c', 'url': 'fig-caption-kriyadocs-F2_0', 'start_index': 169}, {'text': 'figure 2d', 'url': 'fig-caption-kriyadocs-F2_0', 'start_index': 284}, {'text': 'figure 2d', 'url': 'fig-caption-kriyadocs-F2_0', 'start_index': 491}, {'text': '16', 'url': 'ref-para-kriyadocs-R16', 'start_index': 684}, {'text': '17', 'url': 'ref-para-kriyadocs-R17', 'start_index': 792}], 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2580c763689997ed3ca3ccd80a9f42e7', 'category': 'NarrativeText', 'element_id': 'a8a85b5a4fd3844aa463b4b93735f224'}, page_content='PIEZO1 channels change in their properties depending on their context but they usually activate quickly (in milliseconds or less) when there is a rapidly applied force (figure 2b,c). Channel activation is seen through a net flux of cations into cells at negative membrane potentials (figure 2d), which is the usual electrical polarity of cells. In the absence of experimentally applied voltage control, such ion influxes cause depolarization, a lessening of the negative membrane potential (figure 2d) that can increase or decrease cell function depending on the cell type. A cation that enters cells via PIEZO1 channels is Ca2+, which is a pivotal currency of intracellular control [16]. Its cytosolic elevation can trigger many cell activities such as contraction, secretion and migration [17]. The fluxes of two other physiological cations, the sodium ion (Na+) in and the potassium ion (K+) out, can also have specific impact. Therefore, PIEZO1 has the potential to alter cell function through several ionic mechanisms that modulate downstream processes, thereby generating effects.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49862999999988, 285.60588000000007), (42.49862999999988, 366.60588), (552.8388499999999, 366.60588), (552.8388499999999, 285.60588000000007)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': 'figure', 'url': 'fig-caption-kriyadocs-F2_0', 'start_index': 117}, {'text': '2b', 'url': 'fig-caption-kriyadocs-F2_0', 'start_index': 124}, {'text': '18', 'url': 'ref-para-kriyadocs-R18', 'start_index': 215}, {'text': '– 20', 'url': 'ref-para-kriyadocs-R20', 'start_index': 217}, {'text': 'figure  2c', 'url': 'fig-caption-kriyadocs-F2_0', 'start_index': 274}, {'text': '18', 'url': 'ref-para-kriyadocs-R18', 'start_index': 285}, {'text': '20', 'url': 'ref-para-kriyadocs-R20', 'start_index': 425}, {'text': 'figure 2c', 'url': 'fig-caption-kriyadocs-F2_0', 'start_index': 556}], 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2580c763689997ed3ca3ccd80a9f42e7', 'category': 'NarrativeText', 'element_id': '2f0568f41c1f7fb5d5f08b1ef1eed924'}, page_content='Importantly, once the channels are activated, they can desensitize rapidly when there is sustained mechanical force (figure 2b). This property of inactivation is regulated, for example by lipid and protein factors [18–20], and so inactivation may be absent or occur slowly (figure 2c) [18]. Regulated inactivation seems to be a key way in which PIEZO1 channels achieve different functions in diverse cell types and contexts [20]. Once the mechanical stimulus is removed, the channels switch off through another process that is referred to as deactivation (figure 2c). Inactivation and deactivation are both a type of off switch but inactivation differs in its tendency to cause refractoriness to the subsequent opening of the channel when another stimulus arrives. Inactivation thus confers a memory of prior events.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49545999999992, 369.60588), (42.49545999999992, 546.6058800000001), (552.8824699999999, 546.6058800000001), (552.8824699999999, 369.60588)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': 'figure 3a , b', 'url': 'fig-caption-kriyadocs-F3_0', 'start_index': 245}, {'text': 'figure 3c', 'url': 'fig-caption-kriyadocs-F3_0', 'start_index': 296}, {'text': '8 ,', 'url': 'ref-para-kriyadocs-R8', 'start_index': 308}, {'text': '21 ,', 'url': 'ref-para-kriyadocs-R21', 'start_index': 310}, {'text': '22', 'url': 'ref-para-kriyadocs-R22', 'start_index': 313}, {'text': '22 ,', 'url': 'ref-para-kriyadocs-R22', 'start_index': 464}, {'text': '23', 'url': 'ref-para-kriyadocs-R23', 'start_index': 467}, {'text': 'figure 3c', 'url': 'fig-caption-kriyadocs-F3_0', 'start_index': 548}, {'text': 'figure 3a', 'url': 'fig-caption-kriyadocs-F3_0', 'start_index': 794}, {'text': 'figure 3a', 'url': 'fig-caption-kriyadocs-F3_0', 'start_index': 924}, {'text': '22', 'url': 'ref-para-kriyadocs-R22', 'start_index': 1281}, {'text': 'figure 3a', 'url': 'fig-caption-kriyadocs-F3_0', 'start_index': 1286}, {'text': 'figure 3a', 'url': 'fig-caption-kriyadocs-F3_0', 'start_index': 1382}, {'text': '8 ,', 'url': 'ref-para-kriyadocs-R8', 'start_index': 1622}, {'text': '23', 'url': 'ref-para-kriyadocs-R23', 'start_index': 1624}, {'text': '8 ,', 'url': 'ref-para-kriyadocs-R8', 'start_index': 1706}, {'text': '23', 'url': 'ref-para-kriyadocs-R23', 'start_index': 1708}], 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2580c763689997ed3ca3ccd80a9f42e7', 'category': 'NarrativeText', 'element_id': '1d16825e0ce63b3901bab133d593fe76'}, page_content='The PIEZO1 channel is formed by three PIEZO1 proteins, each comprising about 2500 amino acid residues (2521 in human PIEZO1) arranged in chains that loop in and out of the membrane 38 times, leading to 38 membrane-spanning segments per monomer (figure 3a,b) and 114 for the whole channel trimer (figure 3c) [8,21,22]. The channel assembly embeds in and indents the membrane, forming a basket-like (dome) structure when in its closed ion-impermeable configuration [22,23]. Long tentacle-like blade structures span out from the core of each monomer (figure 3c). Each blade comprises nine transmembrane (TM) helical units (THU1−9), so there are 27 THUs per channel. Between THU7 and THU8 of each monomer is a linker on the intracellular side that includes a beam helix extending towards the pore (figure 3a). The beam is followed by a lateral plug and latch before an unstructured loop region and the clasp sitting below THU8 (figure 3a). After the last THU (THU9) is the anchor and then the outer helix formed by TM37, which lines the central ion pore with the inner helix of TM38. These two pore-lining helices are intervened by the C-terminal extracellular domain (CED), which sits above the central core, forming a cap that has foot-like structures touching nearby blade regions [22] (figure 3a). The pore is lastly cuffed on the intracellular side by the C-terminal domain (CTD) (figure 3a). Lateral tension in the membrane causes the dome to flatten and the blades to spread out radially. With flattening and expansion comes opening in the core of the channel and ion permeation, and thus a signal for the cell to act [8,23]. A more in-depth description of the channel structure can be found elsewhere [8,23], although much remains to be understood.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.50168999999998, 567.90002), (42.50168999999998, 581.90002), (113.29969, 581.90002), (113.29969, 567.90002)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '92d37b75024b480c3b341f726390133c', 'category': 'Title', 'element_id': '600aa5bd43f437e25f4cd121864baf84'}, page_content='1.2. PIEZO1 roles'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49542999999999, 591.1059100000001), (42.49542999999999, 732.1059100000001), (552.8856699999999, 732.1059100000001), (552.8856699999999, 591.1059100000001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': '8', 'url': 'ref-para-kriyadocs-R8', 'start_index': 289}, {'text': '24', 'url': 'ref-para-kriyadocs-R24', 'start_index': 449}, {'text': '14 ,', 'url': 'ref-para-kriyadocs-R14', 'start_index': 653}, {'text': '25', 'url': 'ref-para-kriyadocs-R25', 'start_index': 656}, {'text': '– 30', 'url': 'ref-para-kriyadocs-R30', 'start_index': 658}, {'text': '24', 'url': 'ref-para-kriyadocs-R24', 'start_index': 1087}], 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '600aa5bd43f437e25f4cd121864baf84', 'category': 'NarrativeText', 'element_id': '06aaa1241ff5be96874397c96e53d293'}, page_content='PIEZO1 has roles in contractile, secretory and many other cell types of all major systems that include adipose, cardiovascular, dermal, gastrointestinal, haematological, hepatobiliary, immune, musculoskeletal, neurological, reproductive, respiratory and urinary organs as well as cancers [8]. The roles are sometimes inferred based on the responses of cultured cells to the synthetic chemical agonist of PIEZO1 channels, Yoda1 or an analogue of it [24]. Responses to these agonists may not be physiological but studies of Piezo1 gene-modified mice also support the idea of diverse functions of PIEZO1 in physiology and disease as modelled in the mouse [14,25–30], all of which are likely to be due to mechanical activation of the channels by endogenous forces. Thus, there might be wide-ranging functional importance of PIEZO1 in human health and disease, but direct testing of such a hypothesis is challenging. We do not currently have PIEZO1 modulators that can be investigated in clinical studies. In particular, we do not have suitable PIEZO1 inhibitors to test the roles of PIEZO1 [24]. However, the PIEZO1 gene is naturally varied in sequence, and this presents opportunities to explore associations between its variation and phenotypes in familial genetic, genome-wide association and candidate gene association studies. Hundreds of PIEZO1 variants have so far been linked strongly or tentatively to human traits, diseases and susceptibilities to disease or therapies.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.494929999999975, 753.40002), (42.494929999999975, 767.40002), (142.76292999999995, 767.40002), (142.76292999999995, 753.40002)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '92d37b75024b480c3b341f726390133c', 'category': 'Title', 'element_id': 'f1a73b5dddcfda4528c2181f4a63e2e2'}, page_content='1.3. Focus of this article'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49828999999994, 776.6059100000001), (42.49828999999994, 797.6059100000001), (552.8040399999999, 797.6059100000001), (552.8040399999999, 776.6059100000001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': 'www . ukbiobank . ac . uk', 'url': 'http://www.ukbiobank.ac.uk', 'start_index': 219}, {'text': 'table 1', 'url': 'table-caption-kriyadocs-T1_0', 'start_index': 111}], 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Footer', 'element_id': '9aecee315121900f9223b9613e9ea968'}, page_content='Here, we review reports of PIEZO1 variants published in the scientific literature, cataloguing 179 variants in table 1. Addi- tional variants and potential association data may be found in databases such as UK Biobank (www.ukbiobank.ac.uk),'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((567.914, 44.400010000000066), (567.914, 53.400010000000066), (571.7479999999999, 53.400010000000066), (571.7479999999999, 44.400010000000066)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Header', 'element_id': 'e69a87eded0be8f82eb767777469c6d7'}, page_content='2'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 65.596), (566.70999, 117.54899999999998), (577.70999, 117.54899999999998), (577.70999, 65.596)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'e69a87eded0be8f82eb767777469c6d7', 'category': 'NarrativeText', 'element_id': 'c68f09b55fa905d445714265220d2145'}, page_content='r o y a l s o c i e t y p u b'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 117.45000000000005), (566.70999, 195.87999999999988), (577.70999, 195.87999999999988), (577.70999, 117.45000000000005)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'e69a87eded0be8f82eb767777469c6d7', 'category': 'NarrativeText', 'element_id': '32c9da4a423a02425aa3a42003239f8c'}, page_content='l i s h n g . o r g / j o u r n a l / r s o b'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 128.813), (566.70999, 130.837), (577.70999, 130.837), (577.70999, 128.813)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'e69a87eded0be8f82eb767777469c6d7', 'category': 'Title', 'element_id': 'cfb1b8e05bf75379cbb25178e680ea53'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 206.68201), (566.70999, 239.08800999999994), (577.70999, 239.08800999999994), (577.70999, 206.68201)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'e69a87eded0be8f82eb767777469c6d7', 'category': 'Title', 'element_id': '7dbe704b1987b65d9315962c008ac24d'}, page_content='O p e n B o l .'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 229.28701), (566.70999, 231.25601000000006), (577.70999, 231.25601000000006), (577.70999, 229.28701)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'e69a87eded0be8f82eb767777469c6d7', 'category': 'Title', 'element_id': 'efa350488b09792adc6fc5d5e7f36321'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 244.39001999999994), (566.70999, 256.12702), (577.70999, 256.12702), (577.70999, 244.39001999999994)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'efa350488b09792adc6fc5d5e7f36321', 'category': 'UncategorizedText', 'element_id': 'ec26ac147a26d76915bc6c5d10ad8d97'}, page_content='1 5 :'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 261.41801999999996), (566.70999, 286.33302000000003), (577.70999, 286.33302000000003), (577.70999, 261.41801999999996)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 2, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'efa350488b09792adc6fc5d5e7f36321', 'category': 'UncategorizedText', 'element_id': '3b1227077ab6c50241861925966a9d8f'}, page_content='2 4 0 3 4 5'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49992, 213.53981), (42.49992, 247.53981), (552.8001299999997, 247.53981), (552.8001299999997, 213.53981)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'efa350488b09792adc6fc5d5e7f36321', 'category': 'NarrativeText', 'element_id': '7c73b7d2fc91ca04f9ba9249138bc90b'}, page_content='Figure 1. Illustration of the concept of mechanical homeostasis concept. The central box indicates the requirement for mechanical force sensors to detect force, and transducers to convert the amplitude of the detected force into a cellular effect. The molecular details of the sensors and transducers are not fully determined but PIEZO1 channels are both a crucial force sensor and transducer in many cell and tissue types.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49610999999999, 411.15457), (42.49610999999999, 565.1545699999999), (553.1498799999998, 565.1545699999999), (553.1498799999998, 411.15457)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'efa350488b09792adc6fc5d5e7f36321', 'category': 'NarrativeText', 'element_id': 'c28da982c380ea8bf674da0359311fe1'}, page_content='Figure 2. PIEZO1 channel electrophysiology. Patch-clamp data obtained from HEK 293 cells that were not transfected (a) or which were transfected (b–d) with plasmid containing cDNA encoding human PIEZO1 (hPIEZO1) channels. (a–c) Outside-out patch data at a constant transmembrane voltage of −80 mV (millivolt). (a–c) Example pressure steps applied to the patch are shown in the upper traces with associated current traces below for ionic currents across the membrane. All pressure steps were from a background of zero pressure (0 mmHg) to +75 mmHg (positive pressure on the patch pipette, leading to stretch of the membrane). The 100 pA (picoampere) and 100 ms (millisecond) calibrations apply to all current traces. (a) Without PIEZO1 transfection, no current was seen in response to the pressure step. (b,c) With PIEZO1 transfection, inward current (downward deflection) was seen in response to the pressure step. Two examples of the variation in the properties of the PIEZO1 channel currents are shown in (b) and (c). In example 1 (b), there was very fast activation and then slightly slower but still very fast inactivation that was complete within the time course of the pressure step (i.e. no PIEZO1 current remained). In example 2 (c), there was also fast activation but then slower inactivation that was incomplete (i.e. about 25% of the PIEZO1 current remained at the end of the pressure step). After the pressure step ended, deactivation of the remaining current occurred (c). (d) Whole-cell voltage-clamp data for current evoked by the PIEZO1 agonist Yoda1 and obtained in response to a ramp change in voltage from −100 mV to +100 mV. The current progressively changed from inward polarity at −100 mV to outward polarity at +100 mV with the cross-over point from inward to outward current referred to as the reversal potential. In physiology (without voltage-clamp), the resting membrane potential is expected to be in the region of −30 and −80 mV. Therefore, inward current through PIEZO1 channels (i.e. influx of cations, which are positively charged) causes depolarization towards the reversal potential.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49745999999999, 584.6054300000001), (42.49745999999999, 677.6054300000001), (552.86594, 677.6054300000001), (552.86594, 584.6054300000001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': 'www . ncbi . nlm . nih . gov / clinvar', 'url': 'http://www.ncbi.nlm.nih.gov/clinvar', 'start_index': 9}, {'text': 'https :// azphewas . com', 'url': 'https://azphewas.com', 'start_index': 67}, {'text': 'www . finngen . fi ).', 'url': 'http://www.finngen.fi)', 'start_index': 102}, {'text': '125', 'url': 'ref-para-kriyadocs-R125', 'start_index': 136}, {'text': 'table 1', 'url': 'table-caption-kriyadocs-T1_0', 'start_index': 803}], 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'efa350488b09792adc6fc5d5e7f36321', 'category': 'NarrativeText', 'element_id': 'a1180c20da8aefc449c1b593cbf25c67'}, page_content='ClinVar (www.ncbi.nlm.nih.gov/clinvar), AstraZeneca PheWAS Portal (https://azphewas.com) and FinnGen (www.finngen.fi). The PIEZO1 gene [125] is on chromosome 16 at locus 16q24.3, so variants are autosomal and can be dominant or recessive. Hence, some associations are evident only when there is homozygosity of the variant but there are also associations with heterozygous variants and compound heterozygous variants (i.e. when two different variants occur in the same person, one inherited from each parent). The variants include exonic variants that may affect details of the PIEZO1 amino acid sequence (missense, deletion, insertion and duplication variants) or the integrity of the PIEZO1 protein (premature termination codons, frameshifts and splice site variants). These variants are the focus of table 1. There are also synonymous and intronic variants, the effects of which may arise via gene regulatory mechanisms.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.496669999999995, 680.6054300000001), (42.496669999999995, 785.6054300000001), (552.8776699999997, 785.6054300000001), (552.8776699999997, 680.6054300000001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'efa350488b09792adc6fc5d5e7f36321', 'category': 'NarrativeText', 'element_id': '731a96a6ae950e8a6c173a734d0067e1'}, page_content='We begin by addressing the three main disease conditions most associated with PIEZO1 variants, relating to lymphatics, veins and blood. We then consider additional diseases that may arise because of one or more of the main conditions or independently of them. We discuss physiological traits that may be predisposing factors to some diseases or confer survival advantage. We consider what might be learned about how the channels work as structural machines. We address the relevance to human physiology and the apparent paradox of relatively narrow disease phenotypes despite the widespread expression and functional significance of PIEZO1 that is indicated by laboratory studies. We consider the complexities of the genetic data and what the data might tell us about the systems of the PIEZO1 gene and the PIEZO1 protein. We discuss the potential for predicting and correcting or compensating for its abnormal expression or function, and thus the opportunities for patient benefit. We make conclusions and discuss potential future directions for research on this topic.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((567.9139899999999, 44.40000000000009), (567.9139899999999, 53.40000000000009), (571.7479899999998, 53.40000000000009), (571.7479899999998, 44.40000000000009)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Header', 'element_id': '089136b082a9565e5db2cac33485b173'}, page_content='3'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 65.596), (566.70999, 117.54899999999998), (577.70999, 117.54899999999998), (577.70999, 65.596)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '089136b082a9565e5db2cac33485b173', 'category': 'NarrativeText', 'element_id': 'ba8cb00be8da7d166ee04d86124bba68'}, page_content='r o y a l s o c i e t y p u b'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 117.45000000000005), (566.70999, 195.87999999999988), (577.70999, 195.87999999999988), (577.70999, 117.45000000000005)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '089136b082a9565e5db2cac33485b173', 'category': 'NarrativeText', 'element_id': 'e8dd556606779edf1c9271f344eee1c6'}, page_content='l i s h n g . o r g / j o u r n a l / r s o b'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 128.813), (566.70999, 130.837), (577.70999, 130.837), (577.70999, 128.813)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '089136b082a9565e5db2cac33485b173', 'category': 'Title', 'element_id': '5c3e92006bb553383116c8a792b9746f'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 206.68201), (566.70999, 239.08800999999994), (577.70999, 239.08800999999994), (577.70999, 206.68201)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '089136b082a9565e5db2cac33485b173', 'category': 'Title', 'element_id': '068abbcfe0e2d060d972a1f397d2126b'}, page_content='O p e n B o l .'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 229.28701), (566.70999, 231.25601000000006), (577.70999, 231.25601000000006), (577.70999, 229.28701)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '089136b082a9565e5db2cac33485b173', 'category': 'Title', 'element_id': '308d606494cf0c5f9934e3e2ebd90c9e'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 244.39001999999994), (566.70999, 256.12702), (577.70999, 256.12702), (577.70999, 244.39001999999994)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '308d606494cf0c5f9934e3e2ebd90c9e', 'category': 'UncategorizedText', 'element_id': '2a30125d2e65cd45e7c61fb53c68a459'}, page_content='1 5 :'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 261.41801999999996), (566.70999, 286.33302000000003), (577.70999, 286.33302000000003), (577.70999, 261.41801999999996)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 3, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '308d606494cf0c5f9934e3e2ebd90c9e', 'category': 'UncategorizedText', 'element_id': '4c4ad97ac921bd8f15f28f00e06e05d0'}, page_content='2 4 0 3 4 5'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.495339999999956, 374.86407), (42.495339999999956, 468.86407), (553.1454100000001, 468.86407), (553.1454100000001, 374.86407)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': 'https :// alphafold . ebi . ac . uk', 'url': 'https://alphafold.ebi.ac.uk', 'start_index': 51}, {'text': 'www . pymol . org', 'url': 'http://www.pymol.org', 'start_index': 452}, {'text': 'table 1', 'url': 'table-caption-kriyadocs-T1_0', 'start_index': 972}], 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '308d606494cf0c5f9934e3e2ebd90c9e', 'category': 'NarrativeText', 'element_id': '32da1a83011a295d8ed9bcdfa3ea8440'}, page_content='Figure 3. PIEZO1 channel structure: the AlphaFold (https://alphafold.ebi.ac.uk) model of human PIEZO1 AF-Q92508-F1-v4. (a,b) The monomer in side-view (a) and top (helicopter) view (b). The N-terminus is to the right and the C-terminus to the left. (c) The trimer (the complete PIEZO1 channel) assembled by aligning the single chain (monomer) of the AlphaFold model to the coordinates of all three chains of the full-length mouse PIEZO1 model in Pymol (www.pymol.org). The N-termini are at the outer tips of the blades and the C-termini are central. The trimer in (c) is artificially enlarged relative to the size of the monomer in (a,b) for visual presentation. (a–c) Highlighted in red or blue are variants that have been shown by patch-clamp to affect channel function: Red, loss of function (LOF); blue, gain of function (GOF). In the trimer, each variant is repeated three times, thus assuming homozygosity. Details of these variants can be found in the main text and table 1. (a,b) Indicated structural features of the channel: the cap; the pore-lining inner and outer helices (IH and OH) in light pink and the anchor in darker pink; the beam in light grey and the lateral plug and latch in darker grey; and the clasp and the CTD in dark grey. The blade comprises the 9 THUs, THU1−9.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 486.90009), (42.5, 502.90009), (426.85200000000015, 502.90009), (426.85200000000015, 486.90009)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '308d606494cf0c5f9934e3e2ebd90c9e', 'category': 'ListItem', 'element_id': '6c391777c491e5df0970c3f6e00cb3f6'}, page_content='2. Lymphatics: generalized lymphatic dysplasia and non-immune fetal hydrops'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.495750000000015, 511.10597999999993), (42.495750000000015, 604.10598), (552.84782, 604.10598), (552.84782, 511.10597999999993)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': '126', 'url': 'ref-para-kriyadocs-R126', 'start_index': 291}, {'text': '126', 'url': 'ref-para-kriyadocs-R126', 'start_index': 379}, {'text': '126', 'url': 'ref-para-kriyadocs-R126', 'start_index': 923}], 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '308d606494cf0c5f9934e3e2ebd90c9e', 'category': 'NarrativeText', 'element_id': 'f409a3861e7bb6d094c5e607f6ffd753'}, page_content='Generalized lymphatic dysplasia (GLD) is a rare form of primary systemic lymphedema characterized by widespread accumu- lation of fluid within subcutaneous spaces and body compartments resulting from structural or functional abnormalities of the lymphatic system that affect the whole body [126]. It may present antenatally as non-immune fetal hydrops (NIFH) on ultrasound scan [126]. It is defined as abnormal fluid accumulation in at least two body compartments including pericardial or pleural effusion, ascites and subcutaneous oedema. Immune fetal hydrops is secondary to red blood cell (RBC) allo-immuni- zation whereas the aetiology of NIFH includes congenital infection, cardiovascular and genetic abnormalities. The hydropic changes occur due to an increase in interstitial fluid or impaired lymphatic drainage. Therefore, the term lymphatic-related fetal hydrops may be used to refer to NIFH of lymphatic origin [126].'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.504770000000036, 633.40009), (42.504770000000036, 647.40009), (265.77677, 647.40009), (265.77677, 633.40009)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '308d606494cf0c5f9934e3e2ebd90c9e', 'category': 'NarrativeText', 'element_id': 'c0974a0f9e6feec47d9796a55fd66c6c'}, page_content='2.1. PIEZO1 disruption identified by familial genetics'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49576000000003, 656.60598), (42.49576000000003, 737.60598), (552.89672, 737.60598), (552.89672, 656.60598)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': '70 ,', 'url': 'ref-para-kriyadocs-R70', 'start_index': 79}, {'text': '126 ,', 'url': 'ref-para-kriyadocs-R126', 'start_index': 82}, {'text': '127', 'url': 'ref-para-kriyadocs-R127', 'start_index': 86}, {'text': '128', 'url': 'ref-para-kriyadocs-R128', 'start_index': 120}, {'text': '70 ,', 'url': 'ref-para-kriyadocs-R70', 'start_index': 323}, {'text': '126 ,', 'url': 'ref-para-kriyadocs-R126', 'start_index': 326}, {'text': '127', 'url': 'ref-para-kriyadocs-R127', 'start_index': 330}, {'text': '70', 'url': 'ref-para-kriyadocs-R70', 'start_index': 361}, {'text': '80', 'url': 'ref-para-kriyadocs-R80', 'start_index': 403}, {'text': '70 ,', 'url': 'ref-para-kriyadocs-R70', 'start_index': 519}, {'text': '80 ,', 'url': 'ref-para-kriyadocs-R80', 'start_index': 522}, {'text': '91', 'url': 'ref-para-kriyadocs-R91', 'start_index': 525}, {'text': '80', 'url': 'ref-para-kriyadocs-R80', 'start_index': 569}, {'text': '45 ,', 'url': 'ref-para-kriyadocs-R45', 'start_index': 582}, {'text': '80', 'url': 'ref-para-kriyadocs-R80', 'start_index': 585}, {'text': 'table 1', 'url': 'table-caption-kriyadocs-T1_0', 'start_index': 755}, {'text': 'figure 4a', 'url': 'fig-caption-kriyadocs-F4_0', 'start_index': 812}], 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '308d606494cf0c5f9934e3e2ebd90c9e', 'category': 'NarrativeText', 'element_id': '7fc8508c6a5a9e5f7f40ba931f2630ef'}, page_content='A distinct recessive form of GLD was identified and referred to as Fotiou GLD [70,126,127] or lymphatic malformation 6 [128]. It is characterized by a high incidence of NIFH and fetal demise. If the fetus survives, there is transient resolution of the oedema, but recurrence of oedema may occur in childhood or later life [70,126,127]. Studies of six families [70] followed by a further three families [80] identified associations with PIEZO1 variants. These disorders are associated with loss of PIEZO1 protein (LOP) [70,80,91], loss of PIEZO1 channel function (LOF) [80] or both [45,80]. Often, the variants cause premature termination codons early in the reading frame, or frameshifts or deletions resulting in major disruption in at least one allele (table 1), leading to partial or complete loss of PIEZO1 (figure 4a).'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.49758000000003, 740.60598), (42.49758000000003, 797.60598), (552.8329500000001, 797.60598), (552.8329500000001, 740.60598)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'links': [{'text': '126 ,', 'url': 'ref-para-kriyadocs-R126', 'start_index': 12}, {'text': '127', 'url': 'ref-para-kriyadocs-R127', 'start_index': 16}, {'text': '70 ,', 'url': 'ref-para-kriyadocs-R70', 'start_index': 102}, {'text': '80 ,', 'url': 'ref-para-kriyadocs-R80', 'start_index': 105}, {'text': '91', 'url': 'ref-para-kriyadocs-R91', 'start_index': 108}, {'text': 'table', 'url': 'table-caption-kriyadocs-T1_0', 'start_index': 113}, {'text': '1', 'url': 'table-caption-kriyadocs-T1_0', 'start_index': 119}, {'text': '45 ,', 'url': 'ref-para-kriyadocs-R45', 'start_index': 349}, {'text': '70 ,', 'url': 'ref-para-kriyadocs-R70', 'start_index': 352}, {'text': '80', 'url': 'ref-para-kriyadocs-R80', 'start_index': 355}, {'text': '70', 'url': 'ref-para-kriyadocs-R70', 'start_index': 408}, {'text': '80', 'url': 'ref-para-kriyadocs-R80', 'start_index': 531}], 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '308d606494cf0c5f9934e3e2ebd90c9e', 'category': 'NarrativeText', 'element_id': '4ce6387f270163b78c3c8ec71eec364d'}, page_content='Fotiou GLD [126,127] is associated with PIEZO1 variants that are homozygous or compound heterozygous [70,80,91] (table 1). Severe disease may not be associated with heterozygous single PIEZO1 variants, so the parents of the GLD suffers may not necessarily suffer from the disease or they may have only mild disease even though they carry a variant [45,70,80]. Therefore, the disease is classed as recessive [70]. There is, however, emerging evidence of dominance in which a single variant heterozygous parent also has the disease [80].'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((567.9139899999999, 44.40000000000009), (567.9139899999999, 53.40000000000009), (571.7479899999998, 53.40000000000009), (571.7479899999998, 44.40000000000009)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Header', 'element_id': '962c8469547488ea089dbcc2a8902d26'}, page_content='4'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 65.596), (566.70999, 117.54899999999998), (577.70999, 117.54899999999998), (577.70999, 65.596)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '962c8469547488ea089dbcc2a8902d26', 'category': 'NarrativeText', 'element_id': 'ca2f55ab9b1fe5b3f9bba2d4c2133c34'}, page_content='r o y a l s o c i e t y p u b'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 117.45000000000005), (566.70999, 195.87999999999988), (577.70999, 195.87999999999988), (577.70999, 117.45000000000005)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '962c8469547488ea089dbcc2a8902d26', 'category': 'NarrativeText', 'element_id': '3b8032387d405c1a987a11e1549f4b4d'}, page_content='l i s h n g . o r g / j o u r n a l / r s o b'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 128.813), (566.70999, 130.837), (577.70999, 130.837), (577.70999, 128.813)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '962c8469547488ea089dbcc2a8902d26', 'category': 'Title', 'element_id': 'b439b99a1b38121e8aff07fa423d6dea'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 206.68201), (566.70999, 239.08800999999994), (577.70999, 239.08800999999994), (577.70999, 206.68201)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '962c8469547488ea089dbcc2a8902d26', 'category': 'Title', 'element_id': 'd169326615bebd0c502ea8985ea3d3f7'}, page_content='O p e n B o l .'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 229.28701), (566.70999, 231.25601000000006), (577.70999, 231.25601000000006), (577.70999, 229.28701)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '962c8469547488ea089dbcc2a8902d26', 'category': 'Title', 'element_id': 'f8d0b7e1f1b69d1336a30201d1bb2e41'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 244.39001999999994), (566.70999, 256.12702), (577.70999, 256.12702), (577.70999, 244.39001999999994)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'f8d0b7e1f1b69d1336a30201d1bb2e41', 'category': 'UncategorizedText', 'element_id': 'd5347c5a863a48b029c7ff136ecf3444'}, page_content='1 5 :'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((566.70999, 261.41801999999996), (566.70999, 286.33302000000003), (577.70999, 286.33302000000003), (577.70999, 261.41801999999996)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 4, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'f8d0b7e1f1b69d1336a30201d1bb2e41', 'category': 'UncategorizedText', 'element_id': 'd4bb395daa48ac1f90862465104c8706'}, page_content='2 4 0 3 4 5'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 42.50009999999986), (42.5, 74.39009999999985), (52.5, 74.39009999999985), (52.5, 42.50009999999986)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Header', 'element_id': '2d5636250ca2c638b0416496452f3422'}, page_content='l a n o i t i d d A'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 76.33279999999968), (42.5, 78.27279999999973), (52.5, 78.27279999999973), (52.5, 76.33279999999968)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'UncategorizedText', 'element_id': '7089ba3f4825dbed28267676c082edbe'}, page_content='.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 78.27279999999985), (42.5, 110.53549999999973), (52.5, 110.53549999999973), (52.5, 78.27279999999985)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'NarrativeText', 'element_id': '723568791d6185ebe2bc9d9f1498a5f6'}, page_content='n i a m o d c i l'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 110.53549999999973), (42.5, 140.66089999999986), (52.5, 140.66089999999986), (52.5, 110.53549999999973)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': '8c547cb11787b0478eeae625d8b93f0d'}, page_content='b u p e h t n'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 140.66089999999974), (42.5, 142.50089999999977), (52.5, 142.50089999999977), (52.5, 140.66089999999974)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': 'a54f493a9bbd0bddfd583fec101f214d'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 144.44359999999972), (42.5, 263.18439999999987), (52.5, 263.18439999999987), (52.5, 144.44359999999972)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'a54f493a9bbd0bddfd583fec101f214d', 'category': 'NarrativeText', 'element_id': '39ccaae62be9a58892b74a5401cd7e1c'}, page_content='s r e p a p t n i r p e r p r o d e w e i v e r - r e e p m o r f'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 265.1270999999999), (42.5, 274.73709999999994), (52.5, 274.73709999999994), (52.5, 265.1270999999999)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'a54f493a9bbd0bddfd583fec101f214d', 'category': 'NarrativeText', 'element_id': '14646a744ccee93fbac12d5f25ae78bf'}, page_content='e r a'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 276.6797999999999), (42.5, 303.4424999999999), (52.5, 303.4424999999999), (52.5, 276.6797999999999)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': '7bfdd85501b7e0d87b51b25acea7cd64'}, page_content='a t a d e h T'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 305.3851999999998), (42.5, 307.3251999999999), (52.5, 307.3251999999999), (52.5, 305.3851999999998)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '7bfdd85501b7e0d87b51b25acea7cd64', 'category': 'UncategorizedText', 'element_id': '7f0b635257e8c7a5ed4afdbc38bfb78f'}, page_content='.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 307.3252), (42.5, 364.8406), (52.5, 364.8406), (52.5, 307.3252)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '7bfdd85501b7e0d87b51b25acea7cd64', 'category': 'NarrativeText', 'element_id': 'f6522d8f38dcf7a249351ca9a8f0ead8'}, page_content='n i e t o r p 1 O Z E I P e h t'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 366.78330000000005), (42.5, 372.8933), (52.5, 372.8933), (52.5, 366.78330000000005)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': '4598c1e3bef48fcdf3b9539f3dd86ea5'}, page_content='f o'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 374.836), (42.5, 418.0287), (52.5, 418.0287), (52.5, 374.836)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '4598c1e3bef48fcdf3b9539f3dd86ea5', 'category': 'NarrativeText', 'element_id': 'e1784a8ebe53e1a3ea14f492f29a83cf'}, page_content='e c n e u q e s d i c a'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 419.9714), (42.5, 439.53139999999996), (52.5, 439.53139999999996), (52.5, 419.9714)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '4598c1e3bef48fcdf3b9539f3dd86ea5', 'category': 'NarrativeText', 'element_id': '6f23fd74eb5c6dd881ac519f015633bd'}, page_content='o n m a'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 427.8514), (42.5, 429.6914), (52.5, 429.6914), (52.5, 427.8514)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': '8fa26eefb9ca7976b4dbb18d7a9cf5ec'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 441.47409999999996), (42.5, 451.7041), (52.5, 451.7041), (52.5, 441.47409999999996)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': '6e739b53bebe1ccc14cb6196800a1b72'}, page_content='e h t'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 453.6468), (42.5, 471.16679999999997), (52.5, 471.16679999999997), (52.5, 453.6468)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '6e739b53bebe1ccc14cb6196800a1b72', 'category': 'NarrativeText', 'element_id': 'e79b87984ac306d2c4e7a1b77de4f81d'}, page_content='t c e f f a'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 473.10949999999997), (42.5, 485.80949999999996), (52.5, 485.80949999999996), (52.5, 473.10949999999997)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '6e739b53bebe1ccc14cb6196800a1b72', 'category': 'NarrativeText', 'element_id': '80aa7f2c949891c3410f38a7f93862db'}, page_content='t a h t'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 487.7522), (42.5, 512.1622), (52.5, 512.1622), (52.5, 487.7522)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '6e739b53bebe1ccc14cb6196800a1b72', 'category': 'NarrativeText', 'element_id': 'bbee0382f0be52d0eda76825bc46835f'}, page_content='s t n a i r a v'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 514.1049), (42.5, 546.57761), (52.5, 546.57761), (52.5, 514.1049)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': '93255f8c41089d053f3a412aa1009ea8'}, page_content='1 O Z E I P e r a'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 548.5203100000001), (42.5, 564.6903100000001), (52.5, 564.6903100000001), (52.5, 548.5203100000001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '93255f8c41089d053f3a412aa1009ea8', 'category': 'NarrativeText', 'element_id': 'b2e69bb85a00f22e4bcc41839f1a79ac'}, page_content='e l b a T'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 566.6330100000001), (42.5, 582.8557100000002), (52.5, 582.8557100000002), (52.5, 566.6330100000001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': 'e46dc6b0dbbde6b3ecd24577ab756d23'}, page_content='e h t n'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 582.8557100000002), (42.5, 584.6957100000002), (52.5, 584.6957100000002), (52.5, 582.8557100000002)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': 'a8ed92c8f3a60ef728da7a90fcd6fe43'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 586.6384100000001), (42.5, 604.8984100000001), (52.5, 604.8984100000001), (52.5, 586.6384100000001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '2d5636250ca2c638b0416496452f3422', 'category': 'Title', 'element_id': '9de995f064917fe31772685b246c8a84'}, page_content='d e t s i L'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 606.8411100000001), (42.5, 690.7292100000001), (52.5, 690.7292100000001), (52.5, 606.8411100000001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '9de995f064917fe31772685b246c8a84', 'category': 'NarrativeText', 'element_id': '7a3ea79e34e5787303a4de330a05b940'}, page_content='. s t n a i r a v n i e t o r p 1 O Z E I P h t i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 690.7292100000001), (42.5, 730.08191), (52.5, 730.08191), (52.5, 690.7292100000001)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '9de995f064917fe31772685b246c8a84', 'category': 'NarrativeText', 'element_id': 'b1429dd0d5ca7cc3f5bdcc9769cc1124'}, page_content='w d e t a i c o s s a'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((42.5, 732.02461), (42.5, 797.40002), (52.5, 797.40002), (52.5, 732.02461)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '9de995f064917fe31772685b246c8a84', 'category': 'NarrativeText', 'element_id': 'f7971455de261c225a52a9129a4a8301'}, page_content='s e p y t o n e h P . 1 e l b a T'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 42.500119999999924), (54.5, 64.42012), (64.5, 64.42012), (64.5, 42.500119999999924)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'category': 'Header', 'element_id': 'da959fbc1b37a516adaa2123e5026e0d'}, page_content='n a m u h'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 66.93436999999994), (54.5, 75.49436999999989), (64.5, 75.49436999999989), (64.5, 66.93436999999994)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'da959fbc1b37a516adaa2123e5026e0d', 'category': 'Title', 'element_id': 'c0c104ab98acd57cc5d5dec999d6d6f9'}, page_content='r o f'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 78.00861999999995), (54.5, 87.61861999999996), (64.5, 87.61861999999996), (64.5, 78.00861999999995)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'c0c104ab98acd57cc5d5dec999d6d6f9', 'category': 'NarrativeText', 'element_id': '6e774c1b7997a030b235e90fc882c8e4'}, page_content='e r a'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 90.13287000000003), (54.5, 117.35286999999994), (64.5, 117.35286999999994), (64.5, 90.13287000000003)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'da959fbc1b37a516adaa2123e5026e0d', 'category': 'Title', 'element_id': '15a9dabeaaf8ea715be193a4625de2e5'}, page_content='s r e b m u n'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 119.86712), (54.5, 142.16712000000007), (64.5, 142.16712000000007), (64.5, 119.86712)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '15a9dabeaaf8ea715be193a4625de2e5', 'category': 'NarrativeText', 'element_id': 'd3f3af46001a53688986e91fc813c1b3'}, page_content='e u d i s e r'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 144.68137000000002), (54.5, 157.17137000000002), (64.5, 157.17137000000002), (64.5, 144.68137000000002)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '15a9dabeaaf8ea715be193a4625de2e5', 'category': 'NarrativeText', 'element_id': 'abd0ab6eabcd21c0039213759c985df5'}, page_content='d i c a'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 159.68561999999986), (54.5, 179.92561999999998), (64.5, 179.92561999999998), (64.5, 159.68561999999986)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '15a9dabeaaf8ea715be193a4625de2e5', 'category': 'NarrativeText', 'element_id': 'b00698aee5993d0bb0fcb017948483ec'}, page_content='o n m A'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 167.56561999999997), (54.5, 169.40562), (64.5, 169.40562), (64.5, 167.56561999999997)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'da959fbc1b37a516adaa2123e5026e0d', 'category': 'Title', 'element_id': '217a1dc2551a81200ffbd19348f4b3ad'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 182.43986999999993), (54.5, 184.37986999999998), (64.5, 184.37986999999998), (64.5, 182.43986999999993)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '217a1dc2551a81200ffbd19348f4b3ad', 'category': 'UncategorizedText', 'element_id': 'ad795d32fda5a275cebbd3335f3055c9'}, page_content='.'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 184.24987), (54.5, 208.81987000000004), (64.5, 208.81987000000004), (64.5, 184.24987)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'da959fbc1b37a516adaa2123e5026e0d', 'category': 'Title', 'element_id': 'e301463074e92e44d0a8bd4bf12ab932'}, page_content='d e d u l c n'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 208.81986999999992), (54.5, 210.65986999999996), (64.5, 210.65986999999996), (64.5, 208.81986999999992)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'da959fbc1b37a516adaa2123e5026e0d', 'category': 'Title', 'element_id': '2b0029820e1b7519ac5f1591f6fa19d0'}, page_content='i'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 213.17412000000013), (54.5, 223.60411999999997), (64.5, 223.60411999999997), (64.5, 213.17412000000013)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'da959fbc1b37a516adaa2123e5026e0d', 'category': 'Title', 'element_id': '6dcf88833f16404d7c319df5a7080fe4'}, page_content='t o n'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 226.11837000000003), (54.5, 235.72837000000004), (64.5, 235.72837000000004), (64.5, 226.11837000000003)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': '6dcf88833f16404d7c319df5a7080fe4', 'category': 'NarrativeText', 'element_id': '57f45c8c148455807cedf912674a2bac'}, page_content='e r a'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 238.24262), (54.5, 251.93262000000004), (64.5, 251.93262000000004), (64.5, 238.24262)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'da959fbc1b37a516adaa2123e5026e0d', 'category': 'Title', 'element_id': '52389f34f5aefcf0d600dcae1187cab7'}, page_content='y e h t'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 254.44687), (54.5, 261.16687), (64.5, 261.16687), (64.5, 254.44687)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'da959fbc1b37a516adaa2123e5026e0d', 'category': 'Title', 'element_id': 'c0b2698b7872df22bec86f0304d5c377'}, page_content='o s'), Document(metadata={'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'coordinates': {'points': ((54.5, 263.68111999999996), (54.5, 293.84112000000005), (64.5, 293.84112000000005), (64.5, 263.68111999999996)), 'system': 'PixelSpace', 'layout_width': 595.29999, 'layout_height': 841.90002}, 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs', 'filename': 'rsob.240345.pdf', 'last_modified': '2025-08-17T20:28:27', 'page_number': 5, 'languages': ['eng'], 'filetype': 'application/pdf', 'parent_id': 'da959fbc1b37a516adaa2123e5026e0d', 'category': 'Title', 'element_id': '898edb1dbaa49506bf3d178b79d09046'}, page_content=', e r u t a r e t i l')]\n"
     ]
    }
   ],
   "source": [
    "from langchain_unstructured import UnstructuredLoader\n",
    "\n",
    "loaders = UnstructuredLoader(file, mode = \"elements\", strategy = \"fast\")\n",
    "\n",
    "docs = []\n",
    "for doc in loader.lazy_load():\n",
    "    docs.append(doc)\n",
    "\n",
    "print(docs[:100])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "e80e853e-2787-4f64-a4aa-bddc17767c8e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4197"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(docs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "c44a918e-dbbe-4faf-95e8-a33326f7e373",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf',\n",
       " 'coordinates': {'points': ((42.802, 176.68271000000004),\n",
       "   (42.802, 194.68271000000004),\n",
       "   (82.97800000000001, 194.68271000000004),\n",
       "   (82.97800000000001, 176.68271000000004)),\n",
       "  'system': 'PixelSpace',\n",
       "  'layout_width': 595.29999,\n",
       "  'layout_height': 841.90002},\n",
       " 'file_directory': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs',\n",
       " 'filename': 'rsob.240345.pdf',\n",
       " 'last_modified': '2025-08-17T20:28:27',\n",
       " 'page_number': 1,\n",
       " 'languages': ['eng'],\n",
       " 'filetype': 'application/pdf',\n",
       " 'category': 'Title',\n",
       " 'element_id': '842992c400388e72e728939fc2d44f9b'}"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "docs[0].metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "62511e31-279d-44c7-8d4a-17b2993a339c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Review\n",
      "\n",
      "Cite this article: Cheng CW et al. 2025 PIEZO1 variant implications for biological understanding and human health. Open Biol. 15: 240345. https://doi.org/10.1098/rsob.240345\n",
      "\n",
      "Received: 11 November 2024 Accepted: 28 May 2025\n",
      "\n",
      "Subject Areas: genetics, cellular biology, molecular biology, biophysics\n",
      "\n",
      "Keywords: non-selective cation channel, calcium channel, calcium signalling, mechanical force, shear stress, endothelium\n",
      "\n",
      "Author for correspondence: David J. Beech e-mail: d.j.beech@leeds.ac.uk\n",
      "\n",
      "PIEZO1 variant implications for biological understanding and human health\n",
      "\n",
      "Chew W. Cheng1, Sophie L. Earle1,2, Oleksandr V. Povstyan1, Chloe Randall1, Katie A. Smith1, Marjolaine Debant1, Fraser L. Macrae1, Daniel G. Beech3, Anna McGrane1, Fiona Bartoli1, Eulashini Chuntharpursat-Bon1, Richard M. Cubbon1, Kathryn J. Griffin1,4, Marc A. Bailey1,6, Antreas C. Kalli1, Lara C. Morley1,2, Klaus K. Witte1,5 and David J. Beech1\n",
      "\n",
      "1Leeds Institute of Cardiovascular and Metabolic Medicine, School of Medicine, and 2Academic Department of Obstetrics and Gynaecology, School of Medicine, University of Leeds, Leeds LS2 9JT, UK 3Calderdale and Huddersfield NHS Foundation Trust, Huddersfield HD3 3EA, UK 4Department of Histopathology, and 5Department of Cardiology, Leeds Teaching Hospitals NHS Trust, Leeds, UK 6Leeds Vascular Institute, Leeds General Infirmary, Leeds, UK\n",
      "\n",
      "AMG, 0000-0002-7813-7213; DJB, 0000-0002-7683-9422\n",
      "\n",
      "The large membrane protein PIEZO1 assembles as trimers to form exceptional mechanical force-sensing ion channels of eukaryotes. When these channels are activated by force, cell membrane permeability to calcium ions and other ions increases rapidly, coupling force to cell function through ionic control. In humans and other species, PIEZO1 is both widely expressed and functional across major systems that include the cardiovascular, haematological and musculoskeletal systems, thereby serving diverse needs. In this narrative review of the scientific literature, we address what has been learned about PIEZO1 from associations of its gene variation with human characteristics. A particular physiological importance of PIEZO1 is emerging in lymphatics and thus in the control of tissue fluid homeostasis with relevance to the disease conditions of non-immune fetal hydrops and generalized lymphatic dysplasia. Other vascular relevance is seen in lower limb venous varicosities. PIEZO1 may be non-essential in red blood cells but the amplification of its function by gene variation quite selectively alters these cells, leading to haemolytic anaemia and other related disturbances that may be only mildly adverse and confer survival advantage. We speculate on what else might be learned in humans, guided by knowledge from PIEZO1 studies in mice, and describe how knowledge accumulated to date highlights new opportunities for PIEZO1 understanding and pathways to patient benefit.\n",
      "\n",
      "1. Introduction\n",
      "\n",
      "Mechanobiology is the ability of biological molecules, cells and tissues to detect and respond to mechanical forces, thereby ensuring that the mechan- ical states of these structures match the external and internal demands of organisms (figure 1). Mechanobiology has been described as ‘the elephant in the room … overlooked’ [1], but there is now increasing recognition of its importance across animal biology including human health and disease [2– 7]. Moreover, although all biological molecules are potentially mechanically sensitive depending on the intensity and type of stimulus, it has become increasingly clear that specific proteins evolved as dedicated force sensors of physiology, conferring high-fidelity force-dependent responsiveness [3,7]. A key protein of this type is PIEZO1 [8–10]. It has remarkable force sensing and\n",
      "\n",
      "© 2025 The Authors. Published by the Royal Society under the terms of the Creative Commons Attribution License http://creativecommons.org/licenses/by/4.0/, which permits unrestricted use, provided the original author and source are credited.\n",
      "\n",
      "transduction capabilities, serves force sensing needs in many cell and tissue types, and may be a ubiquitous force sensor across eukaryotes [8,10–12].\n",
      "\n",
      "1.1. PIEZO1 properties\n",
      "\n",
      "PIEZO1 was first defined as a force sensor in patch-clamp recordings from nerve cancer cells cultured in an incubator [9]. The studies were possible because PIEZO1 forms calcium ion (Ca2+)-permeable non-selective cationic channels in the outer cell membrane [9]. The channels open in response to the application of experimentally applied forces such as increased pressure in the patch-clamp recording pipette [9,13]. As the channels open, they allow the flow of millions of ions across the cell membrane, creating a measurable ionic current (figure 2a–d). Although artificial methods of this type can activate the channels in experiments, the channels are also sensitive to physiological forces that arise, for example, from fluid flow [14] and tissue stiffness [15].\n",
      "\n",
      "PIEZO1 channels change in their properties depending on their context but they usually activate quickly (in milliseconds or less) when there is a rapidly applied force (figure 2b,c). Channel activation is seen through a net flux of cations into cells at negative membrane potentials (figure 2d), which is the usual electrical polarity of cells. In the absence of experimentally applied voltage control, such ion influxes cause depolarization, a lessening of the negative membrane potential (figure 2d) that can increase or decrease cell function depending on the cell type. A cation that enters cells via PIEZO1 channels is Ca2+, which is a pivotal currency of intracellular control [16]. Its cytosolic elevation can trigger many cell activities such as contraction, secretion and migration [17]. The fluxes of two other physiological cations, the sodium ion (Na+) in and the potassium ion (K+) out, can also have specific impact. Therefore, PIEZO1 has the potential to alter cell function through several ionic mechanisms that modulate downstream processes, thereby generating effects.\n",
      "\n",
      "Importantly, once the channels are activated, they can desensitize rapidly when there is sustained mechanical force (figure 2b). This property of inactivation is regulated, for example by lipid and protein factors [18–20], and so inactivation may be absent or occur slowly (figure 2c) [18]. Regulated inactivation seems to be a key way in which PIEZO1 channels achieve different functions in diverse cell types and contexts [20]. Once the mechanical stimulus is removed, the channels switch off through another process that is referred to as deactivation (figure 2c). Inactivation and deactivation are both a type of off switch but inactivation differs in its tendency to cause refractoriness to the subsequent opening of the channel when another stimulus arrives. Inactivation thus confers a memory of prior events.\n",
      "\n",
      "The PIEZO1 channel is formed by three PIEZO1 proteins, each comprising about 2500 amino acid residues (2521 in human PIEZO1) arranged in chains that loop in and out of the membrane 38 times, leading to 38 membrane-spanning segments per monomer (figure 3a,b) and 114 for the whole channel trimer (figure 3c) [8,21,22]. The channel assembly embeds in and indents the membrane, forming a basket-like (dome) structure when in its closed ion-impermeable configuration [22,23]. Long tentacle-like blade structures span out from the core of each monomer (figure 3c). Each blade comprises nine transmembrane (TM) helical units (THU1−9), so there are 27 THUs per channel. Between THU7 and THU8 of each monomer is a linker on the intracellular side that includes a beam helix extending towards the pore (figure 3a). The beam is followed by a lateral plug and latch before an unstructured loop region and the clasp sitting below THU8 (figure 3a). After the last THU (THU9) is the anchor and then the outer helix formed by TM37, which lines the central ion pore with the inner helix of TM38. These two pore-lining helices are intervened by the C-terminal extracellular domain (CED), which sits above the central core, forming a cap that has foot-like structures touching nearby blade regions [22] (figure 3a). The pore is lastly cuffed on the intracellular side by the C-terminal domain (CTD) (figure 3a). Lateral tension in the membrane causes the dome to flatten and the blades to spread out radially. With flattening and expansion comes opening in the core of the channel and ion permeation, and thus a signal for the cell to act [8,23]. A more in-depth description of the channel structure can be found elsewhere [8,23], although much remains to be understood.\n",
      "\n",
      "1.2. PIEZO1 roles\n",
      "\n",
      "PIEZO1 has roles in contractile, secretory and many other cell types of all major systems that include adipose, cardiovascular, dermal, gastrointestinal, haematological, hepatobiliary, immune, musculoskeletal, neurological, reproductive, respiratory and urinary organs as well as cancers [8]. The roles are sometimes inferred based on the responses of cultured cells to the synthetic chemical agonist of PIEZO1 channels, Yoda1 or an analogue of it [24]. Responses to these agonists may not be physiological but studies of Piezo1 gene-modified mice also support the idea of diverse functions of PIEZO1 in physiology and disease as modelled in the mouse [14,25–30], all of which are likely to be due to mechanical activation of the channels by endogenous forces. Thus, there might be wide-ranging functional importance of PIEZO1 in human health and disease, but direct testing of such a hypothesis is challenging. We do not currently have PIEZO1 modulators that can be investigated in clinical studies. In particular, we do not have suitable PIEZO1 inhibitors to test the roles of PIEZO1 [24]. However, the PIEZO1 gene is naturally varied in sequence, and this presents opportunities to explore associations between its variation and phenotypes in familial genetic, genome-wide association and candidate gene association studies. Hundreds of PIEZO1 variants have so far been linked strongly or tentatively to human traits, diseases and susceptibilities to disease or therapies.\n",
      "\n",
      "1.3. Focus of this article\n",
      "\n",
      "Here, we review reports of PIEZO1 variants published in the scientific literature, cataloguing 179 variants in table 1. Addi- tional variants and potential association data may be found in databases such as UK Biobank (www.ukbiobank.ac.uk),\n",
      "\n",
      "2\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "Figure 1. Illustration of the concept of mechanical homeostasis concept. The central box indicates the requirement for mechanical force sensors to detect force, and transducers to convert the amplitude of the detected force into a cellular effect. The molecular details of the sensors and transducers are not fully determined but PIEZO1 channels are both a crucial force sensor and transducer in many cell and tissue types.\n",
      "\n",
      "Figure 2. PIEZO1 channel electrophysiology. Patch-clamp data obtained from HEK 293 cells that were not transfected (a) or which were transfected (b–d) with plasmid containing cDNA encoding human PIEZO1 (hPIEZO1) channels. (a–c) Outside-out patch data at a constant transmembrane voltage of −80 mV (millivolt). (a–c) Example pressure steps applied to the patch are shown in the upper traces with associated current traces below for ionic currents across the membrane. All pressure steps were from a background of zero pressure (0 mmHg) to +75 mmHg (positive pressure on the patch pipette, leading to stretch of the membrane). The 100 pA (picoampere) and 100 ms (millisecond) calibrations apply to all current traces. (a) Without PIEZO1 transfection, no current was seen in response to the pressure step. (b,c) With PIEZO1 transfection, inward current (downward deflection) was seen in response to the pressure step. Two examples of the variation in the properties of the PIEZO1 channel currents are shown in (b) and (c). In example 1 (b), there was very fast activation and then slightly slower but still very fast inactivation that was complete within the time course of the pressure step (i.e. no PIEZO1 current remained). In example 2 (c), there was also fast activation but then slower inactivation that was incomplete (i.e. about 25% of the PIEZO1 current remained at the end of the pressure step). After the pressure step ended, deactivation of the remaining current occurred (c). (d) Whole-cell voltage-clamp data for current evoked by the PIEZO1 agonist Yoda1 and obtained in response to a ramp change in voltage from −100 mV to +100 mV. The current progressively changed from inward polarity at −100 mV to outward polarity at +100 mV with the cross-over point from inward to outward current referred to as the reversal potential. In physiology (without voltage-clamp), the resting membrane potential is expected to be in the region of −30 and −80 mV. Therefore, inward current through PIEZO1 channels (i.e. influx of cations, which are positively charged) causes depolarization towards the reversal potential.\n",
      "\n",
      "ClinVar (www.ncbi.nlm.nih.gov/clinvar), AstraZeneca PheWAS Portal (https://azphewas.com) and FinnGen (www.finngen.fi). The PIEZO1 gene [125] is on chromosome 16 at locus 16q24.3, so variants are autosomal and can be dominant or recessive. Hence, some associations are evident only when there is homozygosity of the variant but there are also associations with heterozygous variants and compound heterozygous variants (i.e. when two different variants occur in the same person, one inherited from each parent). The variants include exonic variants that may affect details of the PIEZO1 amino acid sequence (missense, deletion, insertion and duplication variants) or the integrity of the PIEZO1 protein (premature termination codons, frameshifts and splice site variants). These variants are the focus of table 1. There are also synonymous and intronic variants, the effects of which may arise via gene regulatory mechanisms.\n",
      "\n",
      "We begin by addressing the three main disease conditions most associated with PIEZO1 variants, relating to lymphatics, veins and blood. We then consider additional diseases that may arise because of one or more of the main conditions or independently of them. We discuss physiological traits that may be predisposing factors to some diseases or confer survival advantage. We consider what might be learned about how the channels work as structural machines. We address the relevance to human physiology and the apparent paradox of relatively narrow disease phenotypes despite the widespread expression and functional significance of PIEZO1 that is indicated by laboratory studies. We consider the complexities of the genetic data and what the data might tell us about the systems of the PIEZO1 gene and the PIEZO1 protein. We discuss the potential for predicting and correcting or compensating for its abnormal expression or function, and thus the opportunities for patient benefit. We make conclusions and discuss potential future directions for research on this topic.\n",
      "\n",
      "3\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "Figure 3. PIEZO1 channel structure: the AlphaFold (https://alphafold.ebi.ac.uk) model of human PIEZO1 AF-Q92508-F1-v4. (a,b) The monomer in side-view (a) and top (helicopter) view (b). The N-terminus is to the right and the C-terminus to the left. (c) The trimer (the complete PIEZO1 channel) assembled by aligning the single chain (monomer) of the AlphaFold model to the coordinates of all three chains of the full-length mouse PIEZO1 model in Pymol (www.pymol.org). The N-termini are at the outer tips of the blades and the C-termini are central. The trimer in (c) is artificially enlarged relative to the size of the monomer in (a,b) for visual presentation. (a–c) Highlighted in red or blue are variants that have been shown by patch-clamp to affect channel function: Red, loss of function (LOF); blue, gain of function (GOF). In the trimer, each variant is repeated three times, thus assuming homozygosity. Details of these variants can be found in the main text and table 1. (a,b) Indicated structural features of the channel: the cap; the pore-lining inner and outer helices (IH and OH) in light pink and the anchor in darker pink; the beam in light grey and the lateral plug and latch in darker grey; and the clasp and the CTD in dark grey. The blade comprises the 9 THUs, THU1−9.\n",
      "\n",
      "2. Lymphatics: generalized lymphatic dysplasia and non-immune fetal hydrops\n",
      "\n",
      "Generalized lymphatic dysplasia (GLD) is a rare form of primary systemic lymphedema characterized by widespread accumu- lation of fluid within subcutaneous spaces and body compartments resulting from structural or functional abnormalities of the lymphatic system that affect the whole body [126]. It may present antenatally as non-immune fetal hydrops (NIFH) on ultrasound scan [126]. It is defined as abnormal fluid accumulation in at least two body compartments including pericardial or pleural effusion, ascites and subcutaneous oedema. Immune fetal hydrops is secondary to red blood cell (RBC) allo-immuni- zation whereas the aetiology of NIFH includes congenital infection, cardiovascular and genetic abnormalities. The hydropic changes occur due to an increase in interstitial fluid or impaired lymphatic drainage. Therefore, the term lymphatic-related fetal hydrops may be used to refer to NIFH of lymphatic origin [126].\n",
      "\n",
      "2.1. PIEZO1 disruption identified by familial genetics\n",
      "\n",
      "A distinct recessive form of GLD was identified and referred to as Fotiou GLD [70,126,127] or lymphatic malformation 6 [128]. It is characterized by a high incidence of NIFH and fetal demise. If the fetus survives, there is transient resolution of the oedema, but recurrence of oedema may occur in childhood or later life [70,126,127]. Studies of six families [70] followed by a further three families [80] identified associations with PIEZO1 variants. These disorders are associated with loss of PIEZO1 protein (LOP) [70,80,91], loss of PIEZO1 channel function (LOF) [80] or both [45,80]. Often, the variants cause premature termination codons early in the reading frame, or frameshifts or deletions resulting in major disruption in at least one allele (table 1), leading to partial or complete loss of PIEZO1 (figure 4a).\n",
      "\n",
      "Fotiou GLD [126,127] is associated with PIEZO1 variants that are homozygous or compound heterozygous [70,80,91] (table 1). Severe disease may not be associated with heterozygous single PIEZO1 variants, so the parents of the GLD suffers may not necessarily suffer from the disease or they may have only mild disease even though they carry a variant [45,70,80]. Therefore, the disease is classed as recessive [70]. There is, however, emerging evidence of dominance in which a single variant heterozygous parent also has the disease [80].\n",
      "\n",
      "4\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "l a n o i t i d d A\n",
      "\n",
      ".\n",
      "\n",
      "n i a m o d c i l\n",
      "\n",
      "b u p e h t n\n",
      "\n",
      "i\n",
      "\n",
      "s r e p a p t n i r p e r p r o d e w e i v e r - r e e p m o r f\n",
      "\n",
      "e r a\n",
      "\n",
      "a t a d e h T\n",
      "\n",
      ".\n",
      "\n",
      "n i e t o r p 1 O Z E I P e h t\n",
      "\n",
      "f o\n",
      "\n",
      "e c n e u q e s d i c a\n",
      "\n",
      "o n m a\n",
      "\n",
      "i\n",
      "\n",
      "e h t\n",
      "\n",
      "t c e f f a\n",
      "\n",
      "t a h t\n",
      "\n",
      "s t n a i r a v\n",
      "\n",
      "1 O Z E I P e r a\n",
      "\n",
      "e l b a T\n",
      "\n",
      "e h t n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i L\n",
      "\n",
      ". s t n a i r a v n i e t o r p 1 O Z E I P h t i\n",
      "\n",
      "w d e t a i c o s s a\n",
      "\n",
      "s e p y t o n e h P . 1 e l b a T\n",
      "\n",
      "n a m u h\n",
      "\n",
      "r o f\n",
      "\n",
      "e r a\n",
      "\n",
      "s r e b m u n\n",
      "\n",
      "e u d i s e r\n",
      "\n",
      "d i c a\n",
      "\n",
      "o n m A\n",
      "\n",
      "i\n",
      "\n",
      ".\n",
      "\n",
      "d e d u l c n\n",
      "\n",
      "i\n",
      "\n",
      "t o n\n",
      "\n",
      "e r a\n",
      "\n",
      "y e h t\n",
      "\n",
      "o s\n",
      "\n",
      ", e r u t a r e t i l\n",
      "\n",
      "w e i v e r - r e e p\n",
      "\n",
      "e h t\n",
      "\n",
      "f o\n",
      "\n",
      "h c r a e s\n",
      "\n",
      "r u o\n",
      "\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "d e m\n",
      "\n",
      "r i f n o c\n",
      "\n",
      "e b\n",
      "\n",
      "t o n\n",
      "\n",
      "d u o c\n",
      "\n",
      "l\n",
      "\n",
      "] 8 [\n",
      "\n",
      "y l s u o i v e r p\n",
      "\n",
      "d e t s i l\n",
      "\n",
      "s t n a i r a v w e f\n",
      "\n",
      "A\n",
      "\n",
      ". t x e t\n",
      "\n",
      "n i a m e h t\n",
      "\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "d e t a c i d n\n",
      "\n",
      "i\n",
      "\n",
      "s a\n",
      "\n",
      "s e s a b a t a d\n",
      "\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "d n u o f\n",
      "\n",
      "e b\n",
      "\n",
      "y a m\n",
      "\n",
      "s t n a i r a v\n",
      "\n",
      "f o\n",
      "\n",
      "s n o g e r\n",
      "\n",
      "i\n",
      "\n",
      "l a r u t c u r t S\n",
      "\n",
      ". s e c n e u q e s\n",
      "\n",
      "d i c a\n",
      "\n",
      "o n m a\n",
      "\n",
      "i\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      ") 8 0 5 2 9 Q ( n a m u h d n a\n",
      "\n",
      ") 2 2 F J 2 E (\n",
      "\n",
      "e s u o m\n",
      "\n",
      ") g r o . t o r p n u w w w\n",
      "\n",
      "i\n",
      "\n",
      ".\n",
      "\n",
      "(\n",
      "\n",
      "t o r P n U d e r a p m o c\n",
      "\n",
      "i\n",
      "\n",
      "t a h t\n",
      "\n",
      "t n e m n g\n",
      "\n",
      "i l a\n",
      "\n",
      "a g e m O\n",
      "\n",
      "l a t s u l C\n",
      "\n",
      "a\n",
      "\n",
      "g n i s u\n",
      "\n",
      "d e n m r e t e d\n",
      "\n",
      "i\n",
      "\n",
      ", s t e k c a r b\n",
      "\n",
      "d e v r u c n\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "e s u o m\n",
      "\n",
      "r o f\n",
      "\n",
      "t n e l a v i u q e\n",
      "\n",
      "e h t h t i\n",
      "\n",
      "w 1 O Z E I P\n",
      "\n",
      "g u p\n",
      "\n",
      "l\n",
      "\n",
      "l a r t n e C\n",
      "\n",
      "; ] 7 6 3 1 − 5 0 3 1 [\n",
      "\n",
      "m a e B\n",
      "\n",
      "; ] 4 0 3 1 − 7 1 1 1\n",
      "\n",
      "; 6 1 1 1 − 9 4 9\n",
      "\n",
      "; 8 4 9 − 1 0 7\n",
      "\n",
      "; 0 0 7 − 2 4 5\n",
      "\n",
      "; 1 4 5 − 5 4 3\n",
      "\n",
      "; 4 3 3 − 3 4 1\n",
      "\n",
      "; 2 4 1 − 1 [\n",
      "\n",
      "7 − 1\n",
      "\n",
      ") s U H T (\n",
      "\n",
      "s t i n U\n",
      "\n",
      "l a c i l e H e n a r b m e m s n a r T\n",
      "\n",
      "e d a l B\n",
      "\n",
      ": ) s t e k c a r b\n",
      "\n",
      "e r a u q s\n",
      "\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "s r e b m u n\n",
      "\n",
      "d i c a\n",
      "\n",
      "o n m a\n",
      "\n",
      "i\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 8 [\n",
      "\n",
      "e r a\n",
      "\n",
      "l e n n a h c\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "n a m u h\n",
      "\n",
      "e h t\n",
      "\n",
      "p a C\n",
      "\n",
      "; ] 3 9 1 2 − 6 6 1 2 [\n",
      "\n",
      "x i l e h\n",
      "\n",
      "r e t u O\n",
      "\n",
      "; ] 5 6 1 2 − 9 9 0 2 [\n",
      "\n",
      "r o h c n A\n",
      "\n",
      "; ] 8 9 0 2 − 3 0 8 1\n",
      "\n",
      "; 2 0 8 1 − 7 5 6 1 [\n",
      "\n",
      "9 − 8\n",
      "\n",
      "s U H T\n",
      "\n",
      "e d a l B\n",
      "\n",
      "; ] 6 5 6 1 − 1 7 5 1 [\n",
      "\n",
      "8 U H T - o t - p s a l C\n",
      "\n",
      "; ] 0 7 5 1 − 5 1 5 1 [\n",
      "\n",
      "p s a l C\n",
      "\n",
      "; ] 4 1 5 1 − 6 2 4 1 [\n",
      "\n",
      "p s a l C - o t - h c t a L\n",
      "\n",
      "; ] 5 2 4 1 − 2 1 4 1 [\n",
      "\n",
      "h c t a L\n",
      "\n",
      "; ] 1 1 4 1 − 2 0 4 1 [\n",
      "\n",
      "g u p\n",
      "\n",
      "l\n",
      "\n",
      "l a r e t a L\n",
      "\n",
      "; ] 1 0 4 1 − 8 6 3 1 [\n",
      "\n",
      "d n a 3 6 5 7 s i\n",
      "\n",
      "l e n n a h c 1 O Z E I P n a m u h r e p s d i c a o n m a f o r e b m u n\n",
      "\n",
      "i\n",
      "\n",
      "l a t o t e h t o s\n",
      "\n",
      ", s n i e t o r p 1 O Z E I P 3 f o r e m\n",
      "\n",
      "i r t a s i\n",
      "\n",
      "l e n n a h c 1 O Z E I P h c a E . ] 1 2 5 2 − 4 7 4 2 [\n",
      "\n",
      ") D T C ( n i a m o d\n",
      "\n",
      "l a n m r e t - C ; ] 3 7 4 2 − 0 4 4 2 [ x i l e h r e n n\n",
      "\n",
      "i\n",
      "\n",
      "I\n",
      "\n",
      "; ] 9 3 4 2 − 4 9 1 2 [\n",
      "\n",
      ") D E C ,\n",
      "\n",
      "n i a m o d r a l u\n",
      "\n",
      "l l e c a r t x e l a n m r e t - C (\n",
      "\n",
      "i\n",
      "\n",
      "a t a d d e w e i v e r - r e e p o n\n",
      "\n",
      ", e g d e l w o n k\n",
      "\n",
      "r u o f o t s e b e h t o t\n",
      "\n",
      ", e r e h w s e t a c i d n\n",
      "\n",
      "i\n",
      "\n",
      "’\n",
      "\n",
      "d e i f i c e p s e n o N\n",
      "\n",
      "‘\n",
      "\n",
      ". ] 8 [ e r e h w e s l e d e i f i c e p s e r a l e n n a h c 1 O Z E I P e s u o m\n",
      "\n",
      "f o s n o g e r\n",
      "\n",
      "i\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "r o f\n",
      "\n",
      "s r e b m u n d i c a o n m A\n",
      "\n",
      "i\n",
      "\n",
      ". y t i s o g y z o m o h n\n",
      "\n",
      "i\n",
      "\n",
      "l e n n a h c\n",
      "\n",
      "r e p s e m\n",
      "\n",
      "i t e e r h t\n",
      "\n",
      "s t a e p e r\n",
      "\n",
      "t n a i r a v h c a e\n",
      "\n",
      "d n a F O G r o f e u b n\n",
      "\n",
      "l\n",
      "\n",
      "i\n",
      "\n",
      "d e t a c i d n\n",
      "\n",
      "i\n",
      "\n",
      "s i\n",
      "\n",
      "t c e f f e e h t\n",
      "\n",
      ",\n",
      "\n",
      "n o i t c n u f\n",
      "\n",
      "l e n n a h c n o t n a i r a v e h t\n",
      "\n",
      "f o t c e f f e e h t\n",
      "\n",
      "r o f e l b a l i a v a e r a a t a d y r o t a r o b a l e r e h W\n",
      "\n",
      ". e c n e d i v e g n i t r o p p u s e h t g n n i a t n o c e c n e r e f e r d e r e b m u n g n i y l r e d n u e h t\n",
      "\n",
      "i\n",
      "\n",
      "s e t a c i d n\n",
      "\n",
      "i\n",
      "\n",
      "n m u o c\n",
      "\n",
      "l\n",
      "\n",
      "' s e c n e r e f e r '\n",
      "\n",
      "e h T . e l b a l i a v a e r a\n",
      "\n",
      ". ) n o i t e l e d (\n",
      "\n",
      ". l e d\n",
      "\n",
      "; ) s u o g y z o r e t e h d n u o p m o c (\n",
      "\n",
      ". t e h - c\n",
      "\n",
      "; ) s u o g y z o r e t e h (\n",
      "\n",
      ". t e h\n",
      "\n",
      "; ) s u o g y z o m o h (\n",
      "\n",
      ".\n",
      "\n",
      "m o h\n",
      "\n",
      ": s n o i t a i v e r b b A P. O L r o F O L r o f d e r\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 1 3 [\n",
      "\n",
      ") H F I N (\n",
      "\n",
      "s p o r d y h\n",
      "\n",
      "l a t e f e n u m m\n",
      "\n",
      "i - n o n\n",
      "\n",
      "e b y a m\n",
      "\n",
      ";\n",
      "\n",
      "n o i t a m r o f l a m\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "1 U H T e d a l b\n",
      "\n",
      "t f i h s e m a r f A o t S\n",
      "\n",
      ") 1 3 ( 1 3\n",
      "\n",
      "k n u r t d n a d a e h n\n",
      "\n",
      "i a m e d e o s u o e n a t u c b u s\n",
      "\n",
      "x a r o h t o r d y h\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      "d e z i l a r e n e g s a d e i f i s s a l c\n",
      "\n",
      ") D L G ( a i s a l p s y d\n",
      "\n",
      "9 4 R h t i\n",
      "\n",
      "e r u t a m e r p\n",
      "\n",
      ") n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "y c n a n g e r p f o n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "] 2 3 [\n",
      "\n",
      "H F I N\n",
      "\n",
      "D L G\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "1 U H T e d a l b\n",
      "\n",
      "n o i t e l e d L\n",
      "\n",
      ") 1 4 ( 1 4\n",
      "\n",
      "8 3 6 1 T h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "e r u t a m e r p\n",
      "\n",
      ") n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "] 1 3 [\n",
      "\n",
      "k n u r t d n a d a e h n\n",
      "\n",
      "i a m e d e o s u o e n a t u c b u s\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "1 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p R\n",
      "\n",
      ") 9 4 ( 9 4\n",
      "\n",
      "x a r o h t o r d y h\n",
      "\n",
      ") ? D L G ( n o i t a m r o f l a m\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "y c n a n g e r p f o n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ") A 1 3 S h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 3 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s i s o t y c o t a m o t s y r a t i d e r e h d e t a r d y h e d\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "1 U H T e d a l b\n",
      "\n",
      "n o i t e l e d K G H A\n",
      "\n",
      ") 5 6 - 2 6 ( 5 6 − 2 6\n",
      "\n",
      ") S H D (\n",
      "\n",
      "] 4 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 1 - D V O C\n",
      "\n",
      "I\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "1 U H T e d a l b\n",
      "\n",
      "T o t\n",
      "\n",
      "I\n",
      "\n",
      ") 3 8 ( 3 8\n",
      "\n",
      "] 5 3 [\n",
      "\n",
      "x a r o h t o l y h c\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a t a n i r e p t u o h t i\n",
      "\n",
      "w\n",
      "\n",
      "r o h t i\n",
      "\n",
      "w S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "1 U H T e d a l b\n",
      "\n",
      "E o t D\n",
      "\n",
      ") 8 8 ( 8 8\n",
      "\n",
      "s i s e t n e c a r o h t\n",
      "\n",
      "l a i r e s\n",
      "\n",
      "l\n",
      "\n",
      "o o n a r p o r p\n",
      "\n",
      "l\n",
      "\n",
      "l a n r e t a m\n",
      "\n",
      "n o i t r o b a c i n e g o r t a I\n",
      "\n",
      "] 6 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "1 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p R\n",
      "\n",
      ") 3 0 1 ( 3 0 1\n",
      "\n",
      "D L G\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ") . t e h - c (\n",
      "\n",
      "] 4 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 1 - D V O C\n",
      "\n",
      "I\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "2 U H T e d a l b\n",
      "\n",
      "L o t P\n",
      "\n",
      ") 2 5 1 ( 2 5 1\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "5\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 5 3 [\n",
      "\n",
      "a m e d e o d o f\n",
      "\n",
      "l\n",
      "\n",
      ") k c e n (\n",
      "\n",
      "l a h c u n d n a n i k s\n",
      "\n",
      ", y d o b\n",
      "\n",
      "l a t o t\n",
      "\n",
      ",\n",
      "\n",
      "p l a c s\n",
      "\n",
      ") ? D L G ( n o i t a m r o f l a m\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "2 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p R\n",
      "\n",
      ") 6 9 1 ( 9 8 1\n",
      "\n",
      "n o i s u f f e l a r u e l p\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "s e t i c s a\n",
      "\n",
      ", s u s o n e v s u t c u d e h t\n",
      "\n",
      "f o s i s e n e g a , y l a g e m o d r a C n o i s u f f e l a i d r a c i r e p\n",
      "\n",
      "i\n",
      "\n",
      "a i m e n a , e u g n o t d e g r a l n e , y l a g e m o t n e c a l p\n",
      "\n",
      "n o i s u f s n a r t h t i\n",
      "\n",
      "w g n\n",
      "\n",
      "i l\n",
      "\n",
      "p m a s d o o b\n",
      "\n",
      "l\n",
      "\n",
      "l a c i l i\n",
      "\n",
      "b m u s u o e n a t u c r e p\n",
      "\n",
      "e s i\n",
      "\n",
      "m e d\n",
      "\n",
      "l a t e o f e n i r e t u a r t n\n",
      "\n",
      "I\n",
      "\n",
      "] 9 3 – 7 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e s a e s i d e v l a v c i t r o a d p s u c i b\n",
      "\n",
      "i\n",
      "\n",
      "n o i t a v i t c a d e s a e r c e d\n",
      "\n",
      "2 U H T e d a l b\n",
      "\n",
      "L o t S\n",
      "\n",
      ") 4 2 2 ( 7 1 2\n",
      "\n",
      "n o i t a d a r g e d n i e t o r p\n",
      "\n",
      "] 1 4 , 0 4 , 4 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      ") V V ( n i e v e s o c i r a v\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "2 U H T e d a l b\n",
      "\n",
      ") ] 0 4 [\n",
      "\n",
      "I\n",
      "\n",
      "r o ( A o t V\n",
      "\n",
      ") 7 5 2 ( 0 5 2\n",
      "\n",
      "9 1 - D V O C\n",
      "\n",
      "I\n",
      "\n",
      "e s a e s i d r e v i l\n",
      "\n",
      "c i l\n",
      "\n",
      "o h o c l a\n",
      "\n",
      "] 2 4 [\n",
      "\n",
      "n g n e b e b o t d e t c i d e r p t n a i r a v\n",
      "\n",
      "i\n",
      "\n",
      "a i n e p o r t u e n\n",
      "\n",
      "l a t i n e g n o c e r e v e s\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "2 U H T e d a l b\n",
      "\n",
      "T o t A\n",
      "\n",
      ") 8 5 2 ( 1 5 2\n",
      "\n",
      "] 3 4 [\n",
      "\n",
      "y t i x a l\n",
      "\n",
      "l l a w\n",
      "\n",
      "l a n m o d b a\n",
      "\n",
      "i\n",
      "\n",
      "e m o r d n y s y l l e b e n u r p\n",
      "\n",
      "g n n e p o d e s a e r c e d\n",
      "\n",
      "i\n",
      "\n",
      "2 U H T e d a l b\n",
      "\n",
      "R o t G\n",
      "\n",
      ") 0 6 2 ( 3 5 2\n",
      "\n",
      "s b i r d e r a l f\n",
      "\n",
      ") 1 O Z E I P e s u o m\n",
      "\n",
      "(\n",
      "\n",
      ") L 5 9 1 2 S h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s e e n k d e l p m d\n",
      "\n",
      "i\n",
      "\n",
      "r e d d a l b d e t a l i\n",
      "\n",
      "d\n",
      "\n",
      "r e t e r u o r d y h\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "s e r u d e c o r p r e h t o d n a e g a n i a r d r e d d a l b y r a n i r u\n",
      "\n",
      "l a c i g r u s\n",
      "\n",
      "s i s o r h p e n o r d y h\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "a i d r a c y h c a t\n",
      "\n",
      "r a l u c i r t n e v a r p u s\n",
      "\n",
      "] 4 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e r u\n",
      "\n",
      "l i a f\n",
      "\n",
      "w o r r a m e n o b d e t i r e h n\n",
      "\n",
      "i\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "2 U H T e d a l b\n",
      "\n",
      "I o t S\n",
      "\n",
      ") 1 2 3 ( 5 1 3\n",
      "\n",
      "] 5 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "l a c i n a h c e m d e s a e r c e d\n",
      "\n",
      "2 U H T e d a l b\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      ". t e h - c ( P o t L\n",
      "\n",
      ") 9 2 3 ( 2 2 3\n",
      "\n",
      "y t i v i t i s n e s\n",
      "\n",
      "e r u t a m e r p 7 9 9 E\n",
      "\n",
      ") n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "3 U H T e d a l b\n",
      "\n",
      "Y o t S\n",
      "\n",
      ") 5 4 3 ( 8 3 3\n",
      "\n",
      "] 7 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "a i m e a n a c i t y l o m e a h\n",
      "\n",
      "l a t i n e g n o c\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "3 U H T e d a l b\n",
      "\n",
      ") . t e h ( A o t P\n",
      "\n",
      ") 3 8 3 ( 6 7 3\n",
      "\n",
      "] 8 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "3 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p Q\n",
      "\n",
      ") 9 2 4 ( 2 2 4\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "6\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 9 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "3 U H T e d a l b\n",
      "\n",
      "n o i t r e s n\n",
      "\n",
      "i L - A\n",
      "\n",
      ") 5 3 4 - 4 3 4 ( 8 2 4 − 7 2 4\n",
      "\n",
      "] 0 5 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "3 U H T e d a l b\n",
      "\n",
      "C o t R\n",
      "\n",
      ") 4 6 4 ( 7 5 4\n",
      "\n",
      "] 2 5 , 1 5 [\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "3 U H T e d a l b\n",
      "\n",
      "I o t V\n",
      "\n",
      ") 5 0 5 ( 9 9 4\n",
      "\n",
      ") ? D L G ( n o i t a m r o f l a m\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      ") H 3 0 3 2 R h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 3 5 , 1 5 [ n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i l\n",
      "\n",
      "y c n e c u l s n a r t\n",
      "\n",
      "l a h c u n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "3 U H T e d a l b\n",
      "\n",
      "t f i h s e m a r f S o t C\n",
      "\n",
      ") 9 1 5 ( 3 1 5\n",
      "\n",
      ") A 0 4 5 1 S h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 4 5 [\n",
      "\n",
      ". y r u n\n",
      "\n",
      "j\n",
      "\n",
      "i\n",
      "\n",
      "t n o\n",
      "\n",
      "i\n",
      "\n",
      "j\n",
      "\n",
      "c i t a m u a r t\n",
      "\n",
      "r o e t u c a m o r f e e r f\n",
      "\n",
      "s i t i r h t r a o e t s o\n",
      "\n",
      "l a i l i\n",
      "\n",
      "m a f\n",
      "\n",
      "c i\n",
      "\n",
      "m o r d n y s - n o n\n",
      "\n",
      "s i t i r h t r a o e t s o\n",
      "\n",
      "g n n e p o d e s a e r c e d\n",
      "\n",
      "i\n",
      "\n",
      "3 U H T e d a l b\n",
      "\n",
      "C o t R\n",
      "\n",
      ") 7 3 5 ( 1 3 5\n",
      "\n",
      "s i t i r h t r a o e t s o t n o\n",
      "\n",
      "j\n",
      "\n",
      "l a e g n a l a h p r e t n\n",
      "\n",
      "i\n",
      "\n",
      "d n a s i t i r h t r a o e t s o d n a h e v i s o r e\n",
      "\n",
      ") 1 O Z E I P e s u o m\n",
      "\n",
      "(\n",
      "\n",
      "] 4 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e r u\n",
      "\n",
      "l i a f\n",
      "\n",
      "w o r r a m e n o b d e t i r e h n\n",
      "\n",
      "i\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "3 U H T e d a l b\n",
      "\n",
      "N o t K\n",
      "\n",
      ") 3 4 5 ( 7 3 5\n",
      "\n",
      "] 0 6 – 5 5 , 9 4 , 6 4 [\n",
      "\n",
      "H F I N\n",
      "\n",
      "S H D\n",
      "\n",
      "] 5 5 [ g n n e p o d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "i\n",
      "\n",
      "4 U H T e d a l b\n",
      "\n",
      "] 6 4 [ L r o ] 5 5 [\n",
      "\n",
      "M o t V\n",
      "\n",
      ") 4 0 6 ( 8 9 5\n",
      "\n",
      "] 7 5 [ y c n a n g e r p f o n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "D L G\n",
      "\n",
      "] 6 5 [ e g n a h c o n\n",
      "\n",
      "] 7 5 [ a v a c a n e v r o i r e p u s e l b u o d\n",
      "\n",
      "] 2 6 , 1 6 , 1 5 [\n",
      "\n",
      "y c n a n g e r p f o n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "4 U H T e d a l b\n",
      "\n",
      "L r o G o t V\n",
      "\n",
      ") 5 0 6 ( 9 9 5\n",
      "\n",
      "s e t i c s a , s n o i s u f f e l a r u e l p\n",
      "\n",
      "H F I N\n",
      "\n",
      "a i m e l a k r e p y h o d u e s p t u o h t i\n",
      "\n",
      "w\n",
      "\n",
      "r o h t i\n",
      "\n",
      "w\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a t a n i r e p t u o h t i\n",
      "\n",
      "w\n",
      "\n",
      "r o h t i\n",
      "\n",
      "w\n",
      "\n",
      "] 3 6 , 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "4 U H T e d a l b\n",
      "\n",
      "I\n",
      "\n",
      "r o V o t\n",
      "\n",
      "M\n",
      "\n",
      ") 1 1 6 ( 5 0 6\n",
      "\n",
      "] 4 6 , 1 5 [\n",
      "\n",
      "a m e d e o n i k s\n",
      "\n",
      ", x a r o h t o l y h c d n a n o i s u f f e l a r u e l p\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "4 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p Y\n",
      "\n",
      ") 1 6 6 ( 5 5 6\n",
      "\n",
      "y c n a n g e r p f o n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "l a c i d e m\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "y h p o r t r e p y h\n",
      "\n",
      "l a t n e c a l p\n",
      "\n",
      ") ?\n",
      "\n",
      ". t e h - c (\n",
      "\n",
      "] 6 4 , 3 3 [\n",
      "\n",
      "] 5 6 [ y h t a p o y m o d r a c h t i\n",
      "\n",
      "i\n",
      "\n",
      "w n o i t a i c o s s a l a i t n e t o p\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "4 U H T e d a l b\n",
      "\n",
      "H r o Y o t D\n",
      "\n",
      ") 5 7 6 ( 9 6 6\n",
      "\n",
      "] 6 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "4 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p E\n",
      "\n",
      ") 5 8 6 ( 9 7 6\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ") .\n",
      "\n",
      "m o h (\n",
      "\n",
      "] 6 5 , 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "e g n a h c o n\n",
      "\n",
      "4 U H T e d a l b\n",
      "\n",
      "S o t F\n",
      "\n",
      ") 7 8 6 ( 1 8 6\n",
      "\n",
      "] 7 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s i s o p y l o p s u o t a m o n e d a\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "Y o t H\n",
      "\n",
      ") 8 0 7 ( 2 0 7\n",
      "\n",
      "] 8 6 , 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "S o t G\n",
      "\n",
      ") 4 2 7 ( 8 1 7\n",
      "\n",
      "] 9 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s n o i t a m r o f l a m\n",
      "\n",
      "s u o n e v o i r e t r a n i a r b\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "L o t P\n",
      "\n",
      ") 8 2 7 ( 2 2 7\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "7\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 3 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "n o i t e l e d Q\n",
      "\n",
      ") 6 4 7 ( 5 4 7\n",
      "\n",
      "] 0 7 [\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "D L G\n",
      "\n",
      "n o i s s e r p x e d e s a e r c e d\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p E\n",
      "\n",
      ") 9 4 7 ( 5 5 7\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "H F I N\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "s k e e w 4 3 t a o r e t u n\n",
      "\n",
      "i\n",
      "\n",
      "d e i d\n",
      "\n",
      "8 2 2 2 Q h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "l a t i b r o i r e p\n",
      "\n",
      ", x a r o h t o l y h c\n",
      "\n",
      ", s n o i s u f f e l a r u e l p h t i\n",
      "\n",
      "w a m e d e o d e z i l a r e n e g d\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l i\n",
      "\n",
      "m\n",
      "\n",
      "e r u t a m e r p\n",
      "\n",
      ") n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ", t c e f e d\n",
      "\n",
      "l a t p e s\n",
      "\n",
      "l a i r t a , a i h t a n g o r c i\n",
      "\n",
      "m\n",
      "\n",
      ", s d o f\n",
      "\n",
      "l\n",
      "\n",
      "c i h t n a c i p e , s r a e e l p m\n",
      "\n",
      "i s d e p p u c\n",
      "\n",
      ", e r u t a t s\n",
      "\n",
      "t r o h s\n",
      "\n",
      ", x u l f e r\n",
      "\n",
      "l a e g a h p o s e o - o r t s a g\n",
      "\n",
      ",\n",
      "\n",
      "m g a r h p a i d e h t\n",
      "\n",
      "f o a i s a l p o y m a\n",
      "\n",
      "y l a g e m o n e l p s\n",
      "\n",
      ", a m e d e o\n",
      "\n",
      "l a t i n e g\n",
      "\n",
      ",\n",
      "\n",
      "m u t a v a c x e s u t c e p\n",
      "\n",
      "] 9 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s n o i t a m r o f l a m\n",
      "\n",
      "s u o n e v o i r e t r a n i a r b\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "n o i t e l e d E E\n",
      "\n",
      ") 0 5 7 - 9 4 7 ( 6 5 7 − 5 5 7\n",
      "\n",
      "] 8 7 – 5 7 , 3 7 – 1 7 , 5 2 [\n",
      "\n",
      "] 3 7 [\n",
      "\n",
      "s n o i t a l u p o p e m o s n\n",
      "\n",
      "i e c n a t s i s e r\n",
      "\n",
      "l a i r a l a m h t i\n",
      "\n",
      "w d e t a i c o s s a t o n\n",
      "\n",
      "s n o i t a l u p o p e m o s n\n",
      "\n",
      "i e c n a t s i s e r a i r a l a m\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "n o i t e l e d E\n",
      "\n",
      ") 0 5 7 ( 6 5 7\n",
      "\n",
      "] 4 7 [ e s a e s i d e l k c i s h t i\n",
      "\n",
      "w d e t a i c o s s a t o n\n",
      "\n",
      "] 2 7 , 1 7 [\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "d e w o l s (\n",
      "\n",
      ". ] 5 7 [ x e d n\n",
      "\n",
      "i\n",
      "\n",
      "s s a m y d o b r o e r u s s e r p d o o b\n",
      "\n",
      "l\n",
      "\n",
      ", a m o c u a l g h t i\n",
      "\n",
      "w d e t a i c o s s a t o n\n",
      "\n",
      "n o i t a r d y h e d C B R\n",
      "\n",
      "] 5 2 [ n o i t a r u t a s n i t i r r e f s n a r t d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "] 5 2 [ n i t i r r e f d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "n\n",
      "\n",
      "i y t i l i\n",
      "\n",
      "b a g n p m u\n",
      "\n",
      "i\n",
      "\n",
      "j\n",
      "\n",
      "d n a s s e n f f i t s n o d n e t\n",
      "\n",
      "s n o i t a l u p o p e m o s\n",
      "\n",
      "] 8 6 , 0 6 , 6 5 , 6 4 , 5 3 [\n",
      "\n",
      "a m o r g y h c i t s y c\n",
      "\n",
      "S H D\n",
      "\n",
      "] 6 5 [ y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "s t n a i r a v h t i\n",
      "\n",
      "w\n",
      "\n",
      "( S o t G\n",
      "\n",
      ") 7 7 7 ( 2 8 7\n",
      "\n",
      "s e r u t a e f\n",
      "\n",
      "n r o b e v i l\n",
      "\n",
      "c i h p r o m s y d\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a t a n i r e p t u o h t i\n",
      "\n",
      "w\n",
      "\n",
      "r o h t i\n",
      "\n",
      "w\n",
      "\n",
      "8 0 8 r o 8 0 8 t a\n",
      "\n",
      ") 4 6 7 1 d n a\n",
      "\n",
      "n i k s\n",
      "\n",
      "l a h c u n t n a d n u d e r\n",
      "\n",
      "y l y t c a d o n\n",
      "\n",
      "i l c e o t h t f i f\n",
      "\n",
      "y l a g e m o u c i r t n e v l a r e t a l i\n",
      "\n",
      "l\n",
      "\n",
      "b\n",
      "\n",
      "] 9 7 , 8 6 , 0 6 , 6 4 , 5 3 [\n",
      "\n",
      "a m o r g y h c i t s y c\n",
      "\n",
      "S H D\n",
      "\n",
      "] 6 5 [\n",
      "\n",
      ") Q 8 0 8 R ( e g n a h c o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "e b y a m\n",
      "\n",
      "( G r o Q o t R\n",
      "\n",
      ") 3 0 8 ( 8 0 8\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a t a n i r e p t u o h t i\n",
      "\n",
      "w\n",
      "\n",
      "r o h t i\n",
      "\n",
      "w\n",
      "\n",
      "n r o b e v i l\n",
      "\n",
      "D L G\n",
      "\n",
      "d n a 2 8 7 h t i\n",
      "\n",
      "w\n",
      "\n",
      ") s t n a i r a v 4 6 7 1\n",
      "\n",
      "s e r u t a e f\n",
      "\n",
      "c i h p r o m s y d\n",
      "\n",
      "n i k s\n",
      "\n",
      "l a h c u n t n a d n u d e r\n",
      "\n",
      "y l y t c a d o n\n",
      "\n",
      "i l c e o t h t f i f\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "8\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "y l a g e m o u c i r t n e v l a r e t a l i\n",
      "\n",
      "l\n",
      "\n",
      "b\n",
      "\n",
      "] 0 8 [\n",
      "\n",
      ") . t e h (\n",
      "\n",
      "s e c a r o h t o r d y h\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "D L G\n",
      "\n",
      "d n a y t i v i t c a d e s a e r c e d\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "r o h t i\n",
      "\n",
      "w\n",
      "\n",
      "( V o t E\n",
      "\n",
      ") 4 2 8 ( 9 2 8\n",
      "\n",
      ") . t e h ( H F I N\n",
      "\n",
      "n o i s s e r p x e\n",
      "\n",
      ") T 0 7 2 2 I\n",
      "\n",
      "t u o h t i\n",
      "\n",
      "w\n",
      "\n",
      ") . t e h (\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      ") y l n o V 9 2 8 E (\n",
      "\n",
      "s b m\n",
      "\n",
      "i l\n",
      "\n",
      "r e w o\n",
      "\n",
      "l\n",
      "\n",
      "f o a m e d e o\n",
      "\n",
      ") . t e h ( a m e d e o\n",
      "\n",
      "l a t o r c s\n",
      "\n",
      ", e c a f\n",
      "\n",
      "f o a m e d e o t n e t t i\n",
      "\n",
      "m r e t n\n",
      "\n",
      "i\n",
      "\n",
      ") . t e h (\n",
      "\n",
      "s e c a r o h t o l y h c / s n o i s u f f e l a r u e l p\n",
      "\n",
      ") . t e h ( y m o t s o r t s a g d n a n o i t a c i l\n",
      "\n",
      "p o d n u f g n i r i u q e r x u l f e r\n",
      "\n",
      "l a e g a h p o s e o - o r t s a g\n",
      "\n",
      "a e o n p a p e e l s e v i t c u r t s b o , s i s o t s o n y s o n a r c\n",
      "\n",
      "i\n",
      "\n",
      ", c i p o t e m\n",
      "\n",
      ", e m o r d n y s\n",
      "\n",
      "r e g r e p s A\n",
      "\n",
      "e t a t c n u p\n",
      "\n",
      "l l a m\n",
      "\n",
      "s h t i\n",
      "\n",
      "w s e g n a h c\n",
      "\n",
      "r a l u c s a v l a v i t c n u n o c d n a l a t i b r o i r e p\n",
      "\n",
      "j\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      ") . t e h (\n",
      "\n",
      "s e g a h r r o m e a h\n",
      "\n",
      "] 7 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "a i m e a n a c i t y l o m e h\n",
      "\n",
      "l a t i n e g n o c\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "M o t V\n",
      "\n",
      ") 5 5 8 ( 0 6 8\n",
      "\n",
      "] 4 6 [\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      ", s o n m a r d y h\n",
      "\n",
      "i\n",
      "\n",
      ", a m e d e o n i k s\n",
      "\n",
      ",\n",
      "\n",
      "n o i s u f f e l a r u e l p\n",
      "\n",
      ", s e t i c s a\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p C\n",
      "\n",
      ") 3 6 8 ( 8 6 8\n",
      "\n",
      "a i s a t c e i g n a h p m y l\n",
      "\n",
      ", a m e d e h p m y l\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "n o i s u f r e p l a m\n",
      "\n",
      "r a l u c s a v l a n r e t a m\n",
      "\n",
      "f o s n g i s\n",
      "\n",
      ") .\n",
      "\n",
      "m o h ( d o h t n o m 1 t a h t a e d\n",
      "\n",
      "l\n",
      "\n",
      "] 1 8 , 9 5 , 1 5 , 2 3 [\n",
      "\n",
      "e v l a v l a r h t e r u r o i r e t s o p d e t c e p s u s\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "I o t\n",
      "\n",
      "M\n",
      "\n",
      ") 5 6 8 ( 0 7 8\n",
      "\n",
      "n o i t c n u f s y d c i t a p e h\n",
      "\n",
      "S H D\n",
      "\n",
      ") . t e h - c (\n",
      "\n",
      "e s a e s i d y c n e i c i f e d e n u m m\n",
      "\n",
      "i y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      ") . t e h (\n",
      "\n",
      "] 4 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e r u\n",
      "\n",
      "l i a f\n",
      "\n",
      "w o r r a m e n o b d e t i r e h n\n",
      "\n",
      "i\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "P o t R\n",
      "\n",
      ") 3 0 9 ( 8 0 9\n",
      "\n",
      "] 2 8 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      ") ? S H D ( a i m e n a c i t y l o m e a h e r e v e s\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "S o t N\n",
      "\n",
      ") 0 1 9 ( 5 1 9\n",
      "\n",
      "D P 6 G h t i\n",
      "\n",
      "w\n",
      "\n",
      ". t e h - c (\n",
      "\n",
      ") t n a i r a v\n",
      "\n",
      "] 0 7 , 6 4 [\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "H F I N\n",
      "\n",
      "l a c i n a h c e m d e s a e r c e d\n",
      "\n",
      "5 U H T e d a l b\n",
      "\n",
      "M o t L\n",
      "\n",
      ") 4 3 9 ( 9 3 9\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "S H D\n",
      "\n",
      "] 8 3 [ y t i v i t i s n e s\n",
      "\n",
      "d n a C 6 5 4 2 R h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      ", k c e n d e b b e w\n",
      "\n",
      ", s e c a r o h t o l y h c / x a r o h t o l y h c\n",
      "\n",
      ",\n",
      "\n",
      "h t r i b t a a m e d e o d e z i l a r e n e g\n",
      "\n",
      ") L 8 5 4 2 F\n",
      "\n",
      "y l l e b e n u r p\n",
      "\n",
      ", a m e d e o\n",
      "\n",
      "l a t i b r o i r e p\n",
      "\n",
      "s e t y c o t a m o t s\n",
      "\n",
      "l a n o i s a c c o ,\n",
      "\n",
      "h t r i b t a d o r y h t o p y h\n",
      "\n",
      "i\n",
      "\n",
      ", x u l f e r\n",
      "\n",
      "l a e g a h p o s e o - o r t s a g\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "9\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 3 8 [\n",
      "\n",
      ", s i s o\n",
      "\n",
      "i l\n",
      "\n",
      "o c s\n",
      "\n",
      "r a b m u o c a r o h t\n",
      "\n",
      "l\n",
      "\n",
      ", s r a e c i t s a l p s y d\n",
      "\n",
      ", y r t e m m y s a l a i c a f\n",
      "\n",
      ", s V V\n",
      "\n",
      ", s e l e c o r d y h\n",
      "\n",
      ") ? D L G ( a i s a l p s y d c i t a h p m y l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "H o t R\n",
      "\n",
      ") 8 4 9 ( 3 5 9\n",
      "\n",
      ", y r o t s i h e r u t c a r f e l p i t l u m , y t i l i\n",
      "\n",
      "b a s i d\n",
      "\n",
      "l a u t c e l l e t n\n",
      "\n",
      "i\n",
      "\n",
      "d\n",
      "\n",
      "l i\n",
      "\n",
      "m\n",
      "\n",
      ", s s o\n",
      "\n",
      ", a i s a l p o p y h e n o b\n",
      "\n",
      "l a i c a f d e d i s - t f e l\n",
      "\n",
      ", s i s o\n",
      "\n",
      "i l\n",
      "\n",
      "o c s\n",
      "\n",
      "r a b m u o c a r o h t\n",
      "\n",
      "l\n",
      "\n",
      "m\n",
      "\n",
      "s i h t a n g o r p\n",
      "\n",
      ",\n",
      "\n",
      "n o i t a m r o f l a m\n",
      "\n",
      "l\n",
      "\n",
      "g n i r a e h d\n",
      "\n",
      "l i\n",
      "\n",
      "m\n",
      "\n",
      ", e r u t a t s\n",
      "\n",
      "t r o h s\n",
      "\n",
      "r a l u c i s s o t f e l\n",
      "\n",
      "0 5 − 1 n o x e h t i\n",
      "\n",
      ") n o i t e l e d\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 4 8 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "M o t V\n",
      "\n",
      ") 1 6 9 ( 6 6 9\n",
      "\n",
      "] 2 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "H o t R\n",
      "\n",
      ") 6 6 9 ( 2 7 9\n",
      "\n",
      "] 5 3 [\n",
      "\n",
      "x a r o h t o l y h c\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "V o t L\n",
      "\n",
      ") 0 7 9 ( 5 7 9\n",
      "\n",
      "s e t i c s a\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a t a n i r e p t u o h t i\n",
      "\n",
      "w\n",
      "\n",
      "r o h t i\n",
      "\n",
      "W\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      "s u o n e v t n a r d a u q r e w o\n",
      "\n",
      "l\n",
      "\n",
      "t h g i r\n",
      "\n",
      ", s s e r t s i d y r o t a r i p s e r\n",
      "\n",
      ",\n",
      "\n",
      "n o i s u f f e l a r u e l p\n",
      "\n",
      ",\n",
      "\n",
      "n o i t a l i t n e v , t n e m e c a l p e b u t\n",
      "\n",
      "t s e h c\n",
      "\n",
      ", s i s e t n e c a r o h t\n",
      "\n",
      "l a i r e s\n",
      "\n",
      ",\n",
      "\n",
      "n o i t a m r o f l a m\n",
      "\n",
      "n r o b e v i l\n",
      "\n",
      ", s i s e r u d\n",
      "\n",
      "i\n",
      "\n",
      ", l o o n a r p o r p\n",
      "\n",
      "l\n",
      "\n",
      "] 5 8 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "n o i t a m r o f l a m\n",
      "\n",
      "s u o n r e v a c\n",
      "\n",
      "l a r b e r e c\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "L o t F\n",
      "\n",
      ") 6 8 9 ( 1 9 9\n",
      "\n",
      "] 5 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p E\n",
      "\n",
      ") 2 9 9 ( 7 9 9\n",
      "\n",
      "c ( n o i t a n m r e t i\n",
      "\n",
      ") P 2 2 3 L h t i\n",
      "\n",
      "w\n",
      "\n",
      ". t e h\n",
      "\n",
      "] 7 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s i s o p y l o p s u o t a m o n e d a\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "M o t\n",
      "\n",
      "I\n",
      "\n",
      ") 2 0 0 1 ( 7 0 0 1\n",
      "\n",
      "] 6 8 [\n",
      "\n",
      "x a r o h t o l y h c\n",
      "\n",
      "D L G\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "H o t R\n",
      "\n",
      ") 5 0 0 1 ( 0 1 0 1\n",
      "\n",
      "a m e d e o b m\n",
      "\n",
      "i l\n",
      "\n",
      "r e w o\n",
      "\n",
      "l\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      ")\n",
      "\n",
      "W 1 7 6 1 R h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 4 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e r u\n",
      "\n",
      "l i a f\n",
      "\n",
      "w o r r a m e n o b d e t i r e h n\n",
      "\n",
      "i\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "I o t\n",
      "\n",
      "M\n",
      "\n",
      ") 6 0 0 1 ( 1 1 0 1\n",
      "\n",
      "] 8 8 , 7 8 [\n",
      "\n",
      "h t r i b\n",
      "\n",
      "l l i t s\n",
      "\n",
      "D L G\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p W\n",
      "\n",
      ") 4 6 0 1 ( 9 6 0 1\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a r o p r o c d n a l a i c a f\n",
      "\n",
      "H F I N\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "y l a h p e c n e o r c a m\n",
      "\n",
      ") Q 0 7 0 2 K h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s g n u\n",
      "\n",
      "l\n",
      "\n",
      "c i t s a l p o p y h\n",
      "\n",
      "n o i t a z i l a n a c\n",
      "\n",
      "l a n m u\n",
      "\n",
      "i\n",
      "\n",
      "l\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      "f o e r u\n",
      "\n",
      "l i a f\n",
      "\n",
      "i l l i v c i n o i r o h c\n",
      "\n",
      "c i p o c s o r d y h\n",
      "\n",
      "n o i s u f r e p l a m\n",
      "\n",
      "r a l u c s a v l a t e f\n",
      "\n",
      "] 5 3 [\n",
      "\n",
      "d o f\n",
      "\n",
      "l\n",
      "\n",
      "l a h c u n d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      ") ? H F I N ( n o i t a m r o f l a m\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "C o t R\n",
      "\n",
      ") 5 6 0 1 ( 0 7 0 1\n",
      "\n",
      "n o i s u f f e l a r u e l p\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "a m e d e o n i k s d n a y d o b\n",
      "\n",
      "l a t o t\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "10\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "s e t i c s a\n",
      "\n",
      "n o i s u f f e l a i d r a c i r e p\n",
      "\n",
      ",\n",
      "\n",
      "n o i s u f f e l a r u e l p\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      ", a c r a s a n a , y l a h p e c o r c a m\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "s s e r t s i d y r o t a r i p s e r d n a , s a i d a p s o p y h\n",
      "\n",
      "t n e m e c a l p e b u t\n",
      "\n",
      "t s e h c\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b d n a n o i t a l i t n e v l a c i n a h c e m\n",
      "\n",
      "l a t a n o e n\n",
      "\n",
      "h t a e d\n",
      "\n",
      "l a t a n o e n\n",
      "\n",
      "] 9 8 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s i s o t y c o r h t y r e\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "W o t R\n",
      "\n",
      ") 0 7 0 1 ( 3 7 0 1\n",
      "\n",
      "] 0 9 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "n e g i t n a r E\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p Q\n",
      "\n",
      ") 6 0 1 1 ( 1 1 1 1\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ") . t e h (\n",
      "\n",
      "] 3 5 , 1 5 [ n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i l\n",
      "\n",
      "H F I N\n",
      "\n",
      "D L G\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "6 U H T e d a l b\n",
      "\n",
      "E o t Q\n",
      "\n",
      ") 9 0 1 1 ( 4 1 1 1\n",
      "\n",
      ") V 5 9 3 2 A h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 8 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "7 U H T e d a l b\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "( L o t S\n",
      "\n",
      ") 2 1 1 1 ( 7 1 1 1\n",
      "\n",
      ") V 0 2 0 2 A\n",
      "\n",
      "] 1 9 [\n",
      "\n",
      "s p o r d y h\n",
      "\n",
      ") ? H F I N ( a i s a l p s y d c i t a h p m y l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "7 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p S\n",
      "\n",
      ") 8 4 1 1 ( 3 5 1 1\n",
      "\n",
      "x a r o h t o l y h c\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "e c a f d n a o s r o t\n",
      "\n",
      ", s g e l\n",
      "\n",
      "f o a m e d e h p m y l\n",
      "\n",
      "m u t o r c s d n a s g e l\n",
      "\n",
      "f o g n\n",
      "\n",
      "i l l e w s\n",
      "\n",
      ") R 9 2 0 2 G h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      ") g n\n",
      "\n",
      "i l\n",
      "\n",
      "b i s e n o ( n o i t a n e g y x o e n a r b m e m\n",
      "\n",
      "l a e r o p r o c a r t x e g n d u l c n\n",
      "\n",
      "i\n",
      "\n",
      "i e r a c e v i s n e t n\n",
      "\n",
      "i\n",
      "\n",
      "] 2 9 [\n",
      "\n",
      "c i n e g o h t a p - n o n e b o t d e t c i d e r p t n a i r a v\n",
      "\n",
      "r e d r o s i d\n",
      "\n",
      "l a t n e m p o l e v e d - o r u e n\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "7 U H T e d a l b\n",
      "\n",
      "M o t T\n",
      "\n",
      ") 6 9 1 1 ( 1 0 2 1\n",
      "\n",
      "] 4 9 , 3 9 [\n",
      "\n",
      "s i s y l o m e a h\n",
      "\n",
      ", y l a g e m o n e l p s\n",
      "\n",
      ",\n",
      "\n",
      "d a o l r e v o n o r i\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "7 U H T e d a l b\n",
      "\n",
      "I o t V\n",
      "\n",
      ") 8 1 2 1 ( 3 2 2 1\n",
      "\n",
      "e m o r d n y s\n",
      "\n",
      "c i t s a l p s y d o l e y m\n",
      "\n",
      "] 0 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "y t i l a m r o n b a s a e r c n a p y r a i l i\n",
      "\n",
      "b r e v i l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "7 U H T e d a l b\n",
      "\n",
      "C o t F\n",
      "\n",
      ") 1 4 2 1 ( 7 4 2 1\n",
      "\n",
      "] 9 7 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "a i m e a n a c i t y l o m e a h d e t i r e h n\n",
      "\n",
      "i\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "7 U H T e d a l b\n",
      "\n",
      "r e h t o . t e h - c ( R o t S\n",
      "\n",
      ") 0 5 2 1 ( 5 5 2 1\n",
      "\n",
      ") s t n a i r a v e n e g\n",
      "\n",
      "] 5 9 [\n",
      "\n",
      "H F I N\n",
      "\n",
      "D L G\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "7 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p W\n",
      "\n",
      ") 2 8 2 1 ( 7 8 2 1\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "a i h t a n g o r c i\n",
      "\n",
      "m\n",
      "\n",
      "0 3 3 2 _ 2 − 0 3 3 2 . c\n",
      "\n",
      ") l e d 1 −\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "11\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "s e v l a c e h t\n",
      "\n",
      "f o a m e d e o\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "s e d i r e c y l g i r t\n",
      "\n",
      "l a r u e l p d e t a v e l e\n",
      "\n",
      "] 6 9 [\n",
      "\n",
      "n o i s u f f e l a r u e l p\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "7 U H T e d a l b\n",
      "\n",
      "C o t R\n",
      "\n",
      ") 4 9 2 1 ( 9 9 2 1\n",
      "\n",
      "a m e d e o n i k s d e z i l a r e n e g\n",
      "\n",
      ") l e d 4 4 3 1 E h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s e t i c s a r o n m\n",
      "\n",
      "i\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "n o i t a n m r e t y c n a n g e r p\n",
      "\n",
      "i\n",
      "\n",
      "a c r a s a n a\n",
      "\n",
      "] 0 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "y t i l a m r o n b a s a e r c n a p y r a i l i\n",
      "\n",
      "b r e v i l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "m a e b\n",
      "\n",
      "T o t N\n",
      "\n",
      ") 7 2 3 1 ( 2 3 3 1\n",
      "\n",
      "] 6 9 [\n",
      "\n",
      "n o i s u f f e l a r u e l p\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "m a e b\n",
      "\n",
      "n o i t e l e d E\n",
      "\n",
      ") 9 3 3 1 ( 4 4 3 1\n",
      "\n",
      "a m e d e o n i k s d e z i l a r e n e g\n",
      "\n",
      ") C 9 9 2 1 R h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s e t i c s a r o n m\n",
      "\n",
      "i\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "n o i t a n m r e t y c n a n g e r p\n",
      "\n",
      "i\n",
      "\n",
      "a c r a s a n A\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "m a e b\n",
      "\n",
      "M\n",
      "\n",
      "r o V o t\n",
      "\n",
      "I\n",
      "\n",
      ") 2 5 3 1 ( 7 5 3 1\n",
      "\n",
      "] 7 9 , 6 9 , 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "m a e b\n",
      "\n",
      "h t i\n",
      "\n",
      "w C r o ( P o t R\n",
      "\n",
      ") 3 5 3 1 ( 8 5 3 1\n",
      "\n",
      "a m e d e h p m y l\n",
      "\n",
      "m r a l a t i n e g n o C\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "r e w o l s (\n",
      "\n",
      "] 6 9 [\n",
      "\n",
      ") V 8 7 6 1 A\n",
      "\n",
      "] 8 9 [\n",
      "\n",
      ", a i m e a n a c i t y l o m e a h c i n o r h c\n",
      "\n",
      ", c i t y c o r c a m d\n",
      "\n",
      "l i\n",
      "\n",
      "m , y l a g e m o n e l p s\n",
      "\n",
      ", e c i d n u a j\n",
      "\n",
      "a i m e a n a\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "m a e b\n",
      "\n",
      "R o t Q\n",
      "\n",
      ") 6 5 3 1 ( 1 6 3 1\n",
      "\n",
      "s i s e i o p o r h t y r e s y d\n",
      "\n",
      ", s e t y c i s s e d\n",
      "\n",
      "] 9 9 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "a m o h p m y l\n",
      "\n",
      "l l e c - T c i t s a l b o n u m m o g n a\n",
      "\n",
      "i\n",
      "\n",
      "i\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "g u p\n",
      "\n",
      "l\n",
      "\n",
      "l a r t n e c\n",
      "\n",
      "F o t S\n",
      "\n",
      ") 0 9 3 1 ( 6 9 3 1\n",
      "\n",
      "] 4 5 [\n",
      "\n",
      ". y r u n\n",
      "\n",
      "j\n",
      "\n",
      "i\n",
      "\n",
      "t n o\n",
      "\n",
      "i\n",
      "\n",
      "j\n",
      "\n",
      "c i t a m u a r t\n",
      "\n",
      "r o e t u c a m o r f e e r f\n",
      "\n",
      "s i t i r h t r a o e t s o\n",
      "\n",
      "l a i l i\n",
      "\n",
      "m a f\n",
      "\n",
      "c i\n",
      "\n",
      "m o r d n y s - n o n\n",
      "\n",
      "s i t i r h t r a o e t s o\n",
      "\n",
      "g n n e p o d e s a e r c e d\n",
      "\n",
      "i\n",
      "\n",
      "g u p\n",
      "\n",
      "l\n",
      "\n",
      "l a r e t a l\n",
      "\n",
      "W o t R\n",
      "\n",
      ") 8 9 3 1 ( 4 0 4 1\n",
      "\n",
      "s i t i r h t r a o e t s o t n o\n",
      "\n",
      "j\n",
      "\n",
      "l a e g n a l a h p r e t n\n",
      "\n",
      "i\n",
      "\n",
      "d n a s i t i r h t r a o e t s o d n a h e v i s o r e\n",
      "\n",
      ") 1 O Z E I P e s u o m\n",
      "\n",
      "(\n",
      "\n",
      "] 7 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "a i m e a n a c i t y l o m e a h\n",
      "\n",
      "l a t i n e g n o c\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "h c t a l\n",
      "\n",
      "R o t G\n",
      "\n",
      ") 0 1 4 1 ( 6 1 4 1\n",
      "\n",
      "] 9 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "o t - h c t a l\n",
      "\n",
      "V o t A\n",
      "\n",
      ") 1 5 4 1 ( 7 5 4 1\n",
      "\n",
      "p s a l c\n",
      "\n",
      "] 1 5 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "o t - h c t a l\n",
      "\n",
      "t f i h s e m a r f A o t V\n",
      "\n",
      ") 2 5 4 1 ( 8 5 4 1\n",
      "\n",
      "p s a l c\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "12\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 6 6 [\n",
      "\n",
      "a H F I N\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "o t - h c t a l\n",
      "\n",
      "V o t A\n",
      "\n",
      ") 7 8 4 1 ( 6 9 4 1\n",
      "\n",
      "p s a l c\n",
      "\n",
      ") t n a i r a v l a i t n e t o p (\n",
      "\n",
      "] 0 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "y t i l a m r o n b a s a e r c n a p y r a i l i\n",
      "\n",
      "b r e v i l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "o t - h c t a l\n",
      "\n",
      "V o t A\n",
      "\n",
      ") 8 8 4 1 ( 7 9 4 1\n",
      "\n",
      "p s a l c\n",
      "\n",
      "] 2 5 , 6 4 [\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p s a l c\n",
      "\n",
      "P o t G\n",
      "\n",
      ") 0 1 5 1 ( 9 1 5 1\n",
      "\n",
      ") ? D L G ( n o i t a m r o f l a m\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      "h t i\n",
      "\n",
      "w e b y a m\n",
      "\n",
      "(\n",
      "\n",
      "d n a K 0 1 9 1 Q\n",
      "\n",
      ")\n",
      "\n",
      "W 0 1 1 2 R\n",
      "\n",
      "] 9 9 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e l g n a - n e p o y r a m\n",
      "\n",
      "i r p t s n i a g a e v i t c e t o r p\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p s a l c\n",
      "\n",
      "H o t R\n",
      "\n",
      ") 8 1 5 1 ( 7 2 5 1\n",
      "\n",
      "a m o c u a l g\n",
      "\n",
      "] 3 5 , 1 5 [ n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i l\n",
      "\n",
      "H F I N\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p s a l c\n",
      "\n",
      "A o t S\n",
      "\n",
      ") 1 3 5 1 ( 0 4 5 1\n",
      "\n",
      "y c n e c u l s n a r t\n",
      "\n",
      "l a h c u N\n",
      "\n",
      ") S 3 1 5 C h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 1 6 , 1 5 [\n",
      "\n",
      ") a h c u n d n a t u p i c c o\n",
      "\n",
      "l a t e f ( g n n e k c i h t n i k s\n",
      "\n",
      "i\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p s a l c\n",
      "\n",
      "H o t R\n",
      "\n",
      ") 8 3 5 1 ( 7 4 5 1\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "D L G\n",
      "\n",
      ") V 7 2 2 2 A h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "n o i s u f f e l a i d r a c i r e p\n",
      "\n",
      "s k e e w 4 3 t a y r e v i l e d n a e r a s e a c\n",
      "\n",
      "] 3 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p s a l c\n",
      "\n",
      "G o t D\n",
      "\n",
      ") 7 5 5 1 ( 6 6 5 1\n",
      "\n",
      "] 3 5 , 1 5 [ n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p s a l c\n",
      "\n",
      "L o t V\n",
      "\n",
      ") 1 6 5 1 ( 0 7 5 1\n",
      "\n",
      ") R 9 2 6 1 G h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 3 5 , 1 5 [ n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "o t - p s a l c\n",
      "\n",
      "K 0 1 9 1 E h t i\n",
      "\n",
      "w\n",
      "\n",
      "( P o t Q\n",
      "\n",
      ") 2 8 5 1 ( 1 9 5 1\n",
      "\n",
      "8 U H T\n",
      "\n",
      ") T 0 1 1 2 R d n a\n",
      "\n",
      "] 0 0 1 [\n",
      "\n",
      "e n i r p o h t a z a , s d o r e t s o c i t r o c o c u g\n",
      "\n",
      "i\n",
      "\n",
      "i\n",
      "\n",
      ", e n i c i h c l o c\n",
      "\n",
      "e v i t c e n n o c e n u m m o t u a d n a s i t i l\n",
      "\n",
      "i\n",
      "\n",
      "u c s a v\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "o t - p s a l c\n",
      "\n",
      "h t i\n",
      "\n",
      "w e b y a m\n",
      "\n",
      "(\n",
      "\n",
      "M o t T\n",
      "\n",
      ") 8 0 6 1 ( 7 1 6 1\n",
      "\n",
      "e s a e s i d e u s s i t\n",
      "\n",
      "8 U H T\n",
      "\n",
      "e n e g r e h t o\n",
      "\n",
      ") s t n a i r a v\n",
      "\n",
      "] 3 5 , 1 5 [ n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "o t - p s a l c\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "( R o t G\n",
      "\n",
      ") 0 2 6 1 ( 9 2 6 1\n",
      "\n",
      "8 U H T\n",
      "\n",
      ") L 0 7 5 1 V\n",
      "\n",
      "] 0 7 [\n",
      "\n",
      "t n e r r u c e r\n",
      "\n",
      ", a i h t a n g o r c i\n",
      "\n",
      "m\n",
      "\n",
      ", s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      ", s e l e c o r d y h\n",
      "\n",
      ",\n",
      "\n",
      "g n\n",
      "\n",
      "i l l e w s k c e n d n a d a e h\n",
      "\n",
      ") ? D L G ( a m e d e h p m y l\n",
      "\n",
      "n o i s s e r p x e d e s a e r c e d\n",
      "\n",
      "o t - p s a l c\n",
      "\n",
      "e r u t a m e r p E\n",
      "\n",
      ") 1 2 6 1 ( 0 3 6 1\n",
      "\n",
      "/ x a r o h t o l y h c\n",
      "\n",
      ", a m e d e o\n",
      "\n",
      "l a t i n e g\n",
      "\n",
      ", s i s o b m o r h t n i e v p e e d\n",
      "\n",
      ", s i t i l\n",
      "\n",
      "u\n",
      "\n",
      "l l e c\n",
      "\n",
      "l a i c a f\n",
      "\n",
      "8 U H T\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      ", s V V\n",
      "\n",
      ", s r a e y 2 e g a t a s n o i s u f f e l a r u e l p\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b d n a s e c a r o h t o l y h c\n",
      "\n",
      "y a l e d\n",
      "\n",
      "l a t n e m p o l e v e d d\n",
      "\n",
      "l i\n",
      "\n",
      "m\n",
      "\n",
      ",\n",
      "\n",
      "m\n",
      "\n",
      "s i d o r y h t o p y h\n",
      "\n",
      "i\n",
      "\n",
      ", s s e n f a e d\n",
      "\n",
      "l a r u e n i r o s n e s\n",
      "\n",
      ") .\n",
      "\n",
      "m o h (\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "13\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "s e t y c o t a m o t s\n",
      "\n",
      "l a n o i s a c c o\n",
      "\n",
      "] 3 5 , 1 5 [ n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "o t - p s a l c\n",
      "\n",
      "e r u t a m e r p Y\n",
      "\n",
      ") 9 2 6 1 ( 8 3 6 1\n",
      "\n",
      "8 U H T\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ") l e d 1 4 L h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 6 8 [\n",
      "\n",
      "x a r o h t o l y h c\n",
      "\n",
      "D L G\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "W o t R\n",
      "\n",
      ") 1 7 6 1 ( 1 7 6 1\n",
      "\n",
      "a m e d e o b m\n",
      "\n",
      "i l\n",
      "\n",
      "r e w o\n",
      "\n",
      "l\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      ") H 0 1 0 1 R h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 6 9 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "a m e d e h p m y l\n",
      "\n",
      "m r a l a t i n e g n o c\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "V o t A\n",
      "\n",
      ") 8 7 6 1 ( 8 7 6 1\n",
      "\n",
      ") C 8 5 3 1 R h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 9 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s n o i t a m r o f l a m\n",
      "\n",
      "s u o n e v o i r e t r a n i a r b\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "V o t A\n",
      "\n",
      ") 4 8 6 1 ( 4 8 6 1\n",
      "\n",
      "] 7 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s i s o p y l o p s u o t a m o n e d a\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "M o t V\n",
      "\n",
      ") 2 1 7 1 ( 2 1 7 1\n",
      "\n",
      "] 1 0 1 [\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "D L G\n",
      "\n",
      "n i e t o r p d e s a e r c e d\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "W o t S\n",
      "\n",
      ") 1 2 7 1 ( 1 2 7 1\n",
      "\n",
      "e s i\n",
      "\n",
      "m e d\n",
      "\n",
      "l a t e f\n",
      "\n",
      "H F I N\n",
      "\n",
      ") .\n",
      "\n",
      "m o h ( n o i s s e r p x e\n",
      "\n",
      "n o i t a n m r e t y c n a n g e r p\n",
      "\n",
      "i\n",
      "\n",
      "] 3 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "M o t T\n",
      "\n",
      ") 2 3 7 1 ( 2 3 7 1\n",
      "\n",
      "] 2 0 1 , 7 6 [\n",
      "\n",
      "n o i s u f s n a r t d o o b f o y r o t s i h\n",
      "\n",
      "l\n",
      "\n",
      "s i s o p y l o p s u o t a m o n e d a\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "e r u t a m e r p Y\n",
      "\n",
      ") 3 6 7 1 ( 3 6 7 1\n",
      "\n",
      "n e g i t n a r E\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ") K 2 9 3 2 E h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 5 3 [\n",
      "\n",
      "a m o r g y h c i t s y c\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "Q o t E\n",
      "\n",
      ") 4 6 7 1 ( 4 6 7 1\n",
      "\n",
      "n r o b e v i l\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a t a n i r e p t u o h t i\n",
      "\n",
      "w\n",
      "\n",
      "r o h t i\n",
      "\n",
      "w\n",
      "\n",
      ") 8 0 8 d n a 2 8 7 h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s e r u t a e f\n",
      "\n",
      "c i h p r o m s y d\n",
      "\n",
      "y l a g e m o u c i r t n e v l a r e t a l i\n",
      "\n",
      "l\n",
      "\n",
      "b\n",
      "\n",
      ", y l y t c a d o n\n",
      "\n",
      "i l c e o t h t f i f\n",
      "\n",
      ",\n",
      "\n",
      "n i k s\n",
      "\n",
      "l a h c u n t n a d n u d e r\n",
      "\n",
      "] 1 6 , 1 5 [\n",
      "\n",
      "h t r i b f o y a d 1 n h t i\n",
      "\n",
      "i\n",
      "\n",
      "w y t i l a t r o m\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "t f i h s e m a r f S o t L\n",
      "\n",
      ") 0 9 7 1 ( 0 9 7 1\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "a m e d e o n i k s\n",
      "\n",
      ") C > G 1 + 9 4 0 7 . c\n",
      "\n",
      ". s n o i s u f f e l a r u e l p\n",
      "\n",
      "t n e m e c a l p t n u h s\n",
      "\n",
      "c i t o n m a o c a r o h t\n",
      "\n",
      "i\n",
      "\n",
      ",\n",
      "\n",
      "n o i s u f f e l a r u e l p f o e g a n i a r d e l d e e n\n",
      "\n",
      "n o i s u f s n a r t e n i r e t u a r t n\n",
      "\n",
      "i\n",
      "\n",
      "] 3 0 1 [\n",
      "\n",
      "c i n e g o h t a p - n o n e b o t d e t c i d e r p t n a i r a v\n",
      "\n",
      "r e d r o s i d\n",
      "\n",
      "l a t n e m p o l e v e d - o r u e n\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "I o t\n",
      "\n",
      "M\n",
      "\n",
      ") 1 9 7 1 ( 1 9 7 1\n",
      "\n",
      "] 0 5 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "8 U H T e d a l b\n",
      "\n",
      "C o t R\n",
      "\n",
      ") 7 9 7 1 ( 7 9 7 1\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "14\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 9 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s n o i t a m r o f l a m\n",
      "\n",
      "s u o n e v o i r e t r a n i a r b\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "Q o t E\n",
      "\n",
      ") 0 4 8 1 ( 8 2 8 1\n",
      "\n",
      "] 4 0 1 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "H o t R\n",
      "\n",
      ") 1 8 8 1 ( 4 6 8 1\n",
      "\n",
      "_ 2 9 4 2 E h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      ") n o i t a c i l\n",
      "\n",
      "p u d 3 9 4 2 L\n",
      "\n",
      "] 5 0 1 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c e d\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "n o i t e l e d K\n",
      "\n",
      ") 4 9 8 1 ( 7 7 8 1\n",
      "\n",
      ",\n",
      "\n",
      "n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "r e t s a f (\n",
      "\n",
      ") t n e r r u c\n",
      "\n",
      "s s e l\n",
      "\n",
      "] 0 5 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "D o t E\n",
      "\n",
      ") 3 1 9 1 ( 8 9 8 1\n",
      "\n",
      "] 9 5 , 1 5 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "t f i h s e m a r f K o t P\n",
      "\n",
      ") 1 2 9 1 ( 6 0 9 1\n",
      "\n",
      ") L 0 7 2 2 I\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 6 0 1 [\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a r e h p i r e p\n",
      "\n",
      "H F I N\n",
      "\n",
      "n i e t o r p d e s a e r c e d\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "t f i h s e m a r f E o t R\n",
      "\n",
      ") 4 2 9 1 ( 9 0 9 1\n",
      "\n",
      "e l e c o r d y h\n",
      "\n",
      "a m e d e h p m y l\n",
      "\n",
      "n o i s s e r p x e\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s e c a r o h t o l y h c\n",
      "\n",
      ") A > G 7 − 5 6 1 6 . c\n",
      "\n",
      "] 1 5 , 6 4 [\n",
      "\n",
      "] 2 5 [ a m e d e o\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "K o t E\n",
      "\n",
      ") 5 2 9 1 ( 0 1 9 1\n",
      "\n",
      "H F I N\n",
      "\n",
      "d n a P 1 9 5 1 Q h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      ")\n",
      "\n",
      "W 0 1 1 2 R\n",
      "\n",
      "] 7 0 1 , 6 4 [\n",
      "\n",
      "a m e d e o n i k s\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "W o t R\n",
      "\n",
      ") 1 4 9 1 ( 5 2 9 1\n",
      "\n",
      "x a r o h t o r d y h\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "H F I N\n",
      "\n",
      "2 2 3 2 E h t i\n",
      "\n",
      "w e b y a m\n",
      "\n",
      "(\n",
      "\n",
      "s e t i c s a\n",
      "\n",
      "e r u t a m e r p\n",
      "\n",
      "s k e e w 1 2 t a h t a e d e n i r e t u a r t n\n",
      "\n",
      "i\n",
      "\n",
      ") n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "e u s s i t\n",
      "\n",
      "s u o e n a t u c b u s d e n e k c i h t\n",
      "\n",
      "e s u f f i d\n",
      "\n",
      ", s e n a r b m e m\n",
      "\n",
      "t h g i r b d n a t h g\n",
      "\n",
      "i l\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      ",\n",
      "\n",
      "d e n e k c i h t\n",
      "\n",
      ",\n",
      "\n",
      "d e g r a l n e a t n e c a l p\n",
      "\n",
      "i l l i v d n a e t a l p c i n o i r o h c\n",
      "\n",
      "f o a m e d e o\n",
      "\n",
      "s l e s s e v d e t a l i\n",
      "\n",
      "d\n",
      "\n",
      ", a m e d e o , s i\n",
      "\n",
      "m r e d p e c i h p o r t a\n",
      "\n",
      "i\n",
      "\n",
      "t n e r r u c e r\n",
      "\n",
      ") . t e h (\n",
      "\n",
      "s i s o t y c o t a m o t s\n",
      "\n",
      "l a n r e t a p\n",
      "\n",
      "] 5 0 1 , 4 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "W\n",
      "\n",
      "r o Q o t R\n",
      "\n",
      ") 9 5 9 1 ( 3 4 9 1\n",
      "\n",
      "e r u\n",
      "\n",
      "l i a f\n",
      "\n",
      "w o r r a m e n o b d e t i r e h n\n",
      "\n",
      "i\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "d e w o l s (\n",
      "\n",
      "] 4 9 , 7 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "s i s o p y l o p s u o t a m o n e d a\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "C o t R\n",
      "\n",
      ") 1 7 9 1 ( 5 5 9 1\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "15\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "e m o r d n y s\n",
      "\n",
      "c i t s a l p s y d o l e y m\n",
      "\n",
      "] 0 8 [\n",
      "\n",
      "H F I N\n",
      "\n",
      "a m e d e h p m y l\n",
      "\n",
      "l a c i n a h c e m d e s a e r c e d\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "D o t G\n",
      "\n",
      ") 4 9 9 1 ( 8 7 9 1\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "y t i v i t i s n e s\n",
      "\n",
      ") Q 5 3 3 2 R h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "m u t o r c s d n a e c a f\n",
      "\n",
      ", ) t e e f y l n i a m\n",
      "\n",
      "(\n",
      "\n",
      "s b m\n",
      "\n",
      "i l\n",
      "\n",
      "f o a m e d e o\n",
      "\n",
      "s e c a r o h t o l y h c\n",
      "\n",
      ", s n o i s u f f e l a r u e l p\n",
      "\n",
      "e d i s\n",
      "\n",
      "t f e l\n",
      "\n",
      "s i t s e t d e d n e c s e d l a m\n",
      "\n",
      "s e t i c s a c i t a p e h i r e p\n",
      "\n",
      "s e r u t a e f\n",
      "\n",
      "c i h p r o m s y d o n\n",
      "\n",
      "] 1 7 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e c n a t s i s e r a i r a l a m\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "V o t A\n",
      "\n",
      ") 4 0 0 2 ( 8 8 9 1\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "d e w o l s (\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "F o t S\n",
      "\n",
      ") 0 1 0 2 ( 4 9 9 1\n",
      "\n",
      "] 8 0 1 , 8 6 , 6 4 [\n",
      "\n",
      "d e t a n m r e t y c n a n g e r p d n a n o i s u f f e l a i d r a c i r e p\n",
      "\n",
      "i\n",
      "\n",
      ": e s a c\n",
      "\n",
      "s e t i c s a l a t e f e h t n\n",
      "\n",
      "i\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "T r o N r o D o t A\n",
      "\n",
      ") 9 1 0 2 ( 3 0 0 2\n",
      "\n",
      "e c i d n u a j\n",
      "\n",
      "d n a r e d r o s i d c i t y l o m e a h f o y r o t s i h\n",
      "\n",
      "l a n r e t a p\n",
      "\n",
      "] 8 0 1 [\n",
      "\n",
      "s e t i c s a l a t e f\n",
      "\n",
      "h t i\n",
      "\n",
      "w e b y a m\n",
      "\n",
      "(\n",
      "\n",
      "l a m r o n y l t n e r a p p a a t n e c a l p\n",
      "\n",
      ") ] 8 0 1 [ S 1 5 2 2 P\n",
      "\n",
      "] 9 0 1 , 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "D r o I o t V\n",
      "\n",
      ") 2 2 0 2 ( 6 0 0 2\n",
      "\n",
      "c 1 A b H h t i\n",
      "\n",
      "w n o i t a i c o s s a e v i t a g e n\n",
      "\n",
      "% 5 . 6 >\n",
      "\n",
      "; c 1 A n b o g o m e a h d e t a c y l g (\n",
      "\n",
      "i\n",
      "\n",
      "l\n",
      "\n",
      ") s e t e b a i d s e t a c i d n\n",
      "\n",
      "i\n",
      "\n",
      "] 0 1 1 , 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "L o t\n",
      "\n",
      "M\n",
      "\n",
      ") 3 2 0 2 ( 7 0 0 2\n",
      "\n",
      "] 0 1 1 , 9 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "I o t T\n",
      "\n",
      ") 0 3 0 2 ( 4 1 0 2\n",
      "\n",
      "] 7 9 , 8 6 , 6 4 [\n",
      "\n",
      "a i m e a l a k r e p y h o d u e s p\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "] 8 6 [ V r o ] 7 9 [ T o t A\n",
      "\n",
      ") 6 3 0 2 ( 0 2 0 2\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a t a n i r e p o n\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "d e w o l s (\n",
      "\n",
      ". ) ] 8 6 [\n",
      "\n",
      "t e h V o t A (\n",
      "\n",
      "T o t A\n",
      "\n",
      "] 8 3 , 7 3 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e s a e s i d e v l a v c i t r o a d p s u c i b\n",
      "\n",
      "i\n",
      "\n",
      "n o i t a v i t c a d e s a e r c e d\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "H o t Y\n",
      "\n",
      ") 8 3 0 2 ( 2 2 0 2\n",
      "\n",
      "] 1 1 1 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "V o t L\n",
      "\n",
      ") 9 3 0 2 ( 3 2 0 2\n",
      "\n",
      "] 1 9 , 9 3 , 8 3 [\n",
      "\n",
      "x a r o h t o l y h c\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b d n a s p o r d y h\n",
      "\n",
      ") ? D L G ( a i s a l p s y d c i t a h p m y l\n",
      "\n",
      "e c a f r u s d e s a e r c e d\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "R o t G\n",
      "\n",
      ") 5 4 0 2 ( 9 2 0 2\n",
      "\n",
      "e c a f d n a o s r o t\n",
      "\n",
      ", s g e l\n",
      "\n",
      "f o a m e d e h p m y l\n",
      "\n",
      "m u t o r c s d n a s g e l\n",
      "\n",
      "f o g n\n",
      "\n",
      "i l l e w s\n",
      "\n",
      "N d n a n o i s s e r p x e\n",
      "\n",
      "3 5 1 1 S h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      ") g n\n",
      "\n",
      "i l\n",
      "\n",
      "b i s e n o ( n o i t a n e g y x o e n a r b m e m\n",
      "\n",
      "l a e r o p r o c a r t x e g n d u l c n\n",
      "\n",
      "i\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "i e r a c e v i s n e t n\n",
      "\n",
      "i\n",
      "\n",
      "n o i t a l y s o c y l g d e k n\n",
      "\n",
      "i l\n",
      "\n",
      "n o i t a d a r g e d n i e t o r p\n",
      "\n",
      "e r u t a m e r p\n",
      "\n",
      ") n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "16\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 2 1 1 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "a i m e s s a l a h t a t e b\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "M o t V\n",
      "\n",
      ") 5 8 0 2 ( 9 6 0 2\n",
      "\n",
      "] 8 8 , 7 8 [\n",
      "\n",
      "h t r i b\n",
      "\n",
      "l l i t s\n",
      "\n",
      "D L G\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "Q o t K\n",
      "\n",
      ") 6 8 0 2 ( 0 7 0 2\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a r o p r o c d n a l a i c a f\n",
      "\n",
      "H F I N\n",
      "\n",
      "9 6 0 1 W h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "y l a h p e c n e o r c a m\n",
      "\n",
      "e r u t a m e r p\n",
      "\n",
      "s g n u\n",
      "\n",
      "l\n",
      "\n",
      "c i t s a l p o p y h\n",
      "\n",
      ") n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "n o i t a z i l a n a c\n",
      "\n",
      "l a n m u\n",
      "\n",
      "i\n",
      "\n",
      "l\n",
      "\n",
      "c i t a h p m y l\n",
      "\n",
      "f o e r u\n",
      "\n",
      "l i a f\n",
      "\n",
      "i l l i v c i n o i r o h c\n",
      "\n",
      "c i p o c s o r d y h\n",
      "\n",
      "n o i s u f r e p l a m\n",
      "\n",
      "r a l u c s a v l a t e f\n",
      "\n",
      "] 5 0 1 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "9 U H T e d a l b\n",
      "\n",
      "G o t R\n",
      "\n",
      ") 4 0 1 2 ( 8 8 0 2\n",
      "\n",
      "d o h s e r h t d e c u d e r (\n",
      "\n",
      "l\n",
      "\n",
      ") n o i t a v i t c a r o f\n",
      "\n",
      "n o i s s e r p x e d e s a e r c e d\n",
      "\n",
      "r a l u\n",
      "\n",
      "l l e c a r t n i (\n",
      "\n",
      ") n o i t a z i l a c o\n",
      "\n",
      "l\n",
      "\n",
      "] 4 1 1 , 3 1 1 , 2 5 , 0 5 , 6 4 [\n",
      "\n",
      "] 2 5 [ a m e d e o\n",
      "\n",
      "S H D\n",
      "\n",
      "o t R ( y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "r o h c n a\n",
      "\n",
      "Q r o W o t R\n",
      "\n",
      ") 6 2 1 2 ( 0 1 1 2\n",
      "\n",
      "c 1 A b H\n",
      "\n",
      ")\n",
      "\n",
      "W\n",
      "\n",
      "d n a P 1 9 5 1 Q h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "H F I N\n",
      "\n",
      ") H F I N n\n",
      "\n",
      "K 0 1 9 1 E\n",
      "\n",
      "] 7 9 , 8 6 , 6 4 [\n",
      "\n",
      "a i m e a l a k r e p y h o d u e s p\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "r o h c n a\n",
      "\n",
      "M o t T\n",
      "\n",
      ") 2 4 1 2 ( 7 2 1 2\n",
      "\n",
      "c 1 A b H\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "d e w o l s (\n",
      "\n",
      "] 7 9 [\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "r o h c n a\n",
      "\n",
      "R o t C\n",
      "\n",
      ") 7 6 1 2 ( 1 5 1 2\n",
      "\n",
      "] 4 6 [\n",
      "\n",
      "H F I N\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "r o h c n a\n",
      "\n",
      "t f i h s e m a r f E o t K\n",
      "\n",
      ") 2 7 1 2 ( 7 5 1 2\n",
      "\n",
      "y c n a n g e r p f o n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "l a c i d e m\n",
      "\n",
      ") . t e h - c (\n",
      "\n",
      "a m e d e h p m y l\n",
      "\n",
      ", a i h t a n g o r c i\n",
      "\n",
      "m\n",
      "\n",
      ", s r a e c i t s a l p s y d\n",
      "\n",
      ", s e r a n d e t r e v e t n a\n",
      "\n",
      "a m e d e o d n a y h p o r t r e p y h\n",
      "\n",
      "l a t n e c a l p\n",
      "\n",
      "x a r o h t o l y h c d n a n o i s u f f e l a r u e l p\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      ", s o n m a r d y h\n",
      "\n",
      "i\n",
      "\n",
      ", a m e d e o n i k s\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "r o h c n a\n",
      "\n",
      "L o t P\n",
      "\n",
      ") 6 7 1 2 ( 0 6 1 2\n",
      "\n",
      "] 8 6 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "x i l e h r e t u o\n",
      "\n",
      "9 6 1 2 o t 6 6 1 2\n",
      "\n",
      ") 2 8 1 2 ( 6 6 1 2\n",
      "\n",
      "n o i t e l e d\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "17\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 5 1 1 [\n",
      "\n",
      "n o i t a r t n e c n o c n b o g o m e a h r a l u c s u p r o c n a e m\n",
      "\n",
      "i\n",
      "\n",
      "l\n",
      "\n",
      "r e h g h\n",
      "\n",
      "i\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "x i l e h r e t u o\n",
      "\n",
      "n o i t e l e d K\n",
      "\n",
      ") 5 8 1 2 ( 9 6 1 2\n",
      "\n",
      "] 0 7 [\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "a m e d e h p m y l\n",
      "\n",
      "n o i s s e r p x e d e s a e r c e d\n",
      "\n",
      "x i l e h r e t u o\n",
      "\n",
      "F o t V\n",
      "\n",
      ") 7 8 1 2 ( 1 7 1 2\n",
      "\n",
      "s e t i c s a\n",
      "\n",
      "H F I N\n",
      "\n",
      "e c i l\n",
      "\n",
      "p s\n",
      "\n",
      "r o n o d h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "e r a l u n n a a m o u n a r g\n",
      "\n",
      "l\n",
      "\n",
      ", a i s a t c e i g n a h p m y l\n",
      "\n",
      "l a n i t s e t n\n",
      "\n",
      "i\n",
      "\n",
      ", x u l f e r\n",
      "\n",
      "l a e g a h p o s e o - o r t s a g\n",
      "\n",
      ") t n a i r a v e t i s\n",
      "\n",
      "s i s o\n",
      "\n",
      "i l\n",
      "\n",
      "o c s\n",
      "\n",
      "t e i d t a f -\n",
      "\n",
      "w o\n",
      "\n",
      "l\n",
      "\n",
      "n o d e v o r p m\n",
      "\n",
      "i\n",
      "\n",
      "d n a y l d p a r d e v l o s e r a m e d e o\n",
      "\n",
      "i\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "x i l e h r e t u o\n",
      "\n",
      "I o t L\n",
      "\n",
      ") 8 0 2 2 ( 2 9 1 2\n",
      "\n",
      "] 3 4 [\n",
      "\n",
      "y t i x a l\n",
      "\n",
      "l l a w\n",
      "\n",
      "l a n m o d b a\n",
      "\n",
      "i\n",
      "\n",
      "e m o r d n y s y l l e b e n u r p\n",
      "\n",
      "g n n e p o d e s a e r c e d\n",
      "\n",
      "i\n",
      "\n",
      "p a c\n",
      "\n",
      "L o t S\n",
      "\n",
      ") 1 1 2 2 ( 5 9 1 2\n",
      "\n",
      "s b i r d e r a l f\n",
      "\n",
      "1 O Z E I P e n i r u m\n",
      "\n",
      "(\n",
      "\n",
      "e c a f r e t n i (\n",
      "\n",
      ") R 3 5 2 G h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s e e n k d e l p m d\n",
      "\n",
      "i\n",
      "\n",
      "r e d d a l b d e t a l i\n",
      "\n",
      "r e t e r u o r d y h\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "d\n",
      "\n",
      "b\n",
      "\n",
      ") l e d o m\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "r e t u o\n",
      "\n",
      ") x i l e h\n",
      "\n",
      "s e r u d e c o r p r e h t o d n a e g a n i a r d r e d d a l b y r a n i r u\n",
      "\n",
      "l a c i g r u s\n",
      "\n",
      "s i s o r h p e n o r d y h\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "a i d r a c y h c a t\n",
      "\n",
      "r a l u c i r t n e v a r p u s\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "F o t V\n",
      "\n",
      ") 7 1 2 2 ( 1 0 2 2\n",
      "\n",
      "] 7 1 1 , 6 1 1 , 5 0 1 , 7 9 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "p a c\n",
      "\n",
      ") . t e h ( R o t\n",
      "\n",
      "M\n",
      "\n",
      ") 1 4 2 2 ( 5 2 2 2\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "d e w o l s (\n",
      "\n",
      "] 1 5 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "V o t A\n",
      "\n",
      ") 3 4 2 2 ( 7 2 2 2\n",
      "\n",
      ") H 7 4 5 1 R h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 8 1 1 , 0 7 [\n",
      "\n",
      "H F I N\n",
      "\n",
      ") ? D L G ( a m e d e h p m y l\n",
      "\n",
      "n o i s s e r p x e d e s a e r c e d\n",
      "\n",
      "p a c\n",
      "\n",
      "e r u t a m e r p Q\n",
      "\n",
      ") 4 4 2 2 ( 8 2 2 2\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "5 5 7 E h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a t i b r o i r e p d n a s n o i s u f f e l a r u e l p h t i\n",
      "\n",
      "s k e e w 4 3 t a o r e t u n\n",
      "\n",
      "i\n",
      "\n",
      "h t a e d\n",
      "\n",
      "m g a r h p a i d e h t\n",
      "\n",
      "f o a i s a l p o y m a\n",
      "\n",
      "w a m e d e o d e z i l a r e n e g d\n",
      "\n",
      "l i\n",
      "\n",
      "m\n",
      "\n",
      "e r u t a m e r p\n",
      "\n",
      "r o n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "A 8 5 4 1 V\n",
      "\n",
      ") t f i h s e m a r f\n",
      "\n",
      "s e c a r o h t o l y h c / x a r o h t o l y h c\n",
      "\n",
      "t c e f e d\n",
      "\n",
      "l a t p e s\n",
      "\n",
      "l a i r t a , a i h t a n g o r c i\n",
      "\n",
      "m\n",
      "\n",
      ", s d o f\n",
      "\n",
      "l\n",
      "\n",
      "c i h t n a c i p e , s r a e e l p m\n",
      "\n",
      "i s d e p p u c\n",
      "\n",
      "x u l f e r\n",
      "\n",
      "l a e g a h p o s e o - o r t s a g\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "18\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "y l a g e m o n e l p s\n",
      "\n",
      ", a m e d e o\n",
      "\n",
      "l a t i n e g\n",
      "\n",
      ",\n",
      "\n",
      "m u t a v a c x e s u t c e p\n",
      "\n",
      ", e r u t a t s\n",
      "\n",
      "t r o h s\n",
      "\n",
      "] 2 0 1 [\n",
      "\n",
      "e s i\n",
      "\n",
      "m e d\n",
      "\n",
      "l a t a n t s o p\n",
      "\n",
      ",\n",
      "\n",
      "n o i t c i r t s e r h t w o r g\n",
      "\n",
      "l a t e f\n",
      "\n",
      ",\n",
      "\n",
      "H F I N\n",
      "\n",
      "n e g i t n a r E\n",
      "\n",
      "e v i s n o p s e r 1 a d o Y\n",
      "\n",
      "p a c\n",
      "\n",
      ") .\n",
      "\n",
      "m o h ( K r o R\n",
      "\n",
      ") 1 6 2 2 ( 5 4 2 2\n",
      "\n",
      "D L G\n",
      "\n",
      "] 8 0 1 [\n",
      "\n",
      "n o i s u f f e l a i d r a c i r e p\n",
      "\n",
      "s e t i c s a l a t e f\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "S o t P\n",
      "\n",
      ") 7 6 2 2 ( 1 5 2 2\n",
      "\n",
      "d e t a n m r e t y c n a n g e r p\n",
      "\n",
      "i\n",
      "\n",
      ") D 3 0 0 2 A h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "e c i d n u a j\n",
      "\n",
      "d n a r e d r o s i d c i t y l o m e a h f o y r o t s i h\n",
      "\n",
      "l a n r e t a p\n",
      "\n",
      "] 0 5 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "I o t K\n",
      "\n",
      ") 2 8 2 2 ( 6 6 2 2\n",
      "\n",
      "] 0 8 , 9 5 [\n",
      "\n",
      "H F I N o n\n",
      "\n",
      "D L G\n",
      "\n",
      "l a c i n a h c e m d e s a e r c e d\n",
      "\n",
      "p a c\n",
      "\n",
      "T o t\n",
      "\n",
      "I\n",
      "\n",
      ") 6 8 2 2 ( 0 7 2 2\n",
      "\n",
      "m u t o r c s d n a s b m\n",
      "\n",
      "i l\n",
      "\n",
      "r e w o\n",
      "\n",
      "l\n",
      "\n",
      "f o a m e d e o\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b t e s n o - e t a l\n",
      "\n",
      "y t i v i t i s n e s\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "r o .\n",
      "\n",
      "m o h (\n",
      "\n",
      "s e t i c s a , s e c a r o h t o l y h c\n",
      "\n",
      ", s n o i s u f f e l a r u e l p\n",
      "\n",
      ", s e l e o c o r d y h\n",
      "\n",
      "r o ] 0 8 [ V 9 2 8 E\n",
      "\n",
      "e s a e s i d\n",
      "\n",
      "l a r u e l p d n a s e c a r o h t o l y h c o t e u d h t a e d\n",
      "\n",
      ", ) e g a n i a r d s u o e n a t u c r e p g n i r i u q e r ( n o i s u f f e l a i d r a c i r e p r o j a m\n",
      "\n",
      "r o d\n",
      "\n",
      "l i\n",
      "\n",
      "m\n",
      "\n",
      "K 6 0 9 1 P h t i\n",
      "\n",
      "w\n",
      "\n",
      "] 9 5 [\n",
      "\n",
      "t f i h s e m a r f\n",
      "\n",
      "e d a n o p m a t\n",
      "\n",
      "c a i d r a c o t g n i s s e r g o r p\n",
      "\n",
      "g n n e k c i h t\n",
      "\n",
      "i\n",
      "\n",
      "l a i d r a c i r e p\n",
      "\n",
      "e c n a r e l o t e s i c r e x e d e c u d e r d n a s s e n s s e l h t a e r b\n",
      "\n",
      "a m e z c e h t i\n",
      "\n",
      "w s V V\n",
      "\n",
      "a i n e p o e t s o , e r u t a t s\n",
      "\n",
      "t r o h s\n",
      "\n",
      ", y l a h p e c o r c i\n",
      "\n",
      "m d\n",
      "\n",
      "l i\n",
      "\n",
      "m\n",
      "\n",
      "] 9 1 1 , 3 1 1 , 6 4 [\n",
      "\n",
      "y r t s e c n a n a e p o r u E h t i\n",
      "\n",
      "w d e r a p m o c\n",
      "\n",
      "s n a i s A h t u o S n\n",
      "\n",
      "i y c n e u q e r f d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "M o t L\n",
      "\n",
      ") 3 9 2 2 ( 7 7 2 2\n",
      "\n",
      "c 1 A b H\n",
      "\n",
      "] 5 0 1 , 2 5 , 1 5 [\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "p a c\n",
      "\n",
      "H o t R\n",
      "\n",
      ") 8 1 3 2 ( 2 0 3 2\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e s a e r c n i (\n",
      "\n",
      "r e h t o r o I 9 9 4 V h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "D L G\n",
      "\n",
      "l a c i n a h c e m\n",
      "\n",
      "o n\n",
      "\n",
      ", y t i v i t i s n e s\n",
      "\n",
      "a i m e s s a l a h t - β\n",
      "\n",
      ") t n a i r a v\n",
      "\n",
      "f o e g n a h c\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "P r O\n",
      "\n",
      "n o i s s e r p x e d e s a e r c e d\n",
      "\n",
      "r a l u\n",
      "\n",
      "l l e c a r t n i (\n",
      "\n",
      ") n o i t a z i l a c o\n",
      "\n",
      "l\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "E o t Q\n",
      "\n",
      ") 4 2 3 2 ( 8 0 3 2\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "19\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 7 0 1 [\n",
      "\n",
      "a m e d e o n i k s\n",
      "\n",
      "D L G\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "e r u t a m e r p E\n",
      "\n",
      ") 8 3 3 2 ( 2 2 3 2\n",
      "\n",
      "x a r o h t o r d y h\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      "H F I N\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "s e t i c s a\n",
      "\n",
      ")\n",
      "\n",
      "W 5 2 9 1 R h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s k e e w 1 2 t a h t a e d e n i r e t u a r t n\n",
      "\n",
      "i\n",
      "\n",
      "e u s s i t\n",
      "\n",
      "s u o e n a t u c b u s d e n e k c i h t\n",
      "\n",
      "e s u f f i d\n",
      "\n",
      ", s e n a r b m e m\n",
      "\n",
      "t h g i r b d n a t h g\n",
      "\n",
      "i l\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      ",\n",
      "\n",
      "d e n e k c i h t\n",
      "\n",
      ",\n",
      "\n",
      "d e g r a l n e a t n e c a l p\n",
      "\n",
      "i l l i v d n a e t a l p c i n o i r o h c\n",
      "\n",
      "f o a m e d e o\n",
      "\n",
      "s l e s s e v d e t a l i\n",
      "\n",
      "d\n",
      "\n",
      ", a m e d e o , s i\n",
      "\n",
      "m r e d p e c i h p o r t a\n",
      "\n",
      "i\n",
      "\n",
      ") . t e h (\n",
      "\n",
      "s i s o t y c o t a m o t s\n",
      "\n",
      "l a n r e t a p t n e r r u c e r\n",
      "\n",
      "] 9 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "T o t K\n",
      "\n",
      ") 9 3 3 2 ( 3 2 3 2\n",
      "\n",
      "] 0 8 [\n",
      "\n",
      "H F I N\n",
      "\n",
      ") ? D L G ( a m e d e h p m y l\n",
      "\n",
      "l a c i n a h c e m d e s a e r c e d\n",
      "\n",
      "p a c\n",
      "\n",
      "Q o t R\n",
      "\n",
      ") 1 5 3 2 ( 5 3 3 2\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "y t i v i t i s n e s\n",
      "\n",
      ") D 8 7 9 1 G h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "m u t o r c s d n a e c a f\n",
      "\n",
      ", ) t e e f y l n i a m\n",
      "\n",
      "(\n",
      "\n",
      "s b m\n",
      "\n",
      "i l\n",
      "\n",
      "f o a m e d e o\n",
      "\n",
      "s e c a r o h t o l y h c\n",
      "\n",
      ", s n o i s u f f e l a r u e l p\n",
      "\n",
      "e d i s\n",
      "\n",
      "t f e l\n",
      "\n",
      "s i t s e t d e d n e c s e d l a m\n",
      "\n",
      "s e t i c s a c i t a p e h i r e p\n",
      "\n",
      "s e r u t a e f\n",
      "\n",
      "c i h p r o m s y d o n\n",
      "\n",
      "] 3 5 , 1 5 [ n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i L\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "W o t R\n",
      "\n",
      ") 2 5 3 2 ( 6 3 3 2\n",
      "\n",
      ")\n",
      "\n",
      "M 8 9 5 V h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 4 6 [\n",
      "\n",
      "y l a g e m o t a p e h\n",
      "\n",
      ", y c n a n g e r p f o n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "l a c i d e m\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "e r u t a m e r p Q\n",
      "\n",
      ") 9 8 3 2 ( 3 7 3 2\n",
      "\n",
      "n o i t a r u t a m\n",
      "\n",
      "s u o\n",
      "\n",
      "l l i v d e y a l e d d n a y h p o r t r e p y h\n",
      "\n",
      "l a t n e c a l p\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "x a r o h t o l y h c d n a n o i s u f f e l a r u e l p\n",
      "\n",
      ", e t i c s a\n",
      "\n",
      "] 0 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e s a e s i d r e v i l\n",
      "\n",
      "c i l\n",
      "\n",
      "o h o c l a\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "Q o t R\n",
      "\n",
      ") 6 0 4 2 ( 0 9 3 2\n",
      "\n",
      "] 2 0 1 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "n e g i t n a r E\n",
      "\n",
      "e v i s n o p s e r 1 a d o Y\n",
      "\n",
      "p a c\n",
      "\n",
      "K o t E\n",
      "\n",
      ") 8 0 4 2 ( 2 9 3 2\n",
      "\n",
      "r o S 4 9 3 2 G h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "e r u t a m e r p\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      ") 3 6 7 1 Y\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "20\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 2 0 1 , 0 9 , 0 5 [\n",
      "\n",
      "t n a f n\n",
      "\n",
      "i\n",
      "\n",
      "n r o b\n",
      "\n",
      "l l i t s 1 f o y r o t s i h\n",
      "\n",
      "S H D\n",
      "\n",
      "e v i s n o p s e r 1 a d o Y\n",
      "\n",
      "p a c\n",
      "\n",
      "c r o .\n",
      "\n",
      "m o h ( S o t G\n",
      "\n",
      ") 1 1 4 2 ( 4 9 3 2\n",
      "\n",
      "n e g i t n a r E\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      ". t e h\n",
      "\n",
      "r o V 5 9 3 2 A\n",
      "\n",
      ") S 9 9 3 2 G\n",
      "\n",
      "] 3 5 , 1 5 [ n\n",
      "\n",
      "i\n",
      "\n",
      "d e t s i L\n",
      "\n",
      "H F I N\n",
      "\n",
      "D L G\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "V o t A\n",
      "\n",
      ") 2 1 4 2 ( 5 9 3 2\n",
      "\n",
      "] 0 9 [ n e g i t n a r E\n",
      "\n",
      ") E 4 1 1 1 Q h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "] 0 9 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "n e g i t n a r E\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      ". t e h - c ( S o t G\n",
      "\n",
      ") 6 1 4 2 ( 9 9 3 2\n",
      "\n",
      ") S 4 9 3 2 G\n",
      "\n",
      "] 0 2 1 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e s a e s i d V V\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "e r u t a m e r p E\n",
      "\n",
      ") 8 2 4 2 ( 2 0 4 2\n",
      "\n",
      "n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "] 2 0 1 [\n",
      "\n",
      "s e i c n a n g e r p 6 f o 1 n\n",
      "\n",
      "i\n",
      "\n",
      "n o i t a t s e g s k e e w 6 2 t a h t a e d e n i r e t u a r t n\n",
      "\n",
      "d n a H F I N\n",
      "\n",
      "n e g i t n a r E\n",
      "\n",
      "e v i s n o p s e r 1 a d o Y\n",
      "\n",
      "p a c\n",
      "\n",
      ") .\n",
      "\n",
      "m o h ( K r o Q o t E\n",
      "\n",
      ") 3 3 4 2 ( 7 0 4 2\n",
      "\n",
      "] 0 7 [\n",
      "\n",
      ") e r e v e s (\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "l a r e t a l i\n",
      "\n",
      "b\n",
      "\n",
      ") D L G ( a m e d e h p m y l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "L o t P\n",
      "\n",
      ") 6 5 4 2 ( 0 3 4 2\n",
      "\n",
      "s d o f\n",
      "\n",
      "l\n",
      "\n",
      "c i h t n a c i p e , y a d 1 n\n",
      "\n",
      "i\n",
      "\n",
      "d e v l o s e r a m e d e o , a m e d e o d e z i l a r e n e g\n",
      "\n",
      "H F I N\n",
      "\n",
      "e c i l\n",
      "\n",
      "p s\n",
      "\n",
      "r o n o d h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "s e t y c o r e h p s\n",
      "\n",
      ", s i t i l\n",
      "\n",
      "u\n",
      "\n",
      "l l e c\n",
      "\n",
      ") t n a i r a v e t i s\n",
      "\n",
      "s k e e w 4 e g a t a d e i d\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "R o t G\n",
      "\n",
      ") 9 5 4 2 ( 3 3 4 2\n",
      "\n",
      "] 1 2 1 [\n",
      "\n",
      "s e i t i s o c i r a v s u o n e v , y l a m o n a e v l a v s u o n e v\n",
      "\n",
      "a m e d e h p m y l\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "p a c\n",
      "\n",
      "E o t G\n",
      "\n",
      ") 5 6 4 2 ( 9 3 4 2\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "r o .\n",
      "\n",
      "m o h (\n",
      "\n",
      "5 2 1 1 Q\n",
      "\n",
      "e r u t a m e r p\n",
      "\n",
      ") n o i t a n m r e t\n",
      "\n",
      "i\n",
      "\n",
      "1 1 , 6 1 1 , 5 0 1 , 8 9 , 7 9 , 8 6 , 0 5 , 6 4 [\n",
      "\n",
      "s e t y c i s s e d\n",
      "\n",
      ", a i m e a n a c i t y l o m e a h c i n o r h c\n",
      "\n",
      ", c i t y c o r c a m d\n",
      "\n",
      "l i\n",
      "\n",
      "m\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "x i l e h r e n n\n",
      "\n",
      "i\n",
      "\n",
      "H o t R\n",
      "\n",
      ") 2 8 4 2 ( 6 5 4 2\n",
      "\n",
      "] 7\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "d e w o l s (\n",
      "\n",
      "] 5 0 1 , 7 9 [\n",
      "\n",
      "] 0 7 [\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "H F I N\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "x i l e h r e n n\n",
      "\n",
      "i\n",
      "\n",
      "H r o C o t R\n",
      "\n",
      ") 2 8 4 2 ( 6 5 4 2\n",
      "\n",
      "s o n m a r d y h y l o p\n",
      "\n",
      "i\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "d n a M 9 3 9 L h t i\n",
      "\n",
      "w\n",
      "\n",
      "(\n",
      "\n",
      "h t r i b t a a m e d e o d e z i l a r e n e g\n",
      "\n",
      ") L 8 5 4 2 F\n",
      "\n",
      "s e c a r o h t o l y h c / x a r o h t o l y h c\n",
      "\n",
      ", s e t y c o t a m o t s\n",
      "\n",
      "l a n o i s a c c o ,\n",
      "\n",
      "h t r i b t a d o r y h t o p y h\n",
      "\n",
      "i\n",
      "\n",
      ", x u l f e r\n",
      "\n",
      "l a e g a h p o s e o - o r t s a g\n",
      "\n",
      "y l l e b e n u r p\n",
      "\n",
      ", a m e d e o\n",
      "\n",
      "l a t i b r o i r e p\n",
      "\n",
      ", k c e n d e b b e w\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "21\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 2 2 1 , 0 7 , 2 6 , 4 5 , 6 4 [\n",
      "\n",
      "s n o i s u f f e l a r u e l p\n",
      "\n",
      "H F I N\n",
      "\n",
      "g n n e p o d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "i\n",
      "\n",
      "x i l e h r e n n\n",
      "\n",
      "i\n",
      "\n",
      "L o t F\n",
      "\n",
      ") 4 8 4 2 ( 8 5 4 2\n",
      "\n",
      "s o n m a r d y h y l o P\n",
      "\n",
      "i\n",
      "\n",
      "a m e d e o\n",
      "\n",
      ") 1 O Z E I P e s u o m\n",
      "\n",
      "(\n",
      "\n",
      "h t i\n",
      "\n",
      "w e b y a m\n",
      "\n",
      "(\n",
      "\n",
      ", k c e n d e b b e w\n",
      "\n",
      ", s e c a r o h t o l y h c / x a r o h t o l y h c\n",
      "\n",
      ",\n",
      "\n",
      "h t r i b t a a m e d e o d e z i l a r e n e g\n",
      "\n",
      "y l l e b e n u r p\n",
      "\n",
      ", a m e d e o\n",
      "\n",
      "l a t i b r o i r e p\n",
      "\n",
      "y r e g r u s g n i r i u q e r\n",
      "\n",
      "s i t i r h t r a o e t s o p h\n",
      "\n",
      "i\n",
      "\n",
      "S H D\n",
      "\n",
      "d n a M 9 3 9 L\n",
      "\n",
      ") C 6 5 4 2 R\n",
      "\n",
      "h t r i b t a d o r y h t o p y h\n",
      "\n",
      "i\n",
      "\n",
      ", x u l f e r\n",
      "\n",
      "l a e g a h p o s e o - o r t s a g\n",
      "\n",
      "s e t y c o t a m o t s\n",
      "\n",
      "l a n o i s a c c o\n",
      "\n",
      "] 4 8 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "x i l e h r e n n\n",
      "\n",
      "i\n",
      "\n",
      "K o t E\n",
      "\n",
      ") 7 8 4 2 ( 1 6 4 2\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "x i l e h r e n n\n",
      "\n",
      "i\n",
      "\n",
      "P o t H\n",
      "\n",
      ") 0 9 4 2 ( 4 6 4 2\n",
      "\n",
      "] 3 1 1 , 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "? c 1 A b H\n",
      "\n",
      ". S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "M o t V\n",
      "\n",
      ") 0 0 5 2 ( 4 7 4 2\n",
      "\n",
      "n i a m o d\n",
      "\n",
      "] 5 0 1 , 8 6 , 9 4 , 6 4 [\n",
      "\n",
      "H F I N\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "( G r o Q o t R\n",
      "\n",
      ") 4 1 5 2 ( 8 8 4 2\n",
      "\n",
      "H F I N\n",
      "\n",
      "d o h s e r h t d e c u d e r (\n",
      "\n",
      "l\n",
      "\n",
      "n i a m o d\n",
      "\n",
      ") S 8 1 7 G\n",
      "\n",
      ") n o i t a v i t c a r o f\n",
      "\n",
      "n o i s s e r p x e d e s a e r c e d\n",
      "\n",
      "r a l u\n",
      "\n",
      "l l e c a r t n i (\n",
      "\n",
      ") n o i t a z i l a c o\n",
      "\n",
      "l\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "D o t Q\n",
      "\n",
      ") 5 1 5 2 ( 9 8 4 2\n",
      "\n",
      "n i a m o d\n",
      "\n",
      "] 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "W o t R\n",
      "\n",
      ") 7 1 5 2 ( 1 9 4 2\n",
      "\n",
      "n i a m o d\n",
      "\n",
      "] 0 1 1 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "V o t E\n",
      "\n",
      ") 0 2 5 2 ( 4 9 4 2\n",
      "\n",
      "n i a m o d\n",
      "\n",
      "] 9 7 , 9 4 , 6 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "h t i\n",
      "\n",
      "w\n",
      "\n",
      "( n o i t a c i l\n",
      "\n",
      "p u d\n",
      "\n",
      "6 9 4 2 d n a 5 9 4 2\n",
      "\n",
      "n i a m o d\n",
      "\n",
      ")\n",
      "\n",
      "M 4 7 4 2 V\n",
      "\n",
      ") 2 2 5 2 d n a 1 2 5 2 (\n",
      "\n",
      "] 3 2 1 , 0 1 1 , 5 0 1 , 7 9 [\n",
      "\n",
      "a m e d e o\n",
      "\n",
      "l a t a n i r e p\n",
      "\n",
      "S H D\n",
      "\n",
      "y t i v i t c a d e s a e r c n\n",
      "\n",
      "i\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "E L E o t E\n",
      "\n",
      ") 2 2 5 2 ( 6 9 4 2\n",
      "\n",
      "a i m e a l a k r e p y h o d u e s p\n",
      "\n",
      "H F I N\n",
      "\n",
      ") n o i t a v i t c a n\n",
      "\n",
      "i\n",
      "\n",
      "d e w o l s (\n",
      "\n",
      "n i a m o d\n",
      "\n",
      "a i m e a n a c i t y l o m e a h e n\n",
      "\n",
      "i l\n",
      "\n",
      "o h c l y d i t a h p s o h p h g h y r a t i d e r e h\n",
      "\n",
      "i\n",
      "\n",
      "] 4 2 1 , 4 5 , 0 5 , 7 3 [\n",
      "\n",
      ". y r u n\n",
      "\n",
      "j\n",
      "\n",
      "i\n",
      "\n",
      "t n o\n",
      "\n",
      "i\n",
      "\n",
      "j\n",
      "\n",
      "c i t a m u a r t\n",
      "\n",
      "r o e t u c a m o r f e e r f\n",
      "\n",
      "s i t i r h t r a o e t s o\n",
      "\n",
      "l a i l i\n",
      "\n",
      "m a f\n",
      "\n",
      "c i\n",
      "\n",
      "m o r d n y s - n o n\n",
      "\n",
      "e s a e s i d e v l a v c i t r o a d p s u c i b\n",
      "\n",
      "i\n",
      "\n",
      "n o i t a v i t c a d e s a e r c e d\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "R o t K\n",
      "\n",
      ") 8 2 5 2 ( 2 0 5 2\n",
      "\n",
      "s i t i r h t r a o e t s o t n o\n",
      "\n",
      "j\n",
      "\n",
      "l a e g n a l a h p r e t n\n",
      "\n",
      "i\n",
      "\n",
      "d n a s i t i r h t r a o e t s o d n a h e v i s o r e\n",
      "\n",
      "S H D\n",
      "\n",
      "g n n e p o d e s a e r c e\n",
      "\n",
      "i\n",
      "\n",
      "n i a m o d\n",
      "\n",
      "s i t i r h t r a o e t s O\n",
      "\n",
      ") 1 O Z E I P e s u o m\n",
      "\n",
      "(\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      "22\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      ") . d e u n i t n o C (\n",
      "\n",
      ". 1 e l b a T\n",
      "\n",
      "s e c n e r e f e r\n",
      "\n",
      "d n a s n o i t n e v r e t n\n",
      "\n",
      "i\n",
      "\n",
      ", s n o i t a v r e s b o l a c i n\n",
      "\n",
      "i l c r e h t o d n a l a t a n e t n a\n",
      "\n",
      "e s a e s i d y r a m\n",
      "\n",
      "i r p\n",
      "\n",
      "r o f a t a d y r o t a r o b a l\n",
      "\n",
      "1 O Z E I P\n",
      "\n",
      "r o d i c a o n m a\n",
      "\n",
      "i\n",
      "\n",
      "o n m a 1 O Z E I P\n",
      "\n",
      "i\n",
      "\n",
      "s e m o c t u o\n",
      "\n",
      "1 O Z E I P n o t c e f f e\n",
      "\n",
      "l a r u t c u r t s\n",
      "\n",
      "e g n a h c r e h t o\n",
      "\n",
      "m o r f\n",
      "\n",
      "r e b m u n d i c a\n",
      "\n",
      "n o i g e r\n",
      "\n",
      "s u n m r e t - N\n",
      "\n",
      "i\n",
      "\n",
      ") e s u o m\n",
      "\n",
      "( n a m u h\n",
      "\n",
      "] 4 4 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "e r u\n",
      "\n",
      "l i a f\n",
      "\n",
      "w o r r a m e n o b d e t i r e h n\n",
      "\n",
      "i\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "F o t L\n",
      "\n",
      ") 6 3 5 2 ( 6 0 5 2\n",
      "\n",
      "n i a m o d\n",
      "\n",
      "] 4 5 , 6 4 [\n",
      "\n",
      ". y r u n\n",
      "\n",
      "j\n",
      "\n",
      "i\n",
      "\n",
      "t n o\n",
      "\n",
      "i\n",
      "\n",
      "j\n",
      "\n",
      "c i t a m u a r t\n",
      "\n",
      "r o e t u c a m o r f e e r f\n",
      "\n",
      "s i t i r h t r a o e t s o\n",
      "\n",
      "l a i l i\n",
      "\n",
      "m a f\n",
      "\n",
      "c i\n",
      "\n",
      "m o r d n y s - n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "g n n e p o d e s a e r c e d\n",
      "\n",
      "i\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "L o t P\n",
      "\n",
      ") 6 3 5 2 ( 0 1 5 2\n",
      "\n",
      "s i t i r h t r a o e t s o t n o\n",
      "\n",
      "j\n",
      "\n",
      "l a e g n a l a h p r e t n\n",
      "\n",
      "i\n",
      "\n",
      "d n a s i t i r h t r a o e t s o d n a h e v i s o r e\n",
      "\n",
      "s i t i r h t r a o e t s o\n",
      "\n",
      ") 1 O Z E I P e s u o m\n",
      "\n",
      "(\n",
      "\n",
      "n i a m o d\n",
      "\n",
      "] 0 5 [\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "S H D\n",
      "\n",
      "d e i f i c e p s e n o n\n",
      "\n",
      "l a n m r e t - C\n",
      "\n",
      "i\n",
      "\n",
      "E o t K\n",
      "\n",
      ") 6 4 5 2 ( 0 2 5 2\n",
      "\n",
      "n i a m o d\n",
      "\n",
      "23\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "Figure 4. PIEZO1 variant effects. Effects of variants that cause loss of PIEZO1 expression or function are depicted in red, and those that increase PIEZO1 function are depicted in blue. Non-variant wild-type (WT) is indicated for comparison. (a) Illustrated is the loss or reduction of PIEZO1 expression by fragmentation of PIEZO1 (LOP-1), intracellular trapping of PIEZO1 (LOP-2) or disrupted PIEZO1 gene transcription (or potentially increased transcript degradation). The WT surface membrane PIEZO1 channel permits flux of cations such as Ca2+. (b) Force-response curves illustrating the increasing PIEZO1 channel activation with increasing mechanical force, up to a limit (maximum). Some variants increase the mechanical sensitivity (shifting the curve to the left, which is a type of gain of function (GOF)), while other variants decrease the mechanical sensitivity (shifting the curve to the right, which is a type of loss of function (LOF)). (c) Slow inactivation and slow deactivation as alternative mechanisms of GOF. Outside-out patch data are shown for WT and GOF variant channels overexpressed in HEK 293 cells. The patches were held at a constant voltage of −80 mV. Two recordings are superimposed and normalized to the same maximum current to emphasize the kinetic differences. The pressure steps were to +75 mmHg in both cases.\n",
      "\n",
      "2.2. Diversity and potential importance beyond rare disease\n",
      "\n",
      "PIEZO1 variant-associated GLD is phenotypically varied (table 1). There may be compartmental effusions of the pleural, abdominal, perihepatic or pericardial spaces. Varying degrees of subcutaneous skin oedema occur, including facial, genital and lower limb regions. Additional features can include chylothorax, webbed neck, prune belly and cranial findings such as craniosynostosis and macrocephaly secondary to ventriculomegaly. Resultant clinical presentations can include respiratory distress, cardiac tamponade, cellulitis and psychiatric morbidity. In pregnancy, there may be placental hypertrophy, chorionic plate oedema and polyhydramnios (table 1). The fetus may die or exhibit increased nuchal translucency, redundant nuchal skin and NIFH. A systematic review of studies identified PIEZO1 as the most common monogenic cause of NIFH in exome sequencing, with variants observed in up to 10% of cases [51]. Nineteen of 191 cases of NIFH were identified that could be attributed to PIEZO1 variants.\n",
      "\n",
      "PIEZO1 is expressed widely by lymphatic endothelial cells, and it is a mechanosensor involved in lymphatic expansion through sprouting lymphangiogenesis and lymphatic valve development [129,130]. Activation of PIEZO1 induces junctional gaps, enhancing lymphatic permeability and drainage, thereby having roles in both physiological lymphatic system formation and function. These mechanisms represent how PIEZO1 variants leading to either LOP or LOF (figure 4a,b) may underpin lymphatic maldevelopment or malfunction in a fetus affected by NIFH. The findings also suggest widespread importance of PIEZO1 in lymphatics in general, and thus with implications beyond GLD and NIFH.\n",
      "\n",
      "The varying degrees of penetrance of the variants result in a range of phenotype severities, presenting either antenatally as NIFH or later in adolescence or adulthood. This is further complicated as some variants are associated with both lymphedema and a blood disorder (see table 1 and below). As such, where PIEZO1 variant GLD is identified antenatally, careful counselling of the phenotypic heterogeneity is required and should be managed by a tertiary fetal medicine unit with genetic counselling.\n",
      "\n",
      "3. Veins: varicose veins\n",
      "\n",
      "Varicose veins (VVs) are a manifestation of chronic lower limb venous hypertension, leading to the incompetence of venous valves and distorted superficial veins [131]. Venous hypertension, with or without varicosities, due to venous valvular incom- petence causes skin changes and predisposes to leg ulcers which adversely affect quality of life, require long-term specialist nursing care and therefore represent a high societal burden. Chronic venous congestion and loss of valvular competence may increase the risk of deep vein thrombosis [131,132]. These problems run in families, suggesting a genetic component in addition to environmental and lifestyle factors [131,132].\n",
      "\n",
      "3.1. Significance identified by genome-wide association\n",
      "\n",
      "Genome-wide investigation of data for 493 519 people in UK Biobank strongly suggested the association of PIEZO1 variants with VV disease [133]. Further analysis of UK Biobank data and self-reported VV disease in 408 969 individuals of the 23andMe (www.23andme.com) database also associated PIEZO1 variants with VVs [41]. Analysis of whole exome sequence data in UK\n",
      "\n",
      "24\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "Biobank suggested associations of predicted disruptive PIEZO1 variants with VV disease [134]. Analysis of data for 131 918 participants in the Geisinger MyCode Community Health Initiative associated VV disease and vein ablation procedures with heterozygous PIEZO1 premature termination and missense variants [120]. A further study of data in UK Biobank and FinnGen strongly associated PIEZO1 variants with VVs [135] with stronger association in females [135]. Therefore, the findings suggest that PIEZO1 is important in lower limb venous structure or function and that PIEZO1 variants, even heterozygous single variants, disrupt the venous structure and its function.\n",
      "\n",
      "Sequence analysis algorithms have suggested that PIEZO1 variants associated with VVs are disruptive but there is relatively little direct laboratory evidence for this. One variant generates a premature termination codon in the sequence encoding the cap region (table 1), which suggests that the ion pore region would be damaged, and this is consistent with a disruptive effect.\n",
      "\n",
      "3.2. VVs and lymphedema\n",
      "\n",
      "There are relationships between VVs and lymphedema. In PIEZO1 variant-associated GLD, venous varicosities are reported (table 1), and VVs have been associated with another type of lymphedema, called lymphedema distichiasis [131]. Moreover, lymphedema can be caused by chronically increased pressure in the leg, although the mechanisms are not entirely clear. Higher pressure leads to venous dilatation and valve incompetence and varicosities. This could reflect inabilities of the veins to constrict and remodel in response to a higher pressure. Therefore, perhaps not surprisingly, higher venous pressure is associated with peripheral lymphedema, and higher rates of lymphedema have been detected in a VV cohort [120]. This type of lymphedema, secondary to VVs, is important because once established, it can be difficult to resolve clinically, although VV stripping may be helpful [136].\n",
      "\n",
      "The relationship between VVs and lymphedema is complex, but we suggest two effects. When the PIEZO1 disruption is strong, there may be early onset lymphedema and then late onset VVs that are both direct independent consequences of PIEZO1 disruption. When the PIEZO1 disruption is partial, there may be only late-onset disease that is characterized by VVs and lymphedema; in this case, the lymphedema could be both a direct consequence of the PIEZO1 disruption and the increased venous pressure. When there is partial PIEZO1 disruption and only late-onset disease, environmental and lifestyle factors are likely to be important contributors to the outcome.\n",
      "\n",
      "4. Blood: dehydrated hereditary stomatocytosis\n",
      "\n",
      "Dehydrated hereditary stomatocytosis (DHS) is an autosomal dominant mild or moderately severe haemolytic anaemia in which the RBC plasma membrane is leaky to cations such as K+ and hence the RBCs dehydrate and may adopt a mouth-like, stomatocyte, shape [137,138].\n",
      "\n",
      "4.1. PIEZO1 enhancement identified by familial genetics\n",
      "\n",
      "Variation in chromosome 16q23-q24 was identified as a potential cause of DHS, leading to a focus on PIEZO1 and the associa- tion of DHS in two families with heterozygous or homozygous PIEZO1 missense variants that encode M2225R in the PIEZO1 cap and R2456H in the inner helix. Further studies identified other DHS families and associated PIEZO1 variants [68,97] (table 1). In many cases, the variants are missense, changing a single amino acid in PIEZO1, but there are also amino acid deletions, insertions and duplications (table 1).\n",
      "\n",
      "Some DHS-associated variants have been recapitulated in PIEZO1 cDNA clones overexpressed in cell lines for patch-clamp recording in the laboratory. The results of these studies suggest that a consequence of the variation is the slowing of the PIEZO1 channel’s inactivation mechanism [97,105] (figure 4c). This is a gain of function (GOF) effect that results in more channel activity and therefore more non-selective cation fluxes across the cell membrane. Part of this cation flux is K+ efflux but there is also Ca2+ influx, which triggers the opening of KCa3.1 Ca2+-activated K+ channels encoded by the KCNN4 gene, further increasing the K+ efflux [138]. This suggests that cases of PIEZO1 variant-associated DHS arise through increased PIEZO1 channel-mediated ion flux and downstream consequences of this through increased Ca2+ entry.\n",
      "\n",
      "In some laboratory studies, DHS-associated variants were found to have no effect on channel function, or they decreased channel expression or its surface localization (table 1). A few of the suggested effects were not replicated in other studies (table 1). For many variants, data are not available on whether channel expression or activity is affected. For some DHS variants that have been studied, no change in mechanical sensitivity was observed [97,105], suggesting that the changes in amino acid residues at the affected positions do not regulate the mechanical sensitivity of the channels. There are exceptions, however. R2088G, R2302H and R2488Q reduced the threshold for mechanical activation (table 1). R2302H had no effect on inactivation. R2488Q caused intracellular trapping of the PIEZO1 protein, which is a disruptive (LOP) effect. The consequence of GOF may exceed that of trapping (LOP) effects, thus conferring an overall increase in PIEZO1 function. It has been suggested that the severity of DHS may depend on the location of affected amino acids in the PIEZO1 structure such that those in the ion pore region are associated with the most severe disease [139,140].\n",
      "\n",
      "It is challenging to understand how a PIEZO1 channel that often inactivates in less than a second (figure 2b) could be relevant to RBC physiology, which involves sustained mechanical stress and roles lasting many minutes, hours and days. As indicated, however, the inactivation rate of PIEZO1 channels may change and be regulated. Indeed, patch-clamp studies of physiological RBCs from mice revealed that native PIEZO1 channel activity in RBCs is relatively slow and not characterized\n",
      "\n",
      "25\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "by fast inactivation [141]. When the M2225R variant was recapitulated in the native mouse Piezo1 gene, the RBC PIEZO1 channels of these mice were more active than those of wild-type RBCs, consistent with a GOF mechanism [141], but, in the absence of inactivation, the effect of the variant was to reduce deactivation [141] (figure 2c). PIEZO1 channels interact with their environment [8,18], and, as such, data obtained from PIEZO1 channels in overexpression systems do not necessarily indicate the behaviour of native PIEZO1 channels. There is relatively little information on the properties of native variant channels in situ in patient RBCs, but these PIEZO1 channels may exhibit continuous (i.e. non-inactivating) behaviour [68].\n",
      "\n",
      "4.2. Hyperferritinaemia\n",
      "\n",
      "A clinical feature identified in some severe cases of European PIEZO1 variant-associated DHS was hyperferritinaemia [46,142]. In this condition, there is excess ferritin (an intracellular iron storage protein) and hepatic iron overload that can have serious adverse implications [142]. These effects may not be due to PIEZO1 in RBCs but rather PIEZO1 in tissue-resident macrophages such as Kupffer cells of the liver that mediate phagocytosis, and thus clearance, of RBCs [25]. Intriguingly, the E756del GOF variant (table 1) is associated with iron overload in African Americans who self-report as healthy, suggesting that there are not necessarily obvious adverse implications of GOF [25]. The implications of hyperferritinaemia are made complex by sex, disease and environmental dependencies and other comorbidities that may include VV disease [132].\n",
      "\n",
      "4.3. Er antigens\n",
      "\n",
      "Extensive investigation identified PIEZO1 variants associated with the rare and high-incidence Er antigen blood groups Era, Erb, Er3, Er4 and Er5 [90,102]. Among the variants are four missense variants that alter amino acid residues in the cap structure, thus presenting extracellular antigens [102]. Some of the variants overlap in associations with DHS and NIFH (table 1).\n",
      "\n",
      "4.4. Haemoglobin A1c\n",
      "\n",
      "Haemoglobin A1c (HbA1c), a glycated form of haemoglobin that increases in type 2 diabetes, is commonly used as a biomarker of diabetes. It is associated with some of the PIEZO1 variants, particularly in people of South Asian origin [113,135]. This effect may occur because of altered RBC turnover, as has been noted for other haemoglobinopathies.\n",
      "\n",
      "4.5. Other implications: adverse and beneficial\n",
      "\n",
      "Other GOF implications, or potential implications are erythrocytosis, β-thalassemia, myelodysplastic syndrome (table 1) and malarial resistance [71]. Sickle cell trait, which may also affect RBC shape, associated with resistance to Plasmodium, the causative agent in malaria, prompting investigation of the relevance of DHS-associated PIEZO1 variants to malaria [71]. A search was performed using Exome Aggregation Consortium data from several sequencing projects and two of the identified variants were found to increase PIEZO1 activity [71] (table 1). One of these variants (E756del) has a high (9–23%) allele frequency in people of recent African descent compared with <1% for other populations [71]. This suggests positive selection in people most exposed to Plasmodium [25]. The idea is supported by results from people in Gabon [72], but not Ghana [73], perhaps differing because of geographical effects [73].\n",
      "\n",
      "4.6. Dehydrated hereditary stomatocytosis and lymphedema\n",
      "\n",
      "Some variants associate with both GLD and DHS [143]. This situation is complicated by compound heterozygosity and lack of information on the functional consequences of the variants in many cases (table 1). GOF may be a common theme here, but further investigation of this matter would be beneficial to inform understanding.\n",
      "\n",
      "5. Other disease and physiological traits\n",
      "\n",
      "5.1. Gain of function\n",
      "\n",
      "People with the E756del GOF variant were studied for non-RBC effects in a small-scale population study, and they were found to have normal blood pressure and normal body mass index [75]. The recapitulation of human GOF variants in mice has enabled more detailed studies though. The mice exhibited DHS and stomatocytes. As well as iron overload [25], there was mild cardiac hypertrophy and fibrosis [144] and reduced bile flow [40]. Potentially related is the observation that mice overexpressing wild-type (non-variant) PIEZO1 in cardiac myocytes exhibit dilated cardiomyopathy [29]. Based on studies of the F2458L mutation recapitulated in overexpressed mouse PIEZO1, GOF may also associated with osteoarthritis [54] (table 1). Therefore, the consequences of GOF variants may extend beyond RBCs and potentially confer long-term health risks.\n",
      "\n",
      "5.2. Disruption: expectations from mouse studies\n",
      "\n",
      "Multiple implications of PIEZO1 disruption are expected based on the results of gene disruption (inhibition) studies in mice that suggest functions of PIEZO1 across multiple systems including, for example, in bone formation [26,145], immunity [27], skeletal\n",
      "\n",
      "26\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "muscle function and repair [30,146,147], blood pressure regulation [148,149], adverse reactions of the heart to pressure overload and ischemia [29,150,151] and memory [152]. The first and most striking phenotype of homozygous Piezo1 gene disruption in mice is embryonic lethality, occurring at about embryonic day 10, apparently because of failed vascular maturation that is normally driven by blood flow from the newly beating heart [14,153]. To study adult phenotypes in mice, therefore, cell type-restricted and both conditional and cell type-restricted Piezo1 disruptions have been implemented. The outcomes of such studies lie behind many of the proposals for physiological roles of PIEZO1. We cannot perform comparable genetic experiments in people but we may in the future be able to do something similar if we develop PIEZO1-selective inhibitors [24], but, for now, we can look more closely at the phenotypes of people with disruptive PIEZO1 variants.\n",
      "\n",
      "Intrauterine fetal demise has been associated with disruptive PIEZO1 variants and NIFH (table 1). This does not, however, resemble the embryonic lethality in mice, which is seen as growth restriction and embryo readsorption [14]. Early embryonic lethality may be difficult to detect in people if it occurs as unexplained miscarriage. However, whole exome sequencing of pregnancy tissue may reveal newly identified pathogenic variants in patients with normal chromosomal analysis, as has been demonstrated in a consanguineous couple with recurrent miscarriage, where a PIEZO1 mutation was detected [154]. What is clear, is that there are some people alive who have, to the best of our knowledge, no PIEZO1 [70]. The severe PIEZO1 deficiency in these people may be compensated in some way but, if it is, it is only partially successful compensation or cell type-specific because these people usually suffer NIFH and GLD. In mouse studies, some PIEZO1-null embryos survive at least a week longer than other null embryos in utero [14], so there may also be mechanisms for compensation in mice. If large population studies were performed in mice, a few PIEZO1-null survivors may be found in this species too.\n",
      "\n",
      "5.3. Other vascular associations\n",
      "\n",
      "There are many clinical features of GLD patients and other PIEZO1 variant carriers that may arise from lymphatic disruption (table 1), or alternatively, they may arise from other physiological roles of PIEZO1 that are similar or comparable to those seen in mice. Some of these features are vascular-related, such as maternal and fetal vascular malformations, venous valve anomaly, double superior vena cava, agenesis of the ductus venosus, bilateral periorbital and conjunctival vascular changes with small punctate haemorrhages, deep vein thrombosis, cardiomegaly and atrial septal defect (table 1). The familial association of disruptive PIEZO1 variants with a case of multi-organ prune belly syndrome [43] is reminiscent of some aspects of prune belly seen in GLD (table 1). We know from studies of people who apparently do not have lymphedema, that there are potential associations with other putatively endothelial-related events such as cerebral cavernous and brain arteriovenous malformations as well as bicuspid aortic valve disease (table 1).\n",
      "\n",
      "5.4. Non-vascular associations\n",
      "\n",
      "Other reported clinical features are bone abnormalities that include osteopenia and osteoarthritis (table 1). There are reports of vasculitis and autoimmune connective tissue disease, metabolic dysfunction-associated fatty liver disease, alcohol-related fatty liver disease, liver biliary pancreas abnormality, obesity in a Han Chinese population, inherited bone marrow failure, angioimmunoblastic T-cell lymphoma, SARS-CoV-2 infection, adenomatous polyposis, antidepressant therapy outcome and protection from glaucoma (table 1). There are reports of hepatic dysfunction and jaundice, a suspected case of posterior urethral valve, elevated pleural triglycerides, obstructive sleep apnoea, primary immune deficiency and Asperger syndrome, and there is quite common gastro-oesophageal reflux (table 1). Some of these associations bear resemblances to phenotypes seen in mouse genetic studies but we cannot be sure of the relationships. Detailed investigation of the associations is often difficult because of the rarities and diversities of PIEZO1 variants, the polygenics of some of the associated complex diseases and the many potential environmental factors.\n",
      "\n",
      "5.5. A hypothesis for single variant heterozygous disruption\n",
      "\n",
      "More common than the PIEZO1 knockout or compound heterozygous disruption seen in GLD sufferers, is a single disruptive heterozygous PIEZO1 variant in which there is partial disruption or deficiency of PIEZO1. In these people, who are apparently healthy, there may effectively be haploinsufficiency in which there is only about half the normal amount of PIEZO1. Examples can be found in table 1, but these may reflect only a tiny fraction of such variants because many more variants are seen in genetic databases without knowledge of the consequences. In a few cases, we know that such variants are disruptive, yet the affected people do not have NIFH or GLD. This is reminiscent of heterozygous PIEZO1 knockout mice that also appear healthy and grow similarly to wild-type mice [14]. There has been relatively little study of the heterozygous mice and only at a young age, but disturbed endothelial cell properties were seen in their reduced alignment to the direction of blood flow and lesser phosphorylation of endothelial nitric oxide synthase [14]. Effects of this type are expected to increase the long-term risks of cardiovascular disease and potentially other diseases such as vascular malfunction-related dementia. Is this also the situation in people? Might haploinsufficiency make people vulnerable to environmental factors such as elevated blood cholesterol, which is a PIEZO1 inhibitor [155]?\n",
      "\n",
      "5.6. Trait associations\n",
      "\n",
      "Traits are distinguishing qualities or characteristics that typically belong to one person or a group of people. Many of them are not diseases but they may confer advantages or disadvantages, and they may confer an increased or decreased risk of disease\n",
      "\n",
      "27\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "Figure 5. Diverse phenotype model. In the model, there is compartment 1 for regulatory and reserve systems that try to ensure compartment 2 is supplied with suitable mechanical sensors for physiological force sensing. Compartment 1 contains: PIEZO1 (gene) regulation, which may occur, for example, via microRNAs and transcriptional enhancers and silencers; mRNA encoding PIEZO1, the abundance of which may be modulated and which is available for translation and thus production of new PIEZO1 (protein); PIEZO1 in intracellular membranes (PIEZO1int), which may translocate to the plasma membrane (PM); PIEZO1 at the PM in closed states C1 or C2 due to mechanical insensitivity or inactivation, or be degraded (PIEZO1deg); and other mechanical force sensors (Mechano2 and Mechano3), an example of which may be PIEZO2. Compartment 2 comprises PIEZO1 at the PM that is available for opening (O) in response to mechanical stimuli (PIEZO1PM-O). Consequences are divided into physiology and pathology effects occurring via compartment 3. Physiology effects, occurring via the normal biochemical signalling pathways of cells and tissues, do not necessarily increase if PIEZO1 increases. Pathology, possibly involving non-physiological pathways, may be triggered by strong depletion or mild excess of PIEZO1, depending on the cell type.\n",
      "\n",
      "depending on the context. PIEZO1 variants associated with physiological traits in people. Notable among them are the strong associations with standing and sitting heights [135], which are, for example, relevant to conditions such as VV, which is more common in taller people [133,135]. There are associations with other physical factors such as forced expiratory volume [135], tendon stiffness and jumping ability [76–78]. Associations are suggested with body mass index, obesity and dyslipidaemia [40,113,156], which are consistent with observations in Piezo1 genetically modified mice [28,40] and relevant to lymphedema and VVs [131,132,157].\n",
      "\n",
      "6. Potential origins of restricted phenotypes\n",
      "\n",
      "6.1. A model\n",
      "\n",
      "Why are the most prominent phenotypes those of lymphatics, lower limb veins and RBCs? Why are there cases of GOF variants with apparently no lymphatic effect, and cases of disruptive variants with apparently no RBC effect? Here, we outline a model of how such consequences might arise (figure 5). In compartment 1 of the model, there is a system for mechanosen- sor production, regulation and degradation with a store of readily available but not necessarily used mechanosensors. This compartment serves compartment 2, the active site of mechanosensing, which is a component of compartment 3, which is where physiological effects arise. Compartment 1 contains PIEZO1 (gene) and PIEZO1 (protein) in different stages of readiness as well as alternative mechanosensors Mechano2 and Mechano3 that might comprise PIEZO2, for example. These elements exchange between compartments 1 and 2 according to need.\n",
      "\n",
      "6.2. Analogy\n",
      "\n",
      "Compartment 1 can be compared with a factory and a warehouse for anemometers, which detect wind speed. It has systems for the exchange of products with compartment 2, which is an element of compartment 3, a power station that measures wind speed in a wind turbine system. In this analogy, compartment 1 contains anemometer type 1 in various stages of production and assembly as well as other devices used to measure wind speed, should anemometer type 1 not materialize. There is communication between the compartments that maximises efficiency and tries to ensure suitable outcomes such as the production of the required amount of wind-driven electrical power. Inadvertent alteration in an anemometer’s sensitivity to airflow velocity through a manufacturing error may be detected if there is a system in place to detect it (e.g. the power station may have four anemometers that are expected to give the same readings within a defined tolerance, so one reading out of range may lead to replacement of the presumed faulty anemometer).\n",
      "\n",
      "6.3. Gain of function in the model\n",
      "\n",
      "With PIEZO1 GOF variants, some of the mechanosensors in compartment 2 may overreact to the mechanical forces experienced by the cells. Compartment 1 may be able to compensate for this problem by producing less PIEZO1. If a mechanism for\n",
      "\n",
      "28\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "producing less PIEZO1 is transcriptional, cell types without nuclei (e.g. RBCs) will be less able to compensate, and so a phenotype will arise that is dominated by the effect on RBCs, and the most obvious phenotype will be anaemia.\n",
      "\n",
      "6.4. Loss of expression or function in the model\n",
      "\n",
      "With homozygous or compound heterozygous disruptive (LOP and LOF) variants, PIEZO1s may not be able to provide mechanosensing. Compartment 1 may try to compensate by delivering an alternative mechanosensor (e.g. PIEZO2). In some cell types (e.g. lymphatic endothelial cells), the alternative may not be sensitive enough to detect the required mechanical forces (e.g. the low flow and low pressure of lymph), in which case, the most obvious phenotype will be lymphedema. In other cell types, the substitute mechanosensors may be sufficient, or insufficient only in certain conditions (e.g. when there is a cofactor or comorbidity).\n",
      "\n",
      "6.5. Partner proteins\n",
      "\n",
      "Another control on the contribution of PIEZO1 is its assembly with other proteins. These proteins include but are not limited to the cell adhesion molecules CDH1, CDH5, PECAM1 and CADM1 [20,158,159], and the MyoD family inhibitor domain-contain- ing protein MDFIC [19]. The expression and importance of these proteins vary in different cell types, and so we can speculate that PIEZO1 variants may affect the interactions of some of these proteins with PIEZO1, thereby creating cell type-specific effects. Variants in the genes that encode the interacting proteins may likewise affect PIEZO1 expression or activity. A MDFIC variant has been associated with lymphedema [160]. PECAM1 variants have been associated with lymphedema [161] and CADM1 variants have been associated with endothelial dysfunction and venous thrombosis [162,163]. It is unknown if these associations are related to interactions with PIEZO1.\n",
      "\n",
      "7. Understanding the effects of variants on PIEZO1 channel properties and PIEZO1 expression\n",
      "\n",
      "Variants that, for example, insert a premature termination codon and thus truncate the PIEZO1 protein are relatively easy to understand but others, such as missense variants that affect the amino acid sequence of the channel and thus potentially the channel properties are more difficult to understand and are often hard to interpret clinically. Are the channel properties affected? Is channel activity increased or decreased?\n",
      "\n",
      "7.1. Patch-clamp electrophysiology and intracellular Ca2+ assays\n",
      "\n",
      "Ideally, we would measure PIEZO1 channel activity non-invasively in specific systems of people such as the lymphatics, but this is not currently possible. Alternatively, we can isolate cells from a patient’s blood or tissue sample and then measure PIEZO1 activity in these cells by patch-clamp, applying a mechanical stimulus to activate the channels. This may be possible but is often challenging and laborious. Even for the relatively accessible RBC, there are few data of this type reported [68]. The development of automated planar patch-clamp methods that incorporate a mechanical stimulus, including for RBCs, has the potential to overcome some of these limitations and transform the evaluation of patient cells [102,164–166]. Intracellular Ca2+ assays can alternatively be used to detect Ca2+ influx through PIEZO1 channels, but even when mechanical stimuli are implemented in such assays, the information quality for the channel properties is inferior to that from patch-clamp assays.\n",
      "\n",
      "To avoid requiring a patient sample, it is possible to recreate the variant in a tractable cell line by gene editing, thereby generating a model system that can be studied by patch-clamp. Substantial investment of funds and researcher time is, however, required for every variant. Even if the cell line is successfully made and validated, the subsequent patch-clamp recordings from the native channels can be difficult because of the small and variable nature of the PIEZO1 channel signals and the presence of other ion channels in the same cells that may complicate the data interpretation.\n",
      "\n",
      "At the present time, the almost exclusively used method for determining the effects of variants on channel properties is the generation of the variant artificially in a cDNA clone by site-directed mutagenesis, followed by its overexpression in a host cell line for subsequent patch-clamp recording. This method is also challenging, and the ionic currents observed can be highly variable. It is quite common to use the mouse PIEZO1 channel as a surrogate of the human channel because it can be more suitable for robust recordings, and it is better understood and more widely studied. Despite the challenges and limitations, this method generates useful data for important channel parameters such as the mechanical sensitivity.\n",
      "\n",
      "7.2. Amino acid variations that affect channel activity\n",
      "\n",
      "The overexpression approach described above, whether using the human channel or a mouse surrogate, lies behind most available data for the effects of variants on PIEZO1 channel function. As detailed in table 1, we know from these data of variants altering residues in THU2 and THU3 that decrease channel activity (S217L, G253R, L322P and R531C), a variant in THU4 that increases channel activity (V598M), variants in THU5 that increase (756Edel or G782S) or decrease (E829V or L939M) channel activity or expression, a variant in the beam that increases channel activity (R1358P), a variant in the lateral plug that decreases channel activity (R1404W), seven variants in the THU9 that increase (R1943Q, A1988V, A2020T, R2088G) or decrease (1877Kdel,\n",
      "\n",
      "29\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "G1978D, Y2022H) channel activity, variants in the anchor that increase channel activity (R2110W and T2127M), a variant in the outer helix that decreases channel activity (S2195L), variants in the cap that increase (M2225R and R2302H) or decrease (I2270T and R2335Q) channel activity, variants in the inner helix that increase channel activity (R2456H, F2458L and R2488Q) and variants in the CTD that increase (2496E-ELE) or decrease (K2502R and P2510L) channel activity. Mapping of these variations to a model of the channel suggests vulnerability of the central channel region including the cap, but not a restriction to this region (figure 3). There is no obvious pattern to the sites of these LOF and GOF variants, and there are opposing effects seen from variants in proximity.\n",
      "\n",
      "The origins of PIEZO1 missense variant-associated disease are therefore complex and mostly poorly understood at the molecular level. Unless we have the patch-clamp data, we are currently unable to predict whether missense variants would increase or decrease channel activity, or indeed simply have no effect, as has been found for some variants (table 1). We are left, therefore, with needing to experimentally test the effect of each new variant unless we can devise new methods that might more quickly determine the effects or accurately predict them. We are also uncertain whether the overexpression data truly reflects what happens in the human body.\n",
      "\n",
      "7.3. Protein structural biology\n",
      "\n",
      "Structural studies of the channel might provide a way forward because a protein’s structure determines its function. Structural understanding of PIEZO1 channel function is still in its infancy, but there is progress, and it has the potential for determining effects of variants. High-resolution structural data have been obtained for the E756del and A1988V GOF channels in complex with MDFIC and, albeit slightly less well resolved, for the R2456H GOF channel [140]. While the E756del and A1988V channels had a similar dome to the non-variant wild-type channel, the R2456H channel was more curved, with more contracted blades, although the inner helix was twisted, resulting in a more dilated pore [140]. Lipid density was seen in the pore region of the wild-type channel [140], as had been seen previously in structural data and molecular dynamics simulations for the mouse PIEZO1 channel [23,167]. The lipids have been suggested to seal the pore, inhibiting or preventing the flux of ions [23,140,167]. The E756del and A1988V channel pores also contained lipid density, but the acyl chains were reoriented, which could explain the GOF [140]. There was lipid head group close to R2456, and so this may be critical for the lipid interaction in the pore [140]. These structural data suggest that GOF variants can indeed act by altering the structural configuration of the channel.\n",
      "\n",
      "7.4. Synonymous and intronic variants\n",
      "\n",
      "At present, we can only speculate about why synonymous and intronic variants are associated with human traits and disease. The most obvious hypothesis is that they act by altering the expression of the PIEZO1 gene. We have little or no evidence, however, and little or no information about the mechanisms that normally regulate the expression of PIEZO1, or the murine gene, Piezo1. The associations of these variants can give us ideas about regions of the gene that are important in its regulation by transcriptional enhancers and repressors, and non-coding RNAs such as microRNAs.\n",
      "\n",
      "8. Potential for patient benefit\n",
      "\n",
      "The genetic variant data discussed here, suggest important roles of PIEZO1 in human disease, and so we should consider the possibilities for patient benefits arising from this new knowledge. GLD, NIFH and DHS are prominent in the reports published so far (figure 6). Even in such rare diseases, understanding and manipulating PIEZO1 could generate considerable benefits for conditions that currently have limited treatment options, but, it is our hypothesis, that the benefits of targeting PIEZO1 will extend beyond these conditions and may include conditions that involve PIEZO1 but are not necessarily caused or exacerbated by PIEZO1 variation such as secondary lymphedema arising from factors such as obesity, inflammation, fibrosis and physical injury [157]. Secondary lymphedema is a major problem that affects 250 000 people in the UK [168] and about 230 million people worldwide (World Health Organization data). Improving lymphatic function by stimulating PIEZO1 could therefore have widespread implications. Data from human and mouse studies strongly support the conclusion that PIEZO1 is important in lymphatic function and that beneficial lymphatic effects can be achieved with a PIEZO1 agonist [24,70,129,130,157,169–171].\n",
      "\n",
      "There may be opportunities for resolving other seemingly unrelated clinical problems, among which we suggest may be traumatic brain injury. Head injury causes significant clinical burden and is the most common cause of injury and disability in people aged 1−40 years in the UK. Each year in England and Wales, approximately 1.4 million people attend emergency departments with head injuries. About 40 000 of these patients show evidence of traumatic brain injury. Many suffer long-term disability and around 0.2% die of their injury. Long-term complications can range from neurological deficits and seizures to endocrine conditions, and mood and behavioural disorders (NICE). Meningeal lymphatic dysfunction exacerbates traumatic brain injury pathogenesis [172], and it has been proposed that PIEZO1 agonists, acting to improve the function of meningeal lymphatics, could reduce excess cerebrospinal fluid and ventricular enlargement [169].\n",
      "\n",
      "Other conditions such as VVs are even more prevalent. While less high profile, there is importance at a societal level due to chronic disability, adverse quality of life and healthcare costs. Preclinical investigation has supported the idea of targeting PIEZO1 in the treatment of VVs [173]. Since the affected veins are often superficial, topical application of a PIEZO1 modulator might be sufficient, and limit potential safety concerns of a systemic application.\n",
      "\n",
      "There are also potential clinical implications for the RBC-related variants, for example, in anaemia and malarial resistance,\n",
      "\n",
      "and for the use of Er antigen identification in reducing acute transfusion reactions.\n",
      "\n",
      "30\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "Figure 6. Reported frequencies of PIEZO1 variant-associated phenotypes: a word cloud for PIEZO1 variant-associated disease and clinical terms from the ‘primary disease’ and ‘antenatal and other clinical observations, interventions and outcomes’ columns of table 1. The size of a word was determined by the frequency of its occurrence in the table. The colours and fonts of the word groups are random. Some of the phenotypes were manually harmonized by grouping synonymous terms to ensure consistent terminology and accurate weighting (e.g. ‘varicose veins’ and ‘veins varicosities’ were both counted as ‘varicose veins’).\n",
      "\n",
      "8.1. Progress with pharmacology\n",
      "\n",
      "A review of PIEZO1 pharmacology can be found elsewhere [24] but we note here the encouraging signs that PIEZO1-selective pharmacology is possible, particularly in the case of PIEZO1 agonists [24,45,80]. Other modulators to consider are dietary lipids such as docosahexaenoic acid, which may slow PIEZO1 inactivation and therefore have a GOF, agonist-like, effect [174]; although it may also have a separate inhibitory effect [175]. Mechanical stimulation of the channels by ultrasound [176] might also be considered. Some microRNAs may create agonist-like effects by stimulating PIEZO1 expression [177]. PIEZO1 pharmacology and other forms of modulation are, however, in their infancy, and only experimental at this stage. The approach is currently unproven with unknown safety implications in the clinical setting.\n",
      "\n",
      "8.2. Progress with computer predictions\n",
      "\n",
      "As discussed earlier, laboratory methods for determining the effects of variants on PIEZO1 channel expression and function are laborious and potentially financially expensive, contributing to limited progress on the effects of the many variants (table 1). Computer algorithms can generate quick interpretations, but they currently depend on predictions from the DNA sequence, which may have relatively little value in clinical diagnosis when predicting variants of uncertain significance (VUS). While proteome-wide missense prediction algorithms have been developed, they infer only the presence of likely pathogenicity [178]. A potentially more informative computational approach is the development and validation of molecular dynamics simulations, as are emerging for the human PIEZO1 channel in relevant membranes such as those of endothelium and RBCs [155,167]. These simulations are based on lipidomic data for membranes and static structural data from laboratory protein structural biology studies of the channel, the capabilities for which are rapidly improving. There is the prospect that combining such data with molecular dynamics simulation approaches could ultimately enable the prediction of the effects of variants on channel properties and thereby transform opportunities for clinical diagnosis.\n",
      "\n",
      "31\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "Figure 7. Primary conclusions. Left (pale yellow colour): roles of PIEZO1 inferred from PIEZO1 variant associations. Top right (pale peach colour): suggestions for future research to address aspects of this biology for which we have particularly limited understanding. Bottom right (pale green colour): promising areas of research for delivering patient benefits based on targeting and further understanding of PIEZO1.\n",
      "\n",
      "9. Conclusions and discussion\n",
      "\n",
      "Figure 7 summarizes key conclusions from this review. One conclusion is that the strongest physiological role of PIEZO1 in humans seems to be in the lymphatics, which is based on the observation that lymphedema is the most striking phenotype associated with disruptive PIEZO1 variants and potentially the most major or even singular disease characteristic in some people who carry such variants. There are almost certainly other roles of PIEZO1, but lymphatic function, with its implications for all major organs, may have overlooked influence on multiple aspects of physiology. Another major physiological role that emerged from these studies is a role of PIEZO1 in the structural resilience of superficial leg veins. This role may have little or no impact in early life, but it is of considerable interest for societies with large ageing multi-morbid populations. The roles of PIEZO1 in lymphatics and leg veins probably both reflect its significance in endothelial cells [14,135] and in turn the roles these cells have in vascular permeability and the control of unidirectional fluid flow mediated by valves.\n",
      "\n",
      "Unlike the essential lymphatic and venous roles, PIEZO1 seems to be mostly non-essential in RBC physiology. Anaemia is often not a feature of PIEZO1 variant-associated NIFH or GLD in which we know PIEZO1 is disrupted. RBCs may not depend as much as lymphatics on PIEZO1 because of their multiple diverse mechanisms for membrane cation permeability [179]. The excess cation permeability inadvertently conferred by PIEZO1 GOF variants nevertheless may disturb RBC physiology. The significance of the disturbance for health varies substantially from the relatively benign and even beneficial, to the quite seriously adverse. The variability may be explained by varying genetic backgrounds that may in some instances make people vulnerable to PIEZO1 GOF, such as variations in other genes, for example, KCNN4.\n",
      "\n",
      "Despite the broad expression of PIEZO1 and the multiple phenotypes of PIEZO1-disrupted mice, PIEZO1 also seems to be non-essential in other human physiology. But this does not mean that PIEZO1 variation is without other consequences, or that PIEZO1 modulators (e.g. PIEZO1 agonists) would be without adverse effects if administered to people. As we discuss in this article, there are already many potential additional associations of PIEZO1 variation. We expect that as studies on this topic continue, additional importance in humans will be recognized. In future studies, we suggest the importance of focussing studies on age-onset cardiovascular and musculoskeletal diseases. Tissue stiffness and other mechanical strains increase with age, and we know that PIEZO1’s role in iron homeostasis is age-related [25]. Iron overload could have broad implications for diseases that are more common in older people such as VVs [132], and arthritis, liver damage and heart failure [25]. We should also consider the importance of lipid homeostasis in PIEZO1 variant-associated phenotype because of the impacts of lipids on PIEZO1 channel function [18,174] and the roles of PIEZO1 in whole body lipid homeostasis [40]. We should also consider the\n",
      "\n",
      "32\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "health implications of PIEZO1 haploinsufficiency caused by disruption to one of the PIEZO1 alleles because of the vulnerability it may confer to disease through a reduced reserve of PIEZO1 channels.\n",
      "\n",
      "In summary, we conclude that PIEZO1 is important in human health and disease, and that studies of the effects of PIEZO1 variants have clinical relevance that is only just starting to be broadly realized and which can in parallel advance understanding of how mechanical force sensing works in humans and eukaryotic biology as a whole.\n",
      "\n",
      "Ethics. This work did not require ethical approval from a human subject or animal welfare committee. Data accessibility. This article has no additional data. Declaration of AI use. We have not used AI-assisted technologies in creating this article. Authors’ contributions. C.W.C.: data curation, formal analysis, methodology, writing—review and editing; S.L.E.: validation, writing—review and editing; O.V.P.: data curation, formal analysis, visualization, writing—review and editing; C.R.: data curation, formal analysis, writing—review and editing; K.A.S.: formal analysis, visualization, writing—review and editing; M.D.: formal analysis, visualization, writing—review and editing; F.L.M.: validation, writing—review and editing; D.G.B.: validation, writing—review and editing; A.M.: writing—review and editing; F.B.: writing—review and editing; E.C.-B.: writing—review and editing; R.M.C.: writing—review and editing; K.J.G.: validation, writing— review and editing; M.A.B.: validation, writing—review and editing; A.C.K.: writing—review and editing; L.C.M.: writing—review and editing; K.K.W.: validation, writing—review and editing; D.J.B.: conceptualization, data curation, investigation, project administration, resources, supervision, visualization, writing—original draft.\n",
      "\n",
      "All authors gave final approval for publication and agreed to be held accountable for the work performed therein.\n",
      "\n",
      "Conflict of interest declaration. We declare we have no competing interests. Funding. C.W.C., M.D., F.B., E.C.-B. and K.J.G. were supported by British Heart Foundation (BHF) Mautner fellowships. L.C.M., S.L.E. and D.J.B. were supported in part by the Tommy’s Preterm Birth Centre. L.C.M. was supported by a NIHR Academic Clinical Lectureship. O.V.P. was supported by BHF grant PG/21/10515. C.R., K.A.S. and A.M. were supported by BHF PhD studentships FS/PhD/22/29312, FS/4yPhD/P/20/34012 and FS/4yPhD/F/22/34182. F.L.M. was supported by a Sir Henry Wellcome career development fellowship 215861/Z/19/Z. M.A.B. was supported by a BHF career development fellowship. R.M.C. and D.J.B. were supported in part by the National Institute for Health and Care Research (NIHR) Leeds Biomedical Research Centre (BRC) NIHR203331. The views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR or the Department of Health and Social Care. A.C.K. and D.J.B. were supported in part by UKRI BBSRC research grant APP22657. Some of the background research was supported by BHF RG/17/11/33042 and Wellcome 110044/Z/15/Z grants to D.J.B.\n",
      "\n",
      "References\n",
      "\n",
      "1. 2.\n",
      "\n",
      "3. 4. 5.\n",
      "\n",
      "6.\n",
      "\n",
      "7.\n",
      "\n",
      "8.\n",
      "\n",
      "9.\n",
      "\n",
      "Fritzsche M. 2020 Thinking multi-scale to advance mechanobiology. Commun. Biol. 3, 469. (doi:10.1038/s42003-020-01197-5) Campàs O, Noordstra I, Yap AS. 2024 Adherens junctions as molecular regulators of emergent tissue mechanics. Nat. Rev. Mol. Cell Biol. 25, 252–269. (doi:10.1038/s41580-023- 00688-7) Douguet D, Honoré E. 2019 Mammalian mechanoelectrical transduction: structure and function of force-gated ion channels. Cell 179, 340–354. (doi:10.1016/j.cell.2019.08.049) Tranter JD, Kumar A, Nair VK, Sah R. 2023 Mechanosensing in metabolism. Compr. Physiol. 14, 5269–5290. (doi:10.1002/cphy.c230005) Mammadova-Bach E, Gudermann T, Braun A. 2023 Platelet mechanotransduction: regulatory cross talk between mechanosensitive receptors and calcium channels. Arterioscler. Thromb. Vasc. Biol. 43, 1339–1348. (doi:10.1161/atvbaha.123.318341) Phuyal S, Romani P, Dupont S, Farhan H. 2023 Mechanobiology of organelles: illuminating their roles in mechanosensing and mechanotransduction. Trends Cell Biol. 33, 1049– 1061. (doi:10.1016/j.tcb.2023.05.001) Goodman MB, Haswell ES, Vásquez V. 2023 Mechanosensitive membrane proteins: usual and unusual suspects in mediating mechanotransduction. J. Gen. Physiol. 155, e202213248. (doi:10.1085/jgp.202213248) Jiang Y, Yang X, Jiang J, Xiao B. 2021 Structural designs and mechanogating mechanisms of the mechanosensitive Piezo channels. Trends Biochem. Sci. 46, 472–488. (doi:10.1016/ j.tibs.2021.01.008) Coste B, Mathur J, Schmidt M, Earley TJ, Ranade S, Petrus MJ, Dubin AE, Patapoutian A. 2010 Piezo1 and Piezo2 are essential components of distinct mechanically activated cation channels. Science 330, 55–60. (doi:10.1126/science.1193270)\n",
      "\n",
      "10. Murthy SE, Dubin AE, Patapoutian A. 2017 Piezos thrive under pressure: mechanically activated ion channels in health and disease. Nat. Rev. Mol. Cell Biol. 18, 771–783. (doi:10.\n",
      "\n",
      "11.\n",
      "\n",
      "1038/nrm.2017.92) Srivastava N, Traynor D, Piel M, Kabla AJ, Kay RR. 2020 Pressure sensing through Piezo channels controls whether cells migrate with blebs or pseudopods. Proc. Natl Acad. Sci. USA 117, 2506–2512. (doi:10.1073/pnas.1905730117)\n",
      "\n",
      "12. Mousavi SAR et al. 2021 PIEZO ion channel is required for root mechanotransduction in Arabidopsis thaliana. Proc. Natl Acad. Sci. USA 118, e2102188118. (doi:10.1073/pnas.\n",
      "\n",
      "2102188118)\n",
      "\n",
      "13. Wu J, Lewis AH, Grandl J. 2017 Touch, tension, and transduction – the function and regulation of Piezo ion channels. Trends Biochem. Sci. 42, 57–71. (doi:10.1016/j.tibs.2016.09.\n",
      "\n",
      "14. 15. 16. 17. 18. 19. 20.\n",
      "\n",
      "21. 22. 23.\n",
      "\n",
      "004) Li J et al. 2014 Piezo1 integration of vascular architecture with physiological force. Nature 515, 279–282. (doi:10.1038/nature13701) Segel M et al. 2019 Niche stiffness underlies the ageing of central nervous system progenitor cells. Nature 573, 130–134. (doi:10.1038/s41586-019-1484-9) Clapham DE. 2007 Calcium signaling. Cell 131, 1047–1058. (doi:10.1016/j.cell.2007.11.028) Berridge MJ, Bootman MD, Roderick HL. 2003 Calcium signalling: dynamics, homeostasis and remodelling. Nat. Rev. Mol. Cell Biol. 4, 517–529. (doi:10.1038/nrm1155) Shi J et al. 2020 Sphingomyelinase disables inactivation in endogenous PIEZO1 channels. Cell Rep. 33, 108225. (doi:10.1016/j.celrep.2020.108225) Zhou Z et al. 2023 MyoD-family inhibitor proteins act as auxiliary subunits of Piezo channels. Science 381, 799–804. (doi:10.1126/science.adh8190) Koster AK, Yarishkin O, Dubin AE, Kefauver JM, Pak RA, Cravatt BF, Patapoutian A. 2024 Chemical mapping of the surface interactome of PIEZO1 identifies CADM1 as a modulator of channel inactivation. Proc. Natl Acad. Sci. USA 121, e2415934121. (doi:10.1073/pnas.2415934121) Zhao Q et al. 2018 Structure and mechanogating mechanism of the Piezo1 channel. Nature 554, 487–492. (doi:10.1038/nature25743) Guo YR, MacKinnon R. 2017 Structure-based membrane dome mechanism for Piezo mechanosensitivity. eLife 6, 33660. (doi:10.7554/elife.33660) Yang X, Lin C, Chen X, Li S, Li X, Xiao B. 2022 Structure deformation and curvature sensing of PIEZO1 in lipid membranes. Nature 604, 377–383. (doi:10.1038/s41586-022-04574-8)\n",
      "\n",
      "33\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "24.\n",
      "\n",
      "Kinsella JA, Debant M, Parsonage G, Morley LC, Bajarwan M, Revill C, Foster R, Beech DJ. 2024 Pharmacology of PIEZO1 channels. Br. J. Pharmacol. 181, 4714–4732. (doi:10.1111/ bph.17351)\n",
      "\n",
      "25. Ma S, Dubin AE, Zhang Y, Mousavi SAR, Wang Y, Coombs AM, Loud M, Andolfo I, Patapoutian A. 2021 A role of PIEZO1 in iron metabolism in mice and humans. Cell 184, 969–982.\n",
      "\n",
      "(doi:10.1016/j.cell.2021.01.024) Sun W et al. 2019 The mechanosensitive Piezo1 channel is required for bone formation. eLife 8, 47454. (doi:10.7554/elife.47454) Solis AG et al. 2019 Mechanosensation of cyclical force by PIEZO1 is essential for innate immunity. Nature 573, 69–74. (doi:10.1038/s41586-019-1485-8)\n",
      "\n",
      "26. 27. 28. Wang S et al. 2020 Adipocyte Piezo1 mediates obesogenic adipogenesis through the FGF1/FGFR1 signaling pathway in mice. Nat. Commun. 11, 2303. (doi:10.1038/s41467-020- 16026-w) Jiang F, Yin K, Wu K, Zhang M, Wang S, Cheng H, Zhou Z, Xiao B. 2021 The mechanosensitive Piezo1 channel mediates heart mechano-chemo transduction. Nat. Commun. 12, 869. (doi:10.1038/s41467-021-21178-4)\n",
      "\n",
      "26. 27. 28. Wang S et al. 2020 Adipocyte Piezo1 mediates obesogenic adipogenesis through the FGF1/FGFR1 signaling pathway in mice. Nat. Commun. 11, 2303. (doi:10.1038/s41467-020- 16026-w) Jiang F, Yin K, Wu K, Zhang M, Wang S, Cheng H, Zhou Z, Xiao B. 2021 The mechanosensitive Piezo1 channel mediates heart mechano-chemo transduction. Nat. Commun. 12, 869. (doi:10.1038/s41467-021-21178-4)\n",
      "\n",
      "30. Ma N et al. 2022 Piezo1 regulates the regenerative capacity of skeletal muscles via orchestration of stem cell morphological states. Sci. Adv. 8, eabn0485. (doi:10.1126/sciadv.\n",
      "\n",
      "31.\n",
      "\n",
      "32.\n",
      "\n",
      "abn0485) Qi Q et al. 2020 Simultaneous detection of CNVs and SNVs improves the diagnostic yield of fetuses with ultrasound anomalies and normal karyotypes. Genes 11, 1397. (doi:10. 3390/genes11121397) Correa ARE, Naini K, Mishra P, Dadhwal V, Agarwal R, Shukla R, Kabra M, Gupta N. 2021 Utility of fetal whole exome sequencing in the etiological evaluation and outcome of nonimmune hydrops fetalis. Prenat. Diagn. 41, 1414–1424. (doi:10.1002/pd.6022)\n",
      "\n",
      "33. Maciak K, Adamowicz-Salach A, Siwicka A, Poznanski J, Urasinski T, Plochocka D, Gora M, Burzynska B. 2020 Hereditary xerocytosis - spectrum and clinical manifestations of\n",
      "\n",
      "variants in the PIEZO1 gene, including co-occurrence with a novel β-globin mutation. Blood Cells Mol. Dis. 80, 102378. (doi:10.1016/j.bcmd.2019.102378) Cheng CW et al. 2020 Genetic variants of PIEZO1 associate with COVID-19 fatality. medRxiv (doi:10.1101/2020.1106.1101.20119651) Rogerson D et al. 2023 Investigation into the genetics of fetal congenital lymphatic anomalies. Prenat. Diagn. 43, 703–716. (doi:10.1002/pd.6345) Vora NL et al. 2017 Prenatal exome sequencing in anomalous fetuses: new opportunities and challenges. Genet. Med. 19, 1207–1216. (doi:10.1038/gim.2017.33) Faucherre A et al. 2020 Piezo1 is required for outflow tract and aortic valve development. J. Mol. Cell. Cardiol. 143, 51–62. (doi:10.1016/j.yjmcc.2020.03.013) Li JV et al. 2021 Modified N-linked glycosylation status predicts trafficking defective human Piezo1 channel mutations. Commun. Biol. 4, 1038. (doi:10.1038/s42003-021-02528-w) Zhou Z, Li JV, Martinac B, Cox CD. 2021 Loss-of-function Piezo1 mutations display altered stability driven by ubiquitination and proteasomal degradation. Front. Pharmacol. 12, 766416. (doi:10.3389/fphar.2021.766416) Lichtenstein L et al. 2024 Endothelial force sensing signals to parenchymal cells to regulate bile and plasma lipids. Sci. Adv 10, eadq3075. (doi:10.1126/sciadv.adq3075) 40. 41. Ahmed WUR et al. 2022 Genome-wide association analysis and replication in 810,625 individuals with varicose veins. Nat. Commun. 13, 3065. (doi:10.1038/s41467-022-30765-y) 42. Moradian N, Zoghi S, Rayzan E, Seyedpour S, Jimenez Heredia R, Boztug K, Rezaei N. 2023 Severe congenital neutropenia due to G6PC3 deficiency: early and delayed phenotype of a\n",
      "\n",
      "34. 35. 36. 37. 38. 39.\n",
      "\n",
      "43.\n",
      "\n",
      "44. 45.\n",
      "\n",
      "46.\n",
      "\n",
      "patient. Allergy Asthma Clin. Immunol. 19, 51. (doi:10.1186/s13223-023-00804-4) Amado NG et al. 2024 PIEZO1 loss-of-function compound heterozygous mutations in the rare congenital human disorder Prune Belly Syndrome. Nat. Commun. 15, 339. (doi:10. 1038/s41467-023-44594-0) Bluteau O et al. 2018 A landscape of germ line mutations in a cohort of inherited bone marrow failure patients. Blood 131, 717–732. (doi:10.1182/blood-2017-09-806489) Jiang J et al. 2025 The fetal hydrops-associated single-residue mutation L322P abolishes mechanical but not chemical activation of the PIEZO1 ion channel. bioRxiv (doi:10.1101/ 2025.02.18.638930) Picard V et al. 2019 Clinical and biological features in PIEZO1-hereditary xerocytosis and Gardos channelopathy: a retrospective series of 126 patients. Haematologica 104, 1554– 1564. (doi:10.3324/haematol.2018.205328)\n",
      "\n",
      "47. Mansour-Hendili L et al. 2020 Exome sequencing for diagnosis of congenital hemolytic anemia. Orphanet J. Rare Dis. 15, 180. (doi:10.1186/s13023-020-01425-5) 48. 49. Shamseldin HE et al. 2018 Molecular autopsy in maternal–fetal medicine. Genet. Med. 20, 420–427. (doi:10.1038/gim.2017.111) Nakahara E et al. 2023 Variant spectrum of PIEZO1 and KCNN4 in Japanese patients with dehydrated hereditary stomatocytosis. Hum. Genome Var. 10, 8. (doi:10.1038/s41439-023- 00235-y) Russo R et al. 2018 Multi‐gene panel testing improves diagnosis and management of patients with hereditary anemias. Am. J. Hematol. 93, 672–682. (doi:10.1002/ajh.25058) Brewer CJ, Makhamreh MM, Shivashankar K, McLaren R, Toro M, Berger SI, Al‐Kouatly HB. 2023 PIEZO1 is the most common monogenic etiology of non‐immune hydrops fetalis detected by prenatal exome sequencing. Prenat. Diagn. 43, 1556–1566. (doi:10.1002/pd.6451) Al-Kouatly HB et al. 2021 High diagnosis rate for nonimmune hydrops fetalis with prenatal clinical exome from the Hydrops-Yielding Diagnostic Results of Prenatal Sequencing (HYDROPS) Study. Genet. Med. 23, 1325–1333. (doi:10.1038/s41436-021-01121-0) Guo W et al. 2020 Trio‐whole‐exome sequencing and preimplantation genetic diagnosis for unexplained recurrent fetal malformations. Hum. Mutat. 41, 432–448. (doi:10.1002/ humu.23935) Jurynec MJ, Nosyreva E, Thompson D, Munoz C, Novak KA, Matheson DJ, Kazmers NH, Syeda R. 2024 PIEZO1 variants that reduce open channel probability are associated with familial osteoarthritis. MedRxiv (doi:10.1101/2024.09.03.24312969) Gnanasambandam R et al. 2018 Increased red cell KCNN4 activity in sporadic hereditary xerocytosis associated with enhanced single channel pressure sensitivity of PIEZO1 mutant V598M. HemaSphere 2, e55. (doi:10.1097/hs9.0000000000000055) Yamaguchi Y, Allegrini B, Rapetti-Mauss R, Picard V, Garçon L, Kohl P, Soriani O, Peyronnet R, Guizouarn H. 2021 Hereditary xerocytosis: differential behavior of PIEZO1 mutations in the N-terminal extracellular domain between red blood cells and HEK cells. Front. Physiol. 12, 736585. (doi:10.3389/fphys.2021.736585) Dempsey E et al. 2021 A report on the impact of rapid prenatal exome sequencing on the clinical management of 52 ongoing pregnancies: a retrospective review. BJOG 128, 1012– 1019. (doi:10.1111/1471-0528.16546) Rapetti-Mauss R, Picard V, Guitton C, Ghazal K, Proulle V, Badens C, Soriani O, Garçon L, Guizouarn H. 2017 Red blood cell Gardos channel (KCNN4): the essential determinant of erythrocyte dehydration in hereditary xerocytosis. Haematologica 102, e415–e418. (doi:10.3324/haematol.2017.171389) Sparks TN et al. 2020 Exome sequencing for prenatal diagnosis in nonimmune hydrops fetalis. N. Engl. J. Med. 383, 1746–1756. (doi:10.1056/nejmoa2023643) Vignes S et al. 2022 PIEZO1-gene gain-of-function mutations with lower limb lymphedema onset in an adult: clinical, scintigraphic, and noncontrast magnetic resonance lymphography findings. Am. J. Med. Genet. Part 188, 243–248. (doi:10.1002/ajmg.a.62476) Zhou X et al. 2021 Value of exome sequencing in diagnosis and management of recurrent non-immune hydrops fetalis: a retrospective analysis. Front. Genet. 12, 616392. (doi:10. 3389/fgene.2021.616392)\n",
      "\n",
      "50. 51.\n",
      "\n",
      "52.\n",
      "\n",
      "53.\n",
      "\n",
      "54.\n",
      "\n",
      "55.\n",
      "\n",
      "56.\n",
      "\n",
      "57.\n",
      "\n",
      "58.\n",
      "\n",
      "59. 60.\n",
      "\n",
      "61.\n",
      "\n",
      "34\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "62. More TA, Dongerdiye R, Devendra R, Warang PP, Kedar PS. 2020 Mechanosensitive Piezo1 ion channel protein (PIEZO1 gene): update and extended mutation analysis of hereditary\n",
      "\n",
      "63.\n",
      "\n",
      "64.\n",
      "\n",
      "65.\n",
      "\n",
      "66.\n",
      "\n",
      "67.\n",
      "\n",
      "68. 69.\n",
      "\n",
      "70.\n",
      "\n",
      "xerocytosis in India. Ann. Hematol. 99, 715–727. (doi:10.1007/s00277-020-03955-1) Zama D, Giulietti G, Muratore E, Andolfo I, Russo R, Iolascon A, Pession A. 2020 A novel PIEZO1 mutation in a patient with dehydrated hereditary stomatocytosis: a case report and a brief review of literature. Ital. J. Pediatr. 46, 102. (doi:10.1186/s13052-020-00864-x) Ghesh L et al. 2023 Perinatal presentations of non‐immune hydrops fetalis due to recessive PIEZO1 disease: a challenging fetal diagnosis. Clin. Genet. 103, 560–565. (doi:10.1111/ cge.14274) Cui S, Lu H, Wei S, Li C, Xu F, Chen Y. 2021 The PIEZO1 gene mutation (c.2005G>T) causes iron overload cardiomyopathy: a case report. Emerg. Crit. Care Med. 1, 86–89. (doi:10. 1097/ec9.0000000000000018) Yates CL, Monaghan KG, Copenheaver D, Retterer K, Scuffins J, Kucera CR, Friedman B, Richard G, Juusola J. 2017 Whole-exome sequencing on deceased fetuses with ultrasound anomalies: expanding our knowledge of genetic disease during fetal development. Genet. Med. 19, 1171–1178. (doi:10.1038/gim.2017.31) Spier I et al. 2016 Exome sequencing identifies potential novel candidate genes in patients with unexplained colorectal adenomatous polyposis. Fam. Cancer 15, 281–288. (doi:10. 1007/s10689-016-9870-z) Andolfo I et al. 2013 Multiple clinical forms of dehydrated hereditary stomatocytosis arise from mutations in PIEZO1. Blood 121, 3925–3935. (doi:10.1182/blood-2013-02-482489) Zhang M et al. 2021 Exome sequencing of 112 trios identifies recessive genetic variants in brain arteriovenous malformations. J. Neurointerventional Surg 13, 568–573. (doi:10. 1136/neurintsurg-2020-016469) Fotiou E et al. 2015 Novel mutations in PIEZO1 cause an autosomal recessive generalized lymphatic dysplasia with non-immune hydrops fetalis. Nat. Commun. 6, 8085. (doi:10. 1038/ncomms9085)\n",
      "\n",
      "71. Ma S et al. 2018 Common PIEZO1 allele in African populations causes RBC dehydration and attenuates Plasmodium infection. Cell 173, 443–455. (doi:10.1016/j.cell.2018.02.047) 72. Nguetse CN, Purington N, Ebel ER, Shakya B, Tetard M, Kremsner PG, Velavan TP, Egan ES. 2020 A common polymorphism in the mechanosensitive ion channel PIEZO1 is associated with protection from severe malaria in humans. Proc. Natl Acad. Sci. USA 117, 9074–9081. (doi:10.1073/pnas.1919843117) Thye T, Evans JA, Ruge G, Loag W, Ansong D, Agbenyega T, Horstmann RD, May J, Schuldt K. 2022 Human genetic variant E756del in the ion channel PIEZO1 not associated with protection from severe malaria in a large Ghanaian study. J. Hum. Genet. 67, 65–67. (doi:10.1038/s10038-021-00958-2) Rooks H, Brewin J, Gardner K, Chakravorty S, Menzel S, Hannemann A, Gibson J, Rees DC. 2019 A gain of function variant in PIEZO1 (E756del) and sickle cell disease. Haematologica 104, e91–e93. (doi:10.3324/haematol.2018.202697) Baxter SL et al. 2020 Investigation of associations between Piezo1 mechanoreceptor gain-of-function variants and glaucoma-related phenotypes in humans and mice. Sci. Rep. 10, 19013. (doi:10.1038/s41598-020-76026-0) Passini FS et al. 2021 Shear-stress sensing by PIEZO1 regulates tendon stiffness in rodents and influences jumping performance in humans. Nat. Biomed. Eng. 5, 1457–1471. (doi: 10.1038/s41551-021-00716-x) Nakamichi R et al. 2022 The mechanosensitive ion channel PIEZO1 is expressed in tendons and regulates physical performance. Sci. Transl. Med. 14, j5557. (doi:10.1126/ scitranslmed.abj5557) Götschi T, Held V, Klucker G, Niederöst B, Aagaard P, Spörri J, Passini FS, Snedeker JG. 2023 PIEZO1 gain-of-function gene variant is associated with elevated tendon stiffness in humans. J. Appl. Physiol. 135, 165–173. (doi:10.1152/japplphysiol.00573.2022) Jamwal M, Aggarwal A, Palodhi A, Sharma P, Bansal D, Trehan A, Malhotra P, Maitra A, Das R. 2020 Next-generation sequencing–based diagnosis of unexplained inherited hemolytic anemias reveals wide genetic and phenotypic heterogeneity. J. Mol. Diagn. 22, 579–590. (doi:10.1016/j.jmoldx.2020.01.007) Ludlow MJ et al. 2023 Small-molecule functional rescue of PIEZO1 channel variants associated with generalised lymphatic dysplasia. medRxiv (doi:10.1101/2023.1108.1101. 23292554) Stray-Pedersen A et al. 2017 Primary immunodeficiency diseases: genomic approaches delineate heterogeneous Mendelian disorders. J. Allergy Clin. Immunol. 139, 232–245. (doi: 10.1016/j.jaci.2016.05.042) Fustino NJ, Beck R. 2024 A 3-year-old Tanzanian female with glucose-6-phosphate dehydrogenase A- and a novel heterozygous PIEZO1 mutation (2744A>G, N915S) presenting with severe hemolytic anemia. Ann. Afr. Med. 23, 743–747. (doi:10.4103/aam.aam_29_24) Lee S, Park S, Kim HY, Chae JH, Ko JM. 2021 Extended phenotypes of PIEZO1-related lymphatic dysplasia caused by two novel compound heterozygous variants. Eur. J. Med. Genet. 64, 104295. (doi:10.1016/j.ejmg.2021.104295) Kedar PS et al. 2019 Study of pathophysiology and molecular characterization of congenital anemia in India using targeted next-generation sequencing approach. Int. J. Hematol. 110, 618–626. (doi:10.1007/s12185-019-02716-9) Scimone C, Donato L, Alibrandi S, D’Angelo R, Sidoti A. 2022 Evidences of PIEZO1 involvement in cerebral cavernous malformation pathogenesis. Microvasc. Res. 141, 104342. (doi: 10.1016/j.mvr.2022.104342) Alhazmi W, Qurban A, Alrashidi E. 2023 Case report of generalized lymphatic dysplasia with PIEZO1 mutation and review of the literature. Respir. Med. Case Rep. 44, 101872. (doi: 10.1016/j.rmcr.2023.101872) Datkhaeva I et al. 2018 Identification of novel PIEZO1 variants using prenatal exome sequencing and correlation to ultrasound and autopsy findings of recurrent hydrops fetalis. Am. J. Med. Genet. Part 176, 2829–2834. (doi:10.1002/ajmg.a.40533) Sparks TN et al. 2019 Nonimmune hydrops fetalis: identifying the underlying genetic etiology. Genet. Med. 21, 1339–1344. (doi:10.1038/s41436-018-0352-6) Lee Y, Seo SH, Kim J, Kim SA, Lee JY, Lee JO, Bang SM, Park KU, Hwang SM. 2024 Diagnostic approaches to investigate JAK2-unmutated erythrocytosis based on a single tertiary center experience. Mol. Diagn. Ther. 28, 311–318. (doi:10.1007/s40291-024-00703-3) Lane WJ, Vege S, Mah HH, Ochoa-Garay G, Lomas-Francis C, Westhoff CM. 2024 Three novel Er blood group system alleles and insights from protein modeling. Transfusion 64, 1633–1639. (doi:10.1111/trf.17965) Lukacs V et al. 2015 Impaired PIEZO1 function in patients with a novel autosomal recessive congenital lymphatic dysplasia. Nat. Commun. 6, 8329. (doi:10.1038/ncomms9329) Al-deri N et al. 2021 A novel homozygous variant in TRAPPC2L results in a neurodevelopmental disorder and disrupts TRAPP complex function. J. Med. Genet. 58, 592–601. (doi:10. 1136/jmedgenet-2020-107016) Filser M et al. 2021 Increased incidence of germline PIEZO1 mutations in individuals with idiopathic erythrocytosis. Blood 137, 1828–1832. (doi:10.1182/blood.2020008424) Attardi E et al. 2024 Prospective genetic germline evaluation in a consecutive group of adult patients aged <60 years with myelodysplastic syndromes. Hemasphere 8, e112. (doi: 10.1002/hem3.112) Kovesi T, Rojas SK, Boycott KM. 2023 Persistent chylothorax associated with lymphatic malformation type 6 due to biallelic pathogenic variants in PIEZO1. Am. J. Med. Genet. A 191, 2188–2192. (doi:10.1002/ajmg.a.63237)\n",
      "\n",
      "73.\n",
      "\n",
      "74.\n",
      "\n",
      "75.\n",
      "\n",
      "76.\n",
      "\n",
      "77.\n",
      "\n",
      "78.\n",
      "\n",
      "79.\n",
      "\n",
      "80.\n",
      "\n",
      "81.\n",
      "\n",
      "82.\n",
      "\n",
      "83.\n",
      "\n",
      "84.\n",
      "\n",
      "85.\n",
      "\n",
      "86.\n",
      "\n",
      "87.\n",
      "\n",
      "88. 89.\n",
      "\n",
      "90.\n",
      "\n",
      "91. 92.\n",
      "\n",
      "93. 94.\n",
      "\n",
      "95.\n",
      "\n",
      "35\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "96.\n",
      "\n",
      "97.\n",
      "\n",
      "98.\n",
      "\n",
      "99.\n",
      "\n",
      "Chen Y, Jiang Y, Chen B, Qian Y, Liu J, Yang M, Zhao B, Luo Q. 2021 Case report: Whole exome sequencing revealed two novel mutations of PIEZO1 implicated in nonimmune hydrops fetalis. Front. Genet. 12, 684555. (doi:10.3389/fgene.2021.684555) Albuisson J et al. 2013 Dehydrated hereditary stomatocytosis linked to gain-of-function mutations in mechanically activated PIEZO1 ion channels. Nat. Commun. 4, 1884. (doi:10. 1038/ncomms2899) Kager L et al. 2018 Targeted mutation screening of 292 candidate genes in 38 children with inborn haematological cytopenias efficiently identifies novel disease‐causing mutations. Br. J. Haematol. 182, 251–258. (doi:10.1111/bjh.15389) Shi Y et al. 2023 Genetic abnormalities assist in pathological diagnosis and EBV-positive cell density impact survival in Chinese angioimmunoblastic T-cell lymphoma patients. Chin. J. Cancer Res. 35, 536–549. (doi:10.21147/j.issn.1000-9604.2023.05.10)\n",
      "\n",
      "100. Fathalla BM, Alsarhan A, Afzal S, El Naofal M, Abou Tayoun A. 2021 The genomic landscape of pediatric rheumatology disorders in the Middle East. Hum. Mutat. 42, e1–e14. (doi:\n",
      "\n",
      "10.1002/humu.24165)\n",
      "\n",
      "101. Han S, Guo X, Wang X, Lin H, Yu Y, Shu J, Dong M, Yang L. 2022 A novel homozygous missense mutation of PIEZO1 leading to lymphatic malformation-6 identified in a family with\n",
      "\n",
      "three adverse pregnancy outcomes due to nonimmune fetal hydrops. Front. Genet. 13, 856046. (doi:10.3389/fgene.2022.856046)\n",
      "\n",
      "102. Karamatic Crew V et al. 2023 Missense mutations in PIEZO1, which encodes the Piezo1 mechanosensor protein, define Er red blood cell antigens. Blood 141, 135–146. (doi:10.\n",
      "\n",
      "1182/blood.2022016504)\n",
      "\n",
      "103. Milev MP et al. 2018 Bi-allelic mutations in TRAPPC2L result in a neurodevelopmental disorder and have an impact on RAB11 in fibroblasts. J. Med. Genet. 55, 753–764. (doi:10.\n",
      "\n",
      "1136/jmedgenet-2018-105441)\n",
      "\n",
      "104. Andolfo I et al. 2018 PIEZO1-R1864H rare variant accounts for a genetic phenotype-modifier role in dehydrated hereditary stomatocytosis. Haematologica 103, e94–e97. (doi:10.\n",
      "\n",
      "3324/haematol.2017.180687)\n",
      "\n",
      "105. Glogowska E et al. 2017 Novel mechanisms of PIEZO1 dysfunction in hereditary xerocytosis. Blood 130, 1845–1856. (doi:10.1182/blood-2017-05-786004) 106. Andolfo I et al. 2019 PIEZO1 hypomorphic variants in congenital lymphatic dysplasia cause shape and hydration alterations of red blood cells. Front. Physiol. 10, 258. (doi:10.3389/ fphys.2019.00258)\n",
      "\n",
      "107. Mastromoro G et al. 2021 Recurrent prenatal PIEZO1-related lymphatic dysplasia: Expanding molecular and ultrasound findings. Eur. J. Med. Genet. 64, 104106. (doi:10.1016/j.\n",
      "\n",
      "ejmg.2020.104106)\n",
      "\n",
      "108. Stolyar H, Berry T, Singh AP, Madan I. 2021 PIEZO1 mutation: a rare aetiology for fetal ascites. BMJ Case Rep. 14, e240682. (doi:10.1136/bcr-2020-240682) 109. Song A, Lu L, Li Y, Lin M, Yuan X, Cheng X, Xia W, Wang O, Xing X. 2020 Low HbA1c with normal hemoglobin in a diabetes patient caused by PIEZO1 gene variant: A case report. Front. Endocrinol. 11, 356. (doi:10.3389/fendo.2020.00356)\n",
      "\n",
      "110. de Meira Oliveira P et al. 2020 Heterogeneous phenotype of hereditary xerocytosis in association with PIEZO1 variants. Blood Cells Mol. Dis. 82, 102413. (doi:10.1016/j.bcmd.2020.\n",
      "\n",
      "102413)\n",
      "\n",
      "111. Risinger M, Glogowska E, Chonat S, Zhang K, Dagaonkar N, Joiner CH, Quinn CT, Kalfa TA, Gallagher PG. 2018 Hereditary xerocytosis: diagnostic considerations. Am. J. Hematol. 93,\n",
      "\n",
      "E67–E69. (doi:10.1002/ajh.24996)\n",
      "\n",
      "112. Pinto VM et al. 2023 Coinheritance of PIEZO1 variants and multi‐locus red blood cell defects account for the symptomatic phenotype in beta‐thalassemia carriers. Am. J. Hematol.\n",
      "\n",
      "98, E130–E133. (doi:10.1002/ajh.26901)\n",
      "\n",
      "113. Monti R, Rautenstrauch P, Ghanbari M, James AR, Kirchler M, Ohler U, Konigorski S, Lippert C. 2022 Identifying interpretable gene-biomarker associations with functionally\n",
      "\n",
      "informed kernel-based tests in 190,000 exomes. Nat. Commun. 13, 5332. (doi:10.1038/s41467-022-32864-2)\n",
      "\n",
      "114. Rotordam MG et al. 2019 A novel gain-of-function mutation of Piezo1 is functionally affirmed in red blood cells by high-throughput patch clamp. Haematologica 104, e179–e183.\n",
      "\n",
      "(doi:10.3324/haematol.2018.201160)\n",
      "\n",
      "115. Hu Y et al. 2021 Whole-genome sequencing association analysis of quantitative red blood cell phenotypes: The NHLBI TOPMed program. Am. J. Hum. Genet. 108, 874–893. (doi:10.\n",
      "\n",
      "1016/j.ajhg.2021.04.003)\n",
      "\n",
      "116. Zarychanski R, Schulz VP, Houston BL, Maksimova Y, Houston DS, Smith B, Rinehart J, Gallagher PG. 2012 Mutations in the mechanotransduction protein PIEZO1 are associated\n",
      "\n",
      "with hereditary xerocytosis. Blood 120, 1908–1915. (doi:10.1182/blood-2012-04-422253)\n",
      "\n",
      "117. Bae C, Gnanasambandam R, Nicolai C, Sachs F, Gottlieb PA. 2013 Xerocytosis is caused by mutations that alter the kinetics of the mechanosensitive channel PIEZO1. Proc. Natl Acad.\n",
      "\n",
      "Sci. USA 110, E1162–E1168. (doi:10.1073/pnas.1219777110)\n",
      "\n",
      "118. Zhang T, Zhu X, Gao Z, Huang W, Kong X. 2021 Analysis of three families with recurrence of non-immune hydrops fetalis by trio whole exome sequencing. Zhonghua Yi Xue Yi Chuan\n",
      "\n",
      "Xue Za Zhi 38, 937–941. (doi:10.3760/cma.j.cn511374-20200729-00564)\n",
      "\n",
      "119. Sun Q et al. 2022 Analyses of biomarker traits in diverse UK Biobank participants identify associations missed by European-centric analysis strategies. J. Hum. Genet. 67, 87–93.\n",
      "\n",
      "(doi:10.1038/s10038-021-00968-0)\n",
      "\n",
      "120. Smelser DT, Haley JS, Ryer EJ, Elmore JR, Cook AM, Carey DJ. 2022 Association of varicose veins with rare protein-truncating variants in PIEZO1 identified by exome sequencing of a\n",
      "\n",
      "large clinical population. J. Vasc. Surg. Venous Lymphat. Disord. 10, 382–389. (doi:10.1016/j.jvsv.2021.07.007)\n",
      "\n",
      "121. Mustacich DJ et al. 2022 Digenic inheritance of a FOXC2 mutation and two PIEZO1 mutations underlies congenital lymphedema in a multigeneration family. Am. J. Med. 135, e31–\n",
      "\n",
      "e41. (doi:10.1016/j.amjmed.2021.09.007)\n",
      "\n",
      "122. Henkel C et al. 2023 Genome-wide association meta-analysis of knee and hip osteoarthritis uncovers genetic differences between patients treated with joint replacement and\n",
      "\n",
      "123.\n",
      "\n",
      "patients without joint replacement. Ann. Rheum. Dis. 82, 384–392. (doi:10.1136/ard-2022-223199) Imashuku S et al. 2016 PIEZO1 gene mutation in a Japanese family with hereditary high phosphatidylcholine hemolytic anemia and hemochromatosis-induced diabetes mellitus. Int. J. Hematol. 104, 125–129. (doi:10.1007/s12185-016-1970-x)\n",
      "\n",
      "124. Del Orbe Barreto R, Arrizabalaga B, De la Hoz Rastrollo AB, García-Orad A, Gonzalez Vallejo I, Bento C, Villegas A, García-Ruiz JC. 2016 Hereditary xerocytosis, a misleading anemia.\n",
      "\n",
      "Ann. Hematol. 95, 1545–1546. (doi:10.1007/s00277-016-2716-9)\n",
      "\n",
      "125. NIH. 2025 PIEZO1: piezo type mechanosensitive ion channel component 1 (Er blood group) Homo sapiens [Homo sapiens (human)]. See https://www.ncbi.nlm.nih.gov/gene/9780. 126. Gordon K, Varney R, Keeley V, Riches K, Jeffery S, Van Zanten M, Mortimer P, Ostergaard P, Mansour S. 2020 Update and audit of the St George’s classification algorithm of primary lymphatic anomalies: a clinical and molecular approach to diagnosis. J. Med. Genet. 57, 653–659. (doi:10.1136/jmedgenet-2019-106084)\n",
      "\n",
      "127. Gordon K, Mortimer PS, van Zanten M, Jeffery S, Ostergaard P, Mansour S. 2021 The St George’s classification algorithm of primary lymphatic anomalies. Lymphat. Res. Biol. 19, 25–\n",
      "\n",
      "30. (doi:10.1089/lrb.2020.0104)\n",
      "\n",
      "128. NIH. 2025 Lymphatic malformation 6. See https://www.ncbi.nlm.nih.gov/gtr/conditions/C4225184.\n",
      "\n",
      "36\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "129. Choi D et al. 2019 Piezo1 incorporates mechanical force signals into the genetic program that governs lymphatic valve development and maintenance. JCI Insight 4, e125068. (doi:\n",
      "\n",
      "10.1172/jci.insight.125068)\n",
      "\n",
      "130. Choi D et al. 2022 Piezo1-regulated mechanotransduction controls flow-activated lymphatic expansion. Circ. Res. 131, e2–e21. (doi:10.1161/CIRCRESAHA.121.320565) 131. Anwar MA, Georgiadis KA, Shalhoub J, Lim CS, Gohel MS, Davies AH. 2012 A review of familial, genetic, and congenital aspects of primary varicose vein disease. Circ. Cardiovasc. Genet. 5, 460–466. (doi:10.1161/CIRCGENETICS.112.963439)\n",
      "\n",
      "132. Yuan S, Bruzelius M, Damrauer SM, Larsson SC. 2021 Cardiometabolic, lifestyle, and nutritional factors in relation to varicose veins: a Mendelian randomization study. J. Am. Heart\n",
      "\n",
      "Assoc. 10, e022286. (doi:10.1161/jaha.121.022286)\n",
      "\n",
      "133. Fukaya E, Flores AM, Lindholm D, Gustafsson S, Zanetti D, Ingelsson E, Leeper NJ. 2018 Clinical and genetic determinants of varicose veins. Circulation 138, 2869–2880. (doi:10.\n",
      "\n",
      "1161/circulationaha.118.035584)\n",
      "\n",
      "134. Van Hout CV et al. 2020 Exome sequencing and characterization of 49,960 individuals in the UK Biobank. Nature 586, 749–756. (doi:10.1038/s41586-020-2853-0) 135. Zhang DD et al. 2024 Exome sequencing identifies novel genetic variants associated with varicose veins. PLoS Genet. 20, e1011339. (doi:10.1371/journal.pgen.1011339) 136. Yoshida S, Koshima I, Imai H, Sasaki A, Fujioka Y, Nagamatsu S, Yokota K. 2019 Lymphatic flow restoration after stripping surgery for varicose veins: a case report. SAGE Open Med.\n",
      "\n",
      "Case Rep. 7, 2050313X1984926. (doi:10.1177/2050313x19849265) 137. Iolascon A, Andolfo I, Russo R. 2019 Advances in understanding the pathogenesis of red cell membrane disorders. Br. J. Haematol. 187, 13–24. (doi:10.1111/bjh.16126) 138. Stewart GW, Gibson JS, Rees DC. 2023 The cation‐leaky hereditary stomatocytosis syndromes: a tale of six proteins. Br. J. Haematol. 203, 509–522. (doi:10.1111/bjh.19093) 139. Andolfo I, Russo R, Rosato BE, Manna F, Gambale A, Brugnara C, Iolascon A. 2018 Genotype‐phenotype correlation and risk stratification in a cohort of 123 hereditary\n",
      "\n",
      "stomatocytosis patients. Am. J. Hematol. 93, 1509–1517. (doi:10.1002/ajh.25276)\n",
      "\n",
      "140. Shan Y, Guo X, Zhang M, Chen M, Li Y, Zhang M, Pei D. 2024 Structure of human PIEZO1 and its slow inactivating channelopathy mutants. bioRxiv. (doi:10.1101/2024.07.14.\n",
      "\n",
      "603468)\n",
      "\n",
      "141. Evans EL et al. 2020 RBCs prevent rapid PIEZO1 inactivation and expose slow deactivation as a mechanism of dehydrated hereditary stomatocytosis. Blood 136, 140–144. (doi:10.\n",
      "\n",
      "1182/blood.2019004174)\n",
      "\n",
      "142. Andolfo I, Rosato BE, Manna F, De Rosa G, Marra R, Gambale A, Girelli D, Russo R, Iolascon A. 2020 Gain‐of‐function mutations in PIEZO1 directly impair hepatic iron metabolism via\n",
      "\n",
      "the inhibition of the BMP/SMADs pathway. Am. J. Hematol. 95, 188–197. (doi:10.1002/ajh.25683)\n",
      "\n",
      "143. Martin‐Almedina S, Mansour S, Ostergaard P. 2018 Human phenotypes caused by PIEZO1 mutations; one gene, two overlapping phenotypes? J. Physiol. 596, 985–992. (doi:10.\n",
      "\n",
      "1113/jp275718)\n",
      "\n",
      "144. Bartoli F et al. 2022 Global PIEZO1 gain-of-function mutation causes cardiac hypertrophy and fibrosis in mice. Cells 11, 1199. (doi:10.3390/cells11071199) 145. Brylka LJ et al. 2024 Piezo1 expression in chondrocytes controls endochondral ossification and osteoarthritis development. Bone Res. 12, 12. (doi:10.1038/s41413-024-00315-x) 146. Hirano K et al. 2023 The mechanosensitive ion channel PIEZO1 promotes satellite cell function in muscle regeneration. Life Sci. Alliance 6, e202201783. (doi:10.26508/lsa. 202201783)\n",
      "\n",
      "147. Tsuchiya M et al. 2018 Cell surface flip-flop of phosphatidylserine is critical for PIEZO1-mediated myotube formation. Nat. Commun. 9, 2049. (doi:10.1038/s41467-018-04436-w) 148. Wang S et al. 2016 Endothelial cation channel PIEZO1 controls blood pressure by mediating flow-induced ATP release. J. Clin. Investig 126, 4527–4536. (doi:10.1172/JCI87343) 149. Rode B et al. 2017 Piezo1 channels sense whole body physical activity to reset cardiovascular homeostasis and enhance performance. Nat. Commun. 8, 350. (doi:10.1038/s41467- 017-00429-3)\n",
      "\n",
      "150. Yu ZY et al. 2022 Piezo1 is the cardiac mechanosensor that initiates the cardiomyocyte hypertrophic response to pressure overload in adult mice. Nat. Cardiovasc. Res. 1, 577–591.\n",
      "\n",
      "(doi:10.1038/s44161-022-00082-0)\n",
      "\n",
      "151. Zhang Y et al. 2021 Piezo1-mediated mechanotransduction promotes cardiac hypertrophy by impairing calcium homeostasis to activate calpain/calcineurin signaling. Hypertension\n",
      "\n",
      "78, 647–660. (doi:10.1161/hypertensionaha.121.17177)\n",
      "\n",
      "152. Lim XR et al. 2024 Endothelial Piezo1 channel mediates mechano-feedback control of brain blood flow. Nat. Commun 15, 8686. (doi:10.1038/s41467-024-52969-0) 153. Ranade SS et al. 2014 Piezo1, a mechanically activated ion channel, is required for vascular development in mice. Proc. Natl Acad. Sci. USA 111, 10347–10352. (doi:10.1073/pnas. 1409233111)\n",
      "\n",
      "154. Najafi K, Mehrjoo Z, Ardalani F, Ghaderi-Sohi S, Kariminejad A, Kariminejad R, Najmabadi H. 2021 Identifying the causes of recurrent pregnancy loss in consanguineous couples\n",
      "\n",
      "using whole exome sequencing on the products of miscarriage with no chromosomal abnormalities. Sci. Rep. 11, 6952. (doi:10.1038/s41598-021-86309-9)\n",
      "\n",
      "155. Chong J, De Vecchis D, Hyman AJ, Povstyan OV, Ludlow MJ, Shi J, Beech DJ, Kalli AC. 2021 Modeling of full-length Piezo1 suggests importance of the proximal N-terminus for dome\n",
      "\n",
      "structure. Biophys. J. 120, 1343–1356. (doi:10.1016/j.bpj.2021.02.003)\n",
      "\n",
      "156. Chen Z et al. 2025 Natural selection shaped the protective effect of the mtDNA lineage against obesity in Han Chinese populations. J. Genet. Genom. 52, 539–548. (doi:10.1016/j.\n",
      "\n",
      "jgg.2024.06.005)\n",
      "\n",
      "157. Martin-Almedina S, Mortimer PS, Ostergaard P. 2021 Development and physiological functions of the lymphatic system: insights from human genetic studies of primary\n",
      "\n",
      "lymphedema. Physiol. Rev. 101, 1809–1871. (doi:10.1152/physrev.00006.2020)\n",
      "\n",
      "158. Wang J, Jiang J, Yang X, Zhou G, Wang L, Xiao B. 2022 Tethering Piezo channels to the actin cytoskeleton for mechanogating via the cadherin-β-catenin mechanotransduction\n",
      "\n",
      "complex. Cell Rep. 38, 110342. (doi:10.1016/j.celrep.2022.110342)\n",
      "\n",
      "159. Chuntharpursat-Bon E et al. 2023 PIEZO1 and PECAM1 interact at cell-cell junctions and partner in endothelial force sensing. Commun. Biol. 6, 358. (doi:10.1038/s42003-023-\n",
      "\n",
      "04706-4)\n",
      "\n",
      "160. Byrne AB et al. 2022 Pathogenic variants in MDFIC cause recessive central conducting lymphatic anomaly with lymphedema. Sci. Transl. Med. 14, eabm4869. (doi:10.1126/\n",
      "\n",
      "scitranslmed.abm4869)\n",
      "\n",
      "161. Michelini S et al. 2020 Rare PECAM1 variants in three families with lymphedema. Lymphology 53, 141–151. (doi:10.2458/lymph.4666) 162. de Haan HG, Bezemer ID, Vossen CY, van Hylckama Vlieg A, Böehringer S, Hasstedt SJ, Levy S, Rosendaal FR, Bovill EG. 2014 Genetic variants in cell adhesion molecule 1 (CADM1): a validation study of a novel endothelial cell venous thrombosis risk factor. Thromb. Res. 134, 1186–1192. (doi:10.1016/j.thromres.2014.09.018)\n",
      "\n",
      "163. Hasstedt SJ, Bezemer ID, Callas PW, Vossen CY, Trotman W, Hebbel RP, Demers C, Rosendaal FR, Bovill EG. 2009 Cell adhesion molecule 1: a novel risk factor for venous thrombosis.\n",
      "\n",
      "Blood 114, 3084–3091. (doi:10.1182/blood-2009-05-219485)\n",
      "\n",
      "164. Murciano N et al. 2023 A high-throughput electrophysiology assay to study the response of PIEZO1 to mechanical stimulation. J. Gen. Physiol. 155, e202213132. (doi:10.1085/jgp.\n",
      "\n",
      "202213132)\n",
      "\n",
      "165. Parsonage G et al. 2023 Improved PIEZO1 agonism through 4‐benzoic acid modification of Yoda1. Br. J. Pharmacol. 180, 2039–2063. (doi:10.1111/bph.15996)\n",
      "\n",
      "37\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n",
      "\n",
      "166. Petkova-Kirova P et al. 2024 The Gárdos channel and Piezo1 revisited: comparison between reticulocytes and mature red blood cells. Int. J. Mol. Sci. 25, 1416. (doi:10.3390/\n",
      "\n",
      "ijms25031416)\n",
      "\n",
      "167. De Vecchis D, Beech DJ, Kalli AC. 2021 Molecular dynamics simulations of Piezo1 channel opening by increases in membrane tension. Biophys. J. 120, 1510–1521. (doi:10.1016/j.\n",
      "\n",
      "bpj.2021.02.006)\n",
      "\n",
      "168. Moffatt CJ, Keeley V, Franks PJ, Rich A, Pinnington LL. 2017 Chronic oedema: a prevalent health care problem for UK health services. Int. Wound J. 14, 772–781. (doi:10.1111/iwj.\n",
      "\n",
      "12694)\n",
      "\n",
      "169. Choi D et al. 2024 Piezo1 regulates meningeal lymphatic vessel drainage and alleviates excessive CSF accumulation. Nat. Neurosci. 27, 913–926. (doi:10.1038/s41593-024-01604-\n",
      "\n",
      "8)\n",
      "\n",
      "170. Du J, Liu P, Zhou Y, Misener S, Sharma I, Leeaw P, Thomson BR, Jin J, Quaggin SE. 2024 The mechanosensory channel PIEZO1 functions upstream of angiopoietin/TIE/FOXO1\n",
      "\n",
      "signaling in lymphatic development. J. Clin. Invest. 134, e176577. (doi:10.1172/JCI176577)\n",
      "\n",
      "171. Matrongolo MJ et al. 2023 Piezo1 agonist restores meningeal lymphatic vessels, drainage, and brain-CSF perfusion in craniosynostosis and aged mice. J. Clin. Invest. 134, e171468.\n",
      "\n",
      "(doi:10.1172/JCI171468)\n",
      "\n",
      "172. Bolte AC et al. 2020 Meningeal lymphatic dysfunction exacerbates traumatic brain injury pathogenesis. Nat. Commun. 11, 4524. (doi:10.1038/s41467-020-18113-4) 173. Zhao J, Xiong Y, Liu Y, Ling J, Liu S, Wang W. 2025 Endothelium Piezo1 deletion alleviates experimental varicose veins by attenuating perivenous inflammation. Mol. Cell. Biochem. 480, 2423–2435. (doi:10.1007/s11010-024-05115-9)\n",
      "\n",
      "174. Romero LO, Massey AE, Mata-Daboin AD, Sierra-Valdez FJ, Chauhan SC, Cordero-Morales JF, Vásquez V. 2019 Dietary fatty acids fine-tune Piezo1 mechanical response. Nat.\n",
      "\n",
      "Commun. 10, 1200. (doi:10.1038/s41467-019-09055-7)\n",
      "\n",
      "175. Glogowska E et al. 2025 Potentiation of macrophage Piezo1 by atherogenic 7-ketocholesterol. Cell Rep. 44, 115542. (doi:10.1016/j.celrep.2025.115542) 176. Zhu J et al. 2023 The mechanosensitive ion channel Piezo1 contributes to ultrasound neuromodulation. Proc. Natl Acad. Sci. USA 120, e2300291120. (doi:10.1073/pnas. 2300291120)\n",
      "\n",
      "177. Chen X, Chen J. 2022 miR-10b-5p-mediated upregulation of PIEZO1 predicts poor prognosis and links to purine metabolism in breast cancer. Genomics 114, 110351. (doi:10.1016/j.\n",
      "\n",
      "ygeno.2022.110351)\n",
      "\n",
      "178. Cheng J et al. 2023 Accurate proteome-wide missense variant effect prediction with AlphaMissense. Science 381, eadg7492. (doi:10.1126/science.adg7492) 179. Gallagher PG. 2017 Disorders of erythrocyte hydration. Blood 130, 2699–2708. (doi:10.1182/blood-2017-04-590810)\n",
      "\n",
      "38\n",
      "\n",
      "r o y a l s o c i e t y p u b\n",
      "\n",
      "l i s h n g . o r g / j o u r n a l / r s o b\n",
      "\n",
      "i\n",
      "\n",
      "O p e n B o l .\n",
      "\n",
      "i\n",
      "\n",
      "1 5 :\n",
      "\n",
      "2 4 0 3 4 5\n"
     ]
    }
   ],
   "source": [
    "with open(\"rsob_240345.txt\", 'r', encoding = 'utf-8') as f:\n",
    "    text = f.read()\n",
    "\n",
    "print(text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "4e8fd32e-45b7-4d14-83b5-8aef101113be",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Warning: No languages specified, defaulting to English.\n",
      "B\n",
      "\n",
      "R\n",
      "\n",
      "AIN COMMUNICATIONS\n",
      "\n",
      "https://doi.org/10.1093/braincomms/fcaf276\n",
      "\n",
      "BRAIN COMMUNICATIONS 2025: fcaf276\n",
      "\n",
      "Latent dimension linking functional connectivity with post-stroke deficits across multiple domains Ke Wu,1 Yaya Jiang,2 Junhao Luo,1,3,4 Yijun Chen,5 Shaoling Peng,6 Xiangyu Kong1 and Gaolang Gong1,7,8\n",
      "\n",
      "Stroke often leads to multiple behavioural impairments. Understanding the neural basis of these deficits is essential for elucidating the mechanisms of functional impairments and optimising therapeutic strategies for stroke patients. Although many studies have revealed that specific behavioural deficits are related to disruptions in distributed functional connectivity across brain networks, these studies typically focus on single behavioural traits, overlooking the multivariate characteristics of deficits after stroke. Recent studies have demonstrated that deficits within and across domains are highly correlated, suggesting a complex many-to-many mapping between brain and behaviour following stroke. Thus, the present study aims to identify meaningful multivariate patterns of functional connect- ivity–behaviour covariation following stroke. Specifically, we employed a multivariate data-driven approach, partial least squares correlation, to examine the relationships between whole-brain functional connectivity and an extensive array of neurological scores (including motor, attention, verbal memory, spatial memory and language domains) in a large cohort of stroke patients at 2 weeks (n = 81), 3 months (n = 78) and 12 months (n = 74) post-injury. This multivariate analysis revealed a significant latent component (LC) from 2-week post-stroke data, capturing a unique pattern of cognitive deficits across multiple domains. This pattern was strongly associated with widespread network dysfunction, characterized by decreased interhemispheric connectivity and increased intrahemi- spheric connectivity. Notably, the identified LC was replicated and generalized to stroke data at the 3-month and 12-month time points. Furthermore, we examined whether structural lesion features, including structural disconnection of white matter pathways and grey matter damage, could explain variance in the identified LC. Structural disconnection outperformed grey matter damage, highlighting its critical role in the functional connectivity–behaviour relationship following stroke. Mediation analysis confirmed that structural disconnection serves as the neuroanatomical basis for the association between functional connectivity and deficits. Overall, this study suggests that stroke-induced white matter disconnections are associated with widespread and consistent disrup- tions in brain network connectivity, which are reflected in a highly correlated behavioural profile. These results provide an integrative insight into the complex relationships among lesions, functional networks, and behavioural outcomes following stroke.\n",
      "\n",
      "1 State Key Laboratory of Cognitive Neuroscience and Learning & IDG/McGovern Institute for Brain Research, Beijing Normal\n",
      "\n",
      "University, Beijing 100875, China\n",
      "\n",
      "2 Artificial Intelligence and Language Cognition Laboratory, Beijing International Studies University, Beijing 100024, China 3 Shenzhen CyberAray Network Technology Co., Ltd, Shenzhen 518038, China 4 Harbin Institute of Technology (Shenzhen), Shenzhen 518055, China 5 Department of Psychology, Georgetown University, Washington, DC 20057, USA 6 Department of Neurology, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA 7 Beijing Key Laboratory of Brain Imaging and Connectomics, Beijing Normal University, Beijing 100875, China 8 Chinese Institute for Brain Research, Beijing 102206, China\n",
      "\n",
      "Correspondence to: Gong Gaolang State Key Laboratory of Cognitive Neuroscience and Learning & IDG/McGovern\n",
      "\n",
      "Received November 19, 2024. Revised June 10, 2025. Accepted July 18, 2025. Advance access publication July 22, 2025 © The Author(s) 2025. Published by Oxford University Press on behalf of the Guarantors of Brain. This is an Open Access article distributed under the terms of the Creative Commons Attribution License (https://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.\n",
      "\n",
      "| 1\n",
      "\n",
      "2\n",
      "\n",
      "| BRAIN COMMUNICATIONS 2025, fcaf276 K. Wu et al.\n",
      "\n",
      "Institute for Brain Research, Beijing Normal University Beijing 100875, China E-mail: gaolang.gong@bnu.edu.cn\n",
      "\n",
      "Keywords: stroke; functional connectivity; recovery; lesion; structural disconnection\n",
      "\n",
      "Graphical Abstract\n",
      "\n",
      "Introduction Stroke is one of the leading causes of disability and death worldwide. Stroke patients generally suffer from behaviour- al deficits in multiple domains, e.g. motor, language, atten- tion, and executive functions.1-4 The occurrence of multiple deficits reflects not merely structural damage at the injury site,5,6 but also widespread dysfunction within and between networks.7-11 Elucidating the associations be- tween brain functional networks and behavioural perform- ance across multiple domains in stroke patients is crucial for understanding the mechanisms of neurological disorders and improving the treatment strategies.12,13\n",
      "\n",
      "Functional magnetic resonance imaging (fMRI) during the performance of specific tasks, e.g. motor, language, and at- tention, has been widely applied in the study of stroke\n",
      "\n",
      "patients, shedding light on functional abnormalities or re- organisation following stroke.11,14,15 However, this tech- nique has certain limitations when used in stroke studies. First, task-based fMRI studies often exclude stroke patients with severe functional deficits due to their inability to com- plete experimental tasks designed for healthy individuals.14 This selection bias may limit the generalizability of findings to the broader stroke population. Moreover, activation para- digms refer to task-based designs that probe specific func- tional systems, but are typically optimized for intact neural circuits following stroke.16,17 Additionally, task-evoked studies provide lim- ited information about temporal interactions between re- gions or networks.16 Resting-state functional MRI has emerged as a powerful tool to map the functional network connectivity of damaged brains by measuring the temporal\n",
      "\n",
      "rather\n",
      "\n",
      "than\n",
      "\n",
      "reorganized\n",
      "\n",
      "systems\n",
      "\n",
      "FC-deficit associations after stroke BRAIN COMMUNICATIONS 2025, fcaf276\n",
      "\n",
      "correlation of blood oxygen level-dependent (BOLD) signals across brain regions at rest [resting-state functional connect- ivity (RSFC)].18,19 Studies have demonstrated that severity of neglect symptoms in hemispatial neglect patients is corre- lated with FC alteration within and between (i.e. interac- tions) multiple resting-state networks other than specific attention networks.8,20,21 A subsequent study explored FC–deficit relationships across six behavioural domains (attention, spatial memory, verbal memory, language, motor and visual) in a large, heterogeneous cohort of stroke patients and revealed that each deficit was predicted by a common RSFC pattern characterized by decreased interhe- mispheric connectivity and increased intrahemispheric con- nectivity.22 Together, these lines of study indicate that RSFC offers crucial insights into the functional network or- ganisation associated with different domains of behavioural performance after stroke.\n",
      "\n",
      "Notably, most resting-state fMRI studies have used univari- ate correlation or multivariate regression approaches to high- light that behavioural dysfunction in a specific domain or task is related to FC disruptions across multiple networks, but they often neglected the potential correlations between behav- ioural dysfunctions across multiple domains or tasks and FC disruptions across multiple networks.8,20-25Evidence at the be- havioural level has demonstrated that focal and diffuse strokes do not typically cause highly specific and distinct syndromes or symptoms but instead result in deficits that are closely corre- lated both within and between behavioural domains at the population level.23,24,26 Traditional explanations for this phe- nomenon suggest that strokes occur within vascular territories involving multiple functional areas.12But correlated deficits have also been observed between regions without obvious vas- cular overlap. For example, language and spatial memory def- icits are significantly correlated (r = 0.59), despite being localized to different hemispheres.23 Another plausible explan- ation is the correlation of multiple deficits coupled with wide- spread and consistent disruption in network connectivity, as evidenced by the interdependent recovery of deficits across mul- tiple domains.12,13,24 This argument supports many-to-many mapping between behavioural deficits and functional network connectivity after stroke. However, until recently, few neuroi- maging studies have simultaneously investigated both the multivariate characteristics of behavioural deficits and the multivariate features of functional network connectivity to identify meaningful patterns of FC-behaviour covariation after stroke.\n",
      "\n",
      "deficits dependent on the normalisation of abnormal network connectivity.21 On this basis, we hypothesized that the behav- iour–connectivity covariance patterns would be maintained from the acute to the chronic stage. Furthermore, post-stroke structural brain lesions, including grey matter damage or white matter disconnection, are closely associated with widespread FC abnormalities and multiple behavioural deficits.33-35 However, few studies have explored the complete relationship linking structural damage to widespread FC disruptions and multidomain behavioural deficits. To address this gap, we con- ducted mediation analyses to deepen the understanding of the relationships among these three variables.\n",
      "\n",
      "Materials and methods Dataset The data used in this study are part of the Washington Stroke Cohort.22-24 The dataset includes 132 patients with a first symptomatic stroke, who were assessed at 2 weeks after stroke onset. Of these, 103 and 88 returned for the subse- quent measurements at 3 and 12 months, respectively. A healthy control (HC) group (n = 31), matched with the study sample for age, gender and years of education, was assessed twice at 3-month intervals. Written informed consent was obtained from all participants in accordance with the Declaration of Helsinki and procedures established by the Washington University in Saint Louis Institutional Review Board. All aspects of this study were approved by the Washington University School of Medicine Internal Review Board.\n",
      "\n",
      "The inclusion criteria of present study were as follows: is- chaemic or haemorrhagic stroke, and clinical evidence of motor, language, attention, visual, or memory deficits based on neurological examination. The exclusion criteria were as follows: (i) participants with data missing from more than one behavioural domain; (ii) bilateral lesions; and (iii) poor- quality T1-weighted images and fMRI data. Finally, 81 pa- tients were eligible at 2 weeks, 78 at 3 months, and 76 at 12 months; 28 and 26 controls were eligible at the first and second assessments, respectively (Table 1).\n",
      "\n",
      "In this study, we employed partial least squares correlation (PLSC) analysis, a multivariate data-driven approach, to exam- ine the relationships between whole-brain FC and an extensive array of neurological assessment scores. This analysis allows multivariate brain features to be mapped to multivariate behav- ioural variables in a single integrated analysis, thereby avoiding the pitfall of focusing on a single a priori behavioural aspect, which hides the overall picture.27-32Additionally, we examined whether the latent component (LC) identified in the 2-week stroke data remained stable throughout the recovery process. Previous studies have suggested the recovery of post-stroke\n",
      "\n",
      "Neuropsychological assessment The neuropsychological battery comprised 41 behavioural scores across five behavioural domains: language, attention, verbal memory, spatial memory, and motor function. Detailed descriptions of the behavioural tests are available in the Supplementary Table 1. To determine whether the missing data were missing completely at random (MCAR) or not, we conducted Little’s MCAR test.36 The test yielded a statistically significant result (P < 0.001), indicating that the missing data were not completely random. In this con- text, we adopted a mean imputation strategy that preserves as much information as possible while maintaining the integ- rity of the analysis.\n",
      "\n",
      "| 3\n",
      "\n",
      "4\n",
      "\n",
      "| BRAIN COMMUNICATIONS 2025, fcaf276 K. Wu et al.\n",
      "\n",
      "Table 1 Demographic data of participants in the present study\n",
      "\n",
      "Stroke patients\n",
      "\n",
      "Control healthy\n",
      "\n",
      "2 weeks\n",
      "\n",
      "3 months\n",
      "\n",
      "12 months\n",
      "\n",
      "Visit 1\n",
      "\n",
      "Visit 2\n",
      "\n",
      "Number Age Gender (M/F) Education (years) Handedness (R/L) Lesion type (ischaemic/haemorrhagic) Lesion side (R/L) Lesion size (cm3)\n",
      "\n",
      "81 52.2 ± 11.4 48/33 13.4 ± 2.6 73/8 68/13 38/43 3.29 ± 5.27\n",
      "\n",
      "78 53.8 ± 10.6 45/33 13.3 ± 2.4 72/6 56/14 35/43 3.2 ± 4.6\n",
      "\n",
      "74 54.1 ± 9.7 43/31 13.6 ± 2.5 68/6 64/10 34/40 2.8 ± 3.5\n",
      "\n",
      "28 55.1 ± 12.3 13/15 13.4 ± 2.6 27/1\n",
      "\n",
      "26 55.7 ± 11.8 16/12 13.5 ± 11.8 24/2\n",
      "\n",
      "Neuroimaging data collection Images across various modalities, including structural, function- al, pulsed arterial spin labelling (PASL), and diffusion tensor scans, were acquired using a Siemens 3T Tim-Trio scanner with a 12-channel head coil. Structural scans included sagittal T1-weighted MP-RAGE [repetition time (TR) = 1950 msec, echo time (TE) = 2.26 msec, flip angle = 90°, voxel size = 1.0× 1.0× 1.0 mm], transverse T2-weighted turbo spin echo (TR = 2500 msec, TE = 435 msec, voxel size = 1.0× 1.0× 1.0 mm) and sagittal fluid attenuated inversion recovery (FLAIR) (TR = 7500 msec, TE = 326 msec, voxel size = 1.5× 1.5× 1.5 mm). The PASL acquisition parameters were as follows: TR = 2600 msec, TE = 13 msec, flip angle = 90°, bandwidth = 2.232 kHz/Px, and field of view (FoV) = 220 mm; 120 volumes were acquired (322 s total), each containing 15 slices with slice thick- nesses of 6 mm and gaps of 23.7 mm. Resting-state functional scans were acquired with a gradient echo planar imaging se- quence (TR = 2000 msec, TE = 27 msec, 32 contiguous 4-mm slices, 4× 4 mm in-plane resolution). The participants fixated on a small cross in a low-luminance environment. A total of six to eight fMRI scans, each consisting of 128 volumes, were col- lected over approximately 30 min.\n",
      "\n",
      "Lesion identification Lesions were manually segmented on individual structural MR images with the Analyze biomedical imaging software system.37 The T1-weighted, T2-weighted, and T2-FLAIR scans were used together to ensure accurate lesion delinea- tion. Two board-certified neurologists reviewed all the seg- mentations, and an additional neurologist confirmed all the segmentations. For haemorrhagic strokes, oedema was in- cluded in the lesion. The final segmentations were used as binary lesion masks for subsequent processing and analysis steps. Lesion masks were transformed to Montreal Neurological Institute atlas space via a combination of linear transformations and non-linear warps and were resampled to achieve isotropic voxel resolution.\n",
      "\n",
      "asynchronous interpolation; (iii) whole-brain intensity normalisation to achieve a mode value of 1000; (iv) spatial realignment within and across functional MRI runs; and (v) cross-modal image registration and normalisa- tion. In the fifth step, functional MRI volumes from each time point were transformed from native space to standard space via deformation fields obtained from our previous longitudinal voxel-based morphometry analyses.38\n",
      "\n",
      "slice acquisition via\n",
      "\n",
      "sinc\n",
      "\n",
      "In preparation for the FC analysis, data underwent several additional preprocessing steps: (i) spatial smoothing consist- ing of a 6 mm full-width at half-maximum Gaussian blur in each direction; (ii) temporal filtering retaining frequencies in the 0.009–0.08 Hz band; (iii) regression of head motion, sig- nals from ventricles and cerebrospinal fluid, signals from white matter, and the global signal; (iv) scrubbing using the DVARS measure; and (v) removal of the mean signal in- tensity over the run. Notably, the calculation of the brain sig- nal in the third step did not consider the signal of the lesion location.\n",
      "\n",
      "FC features We used the Gordon 333 cortical parcellation and network community assignments to obtain region-level and network- level measures of FC. This parcellation was based on FC bound- ary mapping and InfoMap community detection analyses of resting-state fMRI data from 120 healthy individuals.39A total of 333 regions of interest (ROIs) assigned to 13 large-scale func- tional networks were used for subsequent estimation of FC. Region-wise FC matrices were constructed by correlating the average nuisance-regressed BOLD time series of each region with the average nuisance-regressed BOLD time series of every other region and applying the Fisher z-transformation to the re- sulting linear correlation values. For each patient, voxels that fell within the boundaries of the lesion were masked out, and regions with more than 50% lesion overlap were excluded from all analyses, in accordance with previous reports.40\n",
      "\n",
      "fMRI data preprocessing Resting-state fMRI preprocessing consisted of (i) removal of the first four frames and brain extraction; (ii) compensation for\n",
      "\n",
      "Structural lesion features Structural lesion features included region-based damage and region-based structural disconnection (SDC). Region-based damage was calculated as the percentage of voxels within grey matter regions overlapping with the lesion, creating a\n",
      "\n",
      "FC-deficit associations after stroke BRAIN COMMUNICATIONS 2025, fcaf276\n",
      "\n",
      "1-dimensional vector. This measure quantifies the damage to predefined brain regions rather than individual voxels. Region-based SDC was defined as the proportion of stream- lines connecting two regions that were disrupted by the le- sion, which was determined with the Lesion Quantification Toolkit (LQT). The LQT incorporates a publicly available diffusion MRI streamline tractography atlas to create a structural connectome template.41 On the basis of 333 cor- tical parcellation inputs and the Human Connectome Project-842 tractography atlas, a 333×333 region-wise structural connectome was created. By embedding the lesion into the atlas-derived structural connectome, the LQT gener- ated a 333×333 SDC matrix where each value represents the severity of lesion-induced disruption in structural connectiv- ity between two brain parcels. Before conducting the statis- tical analyses, the upper triangles (excluding diagonal elements) of the disconnection matrices were extracted and reshaped into a 1-dimensional vector.\n",
      "\n",
      "Statistical analysis PLSC analysis We used PLSC analysis to examine the relationships between whole-brain FC and multiple neurological measures across five behavioural domains (Fig. 1). Both sets of variables were correlated with each other across participants, and the resulting covariance matrix was subjected to singular va- lue decomposition to reveal the latent brain–behaviour com- ponents. Each LC comprised an RSFC pattern at the node level (RSFC saliences) and a profile of behavioural deficits (behavioural saliences). Individual-specific RSFC and behav- ioural composite scores for each LC were obtained by linear- ly projecting the original RSFC and behavioural data of each participant onto their respective saliences.\n",
      "\n",
      "We also used PLSC analysis to examine the relationships between region-based SDCs and multiple-domain neuro- logical variables. For further details, refer to the above ana- lysis. To visualize the SDC loadings, the top 20% of loadings for LC1 were mapped onto the brain to display the most pre- dictive SDCs using BrainNet Viewer. To assess whether LC1 from the SDC–deficits PLSC model corresponds to LC1 from the FC–deficits PLSC model, we calculated the Pearson cor- relation coefficient between the LC1 behavioural/RSFC loadings of both models.\n",
      "\n",
      "Multivariate ridge regression with structural lesion features To understand the anatomical basis of the FC–deficit LC, we used a ridge regression machine learning algorithm to ex- plore whether structural lesion features could predict FC or behavioural composite scores obtained from the first PLSC analysis. The structural lesion features included region-based SDC measures and region-based lesion damage. Features with zero values across all participants were removed from the analysis. All ridge regression models were trained and tested via leave-one-out cross-validation (LOOCV). In each loop, the regularisation coefficient alpha was optimized by identifying a value between α = 2−10 and 25 that minimized leave-one-out prediction error over the training set. Next, optimal weights were calculated across the entire training set via gradient descent to minimize error in the ridge regres- sion equation. These optimal weights were then applied to the SDC/lesion features of the left-out participants to predict behavioural or RSFC composite scores. A prediction was generated for all participants with this procedure. Model performance was evaluated by calculating Pearson’s correl- ation coefficient between the measured and predicted com- posite scores.\n",
      "\n",
      "We computed Pearson’s correlations between partici- pants’ composite scores and the original data, referred to as loadings, to interpret the contribution of each variable to the LCs.27,42 A strong positive or negative correlation for a specific behavioural measure in a given LC indicates the greater significance of that behavioural measure to the LC. Likewise, a large positive (or negative) correlation for a specific RSFC measure in a given LC highlights the import- ance of the RSFC measure.\n",
      "\n",
      "A permutation test (1000 permutations) was used to de- individual LC.43 termine Additionally, bootstrap resampling (1000 bootstraps) was employed to estimate CIs of the behavioural and RSFC load- ings. Procrustes rotation was applied to the LCs obtained from the permutation and bootstrap analyses to ensure alignment with those from the original analyses.43,44 Bootstrapped z scores were calculated by dividing each load- ing by its bootstrap standard deviation, and then the z scores were converted to P values. Prior to this, bootstrapped RSFC loadings were averaged across ROI pairs within and between 26 networks (left: 13; right: 13), resulting in 26× 26 correl- ation matrices to limit the number of multiple comparisons.\n",
      "\n",
      "the\n",
      "\n",
      "significance of each\n",
      "\n",
      "Mediation analysis among the RSFC, SDC and behaviour composite scores We used mediation analysis to explore the complex relation- ships among SDC, RSFC and behaviours. In the first mediation analysis, the RSFC composite scores from the FC–deficit PLSC model were defined as the mediator variable, the SDC composite scores from the SDC–deficit PLSC model were defined as the independent variable, and the behavioural composite scores from both PLSC models were defined indi- vidually as the dependent variable. In the second mediation analysis, the SDC composite scores were defined as the medi- ator variable, the FC composite scores were defined as the in- dependent variable, and the behavioural composite scores were defined individually as the dependent variable. Theoretically, the relationship between the SDC and behav- iour is mediated by large-scale functional connectivity dis- ruptions. Therefore, we hypothesized that the mediation effect would be significant in the first analysis but not in the second.\n",
      "\n",
      "We employed a non-parametric bootstrapping method to assess the significance of the mediation effect. After 10 000 bias-corrected bootstraps, we estimated the distribution of\n",
      "\n",
      "| 5\n",
      "\n",
      "6\n",
      "\n",
      "| BRAIN COMMUNICATIONS 2025, fcaf276 K. Wu et al.\n",
      "\n",
      ", s t u p n\n",
      "\n",
      "i\n",
      "\n",
      "n i a r b r o F\n",
      "\n",
      ".\n",
      "\n",
      "x i r t a m r u o i v a h e b a d n a x i r t a m C F n i a r b - e o h w a\n",
      "\n",
      "l\n",
      "\n",
      "f\n",
      "\n",
      "o t s i s n o c l\n",
      "\n",
      "e d o m C S L P e h t o t\n",
      "\n",
      "s t u p n\n",
      "\n",
      "i\n",
      "\n",
      "e h t\n",
      "\n",
      ", s n o i t a i c o s s a\n",
      "\n",
      "r u o i v a h e b – n i a r b g n i r o p x e f\n",
      "\n",
      "l\n",
      "\n",
      "o t x e t n o c e h t n I\n",
      "\n",
      ".\n",
      "\n",
      "C S L P\n",
      "\n",
      "f o m a r g a i D 1 e r u g F\n",
      "\n",
      "i\n",
      "\n",
      "s i h t\n",
      "\n",
      "f\n",
      "\n",
      "o ) s t n e m e e\n",
      "\n",
      "l\n",
      "\n",
      "l a n o g a i d g n d u l c x e (\n",
      "\n",
      "i\n",
      "\n",
      "s e l g n a i r t\n",
      "\n",
      "r e p p u e h t d n a\n",
      "\n",
      ", t c e b u s h c a e r o\n",
      "\n",
      "j\n",
      "\n",
      "f d e t c u r t s n o c\n",
      "\n",
      "s i\n",
      "\n",
      "x i r t a m C F n i a r b - e o h w A\n",
      "\n",
      "l\n",
      "\n",
      ". s n o i t a l l\n",
      "\n",
      "e c r a p\n",
      "\n",
      "l a c i t r o c d e n fi e d e r p m o r f d e t c a r t x e e r a\n",
      "\n",
      "s e i r e s e m\n",
      "\n",
      "i t\n",
      "\n",
      "l a n g i s D L O B\n",
      "\n",
      "d n a\n",
      "\n",
      "y r o m e m\n",
      "\n",
      "l a b r e v\n",
      "\n",
      ",\n",
      "\n",
      "n o i t n e t t a\n",
      "\n",
      ",\n",
      "\n",
      "e g a u g n a l\n",
      "\n",
      ", r o t o m e h t\n",
      "\n",
      "g n n n a p s\n",
      "\n",
      "i\n",
      "\n",
      "s t n e m\n",
      "\n",
      "s s e s s a\n",
      "\n",
      "s u o i r a v\n",
      "\n",
      "g n d u l c n\n",
      "\n",
      "i\n",
      "\n",
      "i\n",
      "\n",
      ", t c e b u s h c a e r o\n",
      "\n",
      "j\n",
      "\n",
      "f\n",
      "\n",
      "a t a d\n",
      "\n",
      "l\n",
      "\n",
      "e v e l -\n",
      "\n",
      "m e t i\n",
      "\n",
      ", s t u p n\n",
      "\n",
      "i\n",
      "\n",
      "r u o i v a h e b r o F\n",
      "\n",
      ". r o t c e v\n",
      "\n",
      "l a n o i s n e m d - e n o a o t n\n",
      "\n",
      "i\n",
      "\n",
      "i\n",
      "\n",
      "d e p a h s e r\n",
      "\n",
      "s i\n",
      "\n",
      "x i r t a m\n",
      "\n",
      "d n a C F\n",
      "\n",
      "e h t\n",
      "\n",
      "n\n",
      "\n",
      "i\n",
      "\n",
      "s e b a i r a v\n",
      "\n",
      "l\n",
      "\n",
      "e h t\n",
      "\n",
      "f\n",
      "\n",
      "o s n o i t a n b m o c\n",
      "\n",
      "i\n",
      "\n",
      "r a e n\n",
      "\n",
      "i l\n",
      "\n",
      "t n e s e r p e r\n",
      "\n",
      "h c i h w\n",
      "\n",
      ", s e c n e\n",
      "\n",
      "i l a s\n",
      "\n",
      "l a r u o i v a h e b\n",
      "\n",
      "d n a\n",
      "\n",
      "s e c n e\n",
      "\n",
      "i l a s C F\n",
      "\n",
      "s e fi i t n e d\n",
      "\n",
      "i\n",
      "\n",
      "e d o m e h T\n",
      "\n",
      ". l\n",
      "\n",
      "e d o m C S L P\n",
      "\n",
      "e h t\n",
      "\n",
      "o t n\n",
      "\n",
      "i\n",
      "\n",
      "d e t a r o p r o c n\n",
      "\n",
      "e r a\n",
      "\n",
      ", s n i a m o d\n",
      "\n",
      "y r o m e m\n",
      "\n",
      "l a i t a p s\n",
      "\n",
      "s e c n e\n",
      "\n",
      "i l a s e s e h t d e z i m\n",
      "\n",
      "i t p o\n",
      "\n",
      "l\n",
      "\n",
      "e d o m e h T\n",
      "\n",
      ".\n",
      "\n",
      "d e v i r e d e r a s e r o c s e t i s o p m o c r u o i v a h e b d n a C F\n",
      "\n",
      "l a u d i v i d n\n",
      "\n",
      ", s e c n e\n",
      "\n",
      "i l a s e v i t c e p s e r\n",
      "\n",
      "r i e h t o t n o a t a d r u o i v a h e b d n a C F\n",
      "\n",
      "l a n i g i r o e h t g n i t c e o r p y B\n",
      "\n",
      "j\n",
      "\n",
      ". s e c i r t a m r u o i v a h e b\n",
      "\n",
      ". s e r o c s\n",
      "\n",
      "e t i s o p m o c\n",
      "\n",
      "l a r u o i v a h e b\n",
      "\n",
      "e h t\n",
      "\n",
      "d n a\n",
      "\n",
      "s e r o c s\n",
      "\n",
      "e t i s o p m o c\n",
      "\n",
      "n i a r b\n",
      "\n",
      "e h t\n",
      "\n",
      "n e e w t e b\n",
      "\n",
      "e c n a i r a v o c\n",
      "\n",
      "e h t\n",
      "\n",
      "e z i m x a m o t\n",
      "\n",
      "i\n",
      "\n",
      "FC-deficit associations after stroke BRAIN COMMUNICATIONS 2025, fcaf276\n",
      "\n",
      "the indirect effect and calculated its 95% confidence interval (CI). A mediation effect was considered significant (P < 0.05) if the 95% CI did not contain zero. As mediators and depend- ent variables were assessed simultaneously, the mediation re- sults should be interpreted as statistical associations rather than causal effects.\n",
      "\n",
      "Control and reliability analysis Several analyses were performed to ensure the robustness of our results. First, we constructed networks on the basis of an alternative parcellation developed by Yan et al.45 to demon- strate the robustness of the results independent of parcella- tion choices. FC was computed for 400 cortical regions in 7 large-scale networks. Second, we performed LOOCV of the PLSC analysis. In each fold, one patient was selected as the test set, while the remaining patients were selected as the training set. For the training set, PLSC was computed to obtain brain and behavioural saliences. We subsequently computed the brain and behavioural composite scores of the test patients by projecting the test data onto the saliences derived from the training data. Each patient served as a test patient in 1-fold of the analysis. We subsequently computed Pearson’s correlations between the brain and behavioural composite scores of test patients across all folds and tested the statistical significance of these correlations via a permu- tation test (1000 times). Third, we also conducted internal validation by randomly splitting the dataset into two inde- pendent subsets (41 and 40 participants, respectively) and performing separate PLSC analyses on each subset. Fourth, to reduce feature dimensionality and ensure the robustness of our findings, we employed two distinct approaches. Principal component analysis (PCA) was used to reduce the dimensionality of input features prior to partial least squares (PLS) analysis. Sparse PLS (SPLS) embedded L1-norm regu- larisation into the PLS framework to enable automatic fea- ture selection. Finally, we regressed out the confounding effects, including age, sex, ethnicity, lesion types and smok- ing status from both the brain and behaviour data before the PLSC analysis.\n",
      "\n",
      "Results One robust LC links RSFC and behaviour 2 weeks after stroke We conducted PLSC analysis on whole-brain RSFC data and 41 neuropsychological measures collected 2 weeks after stroke. Given that only the first latent component (LC1) sur- vived permutation testing with false discovery rate (FDR) correction (q < 0.05), LC1 is the focus of the remainder of this article. LC1 accounted for 38.8% of the covariance be- tween the RSFC and behavioural data, with a significant as- sociation (r = 0.76, P < 0.001) between the behavioural and RSFC composite scores (Fig. 2A and B).\n",
      "\n",
      "Figure 2C shows the top correlations between the behav- ioural composite scores of LC1 and the 30 significant\n",
      "\n",
      "behavioural measures (thresholded behavioural loadings). A greater behavioural composite score was associated with greater general-domain behavioural performance (e.g. atten- tion, verbal memory, spatial memory, language, motor). Unthresholded behavioural and RSFC loading patterns are shown in Supplementary Fig. 1. Notably, the accuracy of dis- engagement effect in the Posner orienting task showed a positive correlation, in contrast to the predominantly nega- tive trend among other attention-domain measures. Figure 2D illustrates the significant correlations between the RSFC composite scores of LC1 and the RSFC among 333 cortical parcellations (thresholded RSFC loadings). The significant RSFC loadings were averaged within and be- tween networks (Fig. 2E). A greater RSFC composite score was associated with increased interhemispheric RSFC within the sensorimotor (visual, auditory and somatomotor), cingu- lo–opercular, and attention networks. Conversely, a greater composite score was associated with decreased intrahemi- spheric connectivity between the default mode network or frontoparietal network and other networks. We further summed the RSFC loadings for each region in the Gordon at- las to represent the relative importance of each region in the significant RSFC pattern (Fig. 2F). The FC pattern of general-domain deficits followed the hierarchy gradient of cortical organisation (Fig. 2G). In other words, post-stroke behavioural performance was associated with divergent modu- lation of FC across cortical hierarchies between higher-order cognitive networks and lower-order sensorimotor regions.\n",
      "\n",
      "Replicability and generalizability across stroke stage The LC1 identified from 2-week stroke data was further vali- dated with stroke data at 3-month and 12-month time points. We applied PLSC to whole-brain RSFC and 41 neuropsycho- logical assessments at 3-month and 12-month points. The PLSC analysis consistently revealed that LC1 was significant at both time points (3-month: r = 0.82, permuted P = 0.018; 12-month: r = 0.79, permuted P = 0.023), explaining 23.0%, 23.2% of covariance between the brain and behavioural (Supplementary Fig. 2A and B). data, Importantly, the LC1 obtained from 3-month and 12-month stroke data largely replicated the LC1 identified from 2-week stroke data. This is evidenced by the high correlation between behavioural saliences in the 2-week and 3-month stroke data (r = 0.84, P < 0.001), between behavioural saliences in the 2-week and 12-month stroke data (r = 0.77, P < 0.001), be- tween RSFC saliences in the 2-week and 3-month stroke data (r = 0.45, P < 0.001), between RSFC saliences in the 2-week and 12-month stroke data (r = 0.55, P < 0.001) (Fig. 3A and B). The correlation results of loadings were con- sistent with the results of saliences (Supplementary Fig. 3).\n",
      "\n",
      "respectively\n",
      "\n",
      "We further performed a cross-stage validation analysis by projecting stroke data at 3 months and 12 months onto the salience parameters learned by PLS in stroke data at 2 weeks, and then calculated the correlation between the behavioural and RSFC composite scores in projected data. We found that\n",
      "\n",
      "| 7\n",
      "\n",
      "8\n",
      "\n",
      "| BRAIN COMMUNICATIONS 2025, fcaf276 K. Wu et al.\n",
      "\n",
      "Figure 2 One robust LC linking whole-brain RSFC with multiple neuropsychological scores 2 weeks after stroke. (A) The amount of covariance explained by each LC. Each dot represents an LC, and the red dot represents the first LC, which is the only component that survived after permutation testing with FDR correction (q < 0.05). (B) Correlations between individual-specific RSFC and behavioural composite scores (P < 0.001) (n = 81). The inset shows the null distribution obtained by permutation testing. Note that the null distribution is not centred at zero. The dashed line indicates the actual singular value for LC1. (C) The 30 most significant correlations between behavioural measures and behavioural composite scores. The colours of the bars indicate different behavioural domains. The error bars indicate bootstrapped standard deviations with 1000 bootstrap estimations. (D) Significant correlations between RSFC data and RSFC composite scores; only within-network or between-network blocks with significant bootstrapped Z scores are shown (q < 0.05). Red (or blue) colour indicates that greater RSFC is positively (or negatively) associated with LC1. (E) Correlations between RSFC data and RSFC composite scores of participants, averaged within and between networks with significant bootstrapped Z scores. (F) Summed RSFC loadings for each cortical parcellation to represent the regional importance scores. (G) Correlation between the brain region map of summed RSFC loadings and the principal gradient of functional connectivity in healthy humans69 (P < 0.001). Each dot represents a cortical parcellation (n = 333). Att, attention; AUD, auditory; CON, cingulo-opercular network; CPN, cingulo-parietal network; DAT, dorsal attention; DMN, default mode network; FPN, frontoparietal network; Lan, language; RTN, retrosplenial temporal network; SAL, salience; SMD, somatomotor dorsal; SMem, spatial memory; SMV, somatomotor ventral; VAT, ventral attention; VIS, visual; VMem, verbal memory.\n",
      "\n",
      "the obtained LC1 from stroke data at 2weeks successfully generalized to 3-month and 12-month stroke data, as evi- denced by a significant correlation between the obtained be- havioural and RSFC composite score (3-month: r = 0.37, P < 0.001; 12-month: r = 0.47, P < 0.001) (Fig. 3A and B).\n",
      "\n",
      "(Fig. 3C and D). These results suggest the obtained LC1 can only be replicated and generalized using stroke data ra- ther than control data.\n",
      "\n",
      "Moreover, we applied PLSC to whole-brain RSFC and raw behavioural measures of healthy controls. The first LC was significant for the control group for each visit (visit 1: r = 0.880, permuted P < 0.001; visit 2: r = 0.886, permuted P = 0.025) (Supplementary Fig. 2C and D). But correlations between behavioural and RSFC saliences of the 2-week stroke group and control group were low, indicating the LC1 obtained from the control data is distinct from that of the stroke data (Fig. 3C and D). Moreover, when we pro- jected HC data onto the salience parameters learned by PLS in 2-week stroke data, the correlations between the be- havioural and RSFC composite score were non-significant\n",
      "\n",
      "Structural disconnection better explains FC–deficit LC1 scores than cortical damage To explore the anatomical basis of LC1, we utilized a ridge regression model to predict the FC–deficit composite scores using region-based lesion damage and SDC. When we pre- dicted the composite scores using SDC, the prediction per- formance was significant, as evidenced by the correlation between the predicted and test composite behavioural scores (r = 0.50, P < 0.001, R2 = 0.25, MAE = 2.01) and between the predicted and test composite RSFC scores (r = 0.61,\n",
      "\n",
      "FC-deficit associations after stroke BRAIN COMMUNICATIONS 2025, fcaf276\n",
      "\n",
      "Figure 3 The LC1 obtained from 2-week stroke data was verified with 3-month and 12-month stroke data, as well as HC data. (A) Comparison between 2-week and 3-month stroke data. The left scatter plot shows the Pearson correlation between the behavioural and RSFC composite scores by projecting 3-month stroke data onto the salience parameters learned by PLSC on 2-week stroke data (P < 0.001). Each point represents a 3-month stroke patient (n = 78). The middle scatter plot shows the Pearson correlation between behavioural saliences in the 2-week and 3-month stroke data (P < 0.001). Each point represents a behavioural test. The right scatter plot shows the Pearson correlation between the RSFC saliences in the 2-week and 3-month stroke data (P < 0.001). Each point represents an edge between two cortical regions. (B) Comparison of 2-week and 12-month stroke data. The left scatter plot shows the correlation between the behavioural and RSFC composite scores by projecting 12-month stroke data onto the salience parameters from 2-week stroke data (P < 0.001). Each point represents a 12-month stroke patient (n = 74). The middle scatter plot shows the correlation between behavioural saliences in the 2-week and 12-month stroke data (P < 0.001). The right\n",
      "\n",
      "| 9\n",
      "\n",
      "(continued)\n",
      "\n",
      "10\n",
      "\n",
      "| BRAIN COMMUNICATIONS 2025, fcaf276 K. Wu et al.\n",
      "\n",
      "P < 0.001; R2 = 0.37, MAE = 28.11) after LOOCV (Fig. 4A). However, when the composite scores were pre- dicted using focal lesion damage, the prediction performance was not significant, as evidenced by the correlation between the predicted and test composite behaviour scores (r = 0.18, P = 0.11; R2 = −0.59, MAE = 4.67) and between the pre- dicted and test composite RSFC scores (r = 0.26, P = 0.02; R2 = 0.05, MAE = 34.09) after LOOCV (Fig. 4B). SDC fea- tures outperformed lesion damage features in explaining the core mode of covariation between FC and deficits, indicating that SDC of white matter pathways is the primary anatomic- al factor underlying the association between FC and multiple deficits. To characterize the contributions of SDC, the top 20% of the strongest weights were projected back onto the brain (Supplementary Fig. 4), showing that FC–deficit scores were negatively correlated with interhemispheric SDC across most networks, such as the sensorimotor, cingulo–opercular, attention and default networks.\n",
      "\n",
      "Mediating role of FC in the relationship between SDC and behaviour We conducted PLSC analysis on region-based SDC and 41 neuropsychological measures at the 2-week post-stroke. The PLSC analysis revealed that LC1 was significant (q < 0.05); LC1 accounted for 63.7% of the SDC–deficit covari- ance, with a significant association (r = 0.62, P < 0.001) be- tween the behavioural and SDC composite scores (Fig. 5A and B). Figure 5C shows the top correlations between the be- havioural composite scores and behavioural data of LC1. Figure 5D shows the top 20% of SDC loadings, primarily consisting of interhemispheric SDCs within the sensori- motor, cingulo-opercular, attention and default network. Notably, the behavioural patterns driven by SDC are highly correlated with previous behavioural patterns driven by FC (Fig. 5E), indicating that the LC1 also reflects general cogni- tive performance. The SDC patterns derived from the SDC– behaviour data were highly correlated with the FC patterns derived from the FC–behaviour data (Fig. 5F), supporting that SDC may underlie core FC disruptions associated with stroke. We performed a mediation analysis using RSFC compos- ite scores as the mediating variable, SDC composite scores as the independent variable, and behaviour composite scores\n",
      "\n",
      "as the dependent variable. The results showed that FC scores partially mediated the relationship between the SDC and be- haviour scores [ab = −0.06, 95% confidence interval (CI) (−0.09, −0.04)] (Fig. 5G). The total effect (c) was −0.09, and the proportion of mediation (ab/c) was 66.7%. However, no mediating effect was observed when the medi- ating variable was the SDC composite score, the independent variable was the RSFC composite score, and the dependent variable was the behaviour composite score [ab = −0.01, 95% CI (0, 0.02)]. The total effect (c) was −0.04, and the proportion of mediation (ab/c) was 25%. These results offer compelling evidence that large-scale disruption of FC med- iates the relationship between SDC and multidomain behav- ioural deficits from a data-driven perspective.\n",
      "\n",
      "Control and reliability analyses We performed several additional analyses to ensure the ro- bustness of our results. First, in LOOCV, the PLSC compo- nents estimated from training participants were successfully generalized to the remaining participants (Supplementary Fig. 5). Second, the first LC identified in the full dataset was stable across different subsamples. Third, the results were largely unchanged when the RSFC matrix and region-based structural features were based on an alternative parcellation developed by Yan et al.45 (Supplementary Figs. 6–8). Fourth, to ensure that the results were not affected by the in- terpolation processing method for missing behavioural va- lues, we conducted all analyses after excluding participants with incomplete behavioural data. All the results were un- changed. Fifth, the first LC remained stable regardless of whether PCA or SPLS was used for dimensionality reduction. Finally, the results were replicated after we regressed out the confounding effects from both the RSFC and behaviour data.\n",
      "\n",
      "Discussion Although existing studies have explored the relationship be- tween whole-brain RSFC and neurological deficits after stroke, the degree to which RSFC is associated with deficits as a multivariate dimension remains largely unexplored. Employing a multivariate data-driven approach, we discov- ered covariance patterns of RSFC and neurological deficits that were represented by the LC1. This component shows\n",
      "\n",
      "Figure 3 Continued scatter plot shows the correlation between the RSFC saliences in the 2-week and 12-month stroke data (P < 0.001). (C) Comparison between 2-week stroke data and control data at the first visit (HC1). The left scatter plot shows the correlation between the behavioural and RSFC composite scores by projecting HC1 data onto the salience parameters from 2-week stroke data (P > 0.05). Each point represents a HC1 participant (n = 28). The middle scatter plot shows the correlation between behavioural saliences in the 2-week stroke data and HC1 data (P > 0.05). The right scatter plot shows the correlation between RSFC saliences in the 2-week stroke data and HC1 data (P < 0.001). (D) Comparison between 2-week stroke data and control data at the second visit (HC2). The left scatter plot shows the correlation between the behavioural and RSFC composite scores by projecting HC2 data onto the salience parameters from 2-week stroke data (P > 0.05). Each point represents a HC2 patient (n = 26). The middle scatter plot shows the correlation between behavioural saliences in the 2-week stroke data and HC2 data (P > 0.05). The right scatter plot shows the correlation between the RSFC saliences in the 2-week stroke data and HC2 data (P > 0.05).\n",
      "\n",
      "FC-deficit associations after stroke BRAIN COMMUNICATIONS 2025, fcaf276\n",
      "\n",
      "Figure 4 Prediction performance of structural lesion features. (A) Left: A disconnection streamline map, a brain parcellation map and a structural connectome atlas are used to create a region-based SDC matrix, where each edge represents the proportion of streamlines connecting two regions that are disconnected due to the lesion. Right: Correlations between the predicted and real behaviour/RSFC composite scores when region-based SDC is a predictor (P < 0.001). (B) Left: Lesion topography and brain parcellation are used to compute a region-based damage vector, where each value indicates the percentage of voxels within each grey matter parcel that were damaged by the lesion. Right: Correlations between the predicted and real behaviour/RSFC composite scores when lesion damage is a predictor (P > 0.05). Each point represents a 2-month stroke patient (n = 81).\n",
      "\n",
      "that the severity of behavioural deficits across multiple do- mains is associated with decreased interhemispheric connect- ivity and increased intrahemispheric connectivity. Notably, the LC was maintained over the course of recovery from stroke. Furthermore, this component was better explained by damage to underlying white matter tracts than by focal le- sions. Then we examined the relationships among SDC, RSFC and deficits and found that RSFC might mediate the relationship between SDC and multidomain deficits.\n",
      "\n",
      "Previous studies applied dimensionality reduction techni- ques to an extensive battery of post-stroke behavioural mea- sures and revealed that three robust clusters accounted for most of the variance across behavioural domains.23 These behavioural clusters are considered possible representations of the behavioural output of a common, abnormal function- al network state; however, this hypothesis lacks substantial empirical support.12 In the present study, we identified a comprehensive behavioural cluster by incorporating whole- brain RSFC. This cluster involved 30 neurological measures across five behavioural domains (including attention, lan- guage, spatial memory, verbal memory, and motor), reflect- ing general-domain cognitive performance after stroke. Notably, the accuracy of disengagement effect in the Posner orienting task showed a positive correlation with the FC–behaviour dimension, in contrast to the trend of other attention-domain scores (Supplementary Fig. 1). This opposite direction may reflect the distinct neurocognitive\n",
      "\n",
      "basis of disengagement, which is strongly right-lateralized and particularly vulnerable to right-hemisphere dam- age.46,47 Taken together, these results provide evidence sup- porting the hypothesis that highly correlated deficits resulted from abnormal patterns of synchronisation in functional networks.\n",
      "\n",
      "The abnormal FC pattern was characterized by reduced interhemispheric and increased intrahemispheric connectiv- ity. Such a pattern is consistent with previous research, which has demonstrated its strong association with specific behavioural scores, including motor, language, attention and memory performance.20,22 Researchers have suggested that these alterations in FC may be related to a significant re- duction in the modularity of the whole-brain network com- pared with healthy adults.40,48 From a more theoretical perspective, these FC abnormalities following a stroke may reflect a decrease in the entropy of the brain and the variabil- ity of neural states, which could correspond to a decrease in the number of possible behaviours that can be generated.49-51 Furthermore, we discovered that the abnor- mal FC pattern aligns with the brain’s sensorimotor- association (S-A) axis. The more severe the behavioural def- icits are, the weaker the connectivity in lower-level sensory networks and the stronger the connectivity in higher-level cognitive networks. A series of studies have consistently re- vealed that FC patterns linked to overall behavioural scores (such as the general psychopathology factor) follow a\n",
      "\n",
      "| 11\n",
      "\n",
      "12\n",
      "\n",
      "| BRAIN COMMUNICATIONS 2025, fcaf276 K. Wu et al.\n",
      "\n",
      "Figure 5 One robust component linking structural disconnection with raw behavioural assessments and its correlation with the LC1 obtained from the FC–deficit PLSC model. (A) The amount of covariance explained by each LC. Each dot represents an LC, and the red dot represents the LC that survived after permutation testing with FDR correction (q < 0.05). (B) Correlations between individual-specific SDC and behavioural composite scores (n = 81). The inset shows the null distribution obtained by permutation testing. Note that the null distribution is not centred at zero. The dashed line indicates the actual singular value obtained for LC1. (C) The 30 strongest correlations between behavioural measures and behavioural composite scores. The error bars indicate bootstrapped standard deviations with 1000 bootstrap estimations (n = 1000). (D) The top 20% of correlations between SDC data and SDC composite scores. The blue colour indicates that SDC is negatively associated with LC1. (E) The correlation between behavioural saliences obtained from the SDC–deficit PLSC model and those obtained from the FC–deficit PLSC model (P < 0.001). (F) The correlation between SDC saliences from the SDC–deficit PLSC model and RSFC saliences from the FC–deficit PLSC model (P < 0.001). (G) Mediation analysis for SDC composite scores (independent variable), RSFC composite scores (mediating variable) and behaviour composite scores (dependent variable). The total effect (c = −0.09, P < 0.001) and indirect effect (ab = −0.06, P < 0.001) are very significant, and the direct effect (c′ = −0.03, P = 0.04) is significant.\n",
      "\n",
      "hierarchical gradient from lower-order to higher-order regions.52-54 Lower-order and higher-order regions, which exhibit opposite modulations, are known to be structurally and functionally distinct. These two types of brain regions are structurally different in terms of cortical myelination, cortical layering, cytoarchitecture, cell types, and synaptic physiology, and they serve either simple sensorimotor func- tions or complex abstract cognitive functions.55,56 Together with previous findings, our results demonstrate that lower- order and higher-order regions exhibit opposite effects on general behavioural performance after stroke.\n",
      "\n",
      "The covariance patterns between deficits and FC in the LC1 were maintained up to 12 months after stroke but did not exist in the HC data, which demonstrated the robustness and specificity of the identified component. One possible ex- planation is that the recovery of multiple functions depends on common recovery mechanisms after stroke. As previous behavioural studies reported, the time course and magnitude of recovery were similar across behavioural domains and the covariance structure of behavioural deficits remained stable during recovery from stroke.24,57-60 These shared\n",
      "\n",
      "mechanisms involve fundamental plasticity processes includ- ing the restoration of global functional connectivity patterns and the rebalancing of network-level excitation-inhibition dynamics.21,61 While our findings support domain-general mechanisms, we acknowledge the coexistence of domain- specific adaptations. For example, motor recovery frequently involves topographic reorganisation of perilesional cortex; language recovery may engage distinct patterns of contralat- eral homologue recruitment; attention deficits show particu- lar dependence on the resynchronisation of specific attention networks.62 This evidence suggests that post-stroke recovery involves both local network plasticity and overarching bio- logical repair processes, which may operate in parallel or inter- act in complex ways depending on the functional domain.\n",
      "\n",
      "We observed a slightly higher correlation between 2-week and 12-month data compared with that between 2-week and 3-month data. While a stronger correlation at the shorter interval might be expected, this seemingly counterintuitive pattern likely reflects the dynamic and non-linear nature of post-stroke recovery. The 3-month stage typically represents a transitional period marked by rapid and heterogeneous\n",
      "\n",
      "FC-deficit associations after stroke BRAIN COMMUNICATIONS 2025, fcaf276\n",
      "\n",
      "changes in both behaviour and functional connectivity, which may weaken alignment with early brain–behaviour as- sociations. In contrast, recovery tends to stabilize by 12 months, and individual patterns become more consistent, fa- cilitating stronger correspondence with the early LC.\n",
      "\n",
      "Furthermore, SDC features consistently outperformed fo- cal damage features in predicting FC–deficit PLSC scores, highlighting SDC as a critical structural basis for FC–deficit associations. Previous studies have proposed that damage to hub regions connecting multiple functional networks leads to widespread FC disruptions and extensive behaviour im- pairments.35,48 However, the severe effect of damage to cer- tain cortical hubs might be better explained by damage to the underlying white matter tracts, as SDC contain more com- plex damage information than focal lesions.12,23,34 A recent study demonstrated that indirect SDC of grey matter regions better explained widespread FC abnormalities than focal le- sions did.33 In a healthy human brain, the efficacy of cogni- tive processes relies on communication between multiple areas and damage to the major white matter tracts facilitat- ing this communication is likely to cause multiple deficits.13 Notably, white matter disconnection has a long-lasting det- rimental effect and constrains functional and behavioural outcomes. This may partially explain why the deficit–FC LC identified at 2 weeks persists across recovery.\n",
      "\n",
      "Many studies on white matter damage have associated spe- cific behavioural deficits with SDC.34,63-65 Our study identified a significant LC linking SDC and an extensive battery of defi- cits across multiple domains. The behavioural pattern of this component, which is highly similar to the pattern obtained from FC–deficit PLSC analysis, reflects general-domain cogni- tive performance. This component consists of interhemispheric SDCs, which have been demonstrated to be a key factor in the imbalance of large-scale functional networks. For example, studies of non-human primates have suggested that transection of the corpus callosum results in a decrease in between- hemisphere FC and an increase in within-hemisphere FC.66,67 A human study also revealed that the severity of interhemi- spheric SDC was correlated with both type of abnormal FC patterns.33 Taken together, interhemispheric SDC, multi- domain deficits and two abnormal patterns of FC are closely linked. However, prior studies have not thoroughly examined the complete relationships among SDC, FC and multidomain deficits within a unified analytical framework. The present study conducted a mediation analysis using the PLSC compos- ite score of the three variables and revealed that FC statistically mediated the association between SDC and behavioural defi- cits. It provided a deeper understanding of interrelationships among the three variables.\n",
      "\n",
      "recovery trajectories following stroke. Furthermore, we found that SDC is closely associated with this FC–deficit dimension, which provides empirical evidence for the systematic inter- action between SDC, FC, and multi-domain behavioural defi- cits.12 Rather than replicating prior findings, our study offers novel insights by demonstrating that FC statistically mediates the relationship between SDC and multidomain deficits. Clinically, these findings have potential applications in progno- sis and targeted interventions. Specifically, the FC–deficit di- mension could serve as a neurophysiological biomarker to monitor recovery and inform personalized rehabilitation. Functional connectivity assessments could help tailor adaptive therapy protocols, focusing on task-specific training that tar- gets network dysfunction rather than traditional motor or cog- nitive exercises. Moreover, our findings support the use of multisite neuromodulation strategies (e.g. TMS, tDCS) that target network-level dysfunctions rather than isolated regions, potentially improving treatment efficacy.\n",
      "\n",
      "There are several limitations to this study. First, the relatively small sample sizes in both the stroke and control groups may in- crease the risk of overfitting, particularly in multivariate ana- lyses. To reduce the dimensionality of input features, we used PCA and SPLS to validate the stability of the PLS results. While these approaches reinforce the internal robustness of our findings, the generalizability of PLS outcomes remains limited in the absence of external validation. Moreover, the pre- dictive framework based on LOOCV is inherently constrained by the small sample size and available data, which restricts the generalizability of the results to independent datasets. Future studies involving larger, independent stroke cohorts will be es- sential to further assess the reproducibility and external validity of these multivariant models. Second, we focused on unilateral stroke patients and excluded those with bilateral lesions. This decision aimed to reduce sample heterogeneity and ensure the stability and interpretability of multivariate analyses. Bilateral strokes often involve more complex and widespread damage, which may introduce additional sources of variance that are dif- ficult to model in relatively small samples.68 However, we ac- knowledge that this choice may limit the generalizability of our findings to the broader stroke population. Future stud- ies with larger and more diverse samples will be necessary to capture the full spectrum of network disruptions, including those associated with bilateral lesions. Finally, SDC mea- sures were defined by intersecting patient lesions with a structural connectome atlas, which is assumed to approxi- mate individual structural connectomes. These measures may not account for inter-individual variability in undam- aged fibre pathways. While direct patient SDC measures would be ideal, existing studies suggest that indirect SDC data provide valuable information about structural con- nectivity after brain damage.33,65\n",
      "\n",
      "In this study, we discovered and replicated multivariate pat- terns of intrinsic FC that are highly correlated with a dimension of neuropsychological deficit in a longitudinal cohort of stroke patients. The dimension was composed of a common post- stroke feature of FC and a unique cluster of deficits involving multiple domains. This multivariate framework moves beyond pairwise associations and reveals a large-scale network-level mechanism that may underlie inter-individual differences in\n",
      "\n",
      "Supplementary material Supplementary material is available at Brain Communications online.\n",
      "\n",
      "| 13\n",
      "\n",
      "14\n",
      "\n",
      "| BRAIN COMMUNICATIONS 2025, fcaf276 K. Wu et al.\n",
      "\n",
      "Funding National Natural Science Foundation of China, no. T2325006. National Natural Science Foundation of China, no. 82172016. National Natural Science Foundation of China, no. 82021004. Fundamental Research Funds for the Central Universities, no. 2233200020.\n",
      "\n",
      "14. Saur D, Lange R, Baumgaertner A, et al. Dynamics of language re- organization after stroke. Brain. 2006;129(6):1371-1384.\n",
      "\n",
      "15. Ward NS, Brown MM, Thompson AJ, Frackowiak RSJ. Neural cor- relates of motor recovery after stroke: A longitudinal fMRI study. Brain. 2003;126(11):2476-2496.\n",
      "\n",
      "16. Carter AR, Shulman GL, Corbetta M. Why use a connectivity-based approach to study stroke and recovery of function? NeuroImage. 2012;62(4):2271-2280.\n",
      "\n",
      "17. Meinzer M, Harnish S, Conway T, Crosson B. Recent developments in functional and structural imaging of aphasia recovery after stroke. Aphasiology. 2011;25(3):271-290.\n",
      "\n",
      "Competing interests The authors report no competing interests.\n",
      "\n",
      "18. Fox MD, Snyder AZ, Vincent JL, Corbetta M, Van Essen DC, Raichle ME. The human brain is intrinsically organized into dynam- ic, anticorrelated functional networks. Proc Natl Acad Sci U S A. 2005;102(27):9673-9678.\n",
      "\n",
      "Data availability The full set of neuroimaging and behavioural data are avail- able at http://cnda.wustl.edu/app/template/Login.vm. Data sharing is not applicable to this article as no new data were created in this study. Specific data and analysis scripts can be found at https://github.com/PsyWuke/Latent_dimension_ linking_FC_with_post-stroke_deficits2025.git.\n",
      "\n",
      "References\n",
      "\n",
      "1. Schumacher R, Halai AD, Lambon Ralph MA. Assessing and map- ping language, attention and executive multidimensional deficits in stroke aphasia. Brain. 2019;142(10):3202-3216.\n",
      "\n",
      "2. Villard S, Kiran S. To what extent does attention underlie language in aphasia? Aphasiology. 2017;31(10):1226-1245.\n",
      "\n",
      "19. Van Essen DC, Ugurbil K, Auerbach E, et al. The human connec- tome project: A data acquisition perspective. Neuroimage. 2012; 62(4):2222-2231.\n",
      "\n",
      "20. Baldassarre A, Ramsey L, Hacker CL, et al. Large-scale changes in network interactions as a physiological signature of spatial neglect. Brain. 2014;137(12):3267-3283.\n",
      "\n",
      "21. Ramsey LE, Siegel JS, Baldassarre A, et al. Normalization of net- work connectivity in hemispatial neglect recovery: Network con- nectivity and neglect recovery. Ann Neurol. 2016;80(1):127-141.\n",
      "\n",
      "22. Siegel JS, Ramsey LE, Snyder AZ, et al. Disruptions of network con- nectivity predict impairment in multiple behavioral domains after stroke. Proc Natl Acad Sci U S A. 2016;113(30):E4367-E4376. 23. Corbetta M, Ramsey L, Callejas A, et al. Common behavioral clusters and subcortical anatomy in stroke. Neuron. 2015;85(5):927-941. 24. Ramsey LE, Siegel JS, Lang CE, Strube M, Shulman GL, Corbetta M. Behavioural clusters and predictors of performance during re- covery from stroke. Nat Hum Behav. 2017;1(3):0038.\n",
      "\n",
      "25. Bisogno AL, Favaretto C, Zangrossi A, et al. A low-dimensional structure of neurological impairment in stroke. Brain Commun. 2021;3(2):fcab119.\n",
      "\n",
      "3. Nys GMS, Van Zandvoort MJE, De Kort PLM, Jansen BPW, De Haan EHF, Kappelle LJ. Cognitive disorders in acute stroke: Prevalence and clinical determinants. Cerebrovasc Dis. 2007; 23(5-6):408-416.\n",
      "\n",
      "4. Buxbaum LJ, Ferraro MK, Veramonti T, et al. Hemispatial neglect: Subtypes, neuroanatomy, and disability. Neurology. 2004;62(5): 749-756.\n",
      "\n",
      "5. Broca P. Localisation des fonctions cerebrales: Siegedu langage ar- ticule. Bull Soc D’Anthropol Paris. 1863;4:200-208.\n",
      "\n",
      "26. Facchini S, Favaretto C, Castellaro M, et al. A common low dimen- sional structure of cognitive impairment in stroke and brain tumors. NeuroImage Clin. 2023;40:103518.\n",
      "\n",
      "27. Kebets V, Holmes AJ, Orban C, et al. Somatosensory-motor dys- connectivity spans multiple transdiagnostic dimensions of psycho- pathology. Biol Psychiatry. 2019;86(10):779-791.\n",
      "\n",
      "28. Xia CH, Ma Z, Ciric R, et al. Linked dimensions of psychopath- in functional brain networks. Nat ology and connectivity Commun. 2018;9(1):3003.\n",
      "\n",
      "6. Wernicke C. The symptom complex of aphasia: A psychological study on a anatomical basis. Boston Stud Philos Sci. 1874;4:34-97. 7. Carrera E, Tononi G. Diaschisis: Past, present, future. Brain. 2014; 137(9):2408-2422.\n",
      "\n",
      "8. He BJ, Snyder AZ, Vincent JL, Epstein A, Shulman GL, Corbetta M. Breakdown of functional connectivity in frontoparietal networks underlies behavioral deficits in spatial neglect. Neuron. 2007; 53(6):905-918.\n",
      "\n",
      "9. Baron JC, D’antona R, Pantano P, Serdaru M, Samson Y, Bousser MG. Effects of thalamic stroke on energy metabolism of the cerebral cortex: A positron tomography study in man. Brain. 1986;109(6): 1243-1259.\n",
      "\n",
      "10. Baron JC, Levasseur M, Mazoyer B, et al. Thalamocortical diaschi- sis: Positron emission tomography in humans. J Neurol Neurosurg Psychiatry. 1992;55(10):935-942.\n",
      "\n",
      "11. Corbetta M, Kincade MJ, Lewis C, Snyder AZ, Sapir A. Neural ba- sis and recovery of spatial attention deficits in spatial neglect. Nat Neurosci. 2005;8(11):1603-1610.\n",
      "\n",
      "12. Siegel JS, Shulman GL, Corbetta M. Mapping correlated neuro- logical deficits after stroke to distributed brain networks. Brain Struct Funct. 2022;227(9):3173-3187.\n",
      "\n",
      "29. Genon S, Eickhoff SB, Kharabian S. Linking interindividual vari- ability in brain structure to behaviour. Nat Rev Neurosci. 2022; 23(5):307-318.\n",
      "\n",
      "30. Mihalik A, Chapman J, Adams RA, et al. Canonical correlation ana- lysis and partial least squares for identifying brain–behavior asso- ciations: A tutorial and a comparative study. Biol Psychiatry Cogn Neurosci Neuroimaging. 2022;7(11):1055-1067.\n",
      "\n",
      "31. Modabbernia A, Janiri D, Doucet GE, Reichenberg A, Frangou S. Multivariate patterns of brain-behavior-environment associations in the adolescent brain and cognitive development study. Biol Psychiatry. 2021;89(5):510-520.\n",
      "\n",
      "32. Wang Y, Genon S, Dong D, et al. Covariance patterns between sleep health domains and distributed intrinsic functional connectivity. Nat Commun. 2023;14(1):7133.\n",
      "\n",
      "33. Griffis JC, Metcalf NV, Corbetta M, Shulman GL. Structural dis- connections explain brain network dysfunction after stroke. Cell Rep. 2019;28(10):2527-2540.e9.\n",
      "\n",
      "34. Reber J, Hwang K, Bowren M, et al. Cognitive impairment after fo- cal brain lesions is better predicted by damage to structural than functional network hubs. Proc Natl Acad Sci U S A. 2021; 118(19):e2018784118.\n",
      "\n",
      "13. Corbetta M, Siegel JS, Shulman GL. On the low dimensionality of behavioral deficits and alterations of brain network connectivity after focal injury. Cortex. 2018;107:229-237.\n",
      "\n",
      "35. Warren DE, Power JD, Bruss J, et al. Network measures predict neuropsychological outcome after brain injury. Proc Natl Acad Sci U S A. 2014;111(39):14247-14252.\n",
      "\n",
      "FC-deficit associations after stroke BRAIN COMMUNICATIONS 2025, fcaf276\n",
      "\n",
      "36. Little RJA. A test of missing completely at random for multivariate data with missing values. J Am Stat Assoc. 1988;83(404): 1198-1202.\n",
      "\n",
      "53. Han F, Gu Y, Brown GL, Zhang X, Liu X. Neuroimaging contrast across the cortical hierarchy is the feature maximally linked to be- havior and demographics. NeuroImage. 2020;215:116853.\n",
      "\n",
      "37. Robb RA, Hanson DP. A software system for interactive and quan- titative visualization of multidimensional biomedical images. Australas Phys Eng Sci Med. 1991;14(1):9-30.\n",
      "\n",
      "38. Chen Y, Jiang Y, Kong X, et al. Common and unique structural plasticity after left and right hemisphere stroke. J Cereb Blood Flow Metab. 2021;41(12):3350-3364.\n",
      "\n",
      "39. Gordon EM, Laumann TO, Adeyemo B, Huckins JF, Kelley WM, Petersen SE. Generation and evaluation of a cortical area parcella- tion from resting-state correlations. Cereb Cortex. 2016;26(1): 288-303.\n",
      "\n",
      "54. Langeslag SJE, Schmidt M, Ghassabian A, et al. Functional connect- ivity between parietal and frontal brain regions and intelligence in young children: The generation R study. Hum Brain Mapp. 2013; 34(12):3299-3307.\n",
      "\n",
      "55. Keller AS, Sydnor VJ, Pines A, Fair DA, Bassett DS, Satterthwaite TD. Hierarchical functional system development supports executive function. Trends Cogn Sci. 2023;27(2):160-174.\n",
      "\n",
      "56. Shafiei G, Fulcher BD, Voytek B, Satterthwaite TD, Baillet S, Misic B. Neurophysiological signatures of cortical micro-architecture. Nat Commun. 2023;14(1):6000.\n",
      "\n",
      "40. Siegel JS, Seitzman BA, Ramsey LE, et al. Re-emergence of modular brain networks in stroke recovery. Cortex. 2018;101:44-59. 41. Griffis JC, Metcalf NV, Corbetta M, Shulman GL. Lesion quantifi- cation toolkit: A MATLAB software tool for estimating grey matter damage and white matter disconnections in patients with focal brain lesions. NeuroImage Clin. 2021;30:102639.\n",
      "\n",
      "42. Courville T, Thompson B. Use of structure coefficients in published multiple regression articles: Β is not enough. Educ Psychol Meas. 2001;61(2):229-248.\n",
      "\n",
      "43. Krishnan A, Williams LJ, McIntosh AR, Abdi H. Partial least squares (PLS) methods for neuroimaging: A tutorial and review. NeuroImage. 2011;56(2):455-475.\n",
      "\n",
      "44. McIntosh AR, Lobaugh NJ. Partial least squares analysis of neuroi- maging data: Applications and advances. Neuroimage. 2004;23: S250-S263.\n",
      "\n",
      "45. Yan X, Kong R, Xue A, et al. Homotopic local-global parcellation of the human cerebral cortex from resting-state functional connect- ivity. NeuroImage. 2023;273:120010.\n",
      "\n",
      "46. Michael IP. The attention system of the human brain. Annu Rev Neurosci. 1990;13:25-42.\n",
      "\n",
      "47. Corbetta M, Patel G, Shulman GL. The reorienting system of the hu- man brain: From environment to theory of mind. Neuron. 2008; 58(3):306-324.\n",
      "\n",
      "48. Gratton C, Nomura EM, Pérez F, D’Esposito M. Focal brain lesions to critical locations cause widespread disruption of the modular organization of the brain. J Cogn Neurosci. 2012;24(6): 1275-1285.\n",
      "\n",
      "49. Adhikari MH, Hacker CD, Siegel JS, et al. Decreased integration and information capacity in stroke measured by whole brain models of resting state activity. Brain. 2017;140(4):1068-1085.\n",
      "\n",
      "50. Saenger VM, Ponce-Alvarez A, Adhikari M, Hagmann P, Deco G, Corbetta M. Linking entropy at rest with the underlying structural connectivity in the healthy and lesioned brain. Cereb Cortex. 2018; 28(8):2948-2958.\n",
      "\n",
      "57. Murphy TH, Corbett D. Plasticity during stroke recovery: From synapse to behaviour. Nat Rev Neurosci. 2009;10(12):861-872. 58. Ge S, Yang CH, Hsu KS, Ming GL, Song H. A critical period for en- hanced synaptic plasticity in newly generated neurons of the adult brain. Neuron. 2007;54(4):559-566.\n",
      "\n",
      "59. Stinear CM, Byblow WD, Ackerley SJ, Smith MC, Borges VM, Barber PA. Proportional motor recovery after stroke: Implications for trial design. Stroke. 2017;48(3):795-798.\n",
      "\n",
      "60. Bonkhoff AK, Hope T, Bzdok D, et al. Bringing proportional recov- ery into proportion: Bayesian modelling of post-stroke motor im- pairment. Brain. 2020;143(7):2189-2206.\n",
      "\n",
      "61. Joy MT, Carmichael ST. Encouraging an excitable brain state: Mechanisms of brain repair in stroke. Nat Rev Neurosci. 2021; 22(1):38-53.\n",
      "\n",
      "62. Campos B, Choi H, DeMarco AT, et al. Rethinking remapping: Circuit mechanisms of recovery after stroke. J Neurosci. 2023; 43(45):7489-7500.\n",
      "\n",
      "63. Thiebaut de Schotten M, Tomaiuolo F, Aiello M, et al. Damage to white matter pathways in subacute and chronic spatial neglect: A group study and 2 single-case studies with complete virtual “in vivo” tractography dissection. Cereb Cortex. 2014;24(3):691-706. 64. Thiebaut de Schotten M, Foulon C, Nachev P. Brain disconnections link structural connectivity with function and behaviour. Nat Commun. 2020;11(1):5094.\n",
      "\n",
      "65. Salvalaggio A, De Filippo De Grazia M, Zorzi M, Thiebaut de Schotten M, Corbetta M. Post-stroke deficit prediction from lesion and indirect structural and functional disconnection. Brain. 2020; 143(7):2173-2188.\n",
      "\n",
      "66. Johnston JM, Vaishnavi SN, Smyth MD, et al. Loss of resting inter- hemispheric functional connectivity after complete section of the corpus callosum. J Neurosci. 2008;28(25):6453-6458.\n",
      "\n",
      "67. O’Reilly JX, Croxson PL, Jbabdi S, et al. Causal effect of disconnec- tion lesions on interhemispheric functional connectivity in rhesus monkeys. Proc Natl Acad Sci U S A. 2013;110(34):13982-13987.\n",
      "\n",
      "51. Rocha RP, Koçillari L, Suweis S, et al. Recovery of neural dynamics criticality in personalized whole-brain models of stroke. Nat Commun. 2022;13(1):3683.\n",
      "\n",
      "68. Schapiro AC, McClelland JL, Welbourne SR, Rogers TT, Lambon Ralph MA. Why bilateral damage is worse than unilateral damage to the brain. J Cogn Neurosci. 2013;25(12):2107-2123.\n",
      "\n",
      "52. Elliott ML, Romer A, Knodt AR, Hariri AR. A connectome-wide functional signature of transdiagnostic risk for mental illness. Biol Psychiatry. 2018;84(6):452-459.\n",
      "\n",
      "69. Margulies DS, Ghosh SS, Goulas A, et al. Situating the default-mode network along a principal gradient of macroscale cortical organiza- tion. Proc Natl Acad Sci U S A. 2016;113(44):12574-12579.\n",
      "\n",
      "| 15\n"
     ]
    }
   ],
   "source": [
    "file_path = r\"C:\\Users\\deepa\\Downloads\\Deepanshu Bhatt\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\Data\\raw\\pmc_pdfs\\fcaf276.pdf\"\n",
    "loade = UnstructuredLoader(file_path, mode = \"elements\", strategy = \"fast\")\n",
    "document = loade.lazy_load()\n",
    "\n",
    "full_text = \"\\n\\n\".join([doc.page_content for doc in document])\n",
    "output_file = \"sec.txt\"\n",
    "with open(\"sec.txt\", \"w\", encoding = \"utf-8\") as p:\n",
    "    p.write(full_text)\n",
    "\n",
    "with open(\"sec.txt\", \"r\", encoding = \"utf-8\") as p:\n",
    "    text1 = p.read()\n",
    "\n",
    "print(text1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "eaf707c0-d4ab-475e-931e-3075ba298b36",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "venv",
   "language": "python",
   "name": "venv"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
