{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "2b4f82fa-cc0d-4b4e-8cc6-32b844da1387",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Collecting langchain-unstructured\n",
      "  Downloading langchain_unstructured-0.1.6-py3-none-any.whl.metadata (3.3 kB)\n",
      "Collecting langchain-core<0.4.0,>=0.3.6 (from langchain-unstructured)\n",
      "  Using cached langchain_core-0.3.75-py3-none-any.whl.metadata (5.7 kB)\n",
      "Collecting onnxruntime<=1.19.2,>=1.17.0 (from langchain-unstructured)\n",
      "  Downloading onnxruntime-1.19.2-cp312-cp312-win_amd64.whl.metadata (4.7 kB)\n",
      "Collecting unstructured-client<1,>=0.27.0 (from langchain-unstructured)\n",
      "  Using cached unstructured_client-0.42.3-py3-none-any.whl.metadata (23 kB)\n",
      "Collecting langsmith>=0.3.45 (from langchain-core<0.4.0,>=0.3.6->langchain-unstructured)\n",
      "  Using cached langsmith-0.4.19-py3-none-any.whl.metadata (14 kB)\n",
      "Requirement already satisfied: tenacity!=8.4.0,<10.0.0,>=8.1.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (8.2.2)\n",
      "Requirement already satisfied: jsonpatch<2.0,>=1.33 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (1.33)\n",
      "Requirement already satisfied: PyYAML>=5.3 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (6.0.1)\n",
      "Requirement already satisfied: typing-extensions>=4.7 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (4.14.1)\n",
      "Requirement already satisfied: packaging>=23.2 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (23.2)\n",
      "Requirement already satisfied: pydantic>=2.7.4 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (2.11.7)\n",
      "Requirement already satisfied: jsonpointer>=1.9 in c:\\programdata\\anaconda3\\lib\\site-packages (from jsonpatch<2.0,>=1.33->langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (2.1)\n",
      "Collecting coloredlogs (from onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured)\n",
      "  Using cached coloredlogs-15.0.1-py2.py3-none-any.whl.metadata (12 kB)\n",
      "Collecting flatbuffers (from onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured)\n",
      "  Using cached flatbuffers-25.2.10-py2.py3-none-any.whl.metadata (875 bytes)\n",
      "Requirement already satisfied: numpy>=1.21.6 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (2.3.2)\n",
      "Requirement already satisfied: protobuf in c:\\programdata\\anaconda3\\lib\\site-packages (from onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (3.20.3)\n",
      "Requirement already satisfied: sympy in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (1.14.0)\n",
      "Collecting aiofiles>=24.1.0 (from unstructured-client<1,>=0.27.0->langchain-unstructured)\n",
      "  Using cached aiofiles-24.1.0-py3-none-any.whl.metadata (10 kB)\n",
      "Requirement already satisfied: cryptography>=3.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured-client<1,>=0.27.0->langchain-unstructured) (42.0.5)\n",
      "Collecting httpcore>=1.0.9 (from unstructured-client<1,>=0.27.0->langchain-unstructured)\n",
      "  Using cached httpcore-1.0.9-py3-none-any.whl.metadata (21 kB)\n",
      "Collecting httpx>=0.27.0 (from unstructured-client<1,>=0.27.0->langchain-unstructured)\n",
      "  Using cached httpx-0.28.1-py3-none-any.whl.metadata (7.1 kB)\n",
      "Collecting pypdf>=4.0 (from unstructured-client<1,>=0.27.0->langchain-unstructured)\n",
      "  Using cached pypdf-6.0.0-py3-none-any.whl.metadata (7.1 kB)\n",
      "Requirement already satisfied: requests-toolbelt>=1.0.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured-client<1,>=0.27.0->langchain-unstructured) (1.0.0)\n",
      "Requirement already satisfied: cffi>=1.12 in c:\\programdata\\anaconda3\\lib\\site-packages (from cryptography>=3.1->unstructured-client<1,>=0.27.0->langchain-unstructured) (1.16.0)\n",
      "Requirement already satisfied: pycparser in c:\\programdata\\anaconda3\\lib\\site-packages (from cffi>=1.12->cryptography>=3.1->unstructured-client<1,>=0.27.0->langchain-unstructured) (2.21)\n",
      "Requirement already satisfied: certifi in c:\\programdata\\anaconda3\\lib\\site-packages (from httpcore>=1.0.9->unstructured-client<1,>=0.27.0->langchain-unstructured) (2024.7.4)\n",
      "Collecting h11>=0.16 (from httpcore>=1.0.9->unstructured-client<1,>=0.27.0->langchain-unstructured)\n",
      "  Using cached h11-0.16.0-py3-none-any.whl.metadata (8.3 kB)\n",
      "Requirement already satisfied: anyio in c:\\programdata\\anaconda3\\lib\\site-packages (from httpx>=0.27.0->unstructured-client<1,>=0.27.0->langchain-unstructured) (4.2.0)\n",
      "Requirement already satisfied: idna in c:\\programdata\\anaconda3\\lib\\site-packages (from httpx>=0.27.0->unstructured-client<1,>=0.27.0->langchain-unstructured) (3.7)\n",
      "Collecting orjson>=3.9.14 (from langsmith>=0.3.45->langchain-core<0.4.0,>=0.3.6->langchain-unstructured)\n",
      "  Downloading orjson-3.11.3-cp312-cp312-win_amd64.whl.metadata (43 kB)\n",
      "Requirement already satisfied: requests>=2.0.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from langsmith>=0.3.45->langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (2.32.2)\n",
      "Collecting zstandard>=0.23.0 (from langsmith>=0.3.45->langchain-core<0.4.0,>=0.3.6->langchain-unstructured)\n",
      "  Downloading zstandard-0.24.0-cp312-cp312-win_amd64.whl.metadata (3.2 kB)\n",
      "Requirement already satisfied: annotated-types>=0.6.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from pydantic>=2.7.4->langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (0.6.0)\n",
      "Requirement already satisfied: pydantic-core==2.33.2 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from pydantic>=2.7.4->langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (2.33.2)\n",
      "Requirement already satisfied: typing-inspection>=0.4.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from pydantic>=2.7.4->langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (0.4.1)\n",
      "Requirement already satisfied: charset-normalizer<4,>=2 in c:\\programdata\\anaconda3\\lib\\site-packages (from requests>=2.0.0->langsmith>=0.3.45->langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (2.0.4)\n",
      "Requirement already satisfied: urllib3<3,>=1.21.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from requests>=2.0.0->langsmith>=0.3.45->langchain-core<0.4.0,>=0.3.6->langchain-unstructured) (2.2.2)\n",
      "Requirement already satisfied: sniffio>=1.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from anyio->httpx>=0.27.0->unstructured-client<1,>=0.27.0->langchain-unstructured) (1.3.0)\n",
      "Collecting humanfriendly>=9.1 (from coloredlogs->onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured)\n",
      "  Using cached humanfriendly-10.0-py2.py3-none-any.whl.metadata (9.2 kB)\n",
      "Collecting pyreadline3 (from humanfriendly>=9.1->coloredlogs->onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured)\n",
      "  Using cached pyreadline3-3.5.4-py3-none-any.whl.metadata (4.7 kB)\n",
      "Requirement already satisfied: mpmath<1.4,>=1.1.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from sympy->onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (1.3.0)\n",
      "Downloading langchain_unstructured-0.1.6-py3-none-any.whl (7.0 kB)\n",
      "Using cached langchain_core-0.3.75-py3-none-any.whl (443 kB)\n",
      "Downloading onnxruntime-1.19.2-cp312-cp312-win_amd64.whl (11.1 MB)\n",
      "   ---------------------------------------- 0.0/11.1 MB ? eta -:--:--\n",
      "   ---------- ----------------------------- 2.9/11.1 MB 16.8 MB/s eta 0:00:01\n",
      "   ---------------- ----------------------- 4.5/11.1 MB 11.7 MB/s eta 0:00:01\n",
      "   ---------------------- ----------------- 6.3/11.1 MB 12.9 MB/s eta 0:00:01\n",
      "   ------------------------------- -------- 8.7/11.1 MB 10.5 MB/s eta 0:00:01\n",
      "   ---------------------------------------- 11.1/11.1 MB 11.4 MB/s  0:00:01\n",
      "Using cached unstructured_client-0.42.3-py3-none-any.whl (207 kB)\n",
      "Using cached aiofiles-24.1.0-py3-none-any.whl (15 kB)\n",
      "Using cached httpcore-1.0.9-py3-none-any.whl (78 kB)\n",
      "Using cached h11-0.16.0-py3-none-any.whl (37 kB)\n",
      "Using cached httpx-0.28.1-py3-none-any.whl (73 kB)\n",
      "Using cached langsmith-0.4.19-py3-none-any.whl (376 kB)\n",
      "Downloading orjson-3.11.3-cp312-cp312-win_amd64.whl (131 kB)\n",
      "Using cached pypdf-6.0.0-py3-none-any.whl (310 kB)\n",
      "Downloading zstandard-0.24.0-cp312-cp312-win_amd64.whl (505 kB)\n",
      "Using cached coloredlogs-15.0.1-py2.py3-none-any.whl (46 kB)\n",
      "Using cached humanfriendly-10.0-py2.py3-none-any.whl (86 kB)\n",
      "Using cached flatbuffers-25.2.10-py2.py3-none-any.whl (30 kB)\n",
      "Using cached pyreadline3-3.5.4-py3-none-any.whl (83 kB)\n",
      "Installing collected packages: flatbuffers, zstandard, pyreadline3, pypdf, orjson, h11, aiofiles, humanfriendly, httpcore, httpx, coloredlogs, unstructured-client, onnxruntime, langsmith, langchain-core, langchain-unstructured\n",
      "\n",
      "   ----------------------------------------  0/16 [flatbuffers]\n",
      "   -- -------------------------------------  1/16 [zstandard]\n",
      "   ----- ----------------------------------  2/16 [pyreadline3]\n",
      "   ----- ----------------------------------  2/16 [pyreadline3]\n",
      "   ----- ----------------------------------  2/16 [pyreadline3]\n",
      "   ----- ----------------------------------  2/16 [pyreadline3]\n",
      "   ------- --------------------------------  3/16 [pypdf]\n",
      "   ------- --------------------------------  3/16 [pypdf]\n",
      "   ------- --------------------------------  3/16 [pypdf]\n",
      "   ------- --------------------------------  3/16 [pypdf]\n",
      "   ------- --------------------------------  3/16 [pypdf]\n",
      "   ------- --------------------------------  3/16 [pypdf]\n",
      "   ------- --------------------------------  3/16 [pypdf]\n",
      "   ------- --------------------------------  3/16 [pypdf]\n",
      "   ------- --------------------------------  3/16 [pypdf]\n",
      "   ------------ ---------------------------  5/16 [h11]\n",
      "   --------------- ------------------------  6/16 [aiofiles]\n",
      "   ----------------- ----------------------  7/16 [humanfriendly]\n",
      "   ----------------- ----------------------  7/16 [humanfriendly]\n",
      "   ----------------- ----------------------  7/16 [humanfriendly]\n",
      "   ----------------- ----------------------  7/16 [humanfriendly]\n",
      "   -------------------- -------------------  8/16 [httpcore]\n",
      "   -------------------- -------------------  8/16 [httpcore]\n",
      "   -------------------- -------------------  8/16 [httpcore]\n",
      "   -------------------- -------------------  8/16 [httpcore]\n",
      "   ---------------------- -----------------  9/16 [httpx]\n",
      "   ---------------------- -----------------  9/16 [httpx]\n",
      "   ---------------------- -----------------  9/16 [httpx]\n",
      "   ---------------------- -----------------  9/16 [httpx]\n",
      "   ---------------------- -----------------  9/16 [httpx]\n",
      "   ------------------------- -------------- 10/16 [coloredlogs]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   --------------------------- ------------ 11/16 [unstructured-client]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   ------------------------------ --------- 12/16 [onnxruntime]\n",
      "   -------------------------------- ------- 13/16 [langsmith]\n",
      "   -------------------------------- ------- 13/16 [langsmith]\n",
      "   -------------------------------- ------- 13/16 [langsmith]\n",
      "   -------------------------------- ------- 13/16 [langsmith]\n",
      "   -------------------------------- ------- 13/16 [langsmith]\n",
      "   -------------------------------- ------- 13/16 [langsmith]\n",
      "   -------------------------------- ------- 13/16 [langsmith]\n",
      "   -------------------------------- ------- 13/16 [langsmith]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ----------------------------------- ---- 14/16 [langchain-core]\n",
      "   ---------------------------------------- 16/16 [langchain-unstructured]\n",
      "\n",
      "Successfully installed aiofiles-24.1.0 coloredlogs-15.0.1 flatbuffers-25.2.10 h11-0.16.0 httpcore-1.0.9 httpx-0.28.1 humanfriendly-10.0 langchain-core-0.3.75 langchain-unstructured-0.1.6 langsmith-0.4.19 onnxruntime-1.19.2 orjson-3.11.3 pypdf-6.0.0 pyreadline3-3.5.4 unstructured-client-0.42.3 zstandard-0.24.0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  WARNING: The script humanfriendly.exe is installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: The script httpx.exe is installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: The script coloredlogs.exe is installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: The script onnxruntime_test.exe is installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n"
     ]
    }
   ],
   "source": [
    "!pip install -U langchain-unstructured"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "376b54fd-1127-4681-aed7-7a37149e390c",
   "metadata": {},
   "outputs": [
    {
     "ename": "ModuleNotFoundError",
     "evalue": "No module named 'langchain_unstructured'",
     "output_type": "error",
     "traceback": [
      "\u001b[31m---------------------------------------------------------------------------\u001b[39m",
      "\u001b[31mModuleNotFoundError\u001b[39m                       Traceback (most recent call last)",
      "\u001b[36mCell\u001b[39m\u001b[36m \u001b[39m\u001b[32mIn[21]\u001b[39m\u001b[32m, line 1\u001b[39m\n\u001b[32m----> \u001b[39m\u001b[32m1\u001b[39m \u001b[38;5;28;01mfrom\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01mlangchain_unstructured\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[38;5;28;01mimport\u001b[39;00m UnstructuredLoader\n\u001b[32m      3\u001b[39m file = \u001b[33mr\u001b[39m\u001b[33m\"\u001b[39m\u001b[33mC:\u001b[39m\u001b[33m\\\u001b[39m\u001b[33mUsers\u001b[39m\u001b[33m\\\u001b[39m\u001b[33mdeepa\u001b[39m\u001b[33m\\\u001b[39m\u001b[33mDownloads\u001b[39m\u001b[33m\\\u001b[39m\u001b[33mDeepanshu Bhatt\u001b[39m\u001b[33m\\\u001b[39m\u001b[33mrerprCheck-NLP-base-Biomedical-Research-Checklist-\u001b[39m\u001b[33m\\\u001b[39m\u001b[33mData\u001b[39m\u001b[33m\\\u001b[39m\u001b[33mraw\u001b[39m\u001b[33m\\\u001b[39m\u001b[33mpmc_pdfs\u001b[39m\u001b[33m\\\u001b[39m\u001b[33mrsob.240345.pdf\u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m      5\u001b[39m loader = UnstructuredLoader(file, mode= \u001b[33m\"\u001b[39m\u001b[33melements\u001b[39m\u001b[33m\"\u001b[39m, strategy= \u001b[33m\"\u001b[39m\u001b[33mfast\u001b[39m\u001b[33m\"\u001b[39m)\n",
      "\u001b[31mModuleNotFoundError\u001b[39m: No module named 'langchain_unstructured'"
     ]
    }
   ],
   "source": [
    "from langchain_unstructured import UnstructuredLoader\n",
    "\n",
    "file = r\"C:\\Users\\deepa\\Downloads\\Deepanshu Bhatt\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\Data\\raw\\pmc_pdfs\\rsob.240345.pdf\"\n",
    "\n",
    "loader = UnstructuredLoader(file, mode= \"elements\", strategy= \"fast\")\n",
    "doc = loader.load()\n",
    "\n",
    "print(doc)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "0057a366-aaff-4433-9fbd-5a51a73fa691",
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_community.document_loaders import PyMuPDFLoader"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "8284c33b-4afc-411e-8466-9188bfb7c054",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 0}, page_content='PIEZO1 variant implications for biological\\nunderstanding and human\\xa0health\\nChew W. Cheng1, Sophie L. Earle1,2, Oleksandr V. Povstyan1, Chloe Randall1,\\nKatie A. Smith1, Marjolaine Debant1, Fraser L. Macrae1, Daniel G. Beech3, Anna\\nMcGrane1, Fiona Bartoli1, Eulashini Chuntharpursat-Bon1, Richard M. Cubbon1,\\nKathryn J. Griffin1,4, Marc A. Bailey1,6, Antreas C. Kalli1, Lara C. Morley1,2, Klaus\\nK. Witte1,5 and David J. Beech1\\n1Leeds Institute of Cardiovascular and Metabolic Medicine, School of Medicine, and 2Academic Department of\\nObstetrics and Gynaecology, School of Medicine, University of Leeds, Leeds LS2 9JT, UK\\n3Calderdale and Huddersfield NHS Foundation Trust, Huddersfield HD3 3EA, UK\\n4Department of Histopathology, and 5Department of Cardiology, Leeds Teaching Hospitals NHS Trust, Leeds, UK\\n6Leeds Vascular Institute, Leeds General Infirmary, Leeds, UK\\n AMG,\\xa00000-0002-7813-7213; DJB,\\xa00000-0002-7683-9422\\nThe large membrane protein PIEZO1 assembles as trimers to form\\nexceptional mechanical force-sensing ion channels of eukaryotes. When\\nthese channels are activated by force, cell membrane permeability to\\ncalcium ions and other ions increases rapidly, coupling force to cell\\nfunction through ionic control. In humans and other species, PIEZO1 is\\nboth widely expressed and functional across major systems that include\\nthe cardiovascular, haematological and musculoskeletal systems, thereby\\nserving diverse needs. In this narrative review of the scientific literature, we\\naddress what has been learned about PIEZO1 from associations of its gene\\nvariation with human characteristics. A particular physiological importance\\nof PIEZO1 is emerging in lymphatics and thus in the control of tissue fluid\\nhomeostasis with relevance to the disease conditions of non-immune fetal\\nhydrops and generalized lymphatic dysplasia. Other vascular relevance\\nis seen in lower limb venous varicosities. PIEZO1 may be non-essential\\nin red blood cells but the amplification of its function by gene variation\\nquite selectively alters these cells, leading to haemolytic anaemia and\\nother related disturbances that may be only mildly adverse and confer\\nsurvival advantage. We speculate on what else might be learned in humans,\\nguided by knowledge from PIEZO1 studies in mice, and describe how\\nknowledge accumulated to date highlights new opportunities for PIEZO1\\nunderstanding and pathways to patient benefit.\\n1. Introduction\\nMechanobiology is the ability of biological molecules, cells and tissues to\\ndetect and respond to mechanical forces, thereby ensuring that the mechan-\\nical states of these structures match the external and internal demands of\\norganisms (figure 1). Mechanobiology has been described as ‘the elephant\\nin the room … overlooked’ [1], but there is now increasing recognition of\\nits importance across animal biology including human health and disease [2–\\n7]. Moreover, although all biological molecules are potentially mechanically\\nsensitive depending on the intensity and type of stimulus, it has become\\nincreasingly clear that specific proteins evolved as dedicated force sensors of\\nphysiology, conferring high-fidelity force-dependent responsiveness [3,7]. A\\nkey protein of this type is PIEZO1 [8–10]. It has remarkable force sensing and\\n© 2025 The Authors. Published by the Royal Society under the terms of the Creative Commons Attribution\\nLicense http://creativecommons.org/licenses/by/4.0/, which permits unrestricted use, provided the original\\nauthor and source are credited.\\nReview\\nCite this article: Cheng CW et\\xa0al. 2025 PIEZO1\\nvariant implications for biological understanding\\nand human health. Open Biol. 15: 240345.\\nhttps://doi.org/10.1098/rsob.240345\\nReceived: 11 November 2024\\nAccepted: 28 May 2025\\nSubject Areas:\\ngenetics, cellular biology, molecular biology,\\nbiophysics\\nKeywords:\\nnon-selective cation channel, calcium channel,\\ncalcium signalling, mechanical force, shear stress,\\nendothelium\\nAuthor for correspondence:\\nDavid J. Beech\\ne-mail: d.j.beech@leeds.ac.uk'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 1}, page_content='transduction capabilities, serves force sensing needs in many cell and tissue types, and may be a ubiquitous force sensor across\\neukaryotes [8,10–12].\\n1.1. PIEZO1 properties\\nPIEZO1 was first defined as a force sensor in patch-clamp recordings from nerve cancer cells cultured in an incubator [9]. The studies\\nwere possible because PIEZO1 forms calcium ion (Ca2+)-permeable non-selective cationic channels in the outer cell membrane [9]. The\\nchannels open in response to the application of experimentally applied forces such as increased pressure in the patch-clamp recording\\npipette [9,13]. As the channels open, they allow the flow of millions of ions across the cell membrane, creating a measurable ionic\\ncurrent (figure 2a–d). Although artificial methods of this type can activate the channels in experiments, the channels are also sensitive\\nto physiological forces that arise, for example, from fluid flow [14] and tissue stiffness [15].\\nPIEZO1 channels change in their properties depending on their context but they usually activate quickly (in milliseconds\\nor less) when there is a rapidly applied force (figure 2b,c). Channel activation is seen through a net flux of cations into cells\\nat negative membrane potentials (figure 2d), which is the usual electrical polarity of cells. In the absence of experimentally\\napplied voltage control, such ion influxes cause depolarization, a lessening of the negative membrane potential (figure 2d) that\\ncan increase or decrease cell function depending on the cell type. A cation that enters cells via PIEZO1 channels is Ca2+, which\\nis a pivotal currency of intracellular control [16]. Its cytosolic elevation can trigger many cell activities such as contraction,\\nsecretion and migration [17]. The fluxes of two other physiological cations, the sodium ion (Na+) in and the potassium ion (K+)\\nout, can also have specific impact. Therefore, PIEZO1 has the potential to alter cell function through several ionic mechanisms\\nthat modulate downstream processes, thereby generating effects.\\nImportantly, once the channels are activated, they can desensitize rapidly when there is sustained mechanical force (figure\\n2b). This property of inactivation is regulated, for example by lipid and protein factors [18–20], and so inactivation may be\\nabsent or occur slowly (figure 2c) [18]. Regulated inactivation seems to be a key way in which PIEZO1 channels achieve\\ndifferent functions in diverse cell types and contexts [20]. Once the mechanical stimulus is removed, the channels switch off\\nthrough another process that is referred to as deactivation (figure 2c). Inactivation and deactivation are both a type of off switch\\nbut inactivation differs in its tendency to cause refractoriness to the subsequent opening of the channel when another stimulus\\narrives. Inactivation thus confers a memory of prior events.\\nThe PIEZO1 channel is formed by three PIEZO1 proteins, each comprising about 2500 amino acid residues (2521 in human\\nPIEZO1) arranged in chains that loop in and out of the membrane 38 times, leading to 38 membrane-spanning segments\\nper monomer (figure 3a,b) and 114 for the whole channel trimer (figure 3c) [8,21,22]. The channel assembly embeds in and\\nindents the membrane, forming a basket-like (dome) structure when in its closed ion-impermeable configuration [22,23]. Long\\ntentacle-like blade structures span out from the core of each monomer (figure 3c). Each blade comprises nine transmembrane\\n(TM) helical units (THU1−9), so there are 27 THUs per channel. Between THU7 and THU8 of each monomer is a linker on the\\nintracellular side that includes a beam helix extending towards the pore (figure 3a). The beam is followed by a lateral plug and\\nlatch before an unstructured loop region and the clasp sitting below THU8 (figure 3a). After the last THU (THU9) is the anchor\\nand then the outer helix formed by TM37, which lines the central ion pore with the inner helix of TM38. These two pore-lining\\nhelices are intervened by the C-terminal extracellular domain (CED), which sits above the central core, forming a cap that has\\nfoot-like structures touching nearby blade regions [22] (figure 3a). The pore is lastly cuffed on the intracellular side by the\\nC-terminal domain (CTD) (figure 3a). Lateral tension in the membrane causes the dome to flatten and the blades to spread out\\nradially. With flattening and expansion comes opening in the core of the channel and ion permeation, and thus a signal for the\\ncell to act [8,23]. A more in-depth description of the channel structure can be found elsewhere [8,23], although much remains to\\nbe understood.\\n1.2. PIEZO1 roles\\nPIEZO1 has roles in contractile, secretory and many other cell types of all major systems that include adipose, cardiovascular,\\ndermal, gastrointestinal, haematological, hepatobiliary, immune, musculoskeletal, neurological, reproductive, respiratory and\\nurinary organs as well as cancers [8]. The roles are sometimes inferred based on the responses of cultured cells to the synthetic\\nchemical agonist of PIEZO1 channels, Yoda1 or an analogue of it [24]. Responses to these agonists may not be physiological\\nbut studies of Piezo1 gene-modified mice also support the idea of diverse functions of PIEZO1 in physiology and disease as\\nmodelled in the mouse [14,25–30], all of which are likely to be due to mechanical activation of the channels by endogenous\\nforces. Thus, there might be wide-ranging functional importance of PIEZO1 in human health and disease, but direct testing of\\nsuch a hypothesis is challenging. We do not currently have PIEZO1 modulators that can be investigated in clinical studies. In\\nparticular, we do not have suitable PIEZO1 inhibitors to test the roles of PIEZO1 [24]. However, the PIEZO1 gene is naturally\\nvaried in sequence, and this presents opportunities to explore associations between its variation and phenotypes in familial\\ngenetic, genome-wide association and candidate gene association studies. Hundreds of PIEZO1 variants have so far been linked\\nstrongly or tentatively to human traits, diseases and susceptibilities to disease or therapies.\\n1.3. Focus of this article\\nHere, we review reports of PIEZO1 variants published in the scientific literature, cataloguing 179 variants in table 1. Addi-\\ntional variants and potential association data may be found in databases such as UK Biobank (www.ukbiobank.ac.uk),\\n2\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 2}, page_content='ClinVar (www.ncbi.nlm.nih.gov/clinvar), AstraZeneca PheWAS Portal (https://azphewas.com) and FinnGen (www.finngen.fi).\\nThe PIEZO1 gene [125] is on chromosome 16 at locus 16q24.3, so variants are autosomal and can be dominant or recessive.\\nHence, some associations are evident only when there is homozygosity of the variant but there are also associations with\\nheterozygous variants and compound heterozygous variants (i.e. when two different variants occur in the same person, one\\ninherited from each parent). The variants include exonic variants that may affect details of the PIEZO1 amino acid sequence\\n(missense, deletion, insertion and duplication variants) or the integrity of the PIEZO1 protein (premature termination codons,\\nframeshifts and splice site variants). These variants are the focus of table 1. There are also synonymous and intronic variants, the\\neffects of which may arise via gene regulatory mechanisms.\\nWe begin by addressing the three main disease conditions most associated with PIEZO1 variants, relating to lymphatics,\\nveins and blood. We then consider additional diseases that may arise because of one or more of the main conditions or\\nindependently of them. We discuss physiological traits that may be predisposing factors to some diseases or confer survival\\nadvantage. We consider what might be learned about how the channels work as structural machines. We address the relevance\\nto human physiology and the apparent paradox of relatively narrow disease phenotypes despite the widespread expression\\nand functional significance of PIEZO1 that is indicated by laboratory studies. We consider the complexities of the genetic data\\nand what the data might tell us about the systems of the PIEZO1 gene and the PIEZO1 protein. We discuss the potential\\nfor predicting and correcting or compensating for its abnormal expression or function, and thus the opportunities for patient\\nbenefit. We make conclusions and discuss potential future directions for research on this topic.\\nFigure 1. Illustration of the concept of mechanical homeostasis concept. The central box indicates the requirement for mechanical force sensors to detect force, and\\ntransducers to convert the amplitude of the detected force into a cellular effect. The molecular details of the sensors and transducers are not fully determined but\\nPIEZO1 channels are both a crucial force sensor and transducer in many cell and tissue types.\\nFigure 2. PIEZO1 channel electrophysiology. Patch-clamp data obtained from HEK 293 cells that were not transfected (a) or which were transfected (b–d) with\\nplasmid containing cDNA encoding human PIEZO1 (hPIEZO1) channels. (a–c) Outside-out patch data at a constant transmembrane voltage of −80 mV (millivolt).\\n(a–c) Example pressure steps applied to the patch are shown in the upper traces with associated current traces below for ionic currents across the membrane. All\\npressure steps were from a background of zero pressure (0 mmHg) to +75 mmHg (positive pressure on the patch pipette, leading to stretch of the membrane). The\\n100 pA (picoampere) and 100 ms (millisecond) calibrations apply to all current traces. (a) Without PIEZO1 transfection, no current was seen in response to the pressure\\nstep. (b,c) With PIEZO1 transfection, inward current (downward deflection) was seen in response to the pressure step. Two examples of the variation in the properties\\nof the PIEZO1 channel currents are shown in (b) and (c). In example 1 (b), there was very fast activation and then slightly slower but still very fast inactivation that was\\ncomplete within the time course of the pressure step (i.e. no PIEZO1 current remained). In example 2 (c), there was also fast activation but then slower inactivation that\\nwas incomplete (i.e. about 25% of the PIEZO1 current remained at the end of the pressure step). After the pressure step ended, deactivation of the remaining current\\noccurred (c). (d) Whole-cell voltage-clamp data for current evoked by the PIEZO1 agonist Yoda1 and obtained in response to a ramp change in voltage from −100 mV\\nto +100 mV. The current progressively changed from inward polarity at −100 mV to outward polarity at +100 mV with the cross-over point from inward to outward\\ncurrent referred to as the reversal potential. In physiology (without voltage-clamp), the resting membrane potential is expected to be in the region of −30 and −80\\nmV. Therefore, inward current through PIEZO1 channels (i.e. influx of cations, which are positively charged) causes depolarization towards the reversal potential.\\n3\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 3}, page_content='2. Lymphatics: generalized lymphatic dysplasia and non-immune fetal hydrops\\nGeneralized lymphatic dysplasia (GLD) is a rare form of primary systemic lymphedema characterized by widespread accumu-\\nlation of fluid within subcutaneous spaces and body compartments resulting from structural or functional abnormalities of\\nthe lymphatic system that affect the whole body [126]. It may present antenatally as non-immune fetal hydrops (NIFH) on\\nultrasound scan [126]. It is defined as abnormal fluid accumulation in at least two body compartments including pericardial\\nor pleural effusion, ascites and subcutaneous oedema. Immune fetal hydrops is secondary to red blood cell (RBC) allo-immuni-\\nzation whereas the aetiology of NIFH includes congenital infection, cardiovascular and genetic abnormalities. The hydropic\\nchanges occur due to an increase in interstitial fluid or impaired lymphatic drainage. Therefore, the term lymphatic-related fetal\\nhydrops may be used to refer to NIFH of lymphatic origin [126].\\n2.1. PIEZO1 disruption identified by familial genetics\\nA distinct recessive form of GLD was identified and referred to as Fotiou GLD [70,126,127] or lymphatic malformation 6 [128]. It\\nis characterized by a high incidence of NIFH and fetal demise. If the fetus survives, there is transient resolution of the oedema,\\nbut recurrence of oedema may occur in childhood or later life [70,126,127]. Studies of six families [70] followed by a further\\nthree families [80] identified associations with PIEZO1 variants. These disorders are associated with loss of PIEZO1 protein\\n(LOP) [70,80,91], loss of PIEZO1 channel function (LOF) [80] or both [45,80]. Often, the variants cause premature termination\\ncodons early in the reading frame, or frameshifts or deletions resulting in major disruption in at least one allele (table 1), leading\\nto partial or complete loss of PIEZO1 (figure 4a).\\nFotiou GLD [126,127] is associated with PIEZO1 variants that are homozygous or compound heterozygous [70,80,91] (table\\n1). Severe disease may not be associated with heterozygous single PIEZO1 variants, so the parents of the GLD suffers may\\nnot necessarily suffer from the disease or they may have only mild disease even though they carry a variant [45,70,80].\\nTherefore, the disease is classed as recessive [70]. There is, however, emerging evidence of dominance in which a single variant\\nheterozygous parent also has the disease [80].\\nFigure 3. PIEZO1 channel structure: the AlphaFold (https://alphafold.ebi.ac.uk) model of human PIEZO1 AF-Q92508-F1-v4. (a,b) The monomer in side-view (a) and\\ntop (helicopter) view (b). The N-terminus is to the right and the C-terminus to the left. (c) The trimer (the complete PIEZO1 channel) assembled by aligning the single\\nchain (monomer) of the AlphaFold model to the coordinates of all three chains of the full-length mouse PIEZO1 model in Pymol (www.pymol.org). The N-termini are\\nat the outer tips of the blades and the C-termini are central. The trimer in (c) is artificially enlarged relative to the size of the monomer in (a,b) for visual presentation.\\n(a–c) Highlighted in red or blue are variants that have been shown by patch-clamp to affect channel function: Red, loss of function (LOF); blue, gain of function (GOF).\\nIn the trimer, each variant is repeated three times, thus assuming homozygosity. Details of these variants can be found in the main text and table 1. (a,b) Indicated\\nstructural features of the channel: the cap; the pore-lining inner and outer helices (IH and OH) in light pink and the anchor in darker pink; the beam in light grey and\\nthe lateral plug and latch in darker grey; and the clasp and the CTD in dark grey. The blade comprises the 9 THUs, THU1−9.\\n4\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 4}, page_content=\"Table 1. Phenotypes associated with PIEZO1 protein variants. Listed in the Table are PIEZO1 variants that affect the amino acid sequence of the PIEZO1 protein. The data are from peer-reviewed or preprint papers in the public domain. Additional\\nvariants may be found in databases as indicated in the main text. A few variants listed previously [8] could not be confirmed in our search of the peer-review literature, so they are not included. Amino acid residue numbers are for human\\nPIEZO1 with the equivalent for mouse PIEZO1 in curved brackets, determined using a Clustal Omega alignment that compared UniProt (www.uniprot.org) mouse (E2JF22) and human (Q92508) PIEZO1 amino acid sequences. Structural regions of\\nthe human PIEZO1 channel are [8] (with amino acid numbers in square brackets): Blade Transmembrane Helical Units (THUs) 1−7 [1−142; 143−334; 345−541; 542−700; 701−948; 949−1116; 1117−1304]; Beam [1305−1367]; Central plug\\n[1368−1401]; Lateral plug [1402−1411]; Latch [1412−1425]; Latch-to-Clasp [1426−1514]; Clasp [1515−1570]; Clasp-to-THU8 [1571−1656]; Blade THUs 8−9 [1657−1802; 1803−2098]; Anchor [2099−2165]; Outer helix [2166−2193]; Cap\\n(C-terminal extracellular domain, CED) [2194−2439]; Inner helix [2440−2473]; C-terminal domain (CTD) [2474−2521]. Each PIEZO1 channel is a trimer of 3 PIEZO1 proteins, so the total number of amino acids per human PIEZO1 channel is 7563 and\\neach variant repeats three times per channel in homozygosity. Amino acid numbers for structural regions of mouse PIEZO1 channel are specified elsewhere [8]. ‘None specified’ indicates where, to the best of our knowledge, no peer-reviewed data\\nare available. The 'references' column indicates the underlying numbered reference containing the supporting evidence. Where laboratory data are available for the effect of the variant on channel function, the effect is indicated in blue for GOF and\\nred for LOF or LOP. Abbreviations: hom. (homozygous); het. (heterozygous); c-het. (compound heterozygous); del. (deletion).\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n31 (31)\\nS to A frameshift\\n(with R49\\npremature\\ntermination)\\nblade THU1\\nnone specified\\nlymphatic malformation; may be\\nclassified as generalized lymphatic\\ndysplasia (GLD)\\nnon-immune fetal hydrops (NIFH)\\nsubcutaneous oedema in head and trunk\\nhydrothorax\\ntermination of pregnancy\\n[31]\\n41 (41)\\nL deletion\\n(with T1638\\npremature\\ntermination)\\nblade THU1\\nnone specified\\nGLD\\nNIFH\\n[32]\\n49 (49)\\nR premature\\ntermination\\n(with S31A)\\nblade THU1\\nnone specified\\nNIFH\\nlymphatic malformation (GLD?)\\nsubcutaneous oedema in head and trunk\\nhydrothorax\\ntermination of pregnancy\\n[31]\\n62−65 (62-65)\\nAHGK deletion\\nblade THU1\\nnone specified\\ndehydrated hereditary stomatocytosis\\n(DHS)\\nnone specified\\n[33]\\n83 (83)\\nI to T\\nblade THU1\\nnone specified\\nCOVID-19\\nnone specified\\n[34]\\n88 (88)\\nD to E\\nblade THU1\\nnone specified\\nDHS with or without perinatal oedema\\nchylothorax\\nserial thoracentesis\\nmaternal propranolol\\nIatrogenic abortion\\n[35]\\n103 (103)\\nR premature\\ntermination\\n(c-het.)\\nblade THU1\\nnone specified\\nNIFH\\nGLD\\nnone specified\\n[36]\\n152 (152)\\nP to L\\nblade THU2\\nnone specified\\nCOVID-19\\nnone specified\\n[34]\\n(Continued.)\\n5\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345\"), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 5}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n189 (196)\\nR premature\\ntermination\\nblade THU2\\nnone specified\\nlymphatic malformation (GLD?)\\nscalp, total body, skin and nuchal (neck) fold oedema\\nbilateral pleural effusion\\nascites\\npericardial effusion Cardiomegaly, agenesis of the ductus venosus,\\nplacentomegaly, enlarged tongue, anemia\\npercutaneous umbilical blood sampling with transfusion\\nIntrauterine foetal demise\\n[35]\\n217 (224)\\nS to L\\nblade THU2\\ndecreased activation\\nprotein degradation\\nbicuspid aortic valve disease\\nnone specified\\n[37–39]\\n250 (257)\\nV to A (or I [40])\\nblade THU2\\nnone specified\\nvaricose vein (VV)\\nCOVID-19\\nalcoholic liver disease\\nnone specified\\n[34,40,41]\\n251 (258)\\nA to T\\nblade THU2\\nnone specified\\nsevere congenital neutropenia\\nvariant predicted to be benign\\n[42]\\n253 (260)\\nG to R\\n(with S2195L)\\nblade THU2\\ndecreased opening\\n(mouse PIEZO1)\\nprune belly syndrome\\nabdominal wall laxity\\nflared ribs\\ndimpled knees\\ndilated bladder\\nbilateral hydroureter\\nsurgical urinary bladder drainage and other procedures\\nbilateral hydronephrosis\\nsupraventricular tachycardia\\n[43]\\n315 (321)\\nS to I\\nblade THU2\\nnone specified\\ninherited bone marrow failure\\nnone specified\\n[44]\\n322 (329)\\nL to P (c-het. with\\nE997 premature\\ntermination)\\nblade THU2\\ndecreased mechanical\\nsensitivity\\nNIFH\\nnone specified\\n[45]\\n338 (345)\\nS to Y\\nblade THU3\\nnone specified\\nDHS\\nnone specified\\n[46]\\n376 (383)\\nP to A (het.)\\nblade THU3\\nnone specified\\ncongenital haemolytic anaemia\\nnone specified\\n[47]\\n422 (429)\\nQ premature\\ntermination\\nblade THU3\\nnone specified\\nNIFH\\nnone specified\\n[48]\\n(Continued.)\\n6\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 6}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n427−428 (434-435)\\nA-L insertion\\nblade THU3\\nnone specified\\nDHS\\nnone specified\\n[49]\\n457 (464)\\nR to C\\nblade THU3\\nnone specified\\nDHS\\nnone specified\\n[50]\\n499 (505)\\nV to I\\n(with R2303H)\\nblade THU3\\nnone specified\\nDHS\\nlymphatic malformation (GLD?)\\noedema\\n[51,52]\\n513 (519)\\nC to S frameshift\\n(with S1540A)\\nblade THU3\\nnone specified\\nNIFH\\nnuchal translucency\\nlisted in [51,53]\\n531 (537)\\nR to C\\nblade THU3\\ndecreased opening\\n(mouse PIEZO1)\\nosteoarthritis\\nnon-syndromic familial osteoarthritis free from acute or traumatic joint injury.\\nerosive hand osteoarthritis and interphalangeal joint osteoarthritis\\n[54]\\n537 (543)\\nK to N\\nblade THU3\\nnone specified\\ninherited bone marrow failure\\nnone specified\\n[44]\\n598 (604)\\nV to M [55] or L [46]\\nblade THU4\\nincreased opening [55]\\nno change [56]\\nDHS\\nGLD\\nNIFH\\ntermination of pregnancy [57]\\ndouble superior vena cava [57]\\n[46,49,55–60]\\n599 (605)\\nV to G or L\\nblade THU4\\nnone specified\\nDHS\\nNIFH\\ntermination of pregnancy\\npleural effusions, ascites\\nwith or without pseudohyperkalemia\\nwith or without perinatal oedema\\n[51,61,62]\\n605 (611)\\nM to V or I\\nblade THU4\\nnone specified\\nDHS\\nnone specified\\n[46,63]\\n655 (661)\\nY premature\\ntermination\\n(c-het. ?)\\nblade THU4\\nnone specified\\nNIFH\\npleural effusion and chylothorax, skin oedema\\nmedical termination of pregnancy\\nplacental hypertrophy\\n[51,64]\\n669 (675)\\nD to Y or H\\nblade THU4\\nnone specified\\nDHS\\npotential association with cardiomyopathy [65]\\n[33,46]\\n679 (685)\\nE premature\\ntermination\\n(hom.)\\nblade THU4\\nnone specified\\nNIFH\\nnone specified\\n[66]\\n681 (687)\\nF to S\\nblade THU4\\nno change\\nDHS\\nnone specified\\n[46,56]\\n702 (708)\\nH to Y\\nblade THU5\\nnone specified\\nadenomatous polyposis\\nnone specified\\n[67]\\n718 (724)\\nG to S\\nblade THU5\\nnone specified\\nDHS\\nnone specified\\n[46,68]\\n722 (728)\\nP to L\\nblade THU5\\nnone specified\\nbrain arteriovenous malformations\\nnone specified\\n[69]\\n(Continued.)\\n7\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 7}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n745 (746)\\nQ deletion\\nblade THU5\\nnone specified\\nDHS\\nnone specified\\n[33]\\n755 (749)\\nE premature\\ntermination\\n(with Q2228\\npremature\\ntermination)\\nblade THU5\\ndecreased expression\\nGLD\\nNIFH\\npleural effusions\\npolyhydramnios\\ndied in utero at 34 weeks\\nmild generalized oedema with pleural effusions, chylothorax, periorbital\\noedema\\ncupped simple ears, epicanthic folds, micrognathia, atrial septal defect,\\namyoplasia of the diaphragm, gastro-oesophageal reflux, short stature,\\npectus excavatum, genital oedema, splenomegaly\\n[70]\\n755−756 (749-750)\\nEE deletion\\nblade THU5\\nnone specified\\nbrain arteriovenous malformations\\nnone specified\\n[69]\\n756 (750)\\nE deletion\\nblade THU5\\nincreased activity\\n(slowed inactivation)\\nmalaria resistance in some populations\\n[71,72]\\nRBC dehydration\\nincreased transferritin saturation [25]\\nincreased ferritin [25]\\ntendon stiffness and jumping ability in\\nsome populations\\nnot associated with malarial resistance in some populations [73]\\nnot associated with sickle disease [74]\\nnot associated with glaucoma, blood pressure or body mass index [75].\\n[25,71–73,75–78]\\n782 (777)\\nG to S (with variants\\nat 808 or 808\\nand 1764)\\nblade THU5\\nincreased activity [56]\\nDHS\\nwith or without perinatal oedema\\ncystic hygroma\\nliveborn\\ndysmorphic features\\nredundant nuchal skin\\nfifth toe clinodactyly\\nbilateral ventriculomegaly\\n[35,46,56,60,68]\\n808 (803)\\nR to Q or G (may be\\nwith 782 and\\n1764 variants)\\nblade THU5\\nno change (R808Q) [56]\\nDHS\\nGLD\\ncystic hygroma\\nwith or without perinatal oedema\\nliveborn\\ndysmorphic features\\nredundant nuchal skin\\nfifth toe clinodactyly\\n[35,46,60,68,79]\\n(Continued.)\\n8\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 8}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\nbilateral ventriculomegaly\\n829 (824)\\nE to V (with or\\nwithout I2270T)\\nblade THU5\\ndecreased activity and\\nexpression\\nGLD\\nbilateral hydrothoraces (het.)\\nNIFH (het.)\\npolyhydramnios (het.)\\noedema of lower limbs (E829V only)\\nintermittent oedema of face, scrotal oedema (het.)\\npleural effusions/chylothoraces (het.)\\ngastro-oesophageal reflux requiring fundoplication and gastrostomy (het.)\\nAsperger syndrome, metopic, craniosynostosis, obstructive sleep apnoea\\nbilateral periorbital and conjunctival vascular changes with small punctate\\nhaemorrhages (het.)\\n[80]\\n860 (855)\\nV to M\\nblade THU5\\nnone specified\\ncongenital hemolytic anaemia\\nnone specified\\n[47]\\n868 (863)\\nC premature\\ntermination\\nblade THU5\\nnone specified\\nNIFH\\nascites, pleural effusion, skin oedema, hydramnios, polyhydramnios\\nlymphedema, lymphangiectasia\\nsigns of maternal vascular malperfusion\\ndeath at 1 month old (hom.)\\n[64]\\n870 (865)\\nM to I\\n(c-het.)\\nblade THU5\\nnone specified\\nNIFH\\nDHS\\nprimary immune deficiency disease\\n(het.)\\nsuspected posterior urethral valve\\nhepatic dysfunction\\n[32,51,59,81]\\n908 (903)\\nR to P\\nblade THU5\\nnone specified\\ninherited bone marrow failure\\nnone specified\\n[44]\\n915 (910)\\nN to S\\n(c-het. with G6PD\\nvariant)\\nblade THU5\\nnone specified\\nsevere haemolytic anemia (DHS?)\\nnone specified\\n[82]\\n939 (934)\\nL to M\\n(with R2456C and\\nF2458L)\\nblade THU5\\ndecreased mechanical\\nsensitivity [38]\\nNIFH\\nDHS\\npleural effusions\\npolyhydramnios\\ngeneralized oedema at birth, chylothorax/chylothoraces, webbed neck,\\nperiorbital oedema, prune belly\\ngastro-oesophageal reflux, hypothyroid at birth, occasional stomatocytes\\n[46,70]\\n(Continued.)\\n9\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 9}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n953 (948)\\nR to H\\n(with exon 1−50\\ndeletion)\\nblade THU6\\nnone specified\\nlymphatic dysplasia (GLD?)\\nhydroceles, VVs, facial asymmetry, dysplastic ears, thoracolumbar scoliosis,\\nmild hearing loss, mild intellectual disability, multiple fracture history,\\nshort stature, thoracolumbar scoliosis, left-sided facial bone hypoplasia,\\nleft ossicular malformation, prognathism\\n[83]\\n966 (961)\\nV to M\\nblade THU6\\nnone specified\\nDHS\\nnone specified\\n[84]\\n972 (966)\\nR to H\\nblade THU6\\nnone specified\\nDHS\\nnone specified\\n[62]\\n975 (970)\\nL to V\\nblade THU6\\nnone specified\\nDHS\\nWith or without perinatal oedema\\nbilateral chylothorax\\nascites\\npleural effusion, respiratory distress, right lower quadrant venous lymphatic\\nmalformation, serial thoracentesis, chest tube placement, ventilation,\\npropranolol, diuresis, liveborn\\n[35]\\n991 (986)\\nF to L\\nblade THU6\\nnone specified\\ncerebral cavernous malformation\\nnone specified\\n[85]\\n997 (992)\\nE premature\\ntermination (c-\\nhet. with L322P)\\nblade THU6\\nnone specified\\nNIFH\\nnone specified\\n[45]\\n1007 (1002)\\nI to M\\nblade THU6\\nnone specified\\nadenomatous polyposis\\nnone specified\\n[67]\\n1010 (1005)\\nR to H\\n(with R1671W)\\nblade THU6\\nnone specified\\nGLD\\nchylothorax\\nbilateral lower limb oedema\\n[86]\\n1011 (1006)\\nM to I\\nblade THU6\\nnone specified\\ninherited bone marrow failure\\nnone specified\\n[44]\\n1069 (1064)\\nW premature\\ntermination\\n(with K2070Q)\\nblade THU6\\nnone specified\\nGLD\\nNIFH\\nstillbirth\\nfacial and corporal oedema\\nmacroencephaly\\nhypoplastic lungs\\nfailure of lymphatic luminal canalization\\nhydroscopic chorionic villi\\nfetal vascular malperfusion\\n[87,88]\\n1070 (1065)\\nR to C\\nblade THU6\\nnone specified\\nlymphatic malformation (NIFH?)\\nincreased nuchal fold\\nbilateral pleural effusion\\ntotal body and skin oedema\\n[35]\\n(Continued.)\\n10\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 10}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\nascites\\npericardial effusion\\npolyhydramnios macrocephaly, anasarca, bilateral pleural effusion,\\nhypospadias, and respiratory distress\\nneonatal mechanical ventilation and bilateral chest tube placement\\nneonatal death\\n1073 (1070)\\nR to W\\nblade THU6\\nnone specified\\nerythrocytosis\\nnone specified\\n[89]\\n1111 (1106)\\nQ premature\\ntermination\\n(het.)\\nblade THU6\\nnone specified\\nEr antigen\\nnone specified\\n[90]\\n1114 (1109)\\nQ to E\\n(with A2395V)\\nblade THU6\\nnone specified\\nGLD\\nNIFH\\nlisted in [51,53]\\n1117 (1112)\\nS to L (with\\nA2020V)\\nblade THU7\\nnone specified\\nDHS\\nnone specified\\n[68]\\n1153 (1148)\\nS premature\\ntermination\\n(with G2029R)\\nblade THU7\\nnone specified\\nlymphatic dysplasia (NIFH?)\\nhydrops\\nbilateral chylothorax\\nswelling of legs and scrotum lymphedema of legs, torso and face\\npleural effusions\\nintensive care including extracorporeal membrane oxygenation (one sibling)\\n[91]\\n1201 (1196)\\nT to M\\nblade THU7\\nnone specified\\nneuro-developmental disorder\\nvariant predicted to be non-pathogenic\\n[92]\\n1223 (1218)\\nV to I\\nblade THU7\\nnone specified\\nDHS\\nmyelodysplastic syndrome\\niron overload, splenomegaly, haemolysis\\n[93,94]\\n1247 (1241)\\nF to C\\nblade THU7\\nnone specified\\nliver biliary pancreas abnormality\\nnone specified\\n[40]\\n1255 (1250)\\nS to R (c-het. other\\ngene variants)\\nblade THU7\\nnone specified\\ninherited haemolytic anaemia\\nnone specified\\n[79]\\n1287 (1282)\\nW premature\\ntermination\\n(with\\nc.2330−2_2330\\n−1del)\\nblade THU7\\nnone specified\\nGLD\\nNIFH\\npolyhydramnios\\nbilateral pleural effusions\\nmicrognathia\\n[95]\\n(Continued.)\\n11\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 11}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\nbilateral oedema of the calves\\nelevated pleural triglycerides\\n1299 (1294)\\nR to C\\n(with E1344 del)\\nblade THU7\\nnone specified\\nNIFH\\nbilateral pleural effusion\\ngeneralized skin oedema\\nminor ascites\\npolyhydramnios\\npregnancy termination\\nanasarca\\n[96]\\n1332 (1327)\\nN to T\\nbeam\\nnone specified\\nliver biliary pancreas abnormality\\nnone specified\\n[40]\\n1344 (1339)\\nE deletion\\n(with R1299C)\\nbeam\\nnone specified\\nNIFH\\nbilateral pleural effusion\\ngeneralized skin oedema\\nminor ascites\\npolyhydramnios\\npregnancy termination\\nAnasarca\\n[96]\\n1357 (1352)\\nI to V or M\\nbeam\\nnone specified\\nDHS\\nnone specified\\n[46]\\n1358 (1353)\\nR to P (or C with\\nA1678V) [96]\\nbeam\\nincreased activity\\n(slower inactivation)\\nDHS\\nCongenital arm lymphedema\\nnone specified\\n[46,96,97]\\n1361 (1356)\\nQ to R\\nbeam\\nnone specified\\nanaemia\\njaundice, splenomegaly, mild macrocytic, chronic haemolytic anaemia,\\ndessicytes, dyserythropoiesis\\n[98]\\n1396 (1390)\\nS to F\\ncentral plug\\nnone specified\\nangioimmunoblastic T-cell lymphoma\\nnone specified\\n[99]\\n1404 (1398)\\nR to W\\nlateral plug\\ndecreased opening\\n(mouse PIEZO1)\\nosteoarthritis\\nnon-syndromic familial osteoarthritis free from acute or traumatic joint injury.\\nerosive hand osteoarthritis and interphalangeal joint osteoarthritis\\n[54]\\n1416 (1410)\\nG to R\\nlatch\\nnone specified\\ncongenital haemolytic anaemia\\nnone specified\\n[47]\\n1457 (1451)\\nA to V\\nlatch-to-\\nclasp\\nnone specified\\nDHS\\nnone specified\\n[49]\\n1458 (1452)\\nV to A frameshift\\nlatch-to-\\nclasp\\nnone specified\\nNIFH\\nnone specified\\n[51]\\n(Continued.)\\n12\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 12}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n1496 (1487)\\nA to V\\n(potential variant)\\nlatch-to-\\nclasp\\nnone specified\\nNIFH\\nNIFHa\\n[66]\\n1497 (1488)\\nA to V\\nlatch-to-\\nclasp\\nnone specified\\nliver biliary pancreas abnormality\\nnone specified\\n[40]\\n1519 (1510)\\nG to P\\n(maybe with\\nQ1910K and\\nR2110W)\\nclasp\\nnone specified\\nDHS\\nlymphatic malformation (GLD?)\\noedema\\n[46,52]\\n1527 (1518)\\nR to H\\nclasp\\nnone specified\\nprotective against primary open-angle\\nglaucoma\\nnone specified\\n[99]\\n1540 (1531)\\nS to A\\n(with C513S)\\nclasp\\nnone specified\\nNIFH\\nNIFH\\nNuchal translucency\\nlisted in [51,53]\\n1547 (1538)\\nR to H\\n(with A2227V)\\nclasp\\nnone specified\\nNIFH\\nGLD\\nskin thickening (fetal occiput and nucha)\\npleural effusions\\npericardial effusion\\ncaesarean delivery at 34 weeks\\n[51,61]\\n1566 (1557)\\nD to G\\nclasp\\nnone specified\\nDHS\\nnone specified\\n[33]\\n1570 (1561)\\nV to L\\n(with G1629R)\\nclasp\\nnone specified\\nNIFH\\nnone specified\\nlisted in [51,53]\\n1591 (1582)\\nQ to P (with E1910K\\nand R2110T)\\nclasp-to-\\nTHU8\\nnone specified\\nNIFH\\nnone specified\\nlisted in [51,53]\\n1617 (1608)\\nT to M (maybe with\\nother gene\\nvariants)\\nclasp-to-\\nTHU8\\nnone specified\\nvasculitis and autoimmune connective\\ntissue disease\\ncolchicine, glucocorticosteroids, azathioprine\\n[100]\\n1629 (1620)\\nG to R (with\\nV1570L)\\nclasp-to-\\nTHU8\\nnone specified\\nNIFH\\nnone specified\\nlisted in [51,53]\\n1630 (1621)\\nE premature\\ntermination\\n(hom.)\\nclasp-to-\\nTHU8\\ndecreased expression\\nlymphedema (GLD?)\\nhead and neck swelling, hydroceles, polyhydramnios, micrognathia, recurrent\\nfacial cellulitis, deep vein thrombosis, genital oedema, chylothorax/\\nchylothoraces and bilateral pleural effusions at age 2 years, VVs, bilateral\\nsensorineural deafness, hypothyroidism, mild developmental delay\\n[70]\\n(Continued.)\\n13\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 13}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\noccasional stomatocytes\\n1638 (1629)\\nY premature\\ntermination\\n(with L41del)\\nclasp-to-\\nTHU8\\nnone specified\\nNIFH\\nnone specified\\nlisted in [51,53]\\n1671 (1671)\\nR to W\\n(with R1010H)\\nblade THU8\\nnone specified\\nGLD\\nchylothorax\\nbilateral lower limb oedema\\n[86]\\n1678 (1678)\\nA to V\\n(with R1358C)\\nblade THU8\\nnone specified\\ncongenital arm lymphedema\\nnone specified\\n[96]\\n1684 (1684)\\nA to V\\nblade THU8\\nnone specified\\nbrain arteriovenous malformations\\nnone specified\\n[69]\\n1712 (1712)\\nV to M\\nblade THU8\\nnone specified\\nadenomatous polyposis\\nnone specified\\n[67]\\n1721 (1721)\\nS to W\\nblade THU8\\ndecreased protein\\nexpression (hom.)\\nGLD\\nNIFH\\npleural effusions\\nfetal demise\\npregnancy termination\\n[101]\\n1732 (1732)\\nT to M\\nblade THU8\\nnone specified\\nDHS\\nnone specified\\n[33]\\n1763 (1763)\\nY premature\\ntermination\\n(with E2392K)\\nblade THU8\\nnone specified\\nadenomatous polyposis\\nEr antigen\\nhistory of blood transfusion\\n[67,102]\\n1764 (1764)\\nE to Q\\n(with 782 and 808)\\nblade THU8\\nnone specified\\nDHS\\nwith or without perinatal oedema\\ncystic hygroma\\nliveborn\\ndysmorphic features\\nredundant nuchal skin, fifth toe clinodactyly, bilateral ventriculomegaly\\n[35]\\n1790 (1790)\\nL to S frameshift\\n(with\\nc.7049+1G>C)\\nblade THU8\\nnone specified\\nNIFH\\nmortality within 1 day of birth\\npolyhydramnios\\nskin oedema\\npleural effusions.\\nneedle drainage of pleural effusion, thoracoamniotic shunt placement\\nintrauterine transfusion\\n[51,61]\\n1791 (1791)\\nM to I\\nblade THU8\\nnone specified\\nneuro-developmental disorder\\nvariant predicted to be non-pathogenic\\n[103]\\n1797 (1797)\\nR to C\\nblade THU8\\nnone specified\\nDHS\\nnone specified\\n[50]\\n(Continued.)\\n14\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 14}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n1828 (1840)\\nE to Q\\nblade THU9\\nnone specified\\nbrain arteriovenous malformations\\nnone specified\\n[69]\\n1864 (1881)\\nR to H\\n(with E2492_\\nL2493 duplication)\\nblade THU9\\nnone specified\\nDHS\\nnone specified\\n[104]\\n1877 (1894)\\nK deletion\\nblade THU9\\ndecreased activity\\n(faster inactivation,\\nless current)\\nDHS\\nnone specified\\n[105]\\n1898 (1913)\\nE to D\\nblade THU9\\nnone specified\\nDHS\\nnone specified\\n[50]\\n1906 (1921)\\nP to K frameshift\\n(with I2270L)\\nblade THU9\\nnone specified\\nNIFH\\nnone specified\\n[51,59]\\n1909 (1924)\\nR to E frameshift\\n(with\\nc.6165−7G>A)\\nblade THU9\\ndecreased protein\\nexpression\\nNIFH\\nlymphedema\\nperipheral oedema\\nhydrocele\\nchylothoraces\\n[106]\\n1910 (1925)\\nE to K\\n(with Q1591P and\\nR2110W)\\nblade THU9\\nnone specified\\nDHS\\nNIFH\\noedema [52]\\n[46,51]\\n1925 (1941)\\nR to W\\n(may be with E2322\\npremature\\ntermination)\\nblade THU9\\nnone specified\\nDHS\\nNIFH\\nskin oedema\\nbilateral hydrothorax\\nascites\\nintrauterine death at 21 weeks\\nthickened subcutaneous tissue\\nplacenta enlarged, thickened, with light and bright membranes, diffuse\\noedema of chorionic plate and villi\\natrophic epidermis, oedema, dilated vessels\\nrecurrent\\npaternal stomatocytosis (het.)\\n[46,107]\\n1943 (1959)\\nR to Q or W\\nblade THU9\\nincreased activity\\n(slowed inactivation)\\nDHS\\ninherited bone marrow failure\\nnone specified\\n[44,105]\\n1955 (1971)\\nR to C\\nblade THU9\\nnone specified\\nadenomatous polyposis\\nnone specified\\n[67,94]\\n(Continued.)\\n15\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 15}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\nmyelodysplastic syndrome\\n1978 (1994)\\nG to D\\n(with R2335Q)\\nblade THU9\\ndecreased mechanical\\nsensitivity\\nlymphedema\\nNIFH\\npolyhydramnios\\noedema of limbs (mainly feet), face and scrotum\\npleural effusions, chylothoraces\\nmaldescended testis left side\\nperihepatic ascites\\nno dysmorphic features\\n[80]\\n1988 (2004)\\nA to V\\nblade THU9\\nincreased activity\\n(slowed inactivation)\\nmalaria resistance\\nnone specified\\n[71]\\n1994 (2010)\\nS to F\\nblade THU9\\nnone specified\\nDHS\\nnone specified\\n[46]\\n2003 (2019)\\nA to D or N or T\\n(may be with\\nP2251S [108])\\nblade THU9\\nnone specified\\nDHS\\nfetal ascites [108]\\nin the fetal ascites case: pericardial effusion and pregnancy terminated\\npaternal history of haemolytic disorder and jaundice\\nplacenta apparently normal\\n[46,68,108]\\n2006 (2022)\\nV to I or D\\nblade THU9\\nnone specified\\nDHS\\nnegative association with HbA1c\\n(glycated haemoglobin A1c; >6.5%\\nindicates diabetes)\\nnone specified\\n[46,109]\\n2007 (2023)\\nM to L\\nblade THU9\\nnone specified\\nDHS\\nnone specified\\n[46,110]\\n2014 (2030)\\nT to I\\nblade THU9\\nnone specified\\nDHS\\nnone specified\\n[49,110]\\n2020 (2036)\\nA to T [97] or V [68]\\n(A to V het [68]).\\nblade THU9\\nincreased activity\\n(slowed inactivation)\\nA to T\\nDHS\\npseudohyperkalaemia\\nno perinatal oedema\\n[46,68,97]\\n2022 (2038)\\nY to H\\nblade THU9\\ndecreased activation\\nbicuspid aortic valve disease\\nnone specified\\n[37,38]\\n2023 (2039)\\nL to V\\nblade THU9\\nnone specified\\nDHS\\nnone specified\\n[111]\\n2029 (2045)\\nG to R\\n(with S1153\\npremature\\ntermination)\\nblade THU9\\ndecreased surface\\nexpression and N-\\nlinked glycosylation\\nprotein degradation\\nlymphatic dysplasia (GLD?)\\nhydrops and bilateral chylothorax\\nswelling of legs and scrotum lymphedema of legs, torso and face\\npleural effusions\\nintensive care including extracorporeal membrane oxygenation (one sibling)\\n[38,39,91]\\n(Continued.)\\n16\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 16}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n2069 (2085)\\nV to M\\nblade THU9\\nnone specified\\nbeta thalassemia\\nnone specified\\n[112]\\n2070 (2086)\\nK to Q\\n(with W1069\\npremature\\ntermination)\\nblade THU9\\nnone specified\\nGLD\\nNIFH\\nstillbirth\\nfacial and corporal oedema\\nmacroencephaly\\nhypoplastic lungs\\nfailure of lymphatic luminal canalization\\nhydroscopic chorionic villi\\nfetal vascular malperfusion\\n[87,88]\\n2088 (2104)\\nR to G\\nblade THU9\\nincreased activity\\n(reduced threshold\\nfor activation)\\ndecreased expression\\n(intracellular\\nlocalization)\\nDHS\\nnone specified\\n[105]\\n2110 (2126)\\nR to W or Q\\n(with Q1591P and\\nE1910K in NIFH)\\nanchor\\nincreased activity (R to\\nW)\\nDHS\\nHbA1c\\nNIFH\\noedema [52]\\n[46,50,52,113,114]\\n2127(2142)\\nT to M\\nanchor\\nincreased activity\\n(slowed inactivation)\\n[97]\\nDHS\\nHbA1c\\npseudohyperkalaemia\\n[46,68,97]\\n2151 (2167)\\nC to R\\nanchor\\nnone specified\\nDHS\\nnone specified\\n[46]\\n2157 (2172)\\nK to E frameshift\\n(c-het.)\\nanchor\\nnone specified\\nNIFH\\nNIFH\\nmedical termination of pregnancy\\nanteverted nares, dysplastic ears, micrognathia, lymphedema\\nplacental hypertrophy and oedema\\npleural effusion and chylothorax\\nskin oedema, hydramnios, polyhydramnios\\n[64]\\n2160 (2176)\\nP to L\\nanchor\\nnone specified\\nDHS\\nnone specified\\n[46]\\n2166 (2182)\\n2166 to 2169\\ndeletion\\nouter helix\\nnone specified\\nDHS\\nnone specified\\n[68]\\n(Continued.)\\n17\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 17}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n2169 (2185)\\nK deletion\\nouter helix\\nnone specified\\nDHS\\nhigher mean corpuscular haemoglobin concentration\\n[115]\\n2171 (2187)\\nV to F\\n(with donor splice\\nsite variant)\\nouter helix\\ndecreased expression\\nlymphedema\\nNIFH\\npleural effusions\\nascites\\ngastro-oesophageal reflux, intestinal lymphangiectasia, granuloma annulare\\nscoliosis\\noedema resolved rapidly and improved on low-fat diet\\n[70]\\n2192 (2208)\\nL to I\\nouter helix\\nnone specified\\nDHS\\nnone specified\\n[46]\\n2195 (2211)\\nS to L\\n(with G253R)\\ncap\\n(interface\\nwith\\nouter\\nhelix)\\ndecreased opening\\n(murine PIEZO1\\nmodel)\\nprune belly syndrome\\nabdominal wall laxity\\nflared ribs\\ndimpled knees\\ndilated bladder\\nbilateral hydroureter\\nsurgical urinary bladder drainage and other procedures\\nbilateral hydronephrosis\\nsupraventricular tachycardia\\n[43]\\n2201 (2217)\\nV to F\\ncap\\nnone specified\\nDHS\\nnone specified\\n[46]\\n2225 (2241)\\nM to R (het.)\\ncap\\nincreased activity\\n(slowed inactivation)\\nDHS\\nnone specified\\n[97,105,116,117]\\n2227 (2243)\\nA to V\\n(with R1547H)\\ncap\\nnone specified\\nNIFH\\nnone specified\\n[51]\\n2228 (2244)\\nQ premature\\ntermination\\n(with E755\\npremature\\ntermination or\\nV1458A\\nframeshift)\\ncap\\ndecreased expression\\nlymphedema (GLD?)\\nNIFH\\npleural effusions\\npolyhydramnios\\ndeath in utero at 34 weeks\\namyoplasia of the diaphragm\\nmild generalized oedema with pleural effusions and periorbital oedema\\nchylothorax/chylothoraces\\ncupped simple ears, epicanthic folds, micrognathia, atrial septal defect\\ngastro-oesophageal reflux\\n[70,118]\\n(Continued.)\\n18\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 18}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\nshort stature, pectus excavatum, genital oedema, splenomegaly\\n2245 (2261)\\nR or K (hom.)\\ncap\\nYoda1 responsive\\nEr antigen\\nGLD\\nNIFH, fetal growth restriction, postnatal demise\\n[102]\\n2251 (2267)\\nP to S\\n(with A2003D)\\ncap\\nnone specified\\nfetal ascites\\npericardial effusion\\npregnancy terminated\\npaternal history of haemolytic disorder and jaundice\\n[108]\\n2266 (2282)\\nK to I\\ncap\\nnone specified\\nDHS\\nnone specified\\n[50]\\n2270 (2286)\\nI to T\\n(hom. or with\\nE829V [80] or\\nwith P1906K\\nframeshift [59]\\ncap\\ndecreased mechanical\\nsensitivity\\nGLD\\nno NIFH\\nlate-onset bilateral oedema of lower limbs and scrotum\\nhydrocoeles, pleural effusions, chylothoraces, ascites\\ndeath due to chylothoraces and pleural disease\\nmild or major pericardial effusion (requiring percutaneous drainage),\\nprogressing to cardiac tamponade\\npericardial thickening\\nbreathlessness and reduced exercise tolerance\\nVVs with eczema\\nmild microcephaly, short stature, osteopenia\\n[59,80]\\n2277 (2293)\\nL to M\\ncap\\nnone specified\\nDHS\\nHbA1c\\nincreased frequency in South Asians compared with European ancestry\\n[46,113,119]\\n2302 (2318)\\nR to H\\n(with V499I or other\\nβ-thalassemia\\nvariant)\\nOr P\\ncap\\nincreased activity\\n(increased\\nmechanical\\nsensitivity, no\\nchange of\\ninactivation)\\ndecreased expression\\n(intracellular\\nlocalization)\\nDHS\\nNIFH\\nGLD\\noedema\\n[51,52,105]\\n2308 (2324)\\nQ to E\\ncap\\nnone specified\\nDHS\\nnone specified\\n[46]\\n(Continued.)\\n19\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 19}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n2322 (2338)\\nE premature\\ntermination\\n(with R1925W)\\ncap\\nnone specified\\nGLD\\nNIFH\\nskin oedema\\nbilateral hydrothorax\\nascites\\nintrauterine death at 21 weeks\\nthickened subcutaneous tissue\\nplacenta enlarged, thickened, with light and bright membranes, diffuse\\noedema of chorionic plate and villi\\natrophic epidermis, oedema, dilated vessels\\nrecurrent paternal stomatocytosis (het.)\\n[107]\\n2323 (2339)\\nK to T\\ncap\\nnone specified\\nDHS\\nnone specified\\n[49]\\n2335 (2351)\\nR to Q\\n(with G1978D)\\ncap\\ndecreased mechanical\\nsensitivity\\nlymphedema (GLD?)\\nNIFH\\npolyhydramnios\\noedema of limbs (mainly feet), face and scrotum\\npleural effusions, chylothoraces\\nmaldescended testis left side\\nperihepatic ascites\\nno dysmorphic features\\n[80]\\n2336 (2352)\\nR to W\\n(with V598M)\\ncap\\nnone specified\\nNIFH\\nnone specified\\nListed in [51,53]\\n2373 (2389)\\nQ premature\\ntermination\\ncap\\nnone specified\\nNIFH\\nmedical termination of pregnancy, hepatomegaly\\nplacental hypertrophy and delayed villous maturation\\nascite, pleural effusion and chylothorax\\n[64]\\n2390 (2406)\\nR to Q\\ncap\\nnone specified\\nalcoholic liver disease\\nnone specified\\n[40]\\n2392 (2408)\\nE to K\\n(with G2394S or\\npremature\\ntermination\\nY1763)\\ncap\\nYoda1 responsive\\nEr antigen\\nnone specified\\n[102]\\n(Continued.)\\n20\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 20}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n2394 (2411)\\nG to S (hom. or c-\\nhet. with\\nA2395V or\\nG2399S)\\ncap\\nYoda1 responsive\\nDHS\\nEr antigen\\nhistory of 1 stillborn infant\\n[50,90,102]\\n2395 (2412)\\nA to V\\n(with Q1114E)\\ncap\\nnone specified\\nGLD\\nNIFH\\nEr antigen [90]\\nListed in [51,53]\\n2399 (2416)\\nG to S (c-het. with\\nG2394S)\\ncap\\nnone specified\\nEr antigen\\nnone specified\\n[90]\\n2402 (2428)\\nE premature\\ntermination\\ncap\\nnone specified\\nVV disease\\nnone specified\\n[120]\\n2407 (2433)\\nE to Q or K (hom.)\\ncap\\nYoda1 responsive\\nEr antigen\\nNIFH and intrauterine death at 26 weeks gestation in 1 of 6 pregnancies\\n[102]\\n2430 (2456)\\nP to L\\n(with donor splice\\nsite variant)\\ncap\\nnone specified\\nlymphedema (GLD)\\nNIFH\\nbilateral pleural effusions (severe)\\ngeneralized oedema, oedema resolved in 1 day, epicanthic folds\\ncellulitis, spherocytes\\ndied at age 4 weeks\\n[70]\\n2433 (2459)\\nG to R\\ncap\\nnone specified\\nDHS\\nnone specified\\n[46]\\n2439 (2465)\\nG to E\\n(hom. or with\\nQ1125\\npremature\\ntermination)\\ncap\\nnone specified\\nlymphedema\\nvenous valve anomaly, venous varicosities\\n[121]\\n2456 (2482)\\nR to H\\ninner helix\\nincreased activity\\n(slowed inactivation)\\n[97,105]\\nDHS\\nmild macrocytic, chronic haemolytic anaemia, dessicytes\\n[46,50,68,97,98,105,116,11\\n7]\\n2456 (2482)\\nR to C or H\\n(with L939M and\\nF2458L)\\ninner helix\\nnone specified\\nNIFH\\noedema\\npleural effusions\\npolyhydramnios\\ngeneralized oedema at birth\\nchylothorax/chylothoraces\\ngastro-oesophageal reflux, hypothyroid at birth, occasional stomatocytes,\\nwebbed neck, periorbital oedema, prune belly\\n[70]\\n(Continued.)\\n21\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 21}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n2458 (2484)\\nF to L\\n(may be with\\nL939M and\\nR2456C)\\ninner helix\\nincreased opening\\n(mouse PIEZO1)\\nNIFH\\noedema\\nhip osteoarthritis requiring surgery\\nDHS\\npleural effusions\\nPolyhydramnios\\ngeneralized oedema at birth, chylothorax/chylothoraces, webbed neck,\\nperiorbital oedema, prune belly\\ngastro-oesophageal reflux, hypothyroid at birth\\noccasional stomatocytes\\n[46,54,62,70,122]\\n2461 (2487)\\nE to K\\ninner helix\\nnone specified\\nDHS\\nnone specified\\n[84]\\n2464 (2490)\\nH to P\\ninner helix\\nnone specified\\nDHS\\nnone specified\\n[46]\\n2474 (2500)\\nV to M\\nC-terminal\\ndomain\\nnone specified\\nDHS. HbA1c?\\nnone specified\\n[46,113]\\n2488 (2514)\\nR to Q or G (with\\nG718S)\\nC-terminal\\ndomain\\nincreased activity\\n(reduced threshold\\nfor activation)\\ndecreased expression\\n(intracellular\\nlocalization)\\nDHS\\nNIFH\\nNIFH\\n[46,49,68,105]\\n2489 (2515)\\nQ to D\\nC-terminal\\ndomain\\nnone specified\\nDHS\\nnone specified\\n[46]\\n2491 (2517)\\nR to W\\nC-terminal\\ndomain\\nnone specified\\nDHS\\nnone specified\\n[46]\\n2494 (2520)\\nE to V\\nC-terminal\\ndomain\\nnone specified\\nDHS\\nnone specified\\n[110]\\n2495 and 2496\\n(2521 and 2522)\\nduplication (with\\nV2474M)\\nC-terminal\\ndomain\\nnone specified\\nDHS\\nnone specified\\n[46,49,79]\\n2496 (2522)\\nE to ELE\\nC-terminal\\ndomain\\nincreased activity\\n(slowed inactivation)\\nDHS\\nNIFH\\nperinatal oedema\\npseudohyperkalaemia\\nhereditary high phosphatidylcholine haemolytic anaemia\\n[97,105,110,123]\\n2502 (2528)\\nK to R\\nC-terminal\\ndomain\\ndecreased activation\\necreased opening\\n(mouse PIEZO1)\\nbicuspid aortic valve disease\\nDHS\\nOsteoarthritis\\nnon-syndromic familial osteoarthritis free from acute or traumatic joint injury.\\nerosive hand osteoarthritis and interphalangeal joint osteoarthritis\\n[37,50,54,124]\\n(Continued.)\\n22\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 22}, page_content='Table 1. (Continued.)\\nPIEZO1 amino\\nacid number from\\nN-terminus\\nhuman (mouse)\\namino acid or\\nother change\\nPIEZO1\\nstructural\\nregion\\nlaboratory data for\\neffect on PIEZO1\\nprimary disease\\nantenatal and other clinical observations, interventions and\\noutcomes\\nreferences\\n2506 (2536)\\nL to F\\nC-terminal\\ndomain\\nnone specified\\ninherited bone marrow failure\\nnone specified\\n[44]\\n2510 (2536)\\nP to L\\nC-terminal\\ndomain\\ndecreased opening\\n(mouse PIEZO1)\\nDHS\\nosteoarthritis\\nnon-syndromic familial osteoarthritis free from acute or traumatic joint injury.\\nerosive hand osteoarthritis and interphalangeal joint osteoarthritis\\n[46,54]\\n2520 (2546)\\nK to E\\nC-terminal\\ndomain\\nnone specified\\nDHS\\nnone specified\\n[50]\\n23\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 23}, page_content='2.2. Diversity and potential importance beyond rare disease\\nPIEZO1 variant-associated GLD is phenotypically varied (table 1). There may be compartmental effusions of the pleural,\\nabdominal, perihepatic or pericardial spaces. Varying degrees of subcutaneous skin oedema occur, including facial, genital\\nand lower limb regions. Additional features can include chylothorax, webbed neck, prune belly and cranial findings such\\nas craniosynostosis and macrocephaly secondary to ventriculomegaly. Resultant clinical presentations can include respiratory\\ndistress, cardiac tamponade, cellulitis and psychiatric morbidity. In pregnancy, there may be placental hypertrophy, chorionic\\nplate oedema and polyhydramnios (table 1). The fetus may die or exhibit increased nuchal translucency, redundant nuchal\\nskin and NIFH. A systematic review of studies identified PIEZO1 as the most common monogenic cause of NIFH in exome\\nsequencing, with variants observed in up to 10% of cases [51]. Nineteen of 191 cases of NIFH were identified that could be\\nattributed to PIEZO1 variants.\\nPIEZO1 is expressed widely by lymphatic endothelial cells, and it is a mechanosensor involved in lymphatic expansion\\nthrough sprouting lymphangiogenesis and lymphatic valve development [129,130]. Activation of PIEZO1 induces junctional\\ngaps, enhancing lymphatic permeability and drainage, thereby having roles in both physiological lymphatic system formation\\nand function. These mechanisms represent how PIEZO1 variants leading to either LOP or LOF (figure 4a,b) may underpin\\nlymphatic maldevelopment or malfunction in a fetus affected by NIFH. The findings also suggest widespread importance of\\nPIEZO1 in lymphatics in general, and thus with implications beyond GLD and NIFH.\\nThe varying degrees of penetrance of the variants result in a range of phenotype severities, presenting either antenatally as\\nNIFH or later in adolescence or adulthood. This is further complicated as some variants are associated with both lymphedema\\nand a blood disorder (see table 1 and below). As such, where PIEZO1 variant GLD is identified antenatally, careful counselling\\nof the phenotypic heterogeneity is required and should be managed by a tertiary fetal medicine unit with genetic counselling.\\n3. Veins: varicose veins\\nVaricose veins (VVs) are a manifestation of chronic lower limb venous hypertension, leading to the incompetence of venous\\nvalves and distorted superficial veins [131]. Venous hypertension, with or without varicosities, due to venous valvular incom-\\npetence causes skin changes and predisposes to leg ulcers which adversely affect quality of life, require long-term specialist\\nnursing care and therefore represent a high societal burden. Chronic venous congestion and loss of valvular competence may\\nincrease the risk of deep vein thrombosis [131,132]. These problems run in families, suggesting a genetic component in addition\\nto environmental and lifestyle factors [131,132].\\n3.1. Significance identified by genome-wide association\\nGenome-wide investigation of data for 493\\u2009519 people in UK Biobank strongly suggested the association of PIEZO1 variants\\nwith VV disease [133]. Further analysis of UK Biobank data and self-reported VV disease in 408\\u2009969 individuals of the 23andMe\\n(www.23andme.com) database also associated PIEZO1 variants with VVs [41]. Analysis of whole exome sequence data in UK\\nFigure 4. PIEZO1 variant effects. Effects of variants that cause loss of PIEZO1 expression or function are depicted in red, and those that increase PIEZO1 function are\\ndepicted in blue. Non-variant wild-type (WT) is indicated for comparison. (a) Illustrated is the loss or reduction of PIEZO1 expression by fragmentation of PIEZO1\\n(LOP-1), intracellular trapping of PIEZO1 (LOP-2) or disrupted PIEZO1 gene transcription (or potentially increased transcript degradation). The WT surface membrane\\nPIEZO1 channel permits flux of cations such as Ca2+. (b) Force-response curves illustrating the increasing PIEZO1 channel activation with increasing mechanical force, up\\nto a limit (maximum). Some variants increase the mechanical sensitivity (shifting the curve to the left, which is a type of gain of function (GOF)), while other variants\\ndecrease the mechanical sensitivity (shifting the curve to the right, which is a type of loss of function (LOF)). (c) Slow inactivation and slow deactivation as alternative\\nmechanisms of GOF. Outside-out patch data are shown for WT and GOF variant channels overexpressed in HEK 293 cells. The patches were held at a constant voltage of\\n−80 mV. Two recordings are superimposed and normalized to the same maximum current to emphasize the kinetic differences. The pressure steps were to +75 mmHg\\nin both cases.\\n24\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 24}, page_content='Biobank suggested associations of predicted disruptive PIEZO1 variants with VV disease [134]. Analysis of data for 131\\u2009918\\nparticipants in the Geisinger MyCode Community Health Initiative associated VV disease and vein ablation procedures with\\nheterozygous PIEZO1 premature termination and missense variants [120]. A further study of data in UK Biobank and FinnGen\\nstrongly associated PIEZO1 variants with VVs [135] with stronger association in females [135]. Therefore, the findings suggest\\nthat PIEZO1 is important in lower limb venous structure or function and that PIEZO1 variants, even heterozygous single\\nvariants, disrupt the venous structure and its function.\\nSequence analysis algorithms have suggested that PIEZO1 variants associated with VVs are disruptive but there is relatively\\nlittle direct laboratory evidence for this. One variant generates a premature termination codon in the sequence encoding the cap\\nregion (table 1), which suggests that the ion pore region would be damaged, and this is consistent with a disruptive effect.\\n3.2. VVs and lymphedema\\nThere are relationships between VVs and lymphedema. In PIEZO1 variant-associated GLD, venous varicosities are reported\\n(table 1), and VVs have been associated with another type of lymphedema, called lymphedema distichiasis [131]. Moreover,\\nlymphedema can be caused by chronically increased pressure in the leg, although the mechanisms are not entirely clear.\\nHigher pressure leads to venous dilatation and valve incompetence and varicosities. This could reflect inabilities of the veins\\nto constrict and remodel in response to a higher pressure. Therefore, perhaps not surprisingly, higher venous pressure is\\nassociated with peripheral lymphedema, and higher rates of lymphedema have been detected in a VV cohort [120]. This type\\nof lymphedema, secondary to VVs, is important because once established, it can be difficult to resolve clinically, although VV\\nstripping may be helpful [136].\\nThe relationship between VVs and lymphedema is complex, but we suggest two effects. When the PIEZO1 disruption is\\nstrong, there may be early onset lymphedema and then late onset VVs that are both direct independent consequences of PIEZO1\\ndisruption. When the PIEZO1 disruption is partial, there may be only late-onset disease that is characterized by VVs and\\nlymphedema; in this case, the lymphedema could be both a direct consequence of the PIEZO1 disruption and the increased\\nvenous pressure. When there is partial PIEZO1 disruption and only late-onset disease, environmental and lifestyle factors are\\nlikely to be important contributors to the outcome.\\n4. Blood: dehydrated hereditary stomatocytosis\\nDehydrated hereditary stomatocytosis (DHS) is an autosomal dominant mild or moderately severe haemolytic anaemia in\\nwhich the RBC plasma membrane is leaky to cations such as K+ and hence the RBCs dehydrate and may adopt a mouth-like,\\nstomatocyte, shape [137,138].\\n4.1. PIEZO1 enhancement identified by familial genetics\\nVariation in chromosome 16q23-q24 was identified as a potential cause of DHS, leading to a focus on PIEZO1 and the associa-\\ntion of DHS in two families with heterozygous or homozygous PIEZO1 missense variants that encode M2225R in the PIEZO1\\ncap and R2456H in the inner helix. Further studies identified other DHS families and associated PIEZO1 variants [68,97] (table\\n1). In many cases, the variants are missense, changing a single amino acid in PIEZO1, but there are also amino acid deletions,\\ninsertions and duplications (table 1).\\nSome DHS-associated variants have been recapitulated in PIEZO1 cDNA clones overexpressed in cell lines for patch-clamp\\nrecording in the laboratory. The results of these studies suggest that a consequence of the variation is the slowing of the PIEZO1\\nchannel’s inactivation mechanism [97,105] (figure 4c). This is a gain of function (GOF) effect that results in more channel activity\\nand therefore more non-selective cation fluxes across the cell membrane. Part of this cation flux is K+ efflux but there is also Ca2+\\ninflux, which triggers the opening of KCa3.1 Ca2+-activated K+ channels encoded by the KCNN4 gene, further increasing the K+\\nefflux [138]. This suggests that cases of PIEZO1 variant-associated DHS arise through increased PIEZO1 channel-mediated ion\\nflux and downstream consequences of this through increased Ca2+ entry.\\nIn some laboratory studies, DHS-associated variants were found to have no effect on channel function, or they decreased\\nchannel expression or its surface localization (table 1). A few of the suggested effects were not replicated in other studies (table\\n1). For many variants, data are not available on whether channel expression or activity is affected. For some DHS variants\\nthat have been studied, no change in mechanical sensitivity was observed [97,105], suggesting that the changes in amino acid\\nresidues at the affected positions do not regulate the mechanical sensitivity of the channels. There are exceptions, however.\\nR2088G, R2302H and R2488Q reduced the threshold for mechanical activation (table 1). R2302H had no effect on inactivation.\\nR2488Q caused intracellular trapping of the PIEZO1 protein, which is a disruptive (LOP) effect. The consequence of GOF may\\nexceed that of trapping (LOP) effects, thus conferring an overall increase in PIEZO1 function. It has been suggested that the\\nseverity of DHS may depend on the location of affected amino acids in the PIEZO1 structure such that those in the ion pore\\nregion are associated with the most severe disease [139,140].\\nIt is challenging to understand how a PIEZO1 channel that often inactivates in less than a second (figure 2b) could be\\nrelevant to RBC physiology, which involves sustained mechanical stress and roles lasting many minutes, hours and days. As\\nindicated, however, the inactivation rate of PIEZO1 channels may change and be regulated. Indeed, patch-clamp studies of\\nphysiological RBCs from mice revealed that native PIEZO1 channel activity in RBCs is relatively slow and not characterized\\n25\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 25}, page_content='by fast inactivation [141]. When the M2225R variant was recapitulated in the native mouse Piezo1 gene, the RBC PIEZO1\\nchannels of these mice were more active than those of wild-type RBCs, consistent with a GOF mechanism [141], but, in the\\nabsence of inactivation, the effect of the variant was to reduce deactivation [141] (figure 2c). PIEZO1 channels interact with their\\nenvironment [8,18], and, as such, data obtained from PIEZO1 channels in overexpression systems do not necessarily indicate the\\nbehaviour of native PIEZO1 channels. There is relatively little information on the properties of native variant channels in situ in\\npatient RBCs, but these PIEZO1 channels may exhibit continuous (i.e. non-inactivating) behaviour [68].\\n4.2. Hyperferritinaemia\\nA clinical feature identified in some severe cases of European PIEZO1 variant-associated DHS was hyperferritinaemia [46,142].\\nIn this condition, there is excess ferritin (an intracellular iron storage protein) and hepatic iron overload that can have serious\\nadverse implications [142]. These effects may not be due to PIEZO1 in RBCs but rather PIEZO1 in tissue-resident macrophages\\nsuch as Kupffer cells of the liver that mediate phagocytosis, and thus clearance, of RBCs [25]. Intriguingly, the E756del GOF\\nvariant (table 1) is associated with iron overload in African Americans who self-report as healthy, suggesting that there are not\\nnecessarily obvious adverse implications of GOF [25]. The implications of hyperferritinaemia are made complex by sex, disease\\nand environmental dependencies and other comorbidities that may include VV disease [132].\\n4.3. Er antigens\\nExtensive investigation identified PIEZO1 variants associated with the rare and high-incidence Er antigen blood groups Era, Erb,\\nEr3, Er4 and Er5 [90,102]. Among the variants are four missense variants that alter amino acid residues in the cap structure, thus\\npresenting extracellular antigens [102]. Some of the variants overlap in associations with DHS and NIFH (table 1).\\n4.4. Haemoglobin A1c\\nHaemoglobin A1c (HbA1c), a glycated form of haemoglobin that increases in type 2 diabetes, is commonly used as a biomarker\\nof diabetes. It is associated with some of the PIEZO1 variants, particularly in people of South Asian origin [113,135]. This effect\\nmay occur because of altered RBC turnover, as has been noted for other haemoglobinopathies.\\n4.5. Other implications: adverse and beneficial\\nOther GOF implications, or potential implications are erythrocytosis, β-thalassemia, myelodysplastic syndrome (table 1) and\\nmalarial resistance [71]. Sickle cell trait, which may also affect RBC shape, associated with resistance to Plasmodium, the\\ncausative agent in malaria, prompting investigation of the relevance of DHS-associated PIEZO1 variants to malaria [71]. A\\nsearch was performed using Exome Aggregation Consortium data from several sequencing projects and two of the identified\\nvariants were found to increase PIEZO1 activity [71] (table 1). One of these variants (E756del) has a high (9–23%) allele\\nfrequency in people of recent African descent compared with <1% for other populations [71]. This suggests positive selection\\nin people most exposed to Plasmodium [25]. The idea is supported by results from people in Gabon [72], but not Ghana [73],\\nperhaps differing because of geographical effects [73].\\n4.6. Dehydrated hereditary stomatocytosis and lymphedema\\nSome variants associate with both GLD and DHS [143]. This situation is complicated by compound heterozygosity and lack of\\ninformation on the functional consequences of the variants in many cases (table 1). GOF may be a common theme here, but\\nfurther investigation of this matter would be beneficial to inform understanding.\\n5. Other disease and physiological traits\\n5.1. Gain of function\\nPeople with the E756del GOF variant were studied for non-RBC effects in a small-scale population study, and they were\\nfound to have normal blood pressure and normal body mass index [75]. The recapitulation of human GOF variants in mice\\nhas enabled more detailed studies though. The mice exhibited DHS and stomatocytes. As well as iron overload [25], there\\nwas mild cardiac hypertrophy and fibrosis [144] and reduced bile flow [40]. Potentially related is the observation that mice\\noverexpressing wild-type (non-variant) PIEZO1 in cardiac myocytes exhibit dilated cardiomyopathy [29]. Based on studies of\\nthe F2458L mutation recapitulated in overexpressed mouse PIEZO1, GOF may also associated with osteoarthritis [54] (table 1).\\nTherefore, the consequences of GOF variants may extend beyond RBCs and potentially confer long-term health risks.\\n5.2. Disruption: expectations from mouse studies\\nMultiple implications of PIEZO1 disruption are expected based on the results of gene disruption (inhibition) studies in mice that\\nsuggest functions of PIEZO1 across multiple systems including, for example, in bone formation [26,145], immunity [27], skeletal\\n26\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 26}, page_content='muscle function and repair [30,146,147], blood pressure regulation [148,149], adverse reactions of the heart to pressure overload\\nand ischemia [29,150,151] and memory [152]. The first and most striking phenotype of homozygous Piezo1 gene disruption\\nin mice is embryonic lethality, occurring at about embryonic day 10, apparently because of failed vascular maturation that\\nis normally driven by blood flow from the newly beating heart [14,153]. To study adult phenotypes in mice, therefore, cell\\ntype-restricted and both conditional and cell type-restricted Piezo1 disruptions have been implemented. The outcomes of such\\nstudies lie behind many of the proposals for physiological roles of PIEZO1. We cannot perform comparable genetic experiments\\nin people but we may in the future be able to do something similar if we develop PIEZO1-selective inhibitors [24], but, for now,\\nwe can look more closely at the phenotypes of people with disruptive PIEZO1 variants.\\nIntrauterine fetal demise has been associated with disruptive PIEZO1 variants and NIFH (table 1). This does not, however,\\nresemble the embryonic lethality in mice, which is seen as growth restriction and embryo readsorption [14]. Early embryonic\\nlethality may be difficult to detect in people if it occurs as unexplained miscarriage. However, whole exome sequencing of\\npregnancy tissue may reveal newly identified pathogenic variants in patients with normal chromosomal analysis, as has been\\ndemonstrated in a consanguineous couple with recurrent miscarriage, where a PIEZO1 mutation was detected [154]. What is\\nclear, is that there are some people alive who have, to the best of our knowledge, no PIEZO1 [70]. The severe PIEZO1 deficiency\\nin these people may be compensated in some way but, if it is, it is only partially successful compensation or cell type-specific\\nbecause these people usually suffer NIFH and GLD. In mouse studies, some PIEZO1-null embryos survive at least a week\\nlonger than other null embryos in utero [14], so there may also be mechanisms for compensation in mice. If large population\\nstudies were performed in mice, a few PIEZO1-null survivors may be found in this species too.\\n5.3. Other vascular associations\\nThere are many clinical features of GLD patients and other PIEZO1 variant carriers that may arise from lymphatic disruption\\n(table 1), or alternatively, they may arise from other physiological roles of PIEZO1 that are similar or comparable to those\\nseen in mice. Some of these features are vascular-related, such as maternal and fetal vascular malformations, venous valve\\nanomaly, double superior vena cava, agenesis of the ductus venosus, bilateral periorbital and conjunctival vascular changes with\\nsmall punctate haemorrhages, deep vein thrombosis, cardiomegaly and atrial septal defect (table 1). The familial association\\nof disruptive PIEZO1 variants with a case of multi-organ prune belly syndrome [43] is reminiscent of some aspects of prune\\nbelly seen in GLD (table 1). We know from studies of people who apparently do not have lymphedema, that there are potential\\nassociations with other putatively endothelial-related events such as cerebral cavernous and brain arteriovenous malformations\\nas well as bicuspid aortic valve disease (table 1).\\n5.4. Non-vascular associations\\nOther reported clinical features are bone abnormalities that include osteopenia and osteoarthritis (table 1). There are reports\\nof vasculitis and autoimmune connective tissue disease, metabolic dysfunction-associated fatty liver disease, alcohol-related\\nfatty liver disease, liver biliary pancreas abnormality, obesity in a Han Chinese population, inherited bone marrow failure,\\nangioimmunoblastic T-cell lymphoma, SARS-CoV-2 infection, adenomatous polyposis, antidepressant therapy outcome and\\nprotection from glaucoma (table 1). There are reports of hepatic dysfunction and jaundice, a suspected case of posterior urethral\\nvalve, elevated pleural triglycerides, obstructive sleep apnoea, primary immune deficiency and Asperger syndrome, and there\\nis quite common gastro-oesophageal reflux (table 1). Some of these associations bear resemblances to phenotypes seen in mouse\\ngenetic studies but we cannot be sure of the relationships. Detailed investigation of the associations is often difficult because\\nof the rarities and diversities of PIEZO1 variants, the polygenics of some of the associated complex diseases and the many\\npotential environmental factors.\\n5.5. A hypothesis for single variant heterozygous disruption\\nMore common than the PIEZO1 knockout or compound heterozygous disruption seen in GLD sufferers, is a single disruptive\\nheterozygous PIEZO1 variant in which there is partial disruption or deficiency of PIEZO1. In these people, who are apparently\\nhealthy, there may effectively be haploinsufficiency in which there is only about half the normal amount of PIEZO1. Examples can be\\nfound in table 1, but these may reflect only a tiny fraction of such variants because many more variants are seen in genetic databases\\nwithout knowledge of the consequences. In a few cases, we know that such variants are disruptive, yet the affected people do not have\\nNIFH or GLD. This is reminiscent of heterozygous PIEZO1 knockout mice that also appear healthy and grow similarly to wild-type\\nmice [14]. There has been relatively little study of the heterozygous mice and only at a young age, but disturbed endothelial cell\\nproperties were seen in their reduced alignment to the direction of blood flow and lesser phosphorylation of endothelial nitric oxide\\nsynthase [14]. Effects of this type are expected to increase the long-term risks of cardiovascular disease and potentially other diseases\\nsuch as vascular malfunction-related dementia. Is this also the situation in people? Might haploinsufficiency make people vulnerable\\nto environmental factors such as elevated blood cholesterol, which is a PIEZO1 inhibitor [155]?\\n5.6. Trait associations\\nTraits are distinguishing qualities or characteristics that typically belong to one person or a group of people. Many of them are\\nnot diseases but they may confer advantages or disadvantages, and they may confer an increased or decreased risk of disease\\n27\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 27}, page_content='depending on the context. PIEZO1 variants associated with physiological traits in people. Notable among them are the strong\\nassociations with standing and sitting heights [135], which are, for example, relevant to conditions such as VV, which is more\\ncommon in taller people [133,135]. There are associations with other physical factors such as forced expiratory volume [135],\\ntendon stiffness and jumping ability [76–78]. Associations are suggested with body mass index, obesity and dyslipidaemia\\n[40,113,156], which are consistent with observations in Piezo1 genetically modified mice [28,40] and relevant to lymphedema and\\nVVs [131,132,157].\\n6. Potential origins of restricted phenotypes\\n6.1. A model\\nWhy are the most prominent phenotypes those of lymphatics, lower limb veins and RBCs? Why are there cases of GOF\\nvariants with apparently no lymphatic effect, and cases of disruptive variants with apparently no RBC effect? Here, we outline\\na model of how such consequences might arise (figure 5). In compartment 1 of the model, there is a system for mechanosen-\\nsor production, regulation and degradation with a store of readily available but not necessarily used mechanosensors. This\\ncompartment serves compartment 2, the active site of mechanosensing, which is a component of compartment 3, which is where\\nphysiological effects arise. Compartment 1 contains PIEZO1 (gene) and PIEZO1 (protein) in different stages of readiness as well\\nas alternative mechanosensors Mechano2 and Mechano3 that might comprise PIEZO2, for example. These elements exchange\\nbetween compartments 1 and 2 according to need.\\n6.2. Analogy\\nCompartment 1 can be compared with a factory and a warehouse for anemometers, which detect wind speed. It has systems\\nfor the exchange of products with compartment 2, which is an element of compartment 3, a power station that measures\\nwind speed in a wind turbine system. In this analogy, compartment 1 contains anemometer type 1 in various stages of\\nproduction and assembly as well as other devices used to measure wind speed, should anemometer type 1 not materialize.\\nThere is communication between the compartments that maximises efficiency and tries to ensure suitable outcomes such as the\\nproduction of the required amount of wind-driven electrical power. Inadvertent alteration in an anemometer’s sensitivity to\\nairflow velocity through a manufacturing error may be detected if there is a system in place to detect it (e.g. the power station\\nmay have four anemometers that are expected to give the same readings within a defined tolerance, so one reading out of range\\nmay lead to replacement of the presumed faulty anemometer).\\n6.3. Gain of function in the model\\nWith PIEZO1 GOF variants, some of the mechanosensors in compartment 2 may overreact to the mechanical forces experienced\\nby the cells. Compartment 1 may be able to compensate for this problem by producing less PIEZO1. If a mechanism for\\nFigure 5. Diverse phenotype model. In the model, there is compartment 1 for regulatory and reserve systems that try to ensure compartment 2 is supplied with\\nsuitable mechanical sensors for physiological force sensing. Compartment 1 contains: PIEZO1 (gene) regulation, which may occur, for example, via microRNAs and\\ntranscriptional enhancers and silencers; mRNA encoding PIEZO1, the abundance of which may be modulated and which is available for translation and thus production\\nof new PIEZO1 (protein); PIEZO1 in intracellular membranes (PIEZO1int), which may translocate to the plasma membrane (PM); PIEZO1 at the PM in closed states\\nC1 or C2 due to mechanical insensitivity or inactivation, or be degraded (PIEZO1deg); and other mechanical force sensors (Mechano2 and Mechano3), an example of\\nwhich may be PIEZO2. Compartment 2 comprises PIEZO1 at the PM that is available for opening (O) in response to mechanical stimuli (PIEZO1PM-O). Consequences are\\ndivided into physiology and pathology effects occurring via compartment 3. Physiology effects, occurring via the normal biochemical signalling pathways of cells and\\ntissues, do not necessarily increase if PIEZO1 increases. Pathology, possibly involving non-physiological pathways, may be triggered by strong depletion or mild excess\\nof PIEZO1, depending on the cell type.\\n28\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 28}, page_content='producing less PIEZO1 is transcriptional, cell types without nuclei (e.g. RBCs) will be less able to compensate, and so a\\nphenotype will arise that is dominated by the effect on RBCs, and the most obvious phenotype will be anaemia.\\n6.4. Loss of expression or function in the model\\nWith homozygous or compound heterozygous disruptive (LOP and LOF) variants, PIEZO1s may not be able to provide\\nmechanosensing. Compartment 1 may try to compensate by delivering an alternative mechanosensor (e.g. PIEZO2). In some\\ncell types (e.g. lymphatic endothelial cells), the alternative may not be sensitive enough to detect the required mechanical forces\\n(e.g. the low flow and low pressure of lymph), in which case, the most obvious phenotype will be lymphedema. In other cell\\ntypes, the substitute mechanosensors may be sufficient, or insufficient only in certain conditions (e.g. when there is a cofactor or\\ncomorbidity).\\n6.5. Partner proteins\\nAnother control on the contribution of PIEZO1 is its assembly with other proteins. These proteins include but are not limited to\\nthe cell adhesion molecules CDH1, CDH5, PECAM1 and CADM1 [20,158,159], and the MyoD family inhibitor domain-contain-\\ning protein MDFIC [19]. The expression and importance of these proteins vary in different cell types, and so we can speculate\\nthat PIEZO1 variants may affect the interactions of some of these proteins with PIEZO1, thereby creating cell type-specific\\neffects. Variants in the genes that encode the interacting proteins may likewise affect PIEZO1 expression or activity. A MDFIC\\nvariant has been associated with lymphedema [160]. PECAM1 variants have been associated with lymphedema [161] and\\nCADM1 variants have been associated with endothelial dysfunction and venous thrombosis [162,163]. It is unknown if these\\nassociations are related to interactions with PIEZO1.\\n7. Understanding the effects of variants on PIEZO1 channel properties and PIEZO1 expression\\nVariants that, for example, insert a premature termination codon and thus truncate the PIEZO1 protein are relatively easy\\nto understand but others, such as missense variants that affect the amino acid sequence of the channel and thus potentially\\nthe channel properties are more difficult to understand and are often hard to interpret clinically. Are the channel properties\\naffected? Is channel activity increased or decreased?\\n7.1. Patch-clamp electrophysiology and intracellular Ca2+ assays\\nIdeally, we would measure PIEZO1 channel activity non-invasively in specific systems of people such as the lymphatics, but this\\nis not currently possible. Alternatively, we can isolate cells from a patient’s blood or tissue sample and then measure PIEZO1\\nactivity in these cells by patch-clamp, applying a mechanical stimulus to activate the channels. This may be possible but is often\\nchallenging and laborious. Even for the relatively accessible RBC, there are few data of this type reported [68]. The development\\nof automated planar patch-clamp methods that incorporate a mechanical stimulus, including for RBCs, has the potential to\\novercome some of these limitations and transform the evaluation of patient cells [102,164–166]. Intracellular Ca2+ assays can\\nalternatively be used to detect Ca2+ influx through PIEZO1 channels, but even when mechanical stimuli are implemented in\\nsuch assays, the information quality for the channel properties is inferior to that from patch-clamp assays.\\nTo avoid requiring a patient sample, it is possible to recreate the variant in a tractable cell line by gene editing, thereby\\ngenerating a model system that can be studied by patch-clamp. Substantial investment of funds and researcher time is, however,\\nrequired for every variant. Even if the cell line is successfully made and validated, the subsequent patch-clamp recordings from\\nthe native channels can be difficult because of the small and variable nature of the PIEZO1 channel signals and the presence of\\nother ion channels in the same cells that may complicate the data interpretation.\\nAt the present time, the almost exclusively used method for determining the effects of variants on channel properties is the\\ngeneration of the variant artificially in a cDNA clone by site-directed mutagenesis, followed by its overexpression in a host\\ncell line for subsequent patch-clamp recording. This method is also challenging, and the ionic currents observed can be highly\\nvariable. It is quite common to use the mouse PIEZO1 channel as a surrogate of the human channel because it can be more\\nsuitable for robust recordings, and it is better understood and more widely studied. Despite the challenges and limitations, this\\nmethod generates useful data for important channel parameters such as the mechanical sensitivity.\\n7.2. Amino acid variations that affect channel activity\\nThe overexpression approach described above, whether using the human channel or a mouse surrogate, lies behind most\\navailable data for the effects of variants on PIEZO1 channel function. As detailed in table 1, we know from these data of variants\\naltering residues in THU2 and THU3 that decrease channel activity (S217L, G253R, L322P and R531C), a variant in THU4 that\\nincreases channel activity (V598M), variants in THU5 that increase (756Edel or G782S) or decrease (E829V or L939M) channel\\nactivity or expression, a variant in the beam that increases channel activity (R1358P), a variant in the lateral plug that decreases\\nchannel activity (R1404W), seven variants in the THU9 that increase (R1943Q, A1988V, A2020T, R2088G) or decrease (1877Kdel,\\n29\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 29}, page_content='G1978D, Y2022H) channel activity, variants in the anchor that increase channel activity (R2110W and T2127M), a variant in the\\nouter helix that decreases channel activity (S2195L), variants in the cap that increase (M2225R and R2302H) or decrease (I2270T\\nand R2335Q) channel activity, variants in the inner helix that increase channel activity (R2456H, F2458L and R2488Q) and\\nvariants in the CTD that increase (2496E-ELE) or decrease (K2502R and P2510L) channel activity. Mapping of these variations to\\na model of the channel suggests vulnerability of the central channel region including the cap, but not a restriction to this region\\n(figure 3). There is no obvious pattern to the sites of these LOF and GOF variants, and there are opposing effects seen from\\nvariants in proximity.\\nThe origins of PIEZO1 missense variant-associated disease are therefore complex and mostly poorly understood at the\\nmolecular level. Unless we have the patch-clamp data, we are currently unable to predict whether missense variants would\\nincrease or decrease channel activity, or indeed simply have no effect, as has been found for some variants (table 1). We are left,\\ntherefore, with needing to experimentally test the effect of each new variant unless we can devise new methods that might more\\nquickly determine the effects or accurately predict them. We are also uncertain whether the overexpression data truly reflects\\nwhat happens in the human body.\\n7.3. Protein structural biology\\nStructural studies of the channel might provide a way forward because a protein’s structure determines its function. Structural\\nunderstanding of PIEZO1 channel function is still in its infancy, but there is progress, and it has the potential for determining\\neffects of variants. High-resolution structural data have been obtained for the E756del and A1988V GOF channels in complex\\nwith MDFIC and, albeit slightly less well resolved, for the R2456H GOF channel [140]. While the E756del and A1988V channels\\nhad a similar dome to the non-variant wild-type channel, the R2456H channel was more curved, with more contracted blades,\\nalthough the inner helix was twisted, resulting in a more dilated pore [140]. Lipid density was seen in the pore region of the\\nwild-type channel [140], as had been seen previously in structural data and molecular dynamics simulations for the mouse\\nPIEZO1 channel [23,167]. The lipids have been suggested to seal the pore, inhibiting or preventing the flux of ions [23,140,167].\\nThe E756del and A1988V channel pores also contained lipid density, but the acyl chains were reoriented, which could explain\\nthe GOF [140]. There was lipid head group close to R2456, and so this may be critical for the lipid interaction in the pore [140].\\nThese structural data suggest that GOF variants can indeed act by altering the structural configuration of the channel.\\n7.4. Synonymous and intronic variants\\nAt present, we can only speculate about why synonymous and intronic variants are associated with human traits and disease.\\nThe most obvious hypothesis is that they act by altering the expression of the PIEZO1 gene. We have little or no evidence,\\nhowever, and little or no information about the mechanisms that normally regulate the expression of PIEZO1, or the murine\\ngene, Piezo1. The associations of these variants can give us ideas about regions of the gene that are important in its regulation by\\ntranscriptional enhancers and repressors, and non-coding RNAs such as microRNAs.\\n8. Potential for patient benefit\\nThe genetic variant data discussed here, suggest important roles of PIEZO1 in human disease, and so we should consider the\\npossibilities for patient benefits arising from this new knowledge. GLD, NIFH and DHS are prominent in the reports published\\nso far (figure 6). Even in such rare diseases, understanding and manipulating PIEZO1 could generate considerable benefits for\\nconditions that currently have limited treatment options, but, it is our hypothesis, that the benefits of targeting PIEZO1 will\\nextend beyond these conditions and may include conditions that involve PIEZO1 but are not necessarily caused or exacerbated\\nby PIEZO1 variation such as secondary lymphedema arising from factors such as obesity, inflammation, fibrosis and physical\\ninjury [157]. Secondary lymphedema is a major problem that affects 250\\u2009000 people in the UK [168] and about 230 million\\npeople worldwide (World Health Organization data). Improving lymphatic function by stimulating PIEZO1 could therefore\\nhave widespread implications. Data from human and mouse studies strongly support the conclusion that PIEZO1 is important\\nin lymphatic function and that beneficial lymphatic effects can be achieved with a PIEZO1 agonist [24,70,129,130,157,169–171].\\nThere may be opportunities for resolving other seemingly unrelated clinical problems, among which we suggest may be\\ntraumatic brain injury. Head injury causes significant clinical burden and is the most common cause of injury and disability\\nin people aged 1−40 years in the UK. Each year in England and Wales, approximately 1.4 million people attend emergency\\ndepartments with head injuries. About 40\\u2009000 of these patients show evidence of traumatic brain injury. Many suffer long-term\\ndisability and around 0.2% die of their injury. Long-term complications can range from neurological deficits and seizures to\\nendocrine conditions, and mood and behavioural disorders (NICE). Meningeal lymphatic dysfunction exacerbates traumatic\\nbrain injury pathogenesis [172], and it has been proposed that PIEZO1 agonists, acting to improve the function of meningeal\\nlymphatics, could reduce excess cerebrospinal fluid and ventricular enlargement [169].\\nOther conditions such as VVs are even more prevalent. While less high profile, there is importance at a societal level due\\nto chronic disability, adverse quality of life and healthcare costs. Preclinical investigation has supported the idea of targeting\\nPIEZO1 in the treatment of VVs [173]. Since the affected veins are often superficial, topical application of a PIEZO1 modulator\\nmight be sufficient, and limit potential safety concerns of a systemic application.\\nThere are also potential clinical implications for the RBC-related variants, for example, in anaemia and malarial resistance,\\nand for the use of Er antigen identification in reducing acute transfusion reactions.\\n30\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 30}, page_content='8.1. Progress with pharmacology\\nA review of PIEZO1 pharmacology can be found elsewhere [24] but we note here the encouraging signs that PIEZO1-selective\\npharmacology is possible, particularly in the case of PIEZO1 agonists [24,45,80]. Other modulators to consider are dietary\\nlipids such as docosahexaenoic acid, which may slow PIEZO1 inactivation and therefore have a GOF, agonist-like, effect\\n[174]; although it may also have a separate inhibitory effect [175]. Mechanical stimulation of the channels by ultrasound [176]\\nmight also be considered. Some microRNAs may create agonist-like effects by stimulating PIEZO1 expression [177]. PIEZO1\\npharmacology and other forms of modulation are, however, in their infancy, and only experimental at this stage. The approach\\nis currently unproven with unknown safety implications in the clinical setting.\\n8.2. Progress with computer predictions\\nAs discussed earlier, laboratory methods for determining the effects of variants on PIEZO1 channel expression and function\\nare laborious and potentially financially expensive, contributing to limited progress on the effects of the many variants (table\\n1). Computer algorithms can generate quick interpretations, but they currently depend on predictions from the DNA sequence,\\nwhich may have relatively little value in clinical diagnosis when predicting variants of uncertain significance (VUS). While\\nproteome-wide missense prediction algorithms have been developed, they infer only the presence of likely pathogenicity [178].\\nA potentially more informative computational approach is the development and validation of molecular dynamics simulations,\\nas are emerging for the human PIEZO1 channel in relevant membranes such as those of endothelium and RBCs [155,167].\\nThese simulations are based on lipidomic data for membranes and static structural data from laboratory protein structural\\nbiology studies of the channel, the capabilities for which are rapidly improving. There is the prospect that combining such data\\nwith molecular dynamics simulation approaches could ultimately enable the prediction of the effects of variants on channel\\nproperties and thereby transform opportunities for clinical diagnosis.\\nFigure 6. Reported frequencies of PIEZO1 variant-associated phenotypes: a word cloud for PIEZO1 variant-associated disease and clinical terms from the ‘primary\\ndisease’ and ‘antenatal and other clinical observations, interventions and outcomes’ columns of table 1. The size of a word was determined by the frequency of its\\noccurrence in the table. The colours and fonts of the word groups are random. Some of the phenotypes were manually harmonized by grouping synonymous terms to\\nensure consistent terminology and accurate weighting (e.g. ‘varicose veins’ and ‘veins varicosities’ were both counted as ‘varicose veins’).\\n31\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 31}, page_content='9. Conclusions and discussion\\nFigure 7 summarizes key conclusions from this review. One conclusion is that the strongest physiological role of PIEZO1 in\\nhumans seems to be in the lymphatics, which is based on the observation that lymphedema is the most striking phenotype\\nassociated with disruptive PIEZO1 variants and potentially the most major or even singular disease characteristic in some\\npeople who carry such variants. There are almost certainly other roles of PIEZO1, but lymphatic function, with its implications\\nfor all major organs, may have overlooked influence on multiple aspects of physiology. Another major physiological role that\\nemerged from these studies is a role of PIEZO1 in the structural resilience of superficial leg veins. This role may have little or\\nno impact in early life, but it is of considerable interest for societies with large ageing multi-morbid populations. The roles of\\nPIEZO1 in lymphatics and leg veins probably both reflect its significance in endothelial cells [14,135] and in turn the roles these\\ncells have in vascular permeability and the control of unidirectional fluid flow mediated by valves.\\nUnlike the essential lymphatic and venous roles, PIEZO1 seems to be mostly non-essential in RBC physiology. Anaemia is\\noften not a feature of PIEZO1 variant-associated NIFH or GLD in which we know PIEZO1 is disrupted. RBCs may not depend\\nas much as lymphatics on PIEZO1 because of their multiple diverse mechanisms for membrane cation permeability [179].\\nThe excess cation permeability inadvertently conferred by PIEZO1 GOF variants nevertheless may disturb RBC physiology.\\nThe significance of the disturbance for health varies substantially from the relatively benign and even beneficial, to the quite\\nseriously adverse. The variability may be explained by varying genetic backgrounds that may in some instances make people\\nvulnerable to PIEZO1 GOF, such as variations in other genes, for example, KCNN4.\\nDespite the broad expression of PIEZO1 and the multiple phenotypes of PIEZO1-disrupted mice, PIEZO1 also seems to\\nbe non-essential in other human physiology. But this does not mean that PIEZO1 variation is without other consequences, or\\nthat PIEZO1 modulators (e.g. PIEZO1 agonists) would be without adverse effects if administered to people. As we discuss\\nin this article, there are already many potential additional associations of PIEZO1 variation. We expect that as studies on this\\ntopic continue, additional importance in humans will be recognized. In future studies, we suggest the importance of focussing\\nstudies on age-onset cardiovascular and musculoskeletal diseases. Tissue stiffness and other mechanical strains increase with\\nage, and we know that PIEZO1’s role in iron homeostasis is age-related [25]. Iron overload could have broad implications for\\ndiseases that are more common in older people such as VVs [132], and arthritis, liver damage and heart failure [25]. We should\\nalso consider the importance of lipid homeostasis in PIEZO1 variant-associated phenotype because of the impacts of lipids on\\nPIEZO1 channel function [18,174] and the roles of PIEZO1 in whole body lipid homeostasis [40]. We should also consider the\\nFigure 7. Primary conclusions. Left (pale yellow colour): roles of PIEZO1 inferred from PIEZO1 variant associations. Top right (pale peach colour): suggestions for future\\nresearch to address aspects of this biology for which we have particularly limited understanding. Bottom right (pale green colour): promising areas of research for\\ndelivering patient benefits based on targeting and further understanding of PIEZO1.\\n32\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 32}, page_content='health implications of PIEZO1 haploinsufficiency caused by disruption to one of the PIEZO1 alleles because of the vulnerability\\nit may confer to disease through a reduced reserve of PIEZO1 channels.\\nIn summary, we conclude that PIEZO1 is important in human health and disease, and that studies of the effects of PIEZO1\\nvariants have clinical relevance that is only just starting to be broadly realized and which can in parallel advance understanding\\nof how mechanical force sensing works in humans and eukaryotic biology as a whole.\\nEthics. This work did not require ethical approval from a human subject or animal welfare committee.\\nData accessibility. This article has no additional data.\\nDeclaration of AI use. We have not used AI-assisted technologies in creating this article.\\nAuthors’ contributions. C.W.C.: data curation, formal analysis, methodology, writing—review and editing; S.L.E.: validation, writing—review and\\nediting; O.V.P.: data curation, formal analysis, visualization, writing—review and editing; C.R.: data curation, formal analysis, writing—review\\nand editing; K.A.S.: formal analysis, visualization, writing—review and editing; M.D.: formal analysis, visualization, writing—review and\\nediting; F.L.M.: validation, writing—review and editing; D.G.B.: validation, writing—review and editing; A.M.: writing—review and editing;\\nF.B.: writing—review and editing; E.C.-B.: writing—review and editing; R.M.C.: writing—review and editing; K.J.G.: validation, writing—\\nreview and editing; M.A.B.: validation, writing—review and editing; A.C.K.: writing—review and editing; L.C.M.: writing—review and editing;\\nK.K.W.: validation, writing—review and editing; D.J.B.: conceptualization, data curation, investigation, project administration, resources,\\nsupervision, visualization, writing—original draft.\\nAll authors gave final approval for publication and agreed to be held accountable for the work performed therein.\\nConflict of interest declaration. We declare we have no competing interests.\\nFunding. C.W.C., M.D., F.B., E.C.-B. and K.J.G. were supported by British Heart Foundation (BHF) Mautner fellowships. L.C.M., S.L.E. and D.J.B.\\nwere supported in part by the Tommy’s Preterm Birth Centre. L.C.M. was supported by a NIHR Academic Clinical Lectureship. O.V.P. was\\nsupported by BHF grant PG/21/10515. C.R., K.A.S. and A.M. were supported by BHF PhD studentships FS/PhD/22/29312, FS/4yPhD/P/20/34012\\nand FS/4yPhD/F/22/34182. F.L.M. was supported by a Sir Henry Wellcome career development fellowship 215861/Z/19/Z. M.A.B. was supported\\nby a BHF career development fellowship. R.M.C. and D.J.B. were supported in part by the National Institute for Health and Care Research\\n(NIHR) Leeds Biomedical Research Centre (BRC) NIHR203331. The views expressed are those of the author(s) and not necessarily those of\\nthe NHS, the NIHR or the Department of Health and Social Care. A.C.K. and D.J.B. were supported in part by UKRI BBSRC research grant\\nAPP22657. Some of the background research was supported by BHF RG/17/11/33042 and Wellcome 110044/Z/15/Z grants to D.J.B.\\nReferences\\n1.\\nFritzsche M. 2020 Thinking multi-scale to advance mechanobiology. Commun. Biol. 3, 469. (doi:10.1038/s42003-020-01197-5)\\n2.\\nCampàs O, Noordstra I, Yap AS. 2024 Adherens junctions as molecular regulators of emergent tissue mechanics. Nat. Rev. Mol. Cell Biol. 25, 252–269. (doi:10.1038/s41580-023-\\n00688-7)\\n3.\\nDouguet D, Honoré E. 2019 Mammalian mechanoelectrical transduction: structure and function of force-gated ion channels. Cell 179, 340–354. (doi:10.1016/j.cell.2019.08.049)\\n4.\\nTranter JD, Kumar A, Nair VK, Sah R. 2023 Mechanosensing in metabolism. Compr. Physiol. 14, 5269–5290. (doi:10.1002/cphy.c230005)\\n5.\\nMammadova-Bach E, Gudermann T, Braun A. 2023 Platelet mechanotransduction: regulatory cross talk between mechanosensitive receptors and calcium channels. Arterioscler.\\nThromb. Vasc. Biol. 43, 1339–1348. (doi:10.1161/atvbaha.123.318341)\\n6.\\nPhuyal S, Romani P, Dupont S, Farhan H. 2023 Mechanobiology of organelles: illuminating their roles in mechanosensing and mechanotransduction. Trends Cell Biol. 33, 1049–\\n1061. (doi:10.1016/j.tcb.2023.05.001)\\n7.\\nGoodman MB, Haswell ES, Vásquez V. 2023 Mechanosensitive membrane proteins: usual and unusual suspects in mediating mechanotransduction. J. Gen. Physiol. 155,\\ne202213248. (doi:10.1085/jgp.202213248)\\n8.\\nJiang Y, Yang X, Jiang J, Xiao B. 2021 Structural designs and mechanogating mechanisms of the mechanosensitive Piezo channels. Trends Biochem. Sci. 46, 472–488. (doi:10.1016/\\nj.tibs.2021.01.008)\\n9.\\nCoste B, Mathur J, Schmidt M, Earley TJ, Ranade S, Petrus MJ, Dubin AE, Patapoutian A. 2010 Piezo1 and Piezo2 are essential components of distinct mechanically activated cation\\nchannels. Science 330, 55–60. (doi:10.1126/science.1193270)\\n10.\\nMurthy SE, Dubin AE, Patapoutian A. 2017 Piezos thrive under pressure: mechanically activated ion channels in health and disease. Nat. Rev. Mol. Cell Biol. 18, 771–783. (doi:10.\\n1038/nrm.2017.92)\\n11.\\nSrivastava N, Traynor D, Piel M, Kabla AJ, Kay RR. 2020 Pressure sensing through Piezo channels controls whether cells migrate with blebs or pseudopods. Proc. Natl Acad. Sci. USA\\n117, 2506–2512. (doi:10.1073/pnas.1905730117)\\n12.\\nMousavi SAR et al. 2021 PIEZO ion channel is required for root mechanotransduction in Arabidopsis thaliana. Proc. Natl Acad. Sci. USA 118, e2102188118. (doi:10.1073/pnas.\\n2102188118)\\n13.\\nWu J, Lewis AH, Grandl J. 2017 Touch, tension, and transduction – the function and regulation of Piezo ion channels. Trends Biochem. Sci. 42, 57–71. (doi:10.1016/j.tibs.2016.09.\\n004)\\n14.\\nLi J et\\xa0al. 2014 Piezo1 integration of vascular architecture with physiological force. Nature 515, 279–282. (doi:10.1038/nature13701)\\n15.\\nSegel M et\\xa0al. 2019 Niche stiffness underlies the ageing of central nervous system progenitor cells. Nature 573, 130–134. (doi:10.1038/s41586-019-1484-9)\\n16.\\nClapham DE. 2007 Calcium signaling. Cell 131, 1047–1058. (doi:10.1016/j.cell.2007.11.028)\\n17.\\nBerridge MJ, Bootman MD, Roderick HL. 2003 Calcium signalling: dynamics, homeostasis and remodelling. Nat. Rev. Mol. Cell Biol. 4, 517–529. (doi:10.1038/nrm1155)\\n18.\\nShi J et\\xa0al. 2020 Sphingomyelinase disables inactivation in endogenous PIEZO1 channels. Cell Rep. 33, 108225. (doi:10.1016/j.celrep.2020.108225)\\n19.\\nZhou Z et\\xa0al. 2023 MyoD-family inhibitor proteins act as auxiliary subunits of Piezo channels. Science 381, 799–804. (doi:10.1126/science.adh8190)\\n20.\\nKoster AK, Yarishkin O, Dubin AE, Kefauver JM, Pak RA, Cravatt BF, Patapoutian A. 2024 Chemical mapping of the surface interactome of PIEZO1 identifies CADM1 as a modulator of\\nchannel inactivation. Proc. Natl Acad. Sci. USA 121, e2415934121. (doi:10.1073/pnas.2415934121)\\n21.\\nZhao Q et\\xa0al. 2018 Structure and mechanogating mechanism of the Piezo1 channel. Nature 554, 487–492. (doi:10.1038/nature25743)\\n22.\\nGuo YR, MacKinnon R. 2017 Structure-based membrane dome mechanism for Piezo mechanosensitivity. eLife 6, 33660. (doi:10.7554/elife.33660)\\n23.\\nYang X, Lin C, Chen X, Li S, Li X, Xiao B. 2022 Structure deformation and curvature sensing of PIEZO1 in lipid membranes. Nature 604, 377–383. (doi:10.1038/s41586-022-04574-8)\\n33\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 33}, page_content='24.\\nKinsella JA, Debant M, Parsonage G, Morley LC, Bajarwan M, Revill C, Foster R, Beech DJ. 2024 Pharmacology of PIEZO1 channels. Br. J. Pharmacol. 181, 4714–4732. (doi:10.1111/\\nbph.17351)\\n25.\\nMa S, Dubin AE, Zhang Y, Mousavi SAR, Wang Y, Coombs AM, Loud M, Andolfo I, Patapoutian A. 2021 A role of PIEZO1 in iron metabolism in mice and humans. Cell 184, 969–982.\\n(doi:10.1016/j.cell.2021.01.024)\\n26.\\nSun W et\\xa0al. 2019 The mechanosensitive Piezo1 channel is required for bone formation. eLife 8, 47454. (doi:10.7554/elife.47454)\\n27.\\nSolis AG et\\xa0al. 2019 Mechanosensation of cyclical force by PIEZO1 is essential for innate immunity. Nature 573, 69–74. (doi:10.1038/s41586-019-1485-8)\\n28.\\nWang S et al. 2020 Adipocyte Piezo1 mediates obesogenic adipogenesis through the FGF1/FGFR1 signaling pathway in mice. Nat. Commun. 11, 2303. (doi:10.1038/s41467-020-\\n16026-w)\\n29.\\nJiang F, Yin K, Wu K, Zhang M, Wang S, Cheng H, Zhou Z, Xiao B. 2021 The mechanosensitive Piezo1 channel mediates heart mechano-chemo transduction. Nat. Commun. 12, 869.\\n(doi:10.1038/s41467-021-21178-4)\\n30.\\nMa N et al. 2022 Piezo1 regulates the regenerative capacity of skeletal muscles via orchestration of stem cell morphological states. Sci. Adv. 8, eabn0485. (doi:10.1126/sciadv.\\nabn0485)\\n31.\\nQi Q et al. 2020 Simultaneous detection of CNVs and SNVs improves the diagnostic yield of fetuses with ultrasound anomalies and normal karyotypes. Genes 11, 1397. (doi:10.\\n3390/genes11121397)\\n32.\\nCorrea ARE, Naini K, Mishra P, Dadhwal V, Agarwal R, Shukla R, Kabra M, Gupta N. 2021 Utility of fetal whole exome sequencing in the etiological evaluation and outcome of\\nnonimmune hydrops fetalis. Prenat. Diagn. 41, 1414–1424. (doi:10.1002/pd.6022)\\n33.\\nMaciak K, Adamowicz-Salach A, Siwicka A, Poznanski J, Urasinski T, Plochocka D, Gora M, Burzynska B. 2020 Hereditary xerocytosis - spectrum and clinical manifestations of\\nvariants in the PIEZO1 gene, including co-occurrence with a novel β-globin mutation. Blood Cells Mol. Dis. 80, 102378. (doi:10.1016/j.bcmd.2019.102378)\\n34.\\nCheng CW et\\xa0al. 2020 Genetic variants of PIEZO1 associate with COVID-19 fatality. medRxiv (doi:10.1101/2020.1106.1101.20119651)\\n35.\\nRogerson D et\\xa0al. 2023 Investigation into the genetics of fetal congenital lymphatic anomalies. Prenat. Diagn. 43, 703–716. (doi:10.1002/pd.6345)\\n36.\\nVora NL et\\xa0al. 2017 Prenatal exome sequencing in anomalous fetuses: new opportunities and challenges. Genet. Med. 19, 1207–1216. (doi:10.1038/gim.2017.33)\\n37.\\nFaucherre A et\\xa0al. 2020 Piezo1 is required for outflow tract and aortic valve development. J. Mol. Cell. Cardiol. 143, 51–62. (doi:10.1016/j.yjmcc.2020.03.013)\\n38.\\nLi JV et\\xa0al. 2021 Modified N-linked glycosylation status predicts trafficking defective human Piezo1 channel mutations. Commun. Biol. 4, 1038. (doi:10.1038/s42003-021-02528-w)\\n39.\\nZhou Z, Li JV, Martinac B, Cox CD. 2021 Loss-of-function Piezo1 mutations display altered stability driven by ubiquitination and proteasomal degradation. Front. Pharmacol. 12,\\n766416. (doi:10.3389/fphar.2021.766416)\\n40.\\nLichtenstein L et\\xa0al. 2024 Endothelial force sensing signals to parenchymal cells to regulate bile and plasma lipids. Sci. Adv 10, eadq3075. (doi:10.1126/sciadv.adq3075)\\n41.\\nAhmed WUR et\\xa0al. 2022 Genome-wide association analysis and replication in 810,625 individuals with varicose veins. Nat. Commun. 13, 3065. (doi:10.1038/s41467-022-30765-y)\\n42.\\nMoradian N, Zoghi S, Rayzan E, Seyedpour S, Jimenez Heredia R, Boztug K, Rezaei N. 2023 Severe congenital neutropenia due to G6PC3 deficiency: early and delayed phenotype of a\\npatient. Allergy Asthma Clin. Immunol. 19, 51. (doi:10.1186/s13223-023-00804-4)\\n43.\\nAmado NG et al. 2024 PIEZO1 loss-of-function compound heterozygous mutations in the rare congenital human disorder Prune Belly Syndrome. Nat. Commun. 15, 339. (doi:10.\\n1038/s41467-023-44594-0)\\n44.\\nBluteau O et\\xa0al. 2018 A landscape of germ line mutations in a cohort of inherited bone marrow failure patients. Blood 131, 717–732. (doi:10.1182/blood-2017-09-806489)\\n45.\\nJiang J et al. 2025 The fetal hydrops-associated single-residue mutation L322P abolishes mechanical but not chemical activation of the PIEZO1 ion channel. bioRxiv (doi:10.1101/\\n2025.02.18.638930)\\n46.\\nPicard V et al. 2019 Clinical and biological features in PIEZO1-hereditary xerocytosis and Gardos channelopathy: a retrospective series of 126 patients. Haematologica 104, 1554–\\n1564. (doi:10.3324/haematol.2018.205328)\\n47.\\nMansour-Hendili L et\\xa0al. 2020 Exome sequencing for diagnosis of congenital hemolytic anemia. Orphanet J. Rare Dis. 15, 180. (doi:10.1186/s13023-020-01425-5)\\n48.\\nShamseldin HE et\\xa0al. 2018 Molecular autopsy in maternal–fetal medicine. Genet. Med. 20, 420–427. (doi:10.1038/gim.2017.111)\\n49.\\nNakahara E et\\xa0al. 2023 Variant spectrum of PIEZO1 and KCNN4 in Japanese patients with dehydrated hereditary stomatocytosis. Hum. Genome Var. 10, 8. (doi:10.1038/s41439-023-\\n00235-y)\\n50.\\nRusso R et\\xa0al. 2018 Multi‐gene panel testing improves diagnosis and management of patients with hereditary anemias. Am. J. Hematol. 93, 672–682. (doi:10.1002/ajh.25058)\\n51.\\nBrewer CJ, Makhamreh MM, Shivashankar K, McLaren R, Toro M, Berger SI, Al‐Kouatly HB. 2023 PIEZO1 is the most common monogenic etiology of non‐immune hydrops fetalis\\ndetected by prenatal exome sequencing. Prenat. Diagn. 43, 1556–1566. (doi:10.1002/pd.6451)\\n52.\\nAl-Kouatly HB et al. 2021 High diagnosis rate for nonimmune hydrops fetalis with prenatal clinical exome from the Hydrops-Yielding Diagnostic Results of Prenatal Sequencing\\n(HYDROPS) Study. Genet. Med. 23, 1325–1333. (doi:10.1038/s41436-021-01121-0)\\n53.\\nGuo W et al. 2020 Trio‐whole‐exome sequencing and preimplantation genetic diagnosis for unexplained recurrent fetal malformations. Hum. Mutat. 41, 432–448. (doi:10.1002/\\nhumu.23935)\\n54.\\nJurynec MJ, Nosyreva E, Thompson D, Munoz C, Novak KA, Matheson DJ, Kazmers NH, Syeda R. 2024 PIEZO1 variants that reduce open channel probability are associated with\\nfamilial osteoarthritis. MedRxiv (doi:10.1101/2024.09.03.24312969)\\n55.\\nGnanasambandam R et\\xa0al. 2018 Increased red cell KCNN4 activity in sporadic hereditary xerocytosis associated with enhanced single channel pressure sensitivity of PIEZO1 mutant\\nV598M. HemaSphere 2, e55. (doi:10.1097/hs9.0000000000000055)\\n56.\\nYamaguchi Y, Allegrini B, Rapetti-Mauss R, Picard V, Garçon L, Kohl P, Soriani O, Peyronnet R, Guizouarn H. 2021 Hereditary xerocytosis: differential behavior of PIEZO1 mutations in\\nthe N-terminal extracellular domain between red blood cells and HEK cells. Front. Physiol. 12, 736585. (doi:10.3389/fphys.2021.736585)\\n57.\\nDempsey E et\\xa0al. 2021 A report on the impact of rapid prenatal exome sequencing on the clinical management of 52 ongoing pregnancies: a retrospective review. BJOG 128, 1012–\\n1019. (doi:10.1111/1471-0528.16546)\\n58.\\nRapetti-Mauss R, Picard V, Guitton C, Ghazal K, Proulle V, Badens C, Soriani O, Garçon L, Guizouarn H. 2017 Red blood cell Gardos channel (KCNN4): the essential determinant of\\nerythrocyte dehydration in hereditary xerocytosis. Haematologica 102, e415–e418. (doi:10.3324/haematol.2017.171389)\\n59.\\nSparks TN et\\xa0al. 2020 Exome sequencing for prenatal diagnosis in nonimmune hydrops fetalis. N. Engl. J. Med. 383, 1746–1756. (doi:10.1056/nejmoa2023643)\\n60.\\nVignes S et al. 2022 PIEZO1-gene gain-of-function mutations with lower limb lymphedema onset in an adult: clinical, scintigraphic, and noncontrast magnetic resonance\\nlymphography findings. Am. J. Med. Genet. Part 188, 243–248. (doi:10.1002/ajmg.a.62476)\\n61.\\nZhou X et\\xa0al. 2021 Value of exome sequencing in diagnosis and management of recurrent non-immune hydrops fetalis: a retrospective analysis. Front. Genet. 12, 616392. (doi:10.\\n3389/fgene.2021.616392)\\n34\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 34}, page_content='62.\\nMore TA, Dongerdiye R, Devendra R, Warang PP, Kedar PS. 2020 Mechanosensitive Piezo1 ion channel protein (PIEZO1 gene): update and extended mutation analysis of hereditary\\nxerocytosis in India. Ann. Hematol. 99, 715–727. (doi:10.1007/s00277-020-03955-1)\\n63.\\nZama D, Giulietti G, Muratore E, Andolfo I, Russo R, Iolascon A, Pession A. 2020 A novel PIEZO1 mutation in a patient with dehydrated hereditary stomatocytosis: a case report and a\\nbrief review of literature. Ital. J. Pediatr. 46, 102. (doi:10.1186/s13052-020-00864-x)\\n64.\\nGhesh L et\\xa0al. 2023 Perinatal presentations of non‐immune hydrops fetalis due to recessive PIEZO1 disease: a challenging fetal diagnosis. Clin. Genet. 103, 560–565. (doi:10.1111/\\ncge.14274)\\n65.\\nCui S, Lu H, Wei S, Li C, Xu F, Chen Y. 2021 The PIEZO1 gene mutation (c.2005G>T) causes iron overload cardiomyopathy: a case report. Emerg. Crit. Care Med. 1, 86–89. (doi:10.\\n1097/ec9.0000000000000018)\\n66.\\nYates CL, Monaghan KG, Copenheaver D, Retterer K, Scuffins J, Kucera CR, Friedman B, Richard G, Juusola J. 2017 Whole-exome sequencing on deceased fetuses with ultrasound\\nanomalies: expanding our knowledge of genetic disease during fetal development. Genet. Med. 19, 1171–1178. (doi:10.1038/gim.2017.31)\\n67.\\nSpier I et\\xa0al. 2016 Exome sequencing identifies potential novel candidate genes in patients with unexplained colorectal adenomatous polyposis. Fam. Cancer 15, 281–288. (doi:10.\\n1007/s10689-016-9870-z)\\n68.\\nAndolfo I et\\xa0al. 2013 Multiple clinical forms of dehydrated hereditary stomatocytosis arise from mutations in PIEZO1. Blood 121, 3925–3935. (doi:10.1182/blood-2013-02-482489)\\n69.\\nZhang M et al. 2021 Exome sequencing of 112 trios identifies recessive genetic variants in brain arteriovenous malformations. J. Neurointerventional Surg 13, 568–573. (doi:10.\\n1136/neurintsurg-2020-016469)\\n70.\\nFotiou E et al. 2015 Novel mutations in PIEZO1 cause an autosomal recessive generalized lymphatic dysplasia with non-immune hydrops fetalis. Nat. Commun. 6, 8085. (doi:10.\\n1038/ncomms9085)\\n71.\\nMa S et\\xa0al. 2018 Common PIEZO1 allele in African populations causes RBC dehydration and attenuates Plasmodium infection. Cell 173, 443–455. (doi:10.1016/j.cell.2018.02.047)\\n72.\\nNguetse CN, Purington N, Ebel ER, Shakya B, Tetard M, Kremsner PG, Velavan TP, Egan ES. 2020 A common polymorphism in the mechanosensitive ion channel PIEZO1 is associated\\nwith protection from severe malaria in humans. Proc. Natl Acad. Sci. USA 117, 9074–9081. (doi:10.1073/pnas.1919843117)\\n73.\\nThye T, Evans JA, Ruge G, Loag W, Ansong D, Agbenyega T, Horstmann RD, May J, Schuldt K. 2022 Human genetic variant E756del in the ion channel PIEZO1 not associated with\\nprotection from severe malaria in a large Ghanaian study. J. Hum. Genet. 67, 65–67. (doi:10.1038/s10038-021-00958-2)\\n74.\\nRooks H, Brewin J, Gardner K, Chakravorty S, Menzel S, Hannemann A, Gibson J, Rees DC. 2019 A gain of function variant in PIEZO1 (E756del) and sickle cell disease. Haematologica\\n104, e91–e93. (doi:10.3324/haematol.2018.202697)\\n75.\\nBaxter SL et\\xa0al. 2020 Investigation of associations between Piezo1 mechanoreceptor gain-of-function variants and glaucoma-related phenotypes in humans and mice. Sci. Rep. 10,\\n19013. (doi:10.1038/s41598-020-76026-0)\\n76.\\nPassini FS et al. 2021 Shear-stress sensing by PIEZO1 regulates tendon stiffness in rodents and influences jumping performance in humans. Nat. Biomed. Eng. 5, 1457–1471. (doi:\\n10.1038/s41551-021-00716-x)\\n77.\\nNakamichi R et al. 2022 The mechanosensitive ion channel PIEZO1 is expressed in tendons and regulates physical performance. Sci. Transl. Med. 14, j5557. (doi:10.1126/\\nscitranslmed.abj5557)\\n78.\\nGötschi T, Held V, Klucker G, Niederöst B, Aagaard P, Spörri J, Passini FS, Snedeker JG. 2023 PIEZO1 gain-of-function gene variant is associated with elevated tendon stiffness in\\nhumans. J. Appl. Physiol. 135, 165–173. (doi:10.1152/japplphysiol.00573.2022)\\n79.\\nJamwal M, Aggarwal A, Palodhi A, Sharma P, Bansal D, Trehan A, Malhotra P, Maitra A, Das R. 2020 Next-generation sequencing–based diagnosis of unexplained inherited\\nhemolytic anemias reveals wide genetic and phenotypic heterogeneity. J. Mol. Diagn. 22, 579–590. (doi:10.1016/j.jmoldx.2020.01.007)\\n80.\\nLudlow MJ et al. 2023 Small-molecule functional rescue of PIEZO1 channel variants associated with generalised lymphatic dysplasia. medRxiv (doi:10.1101/2023.1108.1101.\\n23292554)\\n81.\\nStray-Pedersen A et\\xa0al. 2017 Primary immunodeficiency diseases: genomic approaches delineate heterogeneous Mendelian disorders. J. Allergy Clin. Immunol. 139, 232–245. (doi:\\n10.1016/j.jaci.2016.05.042)\\n82.\\nFustino NJ, Beck R. 2024 A 3-year-old Tanzanian female with glucose-6-phosphate dehydrogenase A- and a novel heterozygous PIEZO1 mutation (2744A>G, N915S) presenting\\nwith severe hemolytic anemia. Ann. Afr. Med. 23, 743–747. (doi:10.4103/aam.aam_29_24)\\n83.\\nLee S, Park S, Kim HY, Chae JH, Ko JM. 2021 Extended phenotypes of PIEZO1-related lymphatic dysplasia caused by two novel compound heterozygous variants. Eur. J. Med. Genet.\\n64, 104295. (doi:10.1016/j.ejmg.2021.104295)\\n84.\\nKedar PS et\\xa0al. 2019 Study of pathophysiology and molecular characterization of congenital anemia in India using targeted next-generation sequencing approach. Int. J. Hematol.\\n110, 618–626. (doi:10.1007/s12185-019-02716-9)\\n85.\\nScimone C, Donato L, Alibrandi S, D’Angelo R, Sidoti A. 2022 Evidences of PIEZO1 involvement in cerebral cavernous malformation pathogenesis. Microvasc. Res. 141, 104342. (doi:\\n10.1016/j.mvr.2022.104342)\\n86.\\nAlhazmi W, Qurban A, Alrashidi E. 2023 Case report of generalized lymphatic dysplasia with PIEZO1 mutation and review of the literature. Respir. Med. Case Rep. 44, 101872. (doi:\\n10.1016/j.rmcr.2023.101872)\\n87.\\nDatkhaeva I et\\xa0al. 2018 Identification of novel PIEZO1 variants using prenatal exome sequencing and correlation to ultrasound and autopsy findings of recurrent hydrops fetalis. Am.\\nJ. Med. Genet. Part 176, 2829–2834. (doi:10.1002/ajmg.a.40533)\\n88.\\nSparks TN et\\xa0al. 2019 Nonimmune hydrops fetalis: identifying the underlying genetic etiology. Genet. Med. 21, 1339–1344. (doi:10.1038/s41436-018-0352-6)\\n89.\\nLee Y, Seo SH, Kim J, Kim SA, Lee JY, Lee JO, Bang SM, Park KU, Hwang SM. 2024 Diagnostic approaches to investigate JAK2-unmutated erythrocytosis based on a single tertiary\\ncenter experience. Mol. Diagn. Ther. 28, 311–318. (doi:10.1007/s40291-024-00703-3)\\n90.\\nLane WJ, Vege S, Mah HH, Ochoa-Garay G, Lomas-Francis C, Westhoff CM. 2024 Three novel Er blood group system alleles and insights from protein modeling. Transfusion 64,\\n1633–1639. (doi:10.1111/trf.17965)\\n91.\\nLukacs V et\\xa0al. 2015 Impaired PIEZO1 function in patients with a novel autosomal recessive congenital lymphatic dysplasia. Nat. Commun. 6, 8329. (doi:10.1038/ncomms9329)\\n92.\\nAl-deri N et\\xa0al. 2021 A novel homozygous variant in TRAPPC2L results in a neurodevelopmental disorder and disrupts TRAPP complex function. J. Med. Genet. 58, 592–601. (doi:10.\\n1136/jmedgenet-2020-107016)\\n93.\\nFilser M et\\xa0al. 2021 Increased incidence of germline PIEZO1 mutations in individuals with idiopathic erythrocytosis. Blood 137, 1828–1832. (doi:10.1182/blood.2020008424)\\n94.\\nAttardi E et\\xa0al. 2024 Prospective genetic germline evaluation in a consecutive group of adult patients aged <60 years with myelodysplastic syndromes. Hemasphere 8, e112. (doi:\\n10.1002/hem3.112)\\n95.\\nKovesi T, Rojas SK, Boycott KM. 2023 Persistent chylothorax associated with lymphatic malformation type 6 due to biallelic pathogenic variants in PIEZO1. Am. J. Med. Genet. A 191,\\n2188–2192. (doi:10.1002/ajmg.a.63237)\\n35\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 35}, page_content='96.\\nChen Y, Jiang Y, Chen B, Qian Y, Liu J, Yang M, Zhao B, Luo Q. 2021 Case report: Whole exome sequencing revealed two novel mutations of PIEZO1 implicated in nonimmune\\nhydrops fetalis. Front. Genet. 12, 684555. (doi:10.3389/fgene.2021.684555)\\n97.\\nAlbuisson J et al. 2013 Dehydrated hereditary stomatocytosis linked to gain-of-function mutations in mechanically activated PIEZO1 ion channels. Nat. Commun. 4, 1884. (doi:10.\\n1038/ncomms2899)\\n98.\\nKager L et al. 2018 Targeted mutation screening of 292 candidate genes in 38 children with inborn haematological cytopenias efficiently identifies novel disease‐causing\\nmutations. Br. J. Haematol. 182, 251–258. (doi:10.1111/bjh.15389)\\n99.\\nShi Y et\\xa0al. 2023 Genetic abnormalities assist in pathological diagnosis and EBV-positive cell density impact survival in Chinese angioimmunoblastic T-cell lymphoma patients. Chin.\\nJ. Cancer Res. 35, 536–549. (doi:10.21147/j.issn.1000-9604.2023.05.10)\\n100. Fathalla BM, Alsarhan A, Afzal S, El Naofal M, Abou Tayoun A. 2021 The genomic landscape of pediatric rheumatology disorders in the Middle East. Hum. Mutat. 42, e1–e14. (doi:\\n10.1002/humu.24165)\\n101. Han S, Guo X, Wang X, Lin H, Yu Y, Shu J, Dong M, Yang L. 2022 A novel homozygous missense mutation of PIEZO1 leading to lymphatic malformation-6 identified in a family with\\nthree adverse pregnancy outcomes due to nonimmune fetal hydrops. Front. Genet. 13, 856046. (doi:10.3389/fgene.2022.856046)\\n102. Karamatic Crew V et al. 2023 Missense mutations in PIEZO1, which encodes the Piezo1 mechanosensor protein, define Er red blood cell antigens. Blood 141, 135–146. (doi:10.\\n1182/blood.2022016504)\\n103. Milev MP et al. 2018 Bi-allelic mutations in TRAPPC2L result in a neurodevelopmental disorder and have an impact on RAB11 in fibroblasts. J. Med. Genet. 55, 753–764. (doi:10.\\n1136/jmedgenet-2018-105441)\\n104. Andolfo I et al. 2018 PIEZO1-R1864H rare variant accounts for a genetic phenotype-modifier role in dehydrated hereditary stomatocytosis. Haematologica 103, e94–e97. (doi:10.\\n3324/haematol.2017.180687)\\n105. Glogowska E et\\xa0al. 2017 Novel mechanisms of PIEZO1 dysfunction in hereditary xerocytosis. Blood 130, 1845–1856. (doi:10.1182/blood-2017-05-786004)\\n106. Andolfo I et\\xa0al. 2019 PIEZO1 hypomorphic variants in congenital lymphatic dysplasia cause shape and hydration alterations of red blood cells. Front. Physiol. 10, 258. (doi:10.3389/\\nfphys.2019.00258)\\n107. Mastromoro G et al. 2021 Recurrent prenatal PIEZO1-related lymphatic dysplasia: Expanding molecular and ultrasound findings. Eur. J. Med. Genet. 64, 104106. (doi:10.1016/j.\\nejmg.2020.104106)\\n108. Stolyar H, Berry T, Singh AP, Madan I. 2021 PIEZO1 mutation: a rare aetiology for fetal ascites. BMJ Case Rep. 14, e240682. (doi:10.1136/bcr-2020-240682)\\n109. Song A, Lu L, Li Y, Lin M, Yuan X, Cheng X, Xia W, Wang O, Xing X. 2020 Low HbA1c with normal hemoglobin in a diabetes patient caused by PIEZO1 gene variant: A case report.\\nFront. Endocrinol. 11, 356. (doi:10.3389/fendo.2020.00356)\\n110. de Meira Oliveira P et\\xa0al. 2020 Heterogeneous phenotype of hereditary xerocytosis in association with PIEZO1 variants. Blood Cells Mol. Dis. 82, 102413. (doi:10.1016/j.bcmd.2020.\\n102413)\\n111. Risinger M, Glogowska E, Chonat S, Zhang K, Dagaonkar N, Joiner CH, Quinn CT, Kalfa TA, Gallagher PG. 2018 Hereditary xerocytosis: diagnostic considerations. Am. J. Hematol. 93,\\nE67–E69. (doi:10.1002/ajh.24996)\\n112. Pinto VM et al. 2023 Coinheritance of PIEZO1 variants and multi‐locus red blood cell defects account for the symptomatic phenotype in beta‐thalassemia carriers. Am. J. Hematol.\\n98, E130–E133. (doi:10.1002/ajh.26901)\\n113. Monti R, Rautenstrauch P, Ghanbari M, James AR, Kirchler M, Ohler U, Konigorski S, Lippert C. 2022 Identifying interpretable gene-biomarker associations with functionally\\ninformed kernel-based tests in 190,000 exomes. Nat. Commun. 13, 5332. (doi:10.1038/s41467-022-32864-2)\\n114. Rotordam MG et\\xa0al. 2019 A novel gain-of-function mutation of Piezo1 is functionally affirmed in red blood cells by high-throughput patch clamp. Haematologica 104, e179–e183.\\n(doi:10.3324/haematol.2018.201160)\\n115. Hu Y et\\xa0al. 2021 Whole-genome sequencing association analysis of quantitative red blood cell phenotypes: The NHLBI TOPMed program. Am. J. Hum. Genet. 108, 874–893. (doi:10.\\n1016/j.ajhg.2021.04.003)\\n116. Zarychanski R, Schulz VP, Houston BL, Maksimova Y, Houston DS, Smith B, Rinehart J, Gallagher PG. 2012 Mutations in the mechanotransduction protein PIEZO1 are associated\\nwith hereditary xerocytosis. Blood 120, 1908–1915. (doi:10.1182/blood-2012-04-422253)\\n117. Bae C, Gnanasambandam R, Nicolai C, Sachs F, Gottlieb PA. 2013 Xerocytosis is caused by mutations that alter the kinetics of the mechanosensitive channel PIEZO1. Proc. Natl Acad.\\nSci. USA 110, E1162–E1168. (doi:10.1073/pnas.1219777110)\\n118. Zhang T, Zhu X, Gao Z, Huang W, Kong X. 2021 Analysis of three families with recurrence of non-immune hydrops fetalis by trio whole exome sequencing. Zhonghua Yi Xue Yi Chuan\\nXue Za Zhi 38, 937–941. (doi:10.3760/cma.j.cn511374-20200729-00564)\\n119. Sun Q et al. 2022 Analyses of biomarker traits in diverse UK Biobank participants identify associations missed by European-centric analysis strategies. J. Hum. Genet. 67, 87–93.\\n(doi:10.1038/s10038-021-00968-0)\\n120. Smelser DT, Haley JS, Ryer EJ, Elmore JR, Cook AM, Carey DJ. 2022 Association of varicose veins with rare protein-truncating variants in PIEZO1 identified by exome sequencing of a\\nlarge clinical population. J. Vasc. Surg. Venous Lymphat. Disord. 10, 382–389. (doi:10.1016/j.jvsv.2021.07.007)\\n121. Mustacich DJ et\\xa0al. 2022 Digenic inheritance of a FOXC2 mutation and two PIEZO1 mutations underlies congenital lymphedema in a multigeneration family. Am. J. Med. 135, e31–\\ne41. (doi:10.1016/j.amjmed.2021.09.007)\\n122. Henkel C et al. 2023 Genome-wide association meta-analysis of knee and hip osteoarthritis uncovers genetic differences between patients treated with joint replacement and\\npatients without joint replacement. Ann. Rheum. Dis. 82, 384–392. (doi:10.1136/ard-2022-223199)\\n123. Imashuku S et al. 2016 PIEZO1 gene mutation in a Japanese family with hereditary high phosphatidylcholine hemolytic anemia and hemochromatosis-induced diabetes mellitus.\\nInt. J. Hematol. 104, 125–129. (doi:10.1007/s12185-016-1970-x)\\n124. Del Orbe Barreto R, Arrizabalaga B, De la Hoz Rastrollo AB, García-Orad A, Gonzalez Vallejo I, Bento C, Villegas A, García-Ruiz JC. 2016 Hereditary xerocytosis, a misleading anemia.\\nAnn. Hematol. 95, 1545–1546. (doi:10.1007/s00277-016-2716-9)\\n125. NIH. 2025 PIEZO1: piezo type mechanosensitive ion channel component 1 (Er blood group) Homo sapiens [Homo sapiens (human)]. See https://www.ncbi.nlm.nih.gov/gene/9780.\\n126. Gordon K, Varney R, Keeley V, Riches K, Jeffery S, Van Zanten M, Mortimer P, Ostergaard P, Mansour S. 2020 Update and audit of the St George’s classification algorithm of primary\\nlymphatic anomalies: a clinical and molecular approach to diagnosis. J. Med. Genet. 57, 653–659. (doi:10.1136/jmedgenet-2019-106084)\\n127. Gordon K, Mortimer PS, van Zanten M, Jeffery S, Ostergaard P, Mansour S. 2021 The St George’s classification algorithm of primary lymphatic anomalies. Lymphat. Res. Biol. 19, 25–\\n30. (doi:10.1089/lrb.2020.0104)\\n128. NIH. 2025 Lymphatic malformation 6. See https://www.ncbi.nlm.nih.gov/gtr/conditions/C4225184.\\n36\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 36}, page_content='129. Choi D et\\xa0al. 2019 Piezo1 incorporates mechanical force signals into the genetic program that governs lymphatic valve development and maintenance. JCI Insight 4, e125068. (doi:\\n10.1172/jci.insight.125068)\\n130. Choi D et\\xa0al. 2022 Piezo1-regulated mechanotransduction controls flow-activated lymphatic expansion. Circ. Res. 131, e2–e21. (doi:10.1161/CIRCRESAHA.121.320565)\\n131. Anwar MA, Georgiadis KA, Shalhoub J, Lim CS, Gohel MS, Davies AH. 2012 A review of familial, genetic, and congenital aspects of primary varicose vein disease. Circ. Cardiovasc.\\nGenet. 5, 460–466. (doi:10.1161/CIRCGENETICS.112.963439)\\n132. Yuan S, Bruzelius M, Damrauer SM, Larsson SC. 2021 Cardiometabolic, lifestyle, and nutritional factors in relation to varicose veins: a Mendelian randomization study. J. Am. Heart\\nAssoc. 10, e022286. (doi:10.1161/jaha.121.022286)\\n133. Fukaya E, Flores AM, Lindholm D, Gustafsson S, Zanetti D, Ingelsson E, Leeper NJ. 2018 Clinical and genetic determinants of varicose veins. Circulation 138, 2869–2880. (doi:10.\\n1161/circulationaha.118.035584)\\n134. Van Hout CV et\\xa0al. 2020 Exome sequencing and characterization of 49,960 individuals in the UK Biobank. Nature 586, 749–756. (doi:10.1038/s41586-020-2853-0)\\n135. Zhang DD et\\xa0al. 2024 Exome sequencing identifies novel genetic variants associated with varicose veins. PLoS Genet. 20, e1011339. (doi:10.1371/journal.pgen.1011339)\\n136. Yoshida S, Koshima I, Imai H, Sasaki A, Fujioka Y, Nagamatsu S, Yokota K. 2019 Lymphatic flow restoration after stripping surgery for varicose veins: a case report. SAGE Open Med.\\nCase Rep. 7, 2050313X1984926. (doi:10.1177/2050313x19849265)\\n137. Iolascon A, Andolfo I, Russo R. 2019 Advances in understanding the pathogenesis of red cell membrane disorders. Br. J. Haematol. 187, 13–24. (doi:10.1111/bjh.16126)\\n138. Stewart GW, Gibson JS, Rees DC. 2023 The cation‐leaky hereditary stomatocytosis syndromes: a tale of six proteins. Br. J. Haematol. 203, 509–522. (doi:10.1111/bjh.19093)\\n139. Andolfo I, Russo R, Rosato BE, Manna F, Gambale A, Brugnara C, Iolascon A. 2018 Genotype‐phenotype correlation and risk stratification in a cohort of 123 hereditary\\nstomatocytosis patients. Am. J. Hematol. 93, 1509–1517. (doi:10.1002/ajh.25276)\\n140. Shan Y, Guo X, Zhang M, Chen M, Li Y, Zhang M, Pei D. 2024 Structure of human PIEZO1 and its slow inactivating channelopathy mutants. bioRxiv. (doi:10.1101/2024.07.14.\\n603468)\\n141. Evans EL et\\xa0al. 2020 RBCs prevent rapid PIEZO1 inactivation and expose slow deactivation as a mechanism of dehydrated hereditary stomatocytosis. Blood 136, 140–144. (doi:10.\\n1182/blood.2019004174)\\n142. Andolfo I, Rosato BE, Manna F, De Rosa G, Marra R, Gambale A, Girelli D, Russo R, Iolascon A. 2020 Gain‐of‐function mutations in PIEZO1 directly impair hepatic iron metabolism via\\nthe inhibition of the BMP/SMADs pathway. Am. J. Hematol. 95, 188–197. (doi:10.1002/ajh.25683)\\n143. Martin‐Almedina S, Mansour S, Ostergaard P. 2018 Human phenotypes caused by PIEZO1 mutations; one gene, two overlapping phenotypes? J. Physiol. 596, 985–992. (doi:10.\\n1113/jp275718)\\n144. Bartoli F et\\xa0al. 2022 Global PIEZO1 gain-of-function mutation causes cardiac hypertrophy and fibrosis in mice. Cells 11, 1199. (doi:10.3390/cells11071199)\\n145. Brylka LJ et\\xa0al. 2024 Piezo1 expression in chondrocytes controls endochondral ossification and osteoarthritis development. Bone Res. 12, 12. (doi:10.1038/s41413-024-00315-x)\\n146. Hirano K et al. 2023 The mechanosensitive ion channel PIEZO1 promotes satellite cell function in muscle regeneration. Life Sci. Alliance 6, e202201783. (doi:10.26508/lsa.\\n202201783)\\n147. Tsuchiya M et\\xa0al. 2018 Cell surface flip-flop of phosphatidylserine is critical for PIEZO1-mediated myotube formation. Nat. Commun. 9, 2049. (doi:10.1038/s41467-018-04436-w)\\n148. Wang S et\\xa0al. 2016 Endothelial cation channel PIEZO1 controls blood pressure by mediating flow-induced ATP release. J. Clin. Investig 126, 4527–4536. (doi:10.1172/JCI87343)\\n149. Rode B et\\xa0al. 2017 Piezo1 channels sense whole body physical activity to reset cardiovascular homeostasis and enhance performance. Nat. Commun. 8, 350. (doi:10.1038/s41467-\\n017-00429-3)\\n150. Yu ZY et\\xa0al. 2022 Piezo1 is the cardiac mechanosensor that initiates the cardiomyocyte hypertrophic response to pressure overload in adult mice. Nat. Cardiovasc. Res. 1, 577–591.\\n(doi:10.1038/s44161-022-00082-0)\\n151. Zhang Y et\\xa0al. 2021 Piezo1-mediated mechanotransduction promotes cardiac hypertrophy by impairing calcium homeostasis to activate calpain/calcineurin signaling. Hypertension\\n78, 647–660. (doi:10.1161/hypertensionaha.121.17177)\\n152. Lim XR et\\xa0al. 2024 Endothelial Piezo1 channel mediates mechano-feedback control of brain blood flow. Nat. Commun 15, 8686. (doi:10.1038/s41467-024-52969-0)\\n153. Ranade SS et\\xa0al. 2014 Piezo1, a mechanically activated ion channel, is required for vascular development in mice. Proc. Natl Acad. Sci. USA 111, 10347–10352. (doi:10.1073/pnas.\\n1409233111)\\n154. Najafi K, Mehrjoo Z, Ardalani F, Ghaderi-Sohi S, Kariminejad A, Kariminejad R, Najmabadi H. 2021 Identifying the causes of recurrent pregnancy loss in consanguineous couples\\nusing whole exome sequencing on the products of miscarriage with no chromosomal abnormalities. Sci. Rep. 11, 6952. (doi:10.1038/s41598-021-86309-9)\\n155. Chong J, De Vecchis D, Hyman AJ, Povstyan OV, Ludlow MJ, Shi J, Beech DJ, Kalli AC. 2021 Modeling of full-length Piezo1 suggests importance of the proximal N-terminus for dome\\nstructure. Biophys. J. 120, 1343–1356. (doi:10.1016/j.bpj.2021.02.003)\\n156. Chen Z et\\xa0al. 2025 Natural selection shaped the protective effect of the mtDNA lineage against obesity in Han Chinese populations. J. Genet. Genom. 52, 539–548. (doi:10.1016/j.\\njgg.2024.06.005)\\n157. Martin-Almedina S, Mortimer PS, Ostergaard P. 2021 Development and physiological functions of the lymphatic system: insights from human genetic studies of primary\\nlymphedema. Physiol. Rev. 101, 1809–1871. (doi:10.1152/physrev.00006.2020)\\n158. Wang J, Jiang J, Yang X, Zhou G, Wang L, Xiao B. 2022 Tethering Piezo channels to the actin cytoskeleton for mechanogating via the cadherin-β-catenin mechanotransduction\\ncomplex. Cell Rep. 38, 110342. (doi:10.1016/j.celrep.2022.110342)\\n159. Chuntharpursat-Bon E et al. 2023 PIEZO1 and PECAM1 interact at cell-cell junctions and partner in endothelial force sensing. Commun. Biol. 6, 358. (doi:10.1038/s42003-023-\\n04706-4)\\n160. Byrne AB et al. 2022 Pathogenic variants in MDFIC cause recessive central conducting lymphatic anomaly with lymphedema. Sci. Transl. Med. 14, eabm4869. (doi:10.1126/\\nscitranslmed.abm4869)\\n161. Michelini S et\\xa0al. 2020 Rare PECAM1 variants in three families with lymphedema. Lymphology 53, 141–151. (doi:10.2458/lymph.4666)\\n162. de Haan HG, Bezemer ID, Vossen CY, van Hylckama Vlieg A, Böehringer S, Hasstedt SJ, Levy S, Rosendaal FR, Bovill EG. 2014 Genetic variants in cell adhesion molecule 1 (CADM1): a\\nvalidation study of a novel endothelial cell venous thrombosis risk factor. Thromb. Res. 134, 1186–1192. (doi:10.1016/j.thromres.2014.09.018)\\n163. Hasstedt SJ, Bezemer ID, Callas PW, Vossen CY, Trotman W, Hebbel RP, Demers C, Rosendaal FR, Bovill EG. 2009 Cell adhesion molecule 1: a novel risk factor for venous thrombosis.\\nBlood 114, 3084–3091. (doi:10.1182/blood-2009-05-219485)\\n164. Murciano N et\\xa0al. 2023 A high-throughput electrophysiology assay to study the response of PIEZO1 to mechanical stimulation. J. Gen. Physiol. 155, e202213132. (doi:10.1085/jgp.\\n202213132)\\n165. Parsonage G et\\xa0al. 2023 Improved PIEZO1 agonism through 4‐benzoic acid modification of Yoda1. Br. J. Pharmacol. 180, 2039–2063. (doi:10.1111/bph.15996)\\n37\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345'), Document(metadata={'producer': 'Antenna House PDF Output Library 7.4.1901', 'creator': 'Antenna House XSL Formatter V7.4 MR2 Linux : 7.4.3.63807 (2024-03-22T11:49+09)', 'creationdate': '2025-07-03T05:21:37+00:00', 'source': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'file_path': 'C:\\\\Users\\\\deepa\\\\Downloads\\\\Deepanshu Bhatt\\\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\\\Data\\\\raw\\\\pmc_pdfs\\\\rsob.240345.pdf', 'total_pages': 38, 'format': 'PDF 1.7', 'title': '', 'author': '', 'subject': '', 'keywords': '', 'moddate': '2025-07-03T05:21:37+00:00', 'trapped': '', 'modDate': \"D:20250703052137+00'00'\", 'creationDate': \"D:20250703052137+00'00'\", 'page': 37}, page_content='166. Petkova-Kirova P et al. 2024 The Gárdos channel and Piezo1 revisited: comparison between reticulocytes and mature red blood cells. Int. J. Mol. Sci. 25, 1416. (doi:10.3390/\\nijms25031416)\\n167. De Vecchis D, Beech DJ, Kalli AC. 2021 Molecular dynamics simulations of Piezo1 channel opening by increases in membrane tension. Biophys. J. 120, 1510–1521. (doi:10.1016/j.\\nbpj.2021.02.006)\\n168. Moffatt CJ, Keeley V, Franks PJ, Rich A, Pinnington LL. 2017 Chronic oedema: a prevalent health care problem for UK health services. Int. Wound J. 14, 772–781. (doi:10.1111/iwj.\\n12694)\\n169. Choi D et\\xa0al. 2024 Piezo1 regulates meningeal lymphatic vessel drainage and alleviates excessive CSF accumulation. Nat. Neurosci. 27, 913–926. (doi:10.1038/s41593-024-01604-\\n8)\\n170. Du J, Liu P, Zhou Y, Misener S, Sharma I, Leeaw P, Thomson BR, Jin J, Quaggin SE. 2024 The mechanosensory channel PIEZO1 functions upstream of angiopoietin/TIE/FOXO1\\nsignaling in lymphatic development. J. Clin. Invest. 134, e176577. (doi:10.1172/JCI176577)\\n171. Matrongolo MJ et\\xa0al. 2023 Piezo1 agonist restores meningeal lymphatic vessels, drainage, and brain-CSF perfusion in craniosynostosis and aged mice. J. Clin. Invest. 134, e171468.\\n(doi:10.1172/JCI171468)\\n172. Bolte AC et\\xa0al. 2020 Meningeal lymphatic dysfunction exacerbates traumatic brain injury pathogenesis. Nat. Commun. 11, 4524. (doi:10.1038/s41467-020-18113-4)\\n173. Zhao J, Xiong Y, Liu Y, Ling J, Liu S, Wang W. 2025 Endothelium Piezo1 deletion alleviates experimental varicose veins by attenuating perivenous inflammation. Mol. Cell. Biochem.\\n480, 2423–2435. (doi:10.1007/s11010-024-05115-9)\\n174. Romero LO, Massey AE, Mata-Daboin AD, Sierra-Valdez FJ, Chauhan SC, Cordero-Morales JF, Vásquez V. 2019 Dietary fatty acids fine-tune Piezo1 mechanical response. Nat.\\nCommun. 10, 1200. (doi:10.1038/s41467-019-09055-7)\\n175. Glogowska E et\\xa0al. 2025 Potentiation of macrophage Piezo1 by atherogenic 7-ketocholesterol. Cell Rep. 44, 115542. (doi:10.1016/j.celrep.2025.115542)\\n176. Zhu J et al. 2023 The mechanosensitive ion channel Piezo1 contributes to ultrasound neuromodulation. Proc. Natl Acad. Sci. USA 120, e2300291120. (doi:10.1073/pnas.\\n2300291120)\\n177. Chen X, Chen J. 2022 miR-10b-5p-mediated upregulation of PIEZO1 predicts poor prognosis and links to purine metabolism in breast cancer. Genomics 114, 110351. (doi:10.1016/j.\\nygeno.2022.110351)\\n178. Cheng J et\\xa0al. 2023 Accurate proteome-wide missense variant effect prediction with AlphaMissense. Science 381, eadg7492. (doi:10.1126/science.adg7492)\\n179. Gallagher PG. 2017 Disorders of erythrocyte hydration. Blood 130, 2699–2708. (doi:10.1182/blood-2017-04-590810)\\n38\\nroyalsocietypublishing.org/journal/rsob \\nOpen Biol. 15: 240345')]\n"
     ]
    }
   ],
   "source": [
    "file = r\"C:\\Users\\deepa\\Downloads\\Deepanshu Bhatt\\rerprCheck-NLP-base-Biomedical-Research-Checklist-\\Data\\raw\\pmc_pdfs\\rsob.240345.pdf\"\n",
    "\n",
    "loader = PyMuPDFLoader(file, mode = \"page\", pages_delimiter = \"  \")\n",
    "doc = loader.load()\n",
    "print(doc)\n",
    "                       "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "b89d8f4c-2e3b-43ed-bb55-2b36819eb449",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Requirement already satisfied: langchain in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (0.3.27)\n",
      "Requirement already satisfied: langchain-community in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (0.3.28)\n",
      "Collecting unstructured\n",
      "  Using cached unstructured-0.18.14-py3-none-any.whl.metadata (24 kB)\n",
      "Requirement already satisfied: langchain-unstructured in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (0.1.6)\n",
      "Requirement already satisfied: langchain-core<1.0.0,>=0.3.72 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain) (0.3.75)\n",
      "Requirement already satisfied: langchain-text-splitters<1.0.0,>=0.3.9 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain) (0.3.9)\n",
      "Requirement already satisfied: langsmith>=0.1.17 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain) (0.4.19)\n",
      "Requirement already satisfied: pydantic<3.0.0,>=2.7.4 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain) (2.11.7)\n",
      "Requirement already satisfied: SQLAlchemy<3,>=1.4 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain) (2.0.30)\n",
      "Requirement already satisfied: requests<3,>=2 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain) (2.32.5)\n",
      "Requirement already satisfied: PyYAML>=5.3 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain) (6.0.1)\n",
      "Requirement already satisfied: tenacity!=8.4.0,<10.0.0,>=8.1.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain-core<1.0.0,>=0.3.72->langchain) (8.2.2)\n",
      "Requirement already satisfied: jsonpatch<2.0,>=1.33 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain-core<1.0.0,>=0.3.72->langchain) (1.33)\n",
      "Requirement already satisfied: typing-extensions>=4.7 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain-core<1.0.0,>=0.3.72->langchain) (4.14.1)\n",
      "Requirement already satisfied: packaging>=23.2 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain-core<1.0.0,>=0.3.72->langchain) (23.2)\n",
      "Requirement already satisfied: jsonpointer>=1.9 in c:\\programdata\\anaconda3\\lib\\site-packages (from jsonpatch<2.0,>=1.33->langchain-core<1.0.0,>=0.3.72->langchain) (2.1)\n",
      "Requirement already satisfied: annotated-types>=0.6.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from pydantic<3.0.0,>=2.7.4->langchain) (0.6.0)\n",
      "Requirement already satisfied: pydantic-core==2.33.2 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from pydantic<3.0.0,>=2.7.4->langchain) (2.33.2)\n",
      "Requirement already satisfied: typing-inspection>=0.4.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from pydantic<3.0.0,>=2.7.4->langchain) (0.4.1)\n",
      "Requirement already satisfied: charset_normalizer<4,>=2 in c:\\programdata\\anaconda3\\lib\\site-packages (from requests<3,>=2->langchain) (2.0.4)\n",
      "Requirement already satisfied: idna<4,>=2.5 in c:\\programdata\\anaconda3\\lib\\site-packages (from requests<3,>=2->langchain) (3.7)\n",
      "Requirement already satisfied: urllib3<3,>=1.21.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from requests<3,>=2->langchain) (2.2.2)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in c:\\programdata\\anaconda3\\lib\\site-packages (from requests<3,>=2->langchain) (2024.7.4)\n",
      "Requirement already satisfied: greenlet!=0.4.17 in c:\\programdata\\anaconda3\\lib\\site-packages (from SQLAlchemy<3,>=1.4->langchain) (3.0.1)\n",
      "Requirement already satisfied: aiohttp<4.0.0,>=3.8.3 in c:\\programdata\\anaconda3\\lib\\site-packages (from langchain-community) (3.9.5)\n",
      "Requirement already satisfied: dataclasses-json<0.7,>=0.6.7 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain-community) (0.6.7)\n",
      "Requirement already satisfied: pydantic-settings<3.0.0,>=2.10.1 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain-community) (2.10.1)\n",
      "Requirement already satisfied: httpx-sse<1.0.0,>=0.4.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain-community) (0.4.1)\n",
      "Requirement already satisfied: numpy>=1.26.2 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain-community) (2.3.2)\n",
      "Requirement already satisfied: aiosignal>=1.1.2 in c:\\programdata\\anaconda3\\lib\\site-packages (from aiohttp<4.0.0,>=3.8.3->langchain-community) (1.2.0)\n",
      "Requirement already satisfied: attrs>=17.3.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from aiohttp<4.0.0,>=3.8.3->langchain-community) (24.2.0)\n",
      "Requirement already satisfied: frozenlist>=1.1.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from aiohttp<4.0.0,>=3.8.3->langchain-community) (1.4.0)\n",
      "Requirement already satisfied: multidict<7.0,>=4.5 in c:\\programdata\\anaconda3\\lib\\site-packages (from aiohttp<4.0.0,>=3.8.3->langchain-community) (6.0.4)\n",
      "Requirement already satisfied: yarl<2.0,>=1.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from aiohttp<4.0.0,>=3.8.3->langchain-community) (1.9.3)\n",
      "Requirement already satisfied: marshmallow<4.0.0,>=3.18.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from dataclasses-json<0.7,>=0.6.7->langchain-community) (3.26.1)\n",
      "Requirement already satisfied: typing-inspect<1,>=0.4.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from dataclasses-json<0.7,>=0.6.7->langchain-community) (0.9.0)\n",
      "Requirement already satisfied: python-dotenv>=0.21.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from pydantic-settings<3.0.0,>=2.10.1->langchain-community) (0.21.0)\n",
      "Requirement already satisfied: mypy-extensions>=0.3.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from typing-inspect<1,>=0.4.0->dataclasses-json<0.7,>=0.6.7->langchain-community) (1.0.0)\n",
      "Collecting filetype (from unstructured)\n",
      "  Using cached filetype-1.2.0-py2.py3-none-any.whl.metadata (6.5 kB)\n",
      "Collecting python-magic (from unstructured)\n",
      "  Using cached python_magic-0.4.27-py2.py3-none-any.whl.metadata (5.8 kB)\n",
      "Requirement already satisfied: lxml in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured) (5.2.1)\n",
      "Requirement already satisfied: nltk in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured) (3.8.1)\n",
      "Requirement already satisfied: beautifulsoup4 in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured) (4.12.3)\n",
      "Collecting emoji (from unstructured)\n",
      "  Using cached emoji-2.14.1-py3-none-any.whl.metadata (5.7 kB)\n",
      "Collecting python-iso639 (from unstructured)\n",
      "  Using cached python_iso639-2025.2.18-py3-none-any.whl.metadata (14 kB)\n",
      "Collecting langdetect (from unstructured)\n",
      "  Using cached langdetect-1.0.9.tar.gz (981 kB)\n",
      "  Preparing metadata (setup.py): started\n",
      "  Preparing metadata (setup.py): finished with status 'done'\n",
      "Collecting rapidfuzz (from unstructured)\n",
      "  Downloading rapidfuzz-3.13.0-cp312-cp312-win_amd64.whl.metadata (12 kB)\n",
      "Collecting backoff (from unstructured)\n",
      "  Downloading backoff-2.2.1-py3-none-any.whl.metadata (14 kB)\n",
      "Requirement already satisfied: unstructured-client in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured) (0.42.3)\n",
      "Requirement already satisfied: wrapt in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured) (1.14.1)\n",
      "Requirement already satisfied: tqdm in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured) (4.66.4)\n",
      "Requirement already satisfied: psutil in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured) (5.9.0)\n",
      "Collecting python-oxmsg (from unstructured)\n",
      "  Using cached python_oxmsg-0.0.2-py3-none-any.whl.metadata (5.0 kB)\n",
      "Collecting html5lib (from unstructured)\n",
      "  Using cached html5lib-1.1-py2.py3-none-any.whl.metadata (16 kB)\n",
      "Requirement already satisfied: onnxruntime<=1.19.2,>=1.17.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langchain-unstructured) (1.19.2)\n",
      "Requirement already satisfied: coloredlogs in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (15.0.1)\n",
      "Requirement already satisfied: flatbuffers in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (25.2.10)\n",
      "Requirement already satisfied: protobuf in c:\\programdata\\anaconda3\\lib\\site-packages (from onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (3.20.3)\n",
      "Requirement already satisfied: sympy in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (1.14.0)\n",
      "Requirement already satisfied: aiofiles>=24.1.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-client->unstructured) (24.1.0)\n",
      "Requirement already satisfied: cryptography>=3.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured-client->unstructured) (42.0.5)\n",
      "Requirement already satisfied: httpcore>=1.0.9 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-client->unstructured) (1.0.9)\n",
      "Requirement already satisfied: httpx>=0.27.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-client->unstructured) (0.28.1)\n",
      "Requirement already satisfied: pypdf>=4.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-client->unstructured) (6.0.0)\n",
      "Requirement already satisfied: requests-toolbelt>=1.0.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured-client->unstructured) (1.0.0)\n",
      "Requirement already satisfied: cffi>=1.12 in c:\\programdata\\anaconda3\\lib\\site-packages (from cryptography>=3.1->unstructured-client->unstructured) (1.16.0)\n",
      "Requirement already satisfied: pycparser in c:\\programdata\\anaconda3\\lib\\site-packages (from cffi>=1.12->cryptography>=3.1->unstructured-client->unstructured) (2.21)\n",
      "Requirement already satisfied: h11>=0.16 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from httpcore>=1.0.9->unstructured-client->unstructured) (0.16.0)\n",
      "Requirement already satisfied: anyio in c:\\programdata\\anaconda3\\lib\\site-packages (from httpx>=0.27.0->unstructured-client->unstructured) (4.2.0)\n",
      "Requirement already satisfied: orjson>=3.9.14 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langsmith>=0.1.17->langchain) (3.11.3)\n",
      "Requirement already satisfied: zstandard>=0.23.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from langsmith>=0.1.17->langchain) (0.24.0)\n",
      "Requirement already satisfied: sniffio>=1.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from anyio->httpx>=0.27.0->unstructured-client->unstructured) (1.3.0)\n",
      "Requirement already satisfied: soupsieve>1.2 in c:\\programdata\\anaconda3\\lib\\site-packages (from beautifulsoup4->unstructured) (2.5)\n",
      "Requirement already satisfied: humanfriendly>=9.1 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from coloredlogs->onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (10.0)\n",
      "Requirement already satisfied: pyreadline3 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from humanfriendly>=9.1->coloredlogs->onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (3.5.4)\n",
      "Requirement already satisfied: six>=1.9 in c:\\programdata\\anaconda3\\lib\\site-packages (from html5lib->unstructured) (1.16.0)\n",
      "Requirement already satisfied: webencodings in c:\\programdata\\anaconda3\\lib\\site-packages (from html5lib->unstructured) (0.5.1)\n",
      "Requirement already satisfied: click in c:\\programdata\\anaconda3\\lib\\site-packages (from nltk->unstructured) (8.1.7)\n",
      "Requirement already satisfied: joblib in c:\\programdata\\anaconda3\\lib\\site-packages (from nltk->unstructured) (1.4.2)\n",
      "Requirement already satisfied: regex>=2021.8.3 in c:\\programdata\\anaconda3\\lib\\site-packages (from nltk->unstructured) (2023.10.3)\n",
      "Requirement already satisfied: colorama in c:\\programdata\\anaconda3\\lib\\site-packages (from click->nltk->unstructured) (0.4.6)\n",
      "Collecting olefile (from python-oxmsg->unstructured)\n",
      "  Using cached olefile-0.47-py2.py3-none-any.whl.metadata (9.7 kB)\n",
      "Requirement already satisfied: mpmath<1.4,>=1.1.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from sympy->onnxruntime<=1.19.2,>=1.17.0->langchain-unstructured) (1.3.0)\n",
      "Using cached unstructured-0.18.14-py3-none-any.whl (1.8 MB)\n",
      "Downloading backoff-2.2.1-py3-none-any.whl (15 kB)\n",
      "Using cached emoji-2.14.1-py3-none-any.whl (590 kB)\n",
      "Using cached filetype-1.2.0-py2.py3-none-any.whl (19 kB)\n",
      "Using cached html5lib-1.1-py2.py3-none-any.whl (112 kB)\n",
      "Using cached python_iso639-2025.2.18-py3-none-any.whl (167 kB)\n",
      "Using cached python_magic-0.4.27-py2.py3-none-any.whl (13 kB)\n",
      "Using cached python_oxmsg-0.0.2-py3-none-any.whl (31 kB)\n",
      "Using cached olefile-0.47-py2.py3-none-any.whl (114 kB)\n",
      "Downloading rapidfuzz-3.13.0-cp312-cp312-win_amd64.whl (1.6 MB)\n",
      "   ---------------------------------------- 0.0/1.6 MB ? eta -:--:--\n",
      "   ---------------------------------------- 1.6/1.6 MB 17.4 MB/s  0:00:00\n",
      "Building wheels for collected packages: langdetect\n",
      "  Building wheel for langdetect (setup.py): started\n",
      "  Building wheel for langdetect (setup.py): finished with status 'done'\n",
      "  Created wheel for langdetect: filename=langdetect-1.0.9-py3-none-any.whl size=993363 sha256=9bd4ee6b75b0b3092f43d6f35c190b96914eff3ec9689c3f74755c48c80eb338\n",
      "  Stored in directory: c:\\users\\deepa\\appdata\\local\\pip\\cache\\wheels\\c1\\67\\88\\e844b5b022812e15a52e4eaa38a1e709e99f06f6639d7e3ba7\n",
      "Successfully built langdetect\n",
      "Installing collected packages: filetype, rapidfuzz, python-magic, python-iso639, olefile, langdetect, html5lib, emoji, backoff, python-oxmsg, unstructured\n",
      "\n",
      "   ----------------------------------------  0/11 [filetype]\n",
      "   ----------------------------------------  0/11 [filetype]\n",
      "   ----------------------------------------  0/11 [filetype]\n",
      "   --- ------------------------------------  1/11 [rapidfuzz]\n",
      "   --- ------------------------------------  1/11 [rapidfuzz]\n",
      "   --- ------------------------------------  1/11 [rapidfuzz]\n",
      "   --- ------------------------------------  1/11 [rapidfuzz]\n",
      "   --- ------------------------------------  1/11 [rapidfuzz]\n",
      "   ------- --------------------------------  2/11 [python-magic]\n",
      "   -------------- -------------------------  4/11 [olefile]\n",
      "   ------------------ ---------------------  5/11 [langdetect]\n",
      "   ------------------ ---------------------  5/11 [langdetect]\n",
      "   --------------------- ------------------  6/11 [html5lib]\n",
      "   --------------------- ------------------  6/11 [html5lib]\n",
      "   --------------------- ------------------  6/11 [html5lib]\n",
      "   --------------------- ------------------  6/11 [html5lib]\n",
      "   --------------------- ------------------  6/11 [html5lib]\n",
      "   --------------------- ------------------  6/11 [html5lib]\n",
      "   ------------------------- --------------  7/11 [emoji]\n",
      "   ----------------------------- ----------  8/11 [backoff]\n",
      "   -------------------------------- -------  9/11 [python-oxmsg]\n",
      "   -------------------------------- -------  9/11 [python-oxmsg]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ------------------------------------ --- 10/11 [unstructured]\n",
      "   ---------------------------------------- 11/11 [unstructured]\n",
      "\n",
      "Successfully installed backoff-2.2.1 emoji-2.14.1 filetype-1.2.0 html5lib-1.1 langdetect-1.0.9 olefile-0.47 python-iso639-2025.2.18 python-magic-0.4.27 python-oxmsg-0.0.2 rapidfuzz-3.13.0 unstructured-0.18.14\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  DEPRECATION: Building 'langdetect' using the legacy setup.py bdist_wheel mechanism, which will be removed in a future version. pip 25.3 will enforce this behaviour change. A possible replacement is to use the standardized build interface by setting the `--use-pep517` option, (possibly combined with `--no-build-isolation`), or adding a `pyproject.toml` file to the source tree of 'langdetect'. Discussion can be found at https://github.com/pypa/pip/issues/6334\n",
      "  WARNING: The script filetype.exe is installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: The script oxmsg.exe is installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Requirement already satisfied: unstructured[all-docs] in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (0.18.14)\n",
      "Requirement already satisfied: charset-normalizer in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (2.0.4)\n",
      "Requirement already satisfied: filetype in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (1.2.0)\n",
      "Requirement already satisfied: python-magic in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (0.4.27)\n",
      "Requirement already satisfied: lxml in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (5.2.1)\n",
      "Requirement already satisfied: nltk in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (3.8.1)\n",
      "Requirement already satisfied: requests in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (2.32.5)\n",
      "Requirement already satisfied: beautifulsoup4 in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (4.12.3)\n",
      "Requirement already satisfied: emoji in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (2.14.1)\n",
      "Requirement already satisfied: dataclasses-json in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (0.6.7)\n",
      "Requirement already satisfied: python-iso639 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (2025.2.18)\n",
      "Requirement already satisfied: langdetect in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (1.0.9)\n",
      "Requirement already satisfied: numpy in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (2.3.2)\n",
      "Requirement already satisfied: rapidfuzz in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (3.13.0)\n",
      "Requirement already satisfied: backoff in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (2.2.1)\n",
      "Requirement already satisfied: typing-extensions in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (4.14.1)\n",
      "Requirement already satisfied: unstructured-client in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (0.42.3)\n",
      "Requirement already satisfied: wrapt in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (1.14.1)\n",
      "Requirement already satisfied: tqdm in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (4.66.4)\n",
      "Requirement already satisfied: psutil in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (5.9.0)\n",
      "Requirement already satisfied: python-oxmsg in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (0.0.2)\n",
      "Requirement already satisfied: html5lib in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (1.1)\n",
      "Collecting msoffcrypto-tool (from unstructured[all-docs])\n",
      "  Using cached msoffcrypto_tool-5.4.2-py3-none-any.whl.metadata (10 kB)\n",
      "Collecting xlrd (from unstructured[all-docs])\n",
      "  Using cached xlrd-2.0.2-py2.py3-none-any.whl.metadata (3.5 kB)\n",
      "Collecting onnx>=1.17.0 (from unstructured[all-docs])\n",
      "  Downloading onnx-1.19.0-cp312-cp312-win_amd64.whl.metadata (7.2 kB)\n",
      "Collecting pdf2image (from unstructured[all-docs])\n",
      "  Using cached pdf2image-1.17.0-py3-none-any.whl.metadata (6.2 kB)\n",
      "Collecting pi-heif (from unstructured[all-docs])\n",
      "  Downloading pi_heif-1.1.0-cp312-cp312-win_amd64.whl.metadata (6.7 kB)\n",
      "Collecting python-pptx>=1.0.1 (from unstructured[all-docs])\n",
      "  Using cached python_pptx-1.0.2-py3-none-any.whl.metadata (2.5 kB)\n",
      "Collecting google-cloud-vision (from unstructured[all-docs])\n",
      "  Using cached google_cloud_vision-3.10.2-py3-none-any.whl.metadata (9.6 kB)\n",
      "Requirement already satisfied: onnxruntime>=1.19.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (1.19.2)\n",
      "Collecting python-docx>=1.1.2 (from unstructured[all-docs])\n",
      "  Using cached python_docx-1.2.0-py3-none-any.whl.metadata (2.0 kB)\n",
      "Collecting unstructured.pytesseract>=0.3.12 (from unstructured[all-docs])\n",
      "  Using cached unstructured.pytesseract-0.3.15-py3-none-any.whl.metadata (11 kB)\n",
      "Collecting pdfminer.six (from unstructured[all-docs])\n",
      "  Downloading pdfminer_six-20250506-py3-none-any.whl.metadata (4.2 kB)\n",
      "Requirement already satisfied: networkx in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (3.2.1)\n",
      "Requirement already satisfied: pypdf in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (6.0.0)\n",
      "Collecting pypandoc (from unstructured[all-docs])\n",
      "  Using cached pypandoc-1.15-py3-none-any.whl.metadata (16 kB)\n",
      "Requirement already satisfied: openpyxl in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (3.1.2)\n",
      "Collecting effdet (from unstructured[all-docs])\n",
      "  Using cached effdet-0.4.1-py3-none-any.whl.metadata (33 kB)\n",
      "Collecting unstructured-inference>=1.0.5 (from unstructured[all-docs])\n",
      "  Using cached unstructured_inference-1.0.5-py3-none-any.whl.metadata (5.3 kB)\n",
      "Requirement already satisfied: pandas in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured[all-docs]) (2.2.3)\n",
      "Collecting pikepdf (from unstructured[all-docs])\n",
      "  Downloading pikepdf-9.10.2-cp312-cp312-win_amd64.whl.metadata (8.2 kB)\n",
      "Requirement already satisfied: markdown in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured[all-docs]) (3.4.1)\n",
      "Collecting protobuf>=4.25.1 (from onnx>=1.17.0->unstructured[all-docs])\n",
      "  Using cached protobuf-6.32.0-cp310-abi3-win_amd64.whl.metadata (593 bytes)\n",
      "Collecting ml_dtypes (from onnx>=1.17.0->unstructured[all-docs])\n",
      "  Downloading ml_dtypes-0.5.3-cp312-cp312-win_amd64.whl.metadata (9.2 kB)\n",
      "Requirement already satisfied: coloredlogs in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from onnxruntime>=1.19.0->unstructured[all-docs]) (15.0.1)\n",
      "Requirement already satisfied: flatbuffers in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from onnxruntime>=1.19.0->unstructured[all-docs]) (25.2.10)\n",
      "Requirement already satisfied: packaging in c:\\programdata\\anaconda3\\lib\\site-packages (from onnxruntime>=1.19.0->unstructured[all-docs]) (23.2)\n",
      "Requirement already satisfied: sympy in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from onnxruntime>=1.19.0->unstructured[all-docs]) (1.14.0)\n",
      "Requirement already satisfied: Pillow>=3.3.2 in c:\\programdata\\anaconda3\\lib\\site-packages (from python-pptx>=1.0.1->unstructured[all-docs]) (10.3.0)\n",
      "Collecting XlsxWriter>=0.5.7 (from python-pptx>=1.0.1->unstructured[all-docs])\n",
      "  Using cached xlsxwriter-3.2.5-py3-none-any.whl.metadata (2.7 kB)\n",
      "Collecting python-multipart (from unstructured-inference>=1.0.5->unstructured[all-docs])\n",
      "  Using cached python_multipart-0.0.20-py3-none-any.whl.metadata (1.8 kB)\n",
      "Requirement already satisfied: huggingface-hub in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-inference>=1.0.5->unstructured[all-docs]) (0.34.4)\n",
      "Collecting opencv-python!=4.7.0.68 (from unstructured-inference>=1.0.5->unstructured[all-docs])\n",
      "  Using cached opencv_python-4.12.0.88-cp37-abi3-win_amd64.whl.metadata (19 kB)\n",
      "Requirement already satisfied: matplotlib in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-inference>=1.0.5->unstructured[all-docs]) (3.10.0)\n",
      "Requirement already satisfied: torch in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-inference>=1.0.5->unstructured[all-docs]) (2.8.0)\n",
      "Collecting timm (from unstructured-inference>=1.0.5->unstructured[all-docs])\n",
      "  Using cached timm-1.0.19-py3-none-any.whl.metadata (60 kB)\n",
      "Requirement already satisfied: transformers>=4.25.1 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-inference>=1.0.5->unstructured[all-docs]) (4.55.0)\n",
      "Collecting accelerate (from unstructured-inference>=1.0.5->unstructured[all-docs])\n",
      "  Using cached accelerate-1.10.1-py3-none-any.whl.metadata (19 kB)\n",
      "Requirement already satisfied: scipy in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-inference>=1.0.5->unstructured[all-docs]) (1.14.1)\n",
      "Collecting pypdfium2 (from unstructured-inference>=1.0.5->unstructured[all-docs])\n",
      "  Using cached pypdfium2-4.30.0-py3-none-win_amd64.whl.metadata (48 kB)\n",
      "Collecting numpy (from unstructured[all-docs])\n",
      "  Downloading numpy-2.2.6-cp312-cp312-win_amd64.whl.metadata (60 kB)\n",
      "Requirement already satisfied: filelock in c:\\programdata\\anaconda3\\lib\\site-packages (from transformers>=4.25.1->unstructured-inference>=1.0.5->unstructured[all-docs]) (3.13.1)\n",
      "Requirement already satisfied: pyyaml>=5.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from transformers>=4.25.1->unstructured-inference>=1.0.5->unstructured[all-docs]) (6.0.1)\n",
      "Requirement already satisfied: regex!=2019.12.17 in c:\\programdata\\anaconda3\\lib\\site-packages (from transformers>=4.25.1->unstructured-inference>=1.0.5->unstructured[all-docs]) (2023.10.3)\n",
      "Requirement already satisfied: tokenizers<0.22,>=0.21 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from transformers>=4.25.1->unstructured-inference>=1.0.5->unstructured[all-docs]) (0.21.4)\n",
      "Requirement already satisfied: safetensors>=0.4.3 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from transformers>=4.25.1->unstructured-inference>=1.0.5->unstructured[all-docs]) (0.6.2)\n",
      "Requirement already satisfied: fsspec>=2023.5.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from huggingface-hub->unstructured-inference>=1.0.5->unstructured[all-docs]) (2024.3.1)\n",
      "Requirement already satisfied: colorama in c:\\programdata\\anaconda3\\lib\\site-packages (from tqdm->unstructured[all-docs]) (0.4.6)\n",
      "Requirement already satisfied: jinja2 in c:\\programdata\\anaconda3\\lib\\site-packages (from torch->unstructured-inference>=1.0.5->unstructured[all-docs]) (3.1.4)\n",
      "Requirement already satisfied: setuptools in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from torch->unstructured-inference>=1.0.5->unstructured[all-docs]) (80.9.0)\n",
      "Requirement already satisfied: mpmath<1.4,>=1.1.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from sympy->onnxruntime>=1.19.0->unstructured[all-docs]) (1.3.0)\n",
      "Requirement already satisfied: soupsieve>1.2 in c:\\programdata\\anaconda3\\lib\\site-packages (from beautifulsoup4->unstructured[all-docs]) (2.5)\n",
      "Requirement already satisfied: humanfriendly>=9.1 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from coloredlogs->onnxruntime>=1.19.0->unstructured[all-docs]) (10.0)\n",
      "Requirement already satisfied: pyreadline3 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from humanfriendly>=9.1->coloredlogs->onnxruntime>=1.19.0->unstructured[all-docs]) (3.5.4)\n",
      "Requirement already satisfied: marshmallow<4.0.0,>=3.18.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from dataclasses-json->unstructured[all-docs]) (3.26.1)\n",
      "Requirement already satisfied: typing-inspect<1,>=0.4.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from dataclasses-json->unstructured[all-docs]) (0.9.0)\n",
      "Requirement already satisfied: mypy-extensions>=0.3.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from typing-inspect<1,>=0.4.0->dataclasses-json->unstructured[all-docs]) (1.0.0)\n",
      "Collecting torchvision (from effdet->unstructured[all-docs])\n",
      "  Downloading torchvision-0.23.0-cp312-cp312-win_amd64.whl.metadata (6.1 kB)\n",
      "Collecting pycocotools>=2.0.2 (from effdet->unstructured[all-docs])\n",
      "  Downloading pycocotools-2.0.10-cp312-abi3-win_amd64.whl.metadata (1.3 kB)\n",
      "Collecting omegaconf>=2.0 (from effdet->unstructured[all-docs])\n",
      "  Using cached omegaconf-2.3.0-py3-none-any.whl.metadata (3.9 kB)\n",
      "Collecting antlr4-python3-runtime==4.9.* (from omegaconf>=2.0->effdet->unstructured[all-docs])\n",
      "  Using cached antlr4-python3-runtime-4.9.3.tar.gz (117 kB)\n",
      "  Preparing metadata (setup.py): started\n",
      "  Preparing metadata (setup.py): finished with status 'done'\n",
      "Collecting google-api-core!=2.0.*,!=2.1.*,!=2.10.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,<3.0.0,>=1.34.1 (from google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.10.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,<3.0.0,>=1.34.1->google-cloud-vision->unstructured[all-docs])\n",
      "  Using cached google_api_core-2.25.1-py3-none-any.whl.metadata (3.0 kB)\n",
      "Collecting google-auth!=2.24.0,!=2.25.0,<3.0.0,>=2.14.1 (from google-cloud-vision->unstructured[all-docs])\n",
      "  Using cached google_auth-2.40.3-py2.py3-none-any.whl.metadata (6.2 kB)\n",
      "Collecting proto-plus<2.0.0,>=1.22.3 (from google-cloud-vision->unstructured[all-docs])\n",
      "  Using cached proto_plus-1.26.1-py3-none-any.whl.metadata (2.2 kB)\n",
      "Collecting googleapis-common-protos<2.0.0,>=1.56.2 (from google-api-core!=2.0.*,!=2.1.*,!=2.10.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,<3.0.0,>=1.34.1->google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.10.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,<3.0.0,>=1.34.1->google-cloud-vision->unstructured[all-docs])\n",
      "  Using cached googleapis_common_protos-1.70.0-py3-none-any.whl.metadata (9.3 kB)\n",
      "Collecting grpcio<2.0.0,>=1.33.2 (from google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.10.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,<3.0.0,>=1.34.1->google-cloud-vision->unstructured[all-docs])\n",
      "  Downloading grpcio-1.74.0-cp312-cp312-win_amd64.whl.metadata (4.0 kB)\n",
      "Collecting grpcio-status<2.0.0,>=1.33.2 (from google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.10.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,<3.0.0,>=1.34.1->google-cloud-vision->unstructured[all-docs])\n",
      "  Using cached grpcio_status-1.74.0-py3-none-any.whl.metadata (1.1 kB)\n",
      "Requirement already satisfied: cachetools<6.0,>=2.0.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from google-auth!=2.24.0,!=2.25.0,<3.0.0,>=2.14.1->google-cloud-vision->unstructured[all-docs]) (5.3.3)\n",
      "Requirement already satisfied: pyasn1-modules>=0.2.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from google-auth!=2.24.0,!=2.25.0,<3.0.0,>=2.14.1->google-cloud-vision->unstructured[all-docs]) (0.2.8)\n",
      "Collecting rsa<5,>=3.1.4 (from google-auth!=2.24.0,!=2.25.0,<3.0.0,>=2.14.1->google-cloud-vision->unstructured[all-docs])\n",
      "  Using cached rsa-4.9.1-py3-none-any.whl.metadata (5.6 kB)\n",
      "Requirement already satisfied: idna<4,>=2.5 in c:\\programdata\\anaconda3\\lib\\site-packages (from requests->unstructured[all-docs]) (3.7)\n",
      "Requirement already satisfied: urllib3<3,>=1.21.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from requests->unstructured[all-docs]) (2.2.2)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in c:\\programdata\\anaconda3\\lib\\site-packages (from requests->unstructured[all-docs]) (2024.7.4)\n",
      "Requirement already satisfied: pyasn1>=0.1.3 in c:\\programdata\\anaconda3\\lib\\site-packages (from rsa<5,>=3.1.4->google-auth!=2.24.0,!=2.25.0,<3.0.0,>=2.14.1->google-cloud-vision->unstructured[all-docs]) (0.4.8)\n",
      "Requirement already satisfied: six>=1.9 in c:\\programdata\\anaconda3\\lib\\site-packages (from html5lib->unstructured[all-docs]) (1.16.0)\n",
      "Requirement already satisfied: webencodings in c:\\programdata\\anaconda3\\lib\\site-packages (from html5lib->unstructured[all-docs]) (0.5.1)\n",
      "Requirement already satisfied: MarkupSafe>=2.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from jinja2->torch->unstructured-inference>=1.0.5->unstructured[all-docs]) (2.1.3)\n",
      "Requirement already satisfied: contourpy>=1.0.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from matplotlib->unstructured-inference>=1.0.5->unstructured[all-docs]) (1.2.0)\n",
      "Requirement already satisfied: cycler>=0.10 in c:\\programdata\\anaconda3\\lib\\site-packages (from matplotlib->unstructured-inference>=1.0.5->unstructured[all-docs]) (0.11.0)\n",
      "Requirement already satisfied: fonttools>=4.22.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from matplotlib->unstructured-inference>=1.0.5->unstructured[all-docs]) (4.51.0)\n",
      "Requirement already satisfied: kiwisolver>=1.3.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from matplotlib->unstructured-inference>=1.0.5->unstructured[all-docs]) (1.4.4)\n",
      "Requirement already satisfied: pyparsing>=2.3.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from matplotlib->unstructured-inference>=1.0.5->unstructured[all-docs]) (3.0.9)\n",
      "Requirement already satisfied: python-dateutil>=2.7 in c:\\programdata\\anaconda3\\lib\\site-packages (from matplotlib->unstructured-inference>=1.0.5->unstructured[all-docs]) (2.9.0.post0)\n",
      "INFO: pip is looking at multiple versions of contourpy to determine which version is compatible with other requirements. This could take a while.\n",
      "Collecting contourpy>=1.0.1 (from matplotlib->unstructured-inference>=1.0.5->unstructured[all-docs])\n",
      "  Downloading contourpy-1.3.3-cp312-cp312-win_amd64.whl.metadata (5.5 kB)\n",
      "Requirement already satisfied: cryptography>=39.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from msoffcrypto-tool->unstructured[all-docs]) (42.0.5)\n",
      "Requirement already satisfied: olefile>=0.46 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from msoffcrypto-tool->unstructured[all-docs]) (0.47)\n",
      "Requirement already satisfied: cffi>=1.12 in c:\\programdata\\anaconda3\\lib\\site-packages (from cryptography>=39.0->msoffcrypto-tool->unstructured[all-docs]) (1.16.0)\n",
      "Requirement already satisfied: pycparser in c:\\programdata\\anaconda3\\lib\\site-packages (from cffi>=1.12->cryptography>=39.0->msoffcrypto-tool->unstructured[all-docs]) (2.21)\n",
      "Requirement already satisfied: click in c:\\programdata\\anaconda3\\lib\\site-packages (from nltk->unstructured[all-docs]) (8.1.7)\n",
      "Requirement already satisfied: joblib in c:\\programdata\\anaconda3\\lib\\site-packages (from nltk->unstructured[all-docs]) (1.4.2)\n",
      "Requirement already satisfied: et-xmlfile in c:\\programdata\\anaconda3\\lib\\site-packages (from openpyxl->unstructured[all-docs]) (1.1.0)\n",
      "Requirement already satisfied: pytz>=2020.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from pandas->unstructured[all-docs]) (2024.1)\n",
      "Requirement already satisfied: tzdata>=2022.7 in c:\\programdata\\anaconda3\\lib\\site-packages (from pandas->unstructured[all-docs]) (2023.3)\n",
      "Collecting Pillow>=3.3.2 (from python-pptx>=1.0.1->unstructured[all-docs])\n",
      "  Downloading pillow-11.3.0-cp312-cp312-win_amd64.whl.metadata (9.2 kB)\n",
      "Collecting Deprecated (from pikepdf->unstructured[all-docs])\n",
      "  Using cached Deprecated-1.2.18-py2.py3-none-any.whl.metadata (5.7 kB)\n",
      "Requirement already satisfied: aiofiles>=24.1.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-client->unstructured[all-docs]) (24.1.0)\n",
      "Requirement already satisfied: httpcore>=1.0.9 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-client->unstructured[all-docs]) (1.0.9)\n",
      "Requirement already satisfied: httpx>=0.27.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-client->unstructured[all-docs]) (0.28.1)\n",
      "Requirement already satisfied: pydantic>=2.11.2 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from unstructured-client->unstructured[all-docs]) (2.11.7)\n",
      "Requirement already satisfied: requests-toolbelt>=1.0.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from unstructured-client->unstructured[all-docs]) (1.0.0)\n",
      "Requirement already satisfied: h11>=0.16 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from httpcore>=1.0.9->unstructured-client->unstructured[all-docs]) (0.16.0)\n",
      "Requirement already satisfied: anyio in c:\\programdata\\anaconda3\\lib\\site-packages (from httpx>=0.27.0->unstructured-client->unstructured[all-docs]) (4.2.0)\n",
      "Requirement already satisfied: annotated-types>=0.6.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from pydantic>=2.11.2->unstructured-client->unstructured[all-docs]) (0.6.0)\n",
      "Requirement already satisfied: pydantic-core==2.33.2 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from pydantic>=2.11.2->unstructured-client->unstructured[all-docs]) (2.33.2)\n",
      "Requirement already satisfied: typing-inspection>=0.4.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from pydantic>=2.11.2->unstructured-client->unstructured[all-docs]) (0.4.1)\n",
      "Requirement already satisfied: sniffio>=1.1 in c:\\programdata\\anaconda3\\lib\\site-packages (from anyio->httpx>=0.27.0->unstructured-client->unstructured[all-docs]) (1.3.0)\n",
      "Downloading onnx-1.19.0-cp312-cp312-win_amd64.whl (16.5 MB)\n",
      "   ---------------------------------------- 0.0/16.5 MB ? eta -:--:--\n",
      "   ------- -------------------------------- 3.1/16.5 MB 16.8 MB/s eta 0:00:01\n",
      "   --------------- ------------------------ 6.6/16.5 MB 16.8 MB/s eta 0:00:01\n",
      "   ------------------------- -------------- 10.5/16.5 MB 17.2 MB/s eta 0:00:01\n",
      "   ----------------------------------- ---- 14.4/16.5 MB 17.8 MB/s eta 0:00:01\n",
      "   ---------------------------------------- 16.5/16.5 MB 17.3 MB/s  0:00:00\n",
      "Using cached protobuf-6.32.0-cp310-abi3-win_amd64.whl (435 kB)\n",
      "Using cached python_docx-1.2.0-py3-none-any.whl (252 kB)\n",
      "Using cached python_pptx-1.0.2-py3-none-any.whl (472 kB)\n",
      "Using cached unstructured_inference-1.0.5-py3-none-any.whl (48 kB)\n",
      "Using cached opencv_python-4.12.0.88-cp37-abi3-win_amd64.whl (39.0 MB)\n",
      "Downloading numpy-2.2.6-cp312-cp312-win_amd64.whl (12.6 MB)\n",
      "   ---------------------------------------- 0.0/12.6 MB ? eta -:--:--\n",
      "   ----------- ---------------------------- 3.7/12.6 MB 18.2 MB/s eta 0:00:01\n",
      "   ------------------------ --------------- 7.6/12.6 MB 18.1 MB/s eta 0:00:01\n",
      "   ---------------------------------- ----- 11.0/12.6 MB 18.1 MB/s eta 0:00:01\n",
      "   ---------------------------------------- 12.6/12.6 MB 16.8 MB/s  0:00:00\n",
      "Using cached unstructured.pytesseract-0.3.15-py3-none-any.whl (14 kB)\n",
      "Using cached xlsxwriter-3.2.5-py3-none-any.whl (172 kB)\n",
      "Using cached accelerate-1.10.1-py3-none-any.whl (374 kB)\n",
      "Using cached effdet-0.4.1-py3-none-any.whl (112 kB)\n",
      "Using cached omegaconf-2.3.0-py3-none-any.whl (79 kB)\n",
      "Downloading pycocotools-2.0.10-cp312-abi3-win_amd64.whl (76 kB)\n",
      "Using cached timm-1.0.19-py3-none-any.whl (2.5 MB)\n",
      "Using cached google_cloud_vision-3.10.2-py3-none-any.whl (527 kB)\n",
      "Using cached google_api_core-2.25.1-py3-none-any.whl (160 kB)\n",
      "Using cached google_auth-2.40.3-py2.py3-none-any.whl (216 kB)\n",
      "Using cached googleapis_common_protos-1.70.0-py3-none-any.whl (294 kB)\n",
      "Downloading grpcio-1.74.0-cp312-cp312-win_amd64.whl (4.5 MB)\n",
      "   ---------------------------------------- 0.0/4.5 MB ? eta -:--:--\n",
      "   -------------------------------- ------- 3.7/4.5 MB 18.1 MB/s eta 0:00:01\n",
      "   ---------------------------------------- 4.5/4.5 MB 16.9 MB/s  0:00:00\n",
      "Using cached grpcio_status-1.74.0-py3-none-any.whl (14 kB)\n",
      "Using cached proto_plus-1.26.1-py3-none-any.whl (50 kB)\n",
      "Using cached rsa-4.9.1-py3-none-any.whl (34 kB)\n",
      "Downloading contourpy-1.3.3-cp312-cp312-win_amd64.whl (226 kB)\n",
      "Downloading ml_dtypes-0.5.3-cp312-cp312-win_amd64.whl (208 kB)\n",
      "Using cached msoffcrypto_tool-5.4.2-py3-none-any.whl (48 kB)\n",
      "Using cached pdf2image-1.17.0-py3-none-any.whl (11 kB)\n",
      "Downloading pdfminer_six-20250506-py3-none-any.whl (5.6 MB)\n",
      "   ---------------------------------------- 0.0/5.6 MB ? eta -:--:--\n",
      "   ------------------------ --------------- 3.4/5.6 MB 16.7 MB/s eta 0:00:01\n",
      "   ---------------------------------------- 5.6/5.6 MB 17.1 MB/s  0:00:00\n",
      "Downloading pi_heif-1.1.0-cp312-cp312-win_amd64.whl (1.9 MB)\n",
      "   ---------------------------------------- 0.0/1.9 MB ? eta -:--:--\n",
      "   ---------------------------------------- 1.9/1.9 MB 17.4 MB/s  0:00:00\n",
      "Downloading pillow-11.3.0-cp312-cp312-win_amd64.whl (7.0 MB)\n",
      "   ---------------------------------------- 0.0/7.0 MB ? eta -:--:--\n",
      "   ------------------- -------------------- 3.4/7.0 MB 16.7 MB/s eta 0:00:01\n",
      "   ---------------------------------------  6.8/7.0 MB 17.5 MB/s eta 0:00:01\n",
      "   ---------------------------------------- 7.0/7.0 MB 16.6 MB/s  0:00:00\n",
      "Downloading pikepdf-9.10.2-cp312-cp312-win_amd64.whl (3.7 MB)\n",
      "   ---------------------------------------- 0.0/3.7 MB ? eta -:--:--\n",
      "   ------------------------------ --------- 2.9/3.7 MB 15.2 MB/s eta 0:00:01\n",
      "   ---------------------------------------- 3.7/3.7 MB 14.8 MB/s  0:00:00\n",
      "Using cached Deprecated-1.2.18-py2.py3-none-any.whl (10.0 kB)\n",
      "Using cached pypandoc-1.15-py3-none-any.whl (21 kB)\n",
      "Using cached pypdfium2-4.30.0-py3-none-win_amd64.whl (2.9 MB)\n",
      "Using cached python_multipart-0.0.20-py3-none-any.whl (24 kB)\n",
      "Downloading torchvision-0.23.0-cp312-cp312-win_amd64.whl (1.6 MB)\n",
      "   ---------------------------------------- 0.0/1.6 MB ? eta -:--:--\n",
      "   ---------------------------------------- 1.6/1.6 MB 21.6 MB/s  0:00:00\n",
      "Using cached xlrd-2.0.2-py2.py3-none-any.whl (96 kB)\n",
      "Building wheels for collected packages: antlr4-python3-runtime\n",
      "  Building wheel for antlr4-python3-runtime (setup.py): started\n",
      "  Building wheel for antlr4-python3-runtime (setup.py): finished with status 'done'\n",
      "  Created wheel for antlr4-python3-runtime: filename=antlr4_python3_runtime-4.9.3-py3-none-any.whl size=144615 sha256=b7ced839d91e8cb663ada7914989d70effc974953130cf25d14e7af52f7bc764\n",
      "  Stored in directory: c:\\users\\deepa\\appdata\\local\\pip\\cache\\wheels\\1f\\be\\48\\13754633f1d08d1fbfc60d5e80ae1e5d7329500477685286cd\n",
      "Successfully built antlr4-python3-runtime\n",
      "Installing collected packages: antlr4-python3-runtime, XlsxWriter, xlrd, rsa, python-multipart, python-docx, pypdfium2, pypandoc, protobuf, Pillow, omegaconf, numpy, grpcio, Deprecated, unstructured.pytesseract, python-pptx, pycocotools, proto-plus, pikepdf, pi-heif, pdf2image, opencv-python, ml_dtypes, googleapis-common-protos, google-auth, contourpy, torchvision, pdfminer.six, onnx, msoffcrypto-tool, grpcio-status, google-api-core, accelerate, timm, unstructured-inference, google-cloud-vision, effdet\n",
      "\n",
      "   ----------------------------------------  0/37 [antlr4-python3-runtime]\n",
      "   ----------------------------------------  0/37 [antlr4-python3-runtime]\n",
      "   ----------------------------------------  0/37 [antlr4-python3-runtime]\n",
      "   ----------------------------------------  0/37 [antlr4-python3-runtime]\n",
      "   - --------------------------------------  1/37 [XlsxWriter]\n",
      "   - --------------------------------------  1/37 [XlsxWriter]\n",
      "   - --------------------------------------  1/37 [XlsxWriter]\n",
      "   - --------------------------------------  1/37 [XlsxWriter]\n",
      "   - --------------------------------------  1/37 [XlsxWriter]\n",
      "   - --------------------------------------  1/37 [XlsxWriter]\n",
      "   - --------------------------------------  1/37 [XlsxWriter]\n",
      "   -- -------------------------------------  2/37 [xlrd]\n",
      "   -- -------------------------------------  2/37 [xlrd]\n",
      "   --- ------------------------------------  3/37 [rsa]\n",
      "   --- ------------------------------------  3/37 [rsa]\n",
      "   --- ------------------------------------  3/37 [rsa]\n",
      "   --- ------------------------------------  3/37 [rsa]\n",
      "   --- ------------------------------------  3/37 [rsa]\n",
      "   --- ------------------------------------  3/37 [rsa]\n",
      "   ---- -----------------------------------  4/37 [python-multipart]\n",
      "   ----- ----------------------------------  5/37 [python-docx]\n",
      "   ----- ----------------------------------  5/37 [python-docx]\n",
      "   ----- ----------------------------------  5/37 [python-docx]\n",
      "   ----- ----------------------------------  5/37 [python-docx]\n",
      "   ----- ----------------------------------  5/37 [python-docx]\n",
      "   ----- ----------------------------------  5/37 [python-docx]\n",
      "   ----- ----------------------------------  5/37 [python-docx]\n",
      "   ----- ----------------------------------  5/37 [python-docx]\n",
      "   ----- ----------------------------------  5/37 [python-docx]\n",
      "   ------ ---------------------------------  6/37 [pypdfium2]\n",
      "   ------ ---------------------------------  6/37 [pypdfium2]\n",
      "   ------ ---------------------------------  6/37 [pypdfium2]\n",
      "   ------ ---------------------------------  6/37 [pypdfium2]\n",
      "   ------ ---------------------------------  6/37 [pypdfium2]\n",
      "   ------ ---------------------------------  6/37 [pypdfium2]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   -------- -------------------------------  8/37 [protobuf]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   --------- ------------------------------  9/37 [Pillow]\n",
      "   ---------- ----------------------------- 10/37 [omegaconf]\n",
      "   ---------- ----------------------------- 10/37 [omegaconf]\n",
      "   ---------- ----------------------------- 10/37 [omegaconf]\n",
      "   ---------- ----------------------------- 10/37 [omegaconf]\n",
      "  Attempting uninstall: numpy\n",
      "   ---------- ----------------------------- 10/37 [omegaconf]\n",
      "    Found existing installation: numpy 2.3.2\n",
      "   ---------- ----------------------------- 10/37 [omegaconf]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "    Uninstalling numpy-2.3.2:\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "      Successfully uninstalled numpy-2.3.2\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ----------- ---------------------------- 11/37 [numpy]\n",
      "   ------------ --------------------------- 12/37 [grpcio]\n",
      "   ------------ --------------------------- 12/37 [grpcio]\n",
      "   ------------ --------------------------- 12/37 [grpcio]\n",
      "   ------------ --------------------------- 12/37 [grpcio]\n",
      "   ------------ --------------------------- 12/37 [grpcio]\n",
      "   ------------ --------------------------- 12/37 [grpcio]\n",
      "   ------------ --------------------------- 12/37 [grpcio]\n",
      "   -------------- ------------------------- 13/37 [Deprecated]\n",
      "   --------------- ------------------------ 14/37 [unstructured.pytesseract]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ---------------- ----------------------- 15/37 [python-pptx]\n",
      "   ------------------ --------------------- 17/37 [proto-plus]\n",
      "   ------------------ --------------------- 17/37 [proto-plus]\n",
      "   ------------------ --------------------- 17/37 [proto-plus]\n",
      "   ------------------ --------------------- 17/37 [proto-plus]\n",
      "   ------------------- -------------------- 18/37 [pikepdf]\n",
      "   ------------------- -------------------- 18/37 [pikepdf]\n",
      "   ------------------- -------------------- 18/37 [pikepdf]\n",
      "   ------------------- -------------------- 18/37 [pikepdf]\n",
      "   -------------------- ------------------- 19/37 [pi-heif]\n",
      "   -------------------- ------------------- 19/37 [pi-heif]\n",
      "   ---------------------- ----------------- 21/37 [opencv-python]\n",
      "   ---------------------- ----------------- 21/37 [opencv-python]\n",
      "   ---------------------- ----------------- 21/37 [opencv-python]\n",
      "   ---------------------- ----------------- 21/37 [opencv-python]\n",
      "   ---------------------- ----------------- 21/37 [opencv-python]\n",
      "   ---------------------- ----------------- 21/37 [opencv-python]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------ --------------- 23/37 [googleapis-common-protos]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   ------------------------- -------------- 24/37 [google-auth]\n",
      "   --------------------------- ------------ 25/37 [contourpy]\n",
      "   --------------------------- ------------ 25/37 [contourpy]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ---------------------------- ----------- 26/37 [torchvision]\n",
      "   ----------------------------- ---------- 27/37 [pdfminer.six]\n",
      "   ----------------------------- ---------- 27/37 [pdfminer.six]\n",
      "   ----------------------------- ---------- 27/37 [pdfminer.six]\n",
      "   ----------------------------- ---------- 27/37 [pdfminer.six]\n",
      "   ----------------------------- ---------- 27/37 [pdfminer.six]\n",
      "   ----------------------------- ---------- 27/37 [pdfminer.six]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------ --------- 28/37 [onnx]\n",
      "   ------------------------------- -------- 29/37 [msoffcrypto-tool]\n",
      "   ------------------------------- -------- 29/37 [msoffcrypto-tool]\n",
      "   ------------------------------- -------- 29/37 [msoffcrypto-tool]\n",
      "   -------------------------------- ------- 30/37 [grpcio-status]\n",
      "   --------------------------------- ------ 31/37 [google-api-core]\n",
      "   --------------------------------- ------ 31/37 [google-api-core]\n",
      "   --------------------------------- ------ 31/37 [google-api-core]\n",
      "   --------------------------------- ------ 31/37 [google-api-core]\n",
      "   --------------------------------- ------ 31/37 [google-api-core]\n",
      "   --------------------------------- ------ 31/37 [google-api-core]\n",
      "   --------------------------------- ------ 31/37 [google-api-core]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ---------------------------------- ----- 32/37 [accelerate]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ----------------------------------- ---- 33/37 [timm]\n",
      "   ------------------------------------ --- 34/37 [unstructured-inference]\n",
      "   ------------------------------------ --- 34/37 [unstructured-inference]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   ------------------------------------- -- 35/37 [google-cloud-vision]\n",
      "   -------------------------------------- - 36/37 [effdet]\n",
      "   -------------------------------------- - 36/37 [effdet]\n",
      "   -------------------------------------- - 36/37 [effdet]\n",
      "   -------------------------------------- - 36/37 [effdet]\n",
      "   -------------------------------------- - 36/37 [effdet]\n",
      "   -------------------------------------- - 36/37 [effdet]\n",
      "   ---------------------------------------- 37/37 [effdet]\n",
      "\n",
      "Successfully installed Deprecated-1.2.18 Pillow-11.3.0 XlsxWriter-3.2.5 accelerate-1.10.1 antlr4-python3-runtime-4.9.3 contourpy-1.3.3 effdet-0.4.1 google-api-core-2.25.1 google-auth-2.40.3 google-cloud-vision-3.10.2 googleapis-common-protos-1.70.0 grpcio-1.74.0 grpcio-status-1.74.0 ml_dtypes-0.5.3 msoffcrypto-tool-5.4.2 numpy-2.2.6 omegaconf-2.3.0 onnx-1.19.0 opencv-python-4.12.0.88 pdf2image-1.17.0 pdfminer.six-20250506 pi-heif-1.1.0 pikepdf-9.10.2 proto-plus-1.26.1 protobuf-6.32.0 pycocotools-2.0.10 pypandoc-1.15 pypdfium2-4.30.0 python-docx-1.2.0 python-multipart-0.0.20 python-pptx-1.0.2 rsa-4.9.1 timm-1.0.19 torchvision-0.23.0 unstructured-inference-1.0.5 unstructured.pytesseract-0.3.15 xlrd-2.0.2\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  DEPRECATION: Building 'antlr4-python3-runtime' using the legacy setup.py bdist_wheel mechanism, which will be removed in a future version. pip 25.3 will enforce this behaviour change. A possible replacement is to use the standardized build interface by setting the `--use-pep517` option, (possibly combined with `--no-build-isolation`), or adding a `pyproject.toml` file to the source tree of 'antlr4-python3-runtime'. Discussion can be found at https://github.com/pypa/pip/issues/6334\n",
      "  WARNING: The scripts pyrsa-decrypt.exe, pyrsa-encrypt.exe, pyrsa-keygen.exe, pyrsa-priv2pub.exe, pyrsa-sign.exe and pyrsa-verify.exe are installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: The script pypdfium2.exe is installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: The scripts f2py.exe and numpy-config.exe are installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: Failed to remove contents in a temporary directory 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\site-packages\\~.mpy.libs'.\n",
      "  You can safely remove it manually.\n",
      "  WARNING: Failed to remove contents in a temporary directory 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\site-packages\\~.mpy'.\n",
      "  You can safely remove it manually.\n",
      "  WARNING: The script pytesseract.exe is installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: The scripts backend-test-tools.exe, check-model.exe and check-node.exe are installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: The script msoffcrypto-tool.exe is installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "  WARNING: The scripts accelerate-config.exe, accelerate-estimate-memory.exe, accelerate-launch.exe, accelerate-merge-weights.exe and accelerate.exe are installed in 'C:\\Users\\deepa\\AppData\\Roaming\\Python\\Python312\\Scripts' which is not on PATH.\n",
      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\n",
      "ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
      "numba 0.59.1 requires numpy<1.27,>=1.22, but you have numpy 2.2.6 which is incompatible.\n",
      "pywavelets 1.5.0 requires numpy<2.0,>=1.22.4, but you have numpy 2.2.6 which is incompatible.\n",
      "streamlit 1.32.0 requires numpy<2,>=1.19.3, but you have numpy 2.2.6 which is incompatible.\n",
      "streamlit 1.32.0 requires pillow<11,>=7.1.0, but you have pillow 11.3.0 which is incompatible.\n",
      "streamlit 1.32.0 requires protobuf<5,>=3.20, but you have protobuf 6.32.0 which is incompatible.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Requirement already satisfied: pypdf in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (6.0.0)\n",
      "Requirement already satisfied: pdfminer.six in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (20250506)\n",
      "Requirement already satisfied: charset-normalizer>=2.0.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from pdfminer.six) (2.0.4)\n",
      "Requirement already satisfied: cryptography>=36.0.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from pdfminer.six) (42.0.5)\n",
      "Requirement already satisfied: cffi>=1.12 in c:\\programdata\\anaconda3\\lib\\site-packages (from cryptography>=36.0.0->pdfminer.six) (1.16.0)\n",
      "Requirement already satisfied: pycparser in c:\\programdata\\anaconda3\\lib\\site-packages (from cffi>=1.12->cryptography>=36.0.0->pdfminer.six) (2.21)\n",
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Requirement already satisfied: python-docx in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (1.2.0)\n",
      "Requirement already satisfied: pypandoc in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (1.15)\n",
      "Requirement already satisfied: lxml>=3.1.0 in c:\\programdata\\anaconda3\\lib\\site-packages (from python-docx) (5.2.1)\n",
      "Requirement already satisfied: typing_extensions>=4.9.0 in c:\\users\\deepa\\appdata\\roaming\\python\\python312\\site-packages (from python-docx) (4.14.1)\n"
     ]
    }
   ],
   "source": [
    "!pip install -U langchain langchain-community unstructured langchain-unstructured\n",
    "\n",
    "# Add parsers for ALL doc types\n",
    "!pip install \"unstructured[all-docs]\"\n",
    "\n",
    "# Useful PDF parsers\n",
    "!pip install pypdf pdfminer.six\n",
    "\n",
    "# (Optional) if working with DOCX/EPUB\n",
    "!pip install python-docx pypandoc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6648d36f-6ce3-4a2a-b05d-5b0c8bf17484",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "venv",
   "language": "python",
   "name": "venv"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
